<!-- image -->

<!-- image -->

Review

## Current Trends in Sirtuin Activator and Inhibitor Development

Karina L. Bursch 1,2 , Christopher J. Goetz 1 and Brian C. Smith 1,2,3, *

- 1 Department of Biochemistry, Medical College of Wisconsin, Milwaukee, WI 53226, USA; kbursch@mcw.edu (K.L.B.)
- 2 Structural Genomics Unit, Linda T. and John A. Mellowes Center for Genomic Sciences and Precision Medicine, Medical College of Wisconsin, Milwaukee, WI 53226, USA
- 3 Program in Chemical Biology, Medical College of Wisconsin, Milwaukee, WI 53226, USA
- * Correspondence: brismith@mcw.edu; Tel.: +1-(414)-955-5669

Abstract: Sirtuins are NAD + -dependent protein deacylases and key metabolic regulators, coupling the cellular energy state with selective lysine deacylation to regulate many downstream cellular processes. Humans encode seven sirtuin isoforms (Sirt1-7) with diverse subcellular localization and deacylase targets. Sirtuins are considered protective anti-aging proteins since increased sirtuin activity is canonically associated with lifespan extension and decreased activity with developing aging-related diseases. However, sirtuins can also assume detrimental cellular roles where increased activity contributes to pathophysiology. Modulation of sirtuin activity by activators and inhibitors thus holds substantial potential for defining the cellular roles of sirtuins in health and disease and developing therapeutics. Instead of being comprehensive, this review discusses the well-characterized sirtuin activators and inhibitors available to date, particularly those with demonstrated selectivity, potency, and cellular activity. This review also provides recommendations regarding the best-in-class sirtuin activators and inhibitors for practical research as sirtuin modulator discovery and refinement evolve.

Keywords: sirtuins; epigenetics; lysine deacylation; drug development

## 1. Introduction

The sirtuin or silent information regulator 2 (Sir2) protein family was named for the Saccharomyces cerevisiae Sir2 protein [1], a product of the SIR gene family implicated in silencing yeast gene expression [2] by facilitating a less accessible chromatin structure [3]. Since these foundational studies, Sir2 orthologs have been identified across all taxonomic domains (Archaea, Bacteria, and Eukarya) [4]. Sirtuins regulate various pro-survival, lifespan-extending cellular processes, including DNA transcription and repair, metabolism, and stress resistance [5]. Mechanistically, sirtuins are NAD + -dependent protein deacylases that remove acyl modifications on lysine residues of histone and non-histone protein substrates, yielding deacylated protein substrate, nicotinamide, and 2 ' -O -acyl-ADP-ribose ( O AADPr) as products [6,7]. NAD + dependence endows sirtuins with cellular 'energy sensing' capabilities, coupling the energetic needs of the cell with selective lysine deacylation across subcellular organelles [8]. All mammals, including humans, express seven sirtuin isoforms (Sirt1-7) [5] that comprise the class III histone deacetylase (HDAC) family [9] and display diverse subcellular localization, deacylase activities, and protein targets [10,11].

## 1.1. Sirtuin Subcellular Localization and Structure Dictate Function

While sirtuins harbor overlapping substrate specificities in vitro, subcellular localization of the seven mammalian sirtuins confers cellular selectivity for their endogenous protein targets [11]. Sirt1, Sirt6, and Sirt7 are primarily nuclear, Sirt2 is mainly cytosolic, and Sirt3, Sirt4, and Sirt5 are predominantly mitochondrial (Figure 1) [10]. However, sirtuin isoform localization can vary. Depending on specific cellular stimuli, Sirt1 and Sirt2 can shuttle between the nucleus and cytosol [12,13]. Although contested [14,15], cellular

<!-- image -->

Citation: Bursch, K.L.; Goetz, C.J.; Smith, B.C. Current Trends in Sirtuin Activator and Inhibitor Development. Molecules 2024 , 29 , 1185. https:// doi.org/10.3390/molecules29051185

Academic Editor: Enrico Millo

Received: 10 February 2024

Revised: 1 March 2024

Accepted: 4 March 2024

Published: 6 March 2024

<!-- image -->

Copyright: © 2024 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (https:// creativecommons.org/licenses/by/ 4.0/).

Molecules

2024

,

29

, x FOR PEER REVIEW

2 of 39

sirtuin isoform localization can vary. Depending on speci

fi

c cellular stimuli, Sirt1 and Sirt2

can shu

tt

le between the nucleus and cytosol [12,13]. Although contested [14,15], cellular

stress has been reported to cause a long unprocessed isoform of Sirt3 to localize to the nucleus [15-17]. Sirt5 has also been observed to regulate levels of protein malonylation in both the mitochondria and cytosol [18]. stress has been reported to cause a long unprocessed isoform of Sirt3 to localize to the nucleus [15-17]. Sirt5 has also been observed to regulate levels of protein malonylation in both the mitochondria and cytosol [18].

Figure 1. Primary cellular localization and acyl-lysine substrate speci fi cities of human sirtuins. The seven human sirtuin isoforms are di ff erentially distributed between the nucleus (Sirt1, Sirt6, and Sirt7; blue), cytoplasm (Sirt2; white), and mitochondria (Sirt3, Sirt4, and Sirt5; green) and possess distinct acyl-lysine substrate speci fi cities. * = sirtuin deacylase activity con fi rmed in cells. Figure 1. Primary cellular localization and acyl-lysine substrate specificities of human sirtuins. The seven human sirtuin isoforms are differentially distributed between the nucleus (Sirt1, Sirt6, and Sirt7; blue), cytoplasm (Sirt2; white), and mitochondria (Sirt3, Sirt4, and Sirt5; green) and possess distinct acyl-lysine substrate specificities. * = sirtuin deacylase activity confirmed in cells.

<!-- image -->

Sirtuins share an evolutionarily conserved ~275 amino acid catalytic core composed of two domains connected by multiple loops: a larger, highly conserved Rossmann fold domain that binds the sirtuin co-substrate NAD + and a smaller, more variable zinc-binding domain; the acylated substrate binds in a cleft formed by these two domains (Figure 2) [19,20]. Variation in the acylated substrate binding cleft allows for di ff erential substrate preferences and deacylase activities [21,22]. Sirtuins remove a variety of acyl modi fi cations from lysine residues (Figure 1) [23]; Sirt1-3 are strong deacetylases [24] but can also remove propionyl [25], butyryl [25], β -hydroxybutyryl [26], crotonyl [27], and long-chain fa tt y acid (LFA) acyl groups [28]. Sirt5 prefers negatively charged acylations such as malonyl, succinyl, and glutaryl groups [22,29]. Sirt4 removes more complex negativelycharged acylations, such as methylglutaryl, hydroxymethylglutaryl, and methylglutaconyl acyl groups [30] as well as lipoyl and biotinyl acyl groups [31]. Although Sirt6 and Sirt7 are weak deacetylases in vitro [32], their enzymatic activities are stimulated by cellular components. Sirt6 e ffi ciently deacylates LFAs [33], and Sirt6 deacetylase activity is stimulated by biologically relevant free fa tt y acids (FFAs) [28] and by binding to nucleosomes and nucleosomal DNA [34,35]. Similarly, Sirt7 deacetylase activity is stimulated by nucleosome binding and several types of nucleic acids [36-38], while its defa tt y-acylase activity is primarily stimulated by rRNA [38]. Sirtuins share an evolutionarily conserved ~275 amino acid catalytic core composed of two domains connected by multiple loops: a larger, highly conserved Rossmann fold domain that binds the sirtuin co-substrate NAD + and a smaller, more variable zinc-binding domain; the acylated substrate binds in a cleft formed by these two domains (Figure 2) [19,20]. Variation in the acylated substrate binding cleft allows for differential substrate preferences and deacylase activities [21,22]. Sirtuins remove a variety of acyl modifications from lysine residues (Figure 1) [23]; Sirt1-3 are strong deacetylases [24] but can also remove propionyl [25], butyryl [25], β -hydroxybutyryl [26], crotonyl [27], and long-chain fatty acid (LFA) acyl groups [28]. Sirt5 prefers negatively charged acylations such as malonyl, succinyl, and glutaryl groups [22,29]. Sirt4 removes more complex negatively-charged acylations, such as methylglutaryl, hydroxymethylglutaryl, and methylglutaconyl acyl groups [30] as well as lipoyl and biotinyl acyl groups [31]. Although Sirt6 and Sirt7 are weak deacetylases in vitro [32], their enzymatic activities are stimulated by cellular components. Sirt6 efficiently deacylates LFAs [33], and Sirt6 deacetylase activity is stimulated by biologically relevant free fatty acids (FFAs) [28] and by binding to nucleosomes and nucleosomal DNA [34,35]. Similarly, Sirt7 deacetylase activity is stimulated by nucleosome binding and several types of nucleic acids [36-38], while its defatty-acylase activity is primarily stimulated by rRNA [38].

, x FOR PEER REVIEW

3 of

Figure 2. General structure of a human sirtuin deacylase bound to an acylated substrate and co-su strate NAD + . Key structural domains include the acylated substrate binding cleft (red), the NAD + substrate binding pocket (purple), and the zinc-tetrathiolate domain (green). Within and proximal the NAD + binding pocket, subregions involved in binding the adenine moiety (A-site), the nicoti mide ribose moiety (B-site), and the nicotinamide ribose moiety upon acyl-substrate binding (C of NAD + , as well as the extended C-site (EC) and selectivity pocket (SP) exploited by modulators, are outlined by dashed circles and annotated with the corresponding letters. The sirt activating compound (STAC)-binding domain (SBD; blue) is unique to Sirt1. PDB ID: 4ZZJ [39]. Figure 2. General structure of a human sirtuin deacylase bound to an acylated substrate and cosubstrate NAD + . Key structural domains include the acylated substrate binding cleft (red), the NAD + co-substrate binding pocket (purple), and the zinc-tetrathiolate domain (green). Within and proximal to the NAD + binding pocket, subregions involved in binding the adenine moiety (A-site), the nicotinamide ribose moiety (B-site), and the nicotinamide ribose moiety upon acyl-substrate binding (C-site) of NAD + , as well as the extended C-site (EC) and selectivity pocket (SP) exploited by some sirtuin modulators, are outlined by dashed circles and annotated with the corresponding letters. The sirtuin-activating compound (STAC)-binding domain (SBD; blue) is unique to Sirt1. PDB ID: 4ZZJ [39].

<!-- image -->

1.2. Di

ff

erential Modulation of Sirtuin Activity

## 1.2. Differential Modulation of Sirtuin Activity

Initial e ff orts to develop sirtuin activators and inhibitors are encouraging. Sirtui activating compounds (STACs) have been identi fi ed from natural products and optimiz with medicinal chemistry approaches; competitive small molecule inhibitors and pote mechanism-based peptide inhibitors have also been developed for most sirtuin [49-52]. However, these molecular modulators of sirtuin activity have drawbacks. Ma STACs and inhibitors lack cellular activity or selectivity, a ff ecting a va teins in addition to their intended sirtuin targets [52,53]. Therefore, developing activa and inhibitors that modulate sirtuin activity with greater speci fi city and cellular e ffi is imperative to clarify the complex roles of sirtuins in the biology of aging and relat pathophysiologies. Note that the sirtuin modulators discussed in this review are referr Initial efforts to develop sirtuin activators and inhibitors are encouraging. Sirtuinactivating compounds (STACs) have been identified from natural products and optimized with medicinal chemistry approaches; competitive small molecule inhibitors and potent mechanism-based peptide inhibitors have also been developed for most sirtuin isoforms [49-52]. However, these molecular modulators of sirtuin activity have drawbacks. Many STACs and inhibitors lack cellular activity or selectivity, affecting a variety of other proteins in addition to their intended sirtuin targets [52,53]. Therefore, developing activators and inhibitors that modulate sirtuin activity with greater specificity and cellular efficacy is imperative to clarify the complex roles of sirtuins in the biology of aging and related pathophysiologies. Note that the sirtuin modulators discussed in this review are referred to by the names found in the primary literature.

Sirtuin activity is regulated transcriptionally and by protein-protein interactio subcellular localization, and nutrient/cofactor availability [32,40-42]. In additi conserved catalytic core, sirtuins have N - and C -termini that vary in length (Figure Notably, the lengthy N - and C -termini of Sirt1 have been found to enhance Sirt1 de lase activity [43]. Post-translational modi fi cations, including modi fi cations by upon cellular stress [44], can also a ff ect sirtuin deacylase activity [45]. Sirtuin a tt enuati and upregulation have simultaneously been implicated in the pathogenesis of many ing-related diseases, including cancer [46], cardiovascular disease [47], and neur eration [48]. Therefore, small molecule modulators of sirtuin activity have great potent for treating the pathophysiology of aging and extending human lifespans. Sirtuin activity is regulated transcriptionally and by protein-protein interactions, subcellular localization, and nutrient/cofactor availability [32,40-42]. In addition to their conserved catalytic core, sirtuins have N - and C -termini that vary in length (Figure 3). Notably, the lengthy N - and C -termini of Sirt1 have been found to enhance Sirt1 deacetylase activity [43]. Post-translational modifications, including modifications by oxidants upon cellular stress [44], can also affect sirtuin deacylase activity [45]. Sirtuin attenuation and upregulation have simultaneously been implicated in the pathogenesis of many aging-related diseases, including cancer [46], cardiovascular disease [47], and neurodegeneration [48]. Therefore, small molecule modulators of sirtuin activity have great potential for treating the pathophysiology of aging and extending human lifespans.

to by the names found in the primary literature.

Molecules

2024

,

29

, x FOR PEER REVIEW

4 of 39

Molecules

2024

,

29

, x FOR PEER REVIEW

Molecules

2024

,

29

, x FOR PEER REVIEW

4 of 39

, x FOR PEER REVIEW

Figure 3. Human sirtuins have variable N - and C -termini and catalytic cores. Linear representation of Sirt1-7, with tan denoting the catalytic core. Boundaries of the catalytic core were determined from UniProt for each sirtuin isoform. Figure 3. Human sirtuins have variable N - and C -termini and catalytic cores. Linear representation of Sirt1-7, with tan denoting the catalytic core. Boundaries of the catalytic core were determined from UniProt for each sirtuin isoform. Figure 3. Human sirtuins have variable N - and C -termini and catalytic cores. Linear representation of Sirt1-7, with tan denoting the catalytic core. Boundaries of the catalytic core were determined from UniProt for each sirtuin isoform. Figure 3. Human sirtuins have variable N - and C -termini and catalytic cores. Linear representation of Sirt1-7, with tan denoting the catalytic core. Boundaries of the catalytic core were determined from UniProt for each sirtuin isoform. Figure 3. Human sirtuins have variable N - and C -termini and catalytic cores. Linear representation of Sirt1-7, with tan denoting the catalytic core. Boundaries of the catalytic core were determined

<!-- image -->

from UniProt for each sirtuin isoform.

## 2. Sirtuin Activators 2. Sirtuin Activators 2. Sirtuin Activators

aging and a

tt

enuating the pathogenesis of multiple aging-related diseases.

Since the discovery of sirtuin-dependent lifespan extension in yeast [54,55] and mice [56,57], a variety of natural product and synthetic small molecule sirtuin activators have been identi fi ed in a tt empts to extend human lifespan and mitigate aging-related diseases [49,50] (Table 1). Alternative approaches to enhance sirtuin activity are also discussed. Individual or combinatorial strategies to enhance sirtuin activity hold promise for delaying aging and a tt enuating the pathogenesis of multiple aging-related diseases. Since the discovery of sirtuin-dependent lifespan extension in yeast [54,55] and mice [56,57], a variety of natural product and synthetic small molecule sirtuin activators have been identified in attempts to extend human lifespan and mitigate aging-related diseases [49,50] (Table 1). Alternative approaches to enhance sirtuin activity are also discussed. Individual or combinatorial strategies to enhance sirtuin activity hold promise for delaying aging and attenuating the pathogenesis of multiple aging-related diseases. Since the discovery of sirtuin-dependent lifespan extension in yeast [54,55] and mice [56,57], a variety of natural product and synthetic small molecule sirtuin activators have been identi fi ed in a tt empts to extend human lifespan and mitigate aging-related diseases [49,50] (Table 1). Alternative approaches to enhance sirtuin activity are also discussed. Individual or combinatorial strategies to enhance sirtuin activity hold promise for delaying aging and a tt enuating the pathogenesis of multiple aging-related diseases. 2. Sirtuin Activators Since the discovery of sirtuin-dependent lifespan extension in yeast [54,55] and mice [56,57], a variety of natural product and synthetic small molecule sirtuin activators have been identi fi ed in a tt empts to extend human lifespan and mitigate aging-related diseases [49,50] (Table 1). Alternative approaches to enhance sirtuin activity are also discussed. Individual or combinatorial strategies to enhance sirtuin activity hold promise for delaying 2. Sirtuin Activators Since the discovery of sirtuin-dependent lifespan extension in yeast [54,55] and mice [56,57], a variety of natural product and synthetic small molecule sirtuin activators have been identi fi ed in a tt empts to extend human lifespan and mitigate aging-related diseases [49,50] (Table 1). Alternative approaches to enhance sirtuin activity are also discussed. Individual or combinatorial strategies to enhance sirtuin activity hold promise for delaying

aging and a

tt

enuating the pathogenesis of multiple aging-related diseases.

Table 1. Select activators of sirtuin deacylase activity. Table 1. Select activators of sirtuin deacylase activity. Table 1. Select activators of sirtuin deacylase activity.

Table 1.

Select activators of sirtuin deacylase activity.

Select activators of sirtuin deacylase activity.

<!-- image -->

<!-- image -->

<!-- image -->

Table 1.

| Compound Name  Compound Name Compound Name  Compound Name  Compound Name   | Structure  Structure Structure  Structure  Structure   | Targeted Sirtuin  Isoform  Targeted Sirtuin Isoform Targeted Sirtuin  Isoform  Targeted Sirtuin  Isoform  Targeted Sirtuin  Isoform   | (EC50),  Potency (EC 50 ), µ M Potency  (EC50),  μ Potency  (EC50),  μ Potency  (EC50),  μ M   | Potency  μ M  Activation  Fold Activation M  Fold    Activation  M  Fold    Activation  Fold    Activation   | Fold    Cell    Active  Cell Active Cell    Active  Cell    Active  Cell    Active   | Citations  Citations Citations  Citations  Citations   |
|----------------------------------------------------------------------------|--------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|--------------------------------------------------------|
| Resveratrol  Resveratrol Resveratrol  Resveratrol  Resveratrol             |                                                        | Sirt1  Sirt1  Sirt1 Sirt1  Sirt1                                                                                                      | ~100  ~100  ~100 ~100  ~100                                                                    | 13.4  13.4 13.4  13.4                                                                                        | 13.4  Yes  Yes  Yes Yes  Yes                                                         | [58,59]  [58,59]  [58,59] [58,59]  [58,59]             |
| SRT1720  SRT1720  SRT1720 SRT1720  SRT1720                                 |                                                        | Sirt1  Sirt1  Sirt1  Sirt1 Sirt1                                                                                                      | ~0.10  ~0.10  ~0.10  ~0.10 ~0.10                                                               | ~8  ~8  ~8 ~8                                                                                                | ~8  Yes  Yes  Yes  Yes Yes                                                           | [59]  [59]  [59]  [59] [59]                            |
| SRT2104  SRT2104  SRT2104  SRT2104 SRT2104                                 |                                                        | Sirt1  Sirt1  Sirt1  Sirt1  Sirt1                                                                                                     | 0.43  0.43  a 0.43  a 0.43  a 0.43 a                                                           | a ~2  ~2  ~2  ~2                                                                                             | ~2  Yes  Yes  Yes  Yes  Yes                                                          | [60]  [60]  [60]  [60]  [60]                           |

Molecules

2024

,

29

, x FOR PEER REVIEW

5 of 39

Molecules

2024

,

29

, x FOR PEER REVIEW

5 of 39

Molecules

2024

,

29

, x FOR PEER REVIEW

5 of 39

Molecules

Molecules

Molecules

Molecules

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

2.1. Natural Product Sirtuin Activators 2.1. Natural Product Sirtuin Activators 2.1. Natural Product Sirtuin Activators deacetylase activity. deacetylase activity. deacetylase activity. = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** = deacetylase activity. a = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** = deacetylase activity. a = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** = deacetylase activity.

Table 1. Cont.

| Compound Name 2024 ,  29 , x FOR PEER REVIEW  2024 ,  29 , x FOR PEER REVIEW  Molecules  2024 ,  29 2024 ,  29 , x FOR PEER REVIEW  2024 ,  29 , x FOR PEER REVIEW   | Structure , x FOR PEER REVIEW                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Targeted Sirtuin Isoform                                     | Potency (EC 50 ), µ M                                                                                                                | Fold Activation                                                                                                                                                 | Cell Active                                 | Citations 5 of 39  5 of 39  5 of 39  5 of 39  5 of 39   |
|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------|---------------------------------------------------------|
| Honokiol Honokiol  Honokiol  Honokiol  Honokiol  Honokiol  Honokiol  Honokiol  Honokiol                                                                              |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Sirt3  Sirt3 Sirt3  Sirt3  Sirt3  Sirt3  Sirt3  Sirt3  Sirt3 | N.D.  N.D. N.D.  N.D.  N.D.  N.D.  N.D.  N.D.  N.D.                                                                                  | ~2  ~2 ~2  ~2  ~2  ~2  ~2  ~2  ~2                                                                                                                               | Yes  Yes Yes  Yes  Yes  Yes  Yes  Yes  Yes  | [61]  [61] [61]  [61]  [61]  [61]  [61]  [61]  [61]     |
| ADTL-SA1215  ADTL-SA1215 ADTL-SA1215  ADTL-SA1215  ADTL-SA1215  ADTL-SA1215  ADTL-SA1215  ADTL-SA1215  ADTL-SA1215                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Sirt3  Sirt3  Sirt3 Sirt3  Sirt3  Sirt3  Sirt3  Sirt3  Sirt3 | 0.21  0.21  0.21 0.21  0.21  0.21  0.21  0.21  0.21                                                                                  | 2  2  2 2  2  2  2  2  2                                                                                                                                        | Yes  Yes  Yes Yes  Yes  Yes  Yes  Yes  Yes  | [62]  [62]  [62] [62]  [62]  [62]  [62]  [62]  [62]     |
| Compound  31 Compound  31 Compound 31 Compound  31 Compound  31 Compound  31 Compound  31 Compound  31 Compound  31                                                  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Sirt3  Sirt3  Sirt3  Sirt3 Sirt3  Sirt3  Sirt3  Sirt3  Sirt3 | ~100-200  ~100-200  ~100-200  ~100-200 ~100-200  ~100-200  ~100-200  ~100-200                                                        | ~5-1000  ~5-1000  ~5-1000  ~5-1000 ~5-1000  ~5-1000  ~100-200  ~5-1000  ~5-1000  ~5-1000                                                                        | Yes  Yes  Yes  Yes Yes  Yes  Yes  Yes  Yes  | [63]  [63]  [63]  [63] [63]  [63]  [63]  [63]  [63]     |
| Compound  30 Compound  30 Compound  30 Compound 30 Compound  30 Compound  30 Compound  30 Compound  30 Compound  30                                                  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Sirt5  Sirt5  Sirt5  Sirt5  Sirt5 Sirt5  Sirt5  Sirt5  Sirt5 | ~40  ~40  ~40  ~40  ~40 ~40  ~40  ~40  ~40                                                                                           | ~3-5  ~3-5  ~3-5  ~3-5  ~3-5 ~3-5  ~3-5  ~3-5  ~3-5                                                                                                             | Yes  Yes  Yes  Yes  Yes Yes  Yes  Yes  Yes  | [63]  [63]  [63]  [63]  [63] [63]  [63]  [63]  [63]     |
| UBCS039  UBCS039  UBCS039  UBCS039  UBCS039 UBCS039  UBCS039  UBCS039  UBCS039                                                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Sirt6  Sirt6  Sirt6  Sirt6  Sirt6  Sirt6 Sirt6  Sirt6  Sirt6 | 38  38  38  38  38  38 38  38  38                                                                                                    | 2  2  2  2  2  2 2  2  2                                                                                                                                        | Yes  Yes  Yes  Yes  Yes  Yes Yes  Yes  Yes  | [64]  [64]  [64]  [64]  [64]  [64] [64]  [64]  [64]     |
| LPA  LPA  LPA  LPA  LPA  LPA LPA  LPA  LPA                                                                                                                           | Sirt6  Sirt6  Sirt6  Sirt6  Sirt6                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Sirt6  Sirt6  Sirt6 Sirt6                                    | 25  25  25  25  25  25  25 25  25                                                                                                    | 48  48  48  48  48  48  48 48  48                                                                                                                               | Yes  Yes  Yes  Yes  Yes  Yes  Yes Yes  Yes  | [65]  [65]  [65]  [65]  [65]  [65]  [65] [65]  [65]     |
| MDL-811  MDL-811  MDL-811  MDL-811  MDL-811  MDL-811  MDL-811 MDL-811  MDL-811                                                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Sirt6  Sirt6  Sirt6  Sirt6  Sirt6  Sirt6  Sirt6  Sirt6 Sirt6 | 7.1  7.1  7.1  7.1  7.1  7.1  7.1  7.1 7.1                                                                                           | 44  44  44  44  44  44  44  44 44                                                                                                                               | Yes  Yes  Yes  Yes  Yes  Yes  Yes  Yes Yes  | [66]  [66]  [66]  [66]  [66]  [66]  [66]  [66] [66]     |
| Compound  12q Compound  12q Compound  12q Compound  12q Compound  12q Compound  12q Compound  12q Compound 12q Compound  12q                                         | Sirt6  a  = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** =  deacetylase activity.  Sirt6  a  = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** =  deacetylase activity.  Sirt6  a  = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** =  Sirt6  a  = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** =  Sirt6  a  = potency as assessed using EC1.5. N.D. = potency not determined. * = demyristoylase activity. ** = | Sirt6  Sirt6  Sirt6  Sirt6                                   | 8.9 *  5.4 **  8.9 *  5.4 **  8.9 *  5.4 **  8.9 *  5.4 **  8.9 *  5.4 **  8.9 *  5.4 **  8.9 *  5.4 **  8.9 *  5.4 **  8.9 * 5.4 ** | ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 *  ~40 **  ~12-18 * ~40 ** | Yes  Yes  Yes  Yes  Yes  Yes  Yes  Yes  Yes | [67]  [67]  [67]  [67]  [67]  [67]  [67]  [67]  [67]    |

2.1. Natural Product Sirtuin Activators

2.1. Natural Product Sirtuin Activators

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

2.1. Natural Product Sirtuin Activators

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

2.1. Natural Product Sirtuin Activators

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

2.1. Natural Product Sirtuin Activators

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

these observations have been contested [69,70]. Honokiol (Table 1), a biphenolic molecule

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

these observations have been contested [69,70]. Honokiol (Table 1), a biphenolic molecule

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

these observations have been contested [69,70]. Honokiol (Table 1), a biphenolic molecule

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a

these observations have been contested [69,70]. Honokiol (Table 1), a biphenolic molecule

these observations have been contested [69,70]. Honokiol (Table 1), a biphenolic molecule

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although

## 2.1. Natural Product Sirtuin Activators

Several natural products activate sirtuins [58]. For example, resveratrol (Table 1), a polyphenol found in grapes and wine [58,68], is a reported Sirt1 activator [58], although these observations have been contested [69,70]. Honokiol (Table 1), a biphenolic molecule found in the bark of magnolia trees, is an alleged Sirt3 activator [61]; however, we have been unable to demonstrate direct Sirt3 activation by honokiol in vitro (unpublished data). Consistent with Sirt6 activation by biological FFAs [28], a screen of fatty acids and lipids found that oleoyl-lysophosphatidic acid (LPA) (Table 1) increased Sirt6 deacetylase activity ~48-fold [65]. LPA also attenuates Sirt1 protein expression [71]. Considering that most natural product activators of sirtuins exhibit several limitations, particularly low specificity [52], we focus our assessment of current sirtuin activators on synthetic small molecules.

## 2.2. Synthetic Small Molecule Sirt1 Activators

The abundant caveats identified with using resveratrol as a selective Sirt1 activator [59,69,70,72-77] necessitated the development of synthetic Sirt1 STACs with improved potency and pharmacological properties. Initial attempts to generate synthetic Sirt1 STACs involved modifying the 4 ' -hydroxyl of resveratrol to reduce redox potential [78]. These efforts resulted in resveratrol derivatives with moderately enhanced stability or Sirt1 activation [78]. Resveratrol has also been modified with the cationic/lipophilic triphenylphosphonium (TPP + ) group for selective mitochondrial localization [79]. However, given that resveratrol inhibits Sirt3 deacetylase and Sirt5 desuccinylase activities [77], even mitochondrially targeted resveratrol is not predicted to be an effective activator of mitochondrial sirtuins.

The next generation of STACs, structurally unrelated to resveratrol, were identified using high-throughput screening with a fluorescent Sirt1 acetyl-lysine substrate deacetylation assay [80]. This yielded 'hit' molecules ~10-fold more potent than resveratrol in cellular assays [80]. Sirtris (now part of GlaxoSmithKline; GSK) employed similar high-throughput methods using fluorescence polarization and mass spectrometry; 'hit' molecules from this screen were up to 1000-fold more potent than resveratrol [59,81,82]. Although Sirt1-3 are closely related sirtuin isoforms [83], optimized Sirtris Sirt1 STACs SRT1460, SRT1720 (Table 1), and SRT2183 displayed up to 833- and 1875-fold selectivity for Sirt1 over Sirt2/3 in vitro, and demonstrated a Sirt1-dependent decrease in p53 acetylation in the U2OS osteosarcoma cell line [59]. Kinetic analysis revealed that Sirtris compounds increased Sirt1 deacetylase activity by lowering the Michaelis constant ( K M ) of Sirt1 for acetyl-lysine substrates without affecting the K M for NAD + [59]. SRT1720 also inhibits Sirt3 by competing with the acyl-lysine substrate for binding [84], suggesting that the mechanism of Sirt1 activation by SRT1720 is unique.

Despite debates regarding the legitimacy of direct STAC-mediated Sirt1 activation [69,70,85], several studies have used biophysical and structural approaches to deconvolute some of the controversies [39,86,87]. STAC activation of Sirt1 depends on aromatic rings in its substrates [86] from fluorophores or natural aromatic amino acids at specific positions relative to the acylated lysine residue [87]. Consistent with this mechanism, STAC-mediated Sirt1 activation was decreased through the mutation of the aromatic amino acids' C -terminal to the acetyl-lysine residue in acetylated Sirt1 peptide substrate proliferator-activated receptor γ coactivator 1 α (PGC1 α ) and human fork-head box O3a (FOXO3a) [87]. Additionally, N -terminal truncations of Sirt1 adjacent to the catalytic core decreased STAC binding to Sirt1 and corresponding Sirt1 activation, indicating the region N -terminal to the Sirt1 active site (amino acid residues 1-224) is required for STAC-mediated activation [87]. An E230K point mutation also attenuated STAC-mediated activation of Sirt1 but did not affect baseline Sirt1 enzyme kinetics or subcellular localization [87], further demonstrating that STAC-mediated Sirt1 activation requires the Sirt1 N -terminus. Moreover, myoblasts lacking Sirt1 or expressing Sirt1 E230K failed to display markers of STAC-mediated activation [87]. Additional structural analysis using site-directed mutagenesis coupled with hydrogen-deuterium exchange mass spectrometry revealed that Sirt1 contains a unique

STAC-binding domain (SBD) N -terminal to its catalytic core (amino acid residues 183-229) (Figure 2) [39], consistent with the previously reported structural requirements for STACmediated Sirt1 activation [87]. Together, these data suggest that all known Sirt1 activators operate by a direct allosteric activation mechanism that includes binding to the Sirt1 SBD and a Sirt1 substrate preference for hydrophobic and aromatic amino acids C -terminal to the acyl-lysine residue [39,86,87].

The initial use of Sirtris Sirt1 STACs in vivo is promising. Oral gavage administration of SRT1720 to diet-induced or genetically obese diabetic mice reduced insulin resistance and improved glucose tolerance [59]. Furthermore, SRT1720 extended lifespan, improved metabolic and inflammatory markers, and delayed the onset of age-related pathologies in adult mice fed a high-fat or standard diet [88,89]. However, the effects of Sirt1 STACs in vivo may be mediated by proteins other than Sirt1. Notably, the diabetes-mitigating effects of SRT1720 in mice may depend on AMP-activated protein kinase (AMPK) activation since SRT1720 can activate AMPK independently of Sirt1 [90]. Moreover, SRT1460, SRT1720, and SRT2183 have all demonstrated off-target interactions with numerous receptors, enzymes, transporters, and ion channels [85].

Nonetheless, Sirtris Sirt1 STACs have entered clinical trials to assess pharmacokinetics, tolerability, and efficacy in age-related pathologies. Based on pharmacokinetic properties in animal studies, SRT2104 (Table 1) was the first Sirtris Sirt1 STAC deemed to have translational clinical potential [60]. In phase I clinical trials, SRT2104 was well tolerated by healthy volunteers but exhibited poor oral bioavailability [91]. Consistent with Sirt1 activation and despite its low bioavailability, month-long administration of SRT2104 in elderly patients decreased serum cholesterol and triglyceride levels and increased the HDL/LDL ratio [92]. SRT2104 has also been clinically investigated in cardiovascular function [93], type 2 diabetes [94], and other inflammatory diseases [95-97]. Due to the poor bioavailability of SRT2104 [91], these subsequent clinical trials exhibited inter-subject variability in drug exposure and mixed outcomes [93,94,96,97]. Accordingly, once their pharmacological properties are optimized, SRT2104 and analogous Sirt1 STACs will likely demonstrate greater therapeutic benefits for age-related pathologies.

## 2.3. Synthetic Small Molecule Sirt3 and Sirt5 Activators

While Sirt1 small molecule activators have been widely reported, small molecule activators of other sirtuin isoforms are slowly emerging. High-throughput screening and structure-based design generated ADTL-SA1215 (Table 1), an allosteric Sirt3 activator with high selectivity over Sirt2/5 and moderate selectivity over Sirt1 in vitro [62]. Selectivity is likely conferred by the unique interaction of ADTL-SA1215 with Sirt3 via a hydrophobic allosteric pocket near the Sirt3 acyl-lysine binding site [62]. ADTL-SA1215 exhibited Sirt3dependent cellular activity, increasing Sirt3 deacetylase activity ~two-fold and decreasing acetylation of the Sirt3 substrate manganese superoxide dismutase (MnSOD) at lysine-68 and -122 in the MDA-MB-231 triple-negative breast cancer cell line in a concentrationdependent manner [62]. ADTL-SA1215 also induced autophagy of MDA-MB-231 cells both in vitro and in vivo in a murine xenograft model of triple-negative breast cancer via Sirt3mediated regulation of multiple autophagy-related proteins [62], confirming the efficacy of Sirt3 activation in this cancer model.

1,4-dihydropyridines [98,99], first reported as weak Sirt1-3 activators [98], have recently been optimized with structure-activity relationship (SAR) studies to develop nextgeneration Sirt3 activators [63]. Compound 31 , the best identified Sirt3 activator, increased Sirt3 deacetylase activity up to ~1000-fold and exhibited strong Sirt3 selectivity in vitro, with minimal impacts on Sirt1/2/5/6 and only partial inhibition of Sirt4 at 100 µ M[63]. Consistent with cellular Sirt3 activation, Compound 31 decreased acetylation of the Sirt3 target glutamate dehydrogenase in MDA-MB-231 cells [63]. Further SAR studies based on Compound 31 have yielded additional 1,4-dihydropyridine-based Sirt3 activators with demonstrated capacity to decrease viability and colony formation in MDA-MB-231 cells and the CAL-62 thyroid cancer cell line [100].

The same SAR studies that generated 1,4-dihydropyridine Compound 31 also developed Compound 30 as a Sirt5 activator [63]. Compound 30 increased Sirt5 desuccinylase activity up to ~5 fold and maintained Sirt5 selectivity in vitro, showing little effect on Sirt1-3 and only partial inhibition of Sirt4/6 at 100 µ M[63]. Decreased succinylation of the Sirt5 substrates pyruvate dehydrogenase subunit A1 (PDHA1) derived from porcine heart tissue and glutaminase in MDA-MB-231 cells confirmed that Compound 30 is cell active [63]. An additional 1,4-dihydropyridine Sirt5 activator, MC3138, did not impact Sirt1/3 deacetylase activity at a concentration of 100 µ Min vitro and demonstrated on-target cellular Sirt5 activation via decreased acylation of the Sirt5 substrate glutamate oxaloacetate transaminase 1 (GOT1) in pancreatic cancer cell lines [101]. Notably, MC3138 reduced tumor size in a mouse xenograft model of pancreatic cancer alone and in combination with gemcitabine with no significant side effects [101]. Despite the novelty of these new 1,4-dihydropyridine Sirt3 activators, the lack of animal model testing demonstrates the current superiority of ADTL-SA1215 as the Sirt3 activator of choice for probing the roles of Sirt3 in cancer. However, the promising in vitro and in vivo data derived from 1,4-dihydropyridine Sirt5 activators highlights the need for further studies to enhance their solubility and potency.

## 2.4. Synthetic Small Molecule Sirt6 Activators

To further the development of non-Sirt1 STACs, in silico screening of a small molecule library against Sirt2/3/5/6, coupled with hit testing in a fluorescent deacetylation assay, identified a pyrrolo[1,2a ]quinoxaline compound that weakly stimulated Sirt6 deacetylase activity [102]. Derivatization of this parent compound yielded UBCS039 (Table 1), which increased Sirt6 deacetylase activity ~2-3.5 fold without significantly impacting Sirt1-3 deacetylase activity in vitro [64]. Such selectivity is likely derived from the binding of UBCS039 to the hydrophobic portion of the extended Sirt6 acylated substrate binding cleft that facilitates binding to longer acylated substrate chains or fatty acid activators [28,64]. Consistent with a mechanism of Sirt6 activation, UBCS039 increased deacetylation of the Sirt6 substrates histone H3K9 and H3K18 in vitro [64]. Cellular activity of UBCS039 was demonstrated by decreased H3K9 acetylation and Sirt6-mediated increases in autophagosome formation and autophagy-induced cell death in several cancer cell lines [103]. However, as UBCS039 also activates Sirt5 desuccinylase activity by ~two-fold in vitro [64], further optimization is required to enhance UBCS039 sirtuin isoform specificity before it can serve as an effective Sirt6 activator and biological probe. Recent SAR studies based on UBCS039 yielded pyrrolo[1,2a ]quinoxaline analogs that increase Sirt6 deacetylase activity up to ~seven-fold, retain selectivity for Sirt6 over Sirt1-3, and do not impact Sirt5 desuccinylase activity [104].

Allostery prediction, virtual library screening, and optimization of a phenylsulfonamide scaffold have also yielded MDL-800 and MDL-801 as Sirt6 activators [105]. Both compounds increased Sirt6 deacetylase activity up to 22-fold and did not modulate Sirt6 demyristoylase or ADP-ribosyltransferase activities [105]. In addition, neither compound had any effects on Sirt1/3/4 and only minimal impacts on Sirt2/5/7, requiring concentrations at least 17 times higher to induce activation [105]. Modeling demonstrated that MDL-800 and MDL-801 activate Sirt6 by allosterically binding to a hydrophobic N -terminal pocket defined by the N -terminal residues 1-7, V70, E74, F82, F86, V153, and M157 [105]. Notably, MDL-800 decreased acetylation of the Sirt6 substrates histone H3K9 and H3K56 in human hepatocellular carcinoma (HCC) and non-small cell lung cancer (NSCLC) cell lines in a concentration-dependent manner and induced Sirt6-mediated G 0 -G 1 phase cell cycle arrest [105,106]. Furthermore, MDL-800 decreased the growth of HCC and NSCLC xenografts in mice [105,106]. SAR studies based on MDL-800 and MDL-801 have yielded analog MDL-811 (Table 1), which activates Sirt6 deacetylase activity two times better than MDL-800 and retains selectivity for Sirt6 over Sirt1/2/3/5/7 in vitro [107]. MDL-811 also exhibits potent cellular activity, decreasing tumor formation in a spontaneous mouse model of colorectal cancer [107] and attenuating lipopolysaccharide-induced neuroinflammation and stroke-like brain damage in mice [66].

Molecules

2024

,

29

, x FOR PEER REVIEW

9 of 39

Additional virtual screening and SAR analyses based on a quinoline scaffold resulted in Sirt6 activator Compound 12q (Table 1) [67]. Exhibiting nearly 300-fold selectivity for Sirt6 over Sirt1/2/3/5, Compound 12q activated Sirt6 deacetylase activity two times better than MDL-800 and uniquely stimulated Sirt6 demyristoylase activity up to 15 fold [67]. Compound 12q cellular activity was confirmed with a concentrationdependent decrease in acetylation of the Sirt6 substrate histone H3K9 in pancreatic cancer cell lines [67]. Compound 12q also decreased the growth of pancreatic cancer xenografts in mice [67]. The in vivo efficacy of the MDL compound family and Compound 12q demonstrate that selective Sirt6 activation has therapeutic potential in multiple types of cancer and neurological damage. than MDL-800 and uniquely stimulated Sirt6 demyristoylase activity up to 15 fold [67]. Compound 12q cellular activity was con fi rmed with a concentration-dependent decrease in acetylation of the Sirt6 substrate histone H3K9 in pancreatic cancer cell lines [67]. Compound 12q also decreased the growth of pancreatic cancer xenografts in mice [67]. The in vivo e ffi cacy of the MDL compound family and Compound 12q demonstrate that selective Sirt6 activation has therapeutic potential in multiple types of cancer and neurological damage. 2.5. Alternative Mechanisms of Sirtuin Activation

## 2.5. Alternative Mechanisms of Sirtuin Activation While sirtuin activation strategies primarily focus on direct molecular modulation of

While sirtuin activation strategies primarily focus on direct molecular modulation of sirtuin-substrate binding, alternative mechanisms to enhance sirtuin activity exist. Depleting the sirtuin co-substrate NAD + (Figure 4) correlates with the aging-related attenuation of sirtuin activity [108,109]. Therefore, boosting intracellular NAD + levels through supplementation of more cell-permeable NAD + precursors like nicotinamide riboside and nicotinamide mononucleotide (Figure 4) [110,111] or via inhibition of NAD + -consuming enzymes (i.e., poly-ADP-ribose polymerases (PARPs) [112] and NADase CD38 [113]) could fortify sirtuin activity to lessen aging-related pathologies. Additionally, sirtuins are autoinhibited by nicotinamide (NAM; Figure 4), a product of sirtuin-catalyzed deacylation and a noncompetitive inhibitor of sirtuin deacylase activity [114]. Therefore, relieving NAM autoinhibition with isonicotinamide (Figure 4), an NAM isostere that competes with NAM for binding to sirtuins, could have similar restorative effects on sirtuin activity [115]. sirtuin-substrate binding, alternative mechanisms to enhance sirtuin activity exist. Depleting the sirtuin co-substrate NAD + (Figure 4) correlates with the aging-related a tt enuation of sirtuin activity [108,109]. Therefore, boosting intracellular NAD + levels through supplementation of more cell-permeable NAD + precursors like nicotinamide riboside and nicotinamide mononucleotide (Figure 4) [110,111] or via inhibition of NAD + -consuming enzymes (i.e., poly-ADP-ribose polymerases (PARPs) [112] and NADase CD38 [113]) could fortify sirtuin activity to lessen aging-related pathologies. Additionally, sirtuins are autoinhibited by nicotinamide (NAM; Figure 4), a product of sirtuin-catalyzed deacylation and a noncompetitive inhibitor of sirtuin deacylase activity [114]. Therefore, relieving NAM autoinhibition with isonicotinamide (Figure 4), an NAM isostere that competes with NAM for binding to sirtuins, could have similar restorative e ff ects on sirtuin activity [115].

Figure 4. Nicotinamide biomolecules are endogenous regulators of sirtuin deacylation. Structures of key nicotinamide-based biomolecules implicated in sirtuin activity. Figure 4. Nicotinamide biomolecules are endogenous regulators of sirtuin deacylation. Structures of key nicotinamide-based biomolecules implicated in sirtuin activity.

<!-- image -->

Sirtuins can also be modulated by post-translational modi fi cations, including phosphorylation, methylation, and sumoylation [45]. In particular, sirtuin activity can be inhibited  by  post-translational  modi fi cations  resulting  from  cellular  oxidative  stress [44,116]. For example, sirtuins are di ff erentially sensitive to inhibition by cysteine S -nitrosation, glutathionylation, and sulfenylation [116]. We recently reviewed the e ff ects of oxidative post-translational modi fi cations on sirtuin structure and function [44]. Accordingly, using speci fi c oxidant scavengers corresponding to the inhibitory oxidative modi fi -cation, inhibitors of oxidant-producing enzymes (e.g., nitric oxide synthase inhibitors), or using antioxidants more generally could indirectly restore sirtuin activity during oxidaSirtuins can also be modulated by post-translational modifications, including phosphorylation, methylation, and sumoylation [45]. In particular, sirtuin activity can be inhibited by post-translational modifications resulting from cellular oxidative stress [44,116]. For example, sirtuins are differentially sensitive to inhibition by cysteine S -nitrosation, glutathionylation, and sulfenylation [116]. We recently reviewed the effects of oxidative post-translational modifications on sirtuin structure and function [44]. Accordingly, using specific oxidant scavengers corresponding to the inhibitory oxidative modification, inhibitors of oxidant-producing enzymes (e.g., nitric oxide synthase inhibitors), or using antioxidants more generally could indirectly restore sirtuin activity during oxidative stress.

tive stress. The use of epigenetic small molecules to increase sirtuin transcription and protein The use of epigenetic small molecules to increase sirtuin transcription and protein expression provide another potential mechanism for sirtuin activation [117-119]. Indeed,

expression provide another potential mechanism for sirtuin activation [117-119]. Indeed,

selective inhibition of the bromodomain and extraterminal domain (BET) protein family

Molecules

2024

,

29

, x FOR PEER REVIEW

10 of 39

, x FOR PEER REVIEW

10 of 39

, x FOR PEER REVIEW

, x FOR PEER REVIEW

selective inhibition of the bromodomain and extraterminal domain (BET) protein family by BET bromodomain inhibitors (e.g., JQ1, I-BET151, and I-BET762) elevates Sirt1 transcript and protein levels in a variety of cell lines (unpublished data) [117-119]. BET bromodomain inhibition by JQ1 also increases Sirt1-dependent p53 deacetylation without directly regulating Sirt1 deacetylase activity [117]. However, BET bromodomain inhibitors decrease the protein expression of Sirt1 and other sirtuins in additional cell lines [118,119] and broadly impact gene expression across many transcriptional networks [120]; thus, studies with more selective BET bromodomain inhibitors are required to fully elucidate the impact of BET inhibition on sirtuin protein expression and reduce off-target epigenetic effects. by BET bromodomain inhibitors (e.g., JQ1, I-BET151, and I-BET762) elevates Sirt1 transcript and protein levels in a variety of cell lines (unpublished data) [117-119]. BET bromodomain inhibition by JQ1 also increases Sirt1-dependent p53 deacetylation without directly regulating Sirt1 deacetylase activity [117]. However, BET bromodomain inhibitors decrease the protein expression of Sirt1 and other sirtuins in additional cell lines [118,119] and broadly impact gene expression across many transcriptional networks [120]; thus, studies with more selective BET bromodomain inhibitors are required to fully elucidate the impact of BET inhibition on sirtuin protein expression and reduce o ff -target epigenetic e ff ects. by BET bromodomain inhibitors (e.g., JQ1, I-BET151, and I-BET762) elevates Sirt1 transcript and protein levels in a variety of cell lines (unpublished data) [117-119]. BET bromodomain inhibition by JQ1 also increases Sirt1-dependent p53 deacetylation without directly regulating Sirt1 deacetylase activity [117]. However, BET bromodomain inhibitors decrease the protein expression of Sirt1 and other sirtuins in additional cell lines [118,119] and broadly impact gene expression across many transcriptional networks [120]; thus, studies with more selective BET bromodomain inhibitors are required to fully elucidate the impact of BET inhibition on sirtuin protein expression and reduce o ff -target epigenetic e ff ects. by BET bromodomain inhibitors (e.g., JQ1, I-BET151, and I-BET762) elevates Sirt1 transcript and protein levels in a variety of cell lines (unpublished data) [117-119]. BET bromodomain inhibition by JQ1 also increases Sirt1-dependent p53 deacetylation without directly regulating Sirt1 deacetylase activity [117]. However, BET bromodomain inhibitors decrease the protein expression of Sirt1 and other sirtuins in additional cell lines [118,119] and broadly impact gene expression across many transcriptional networks [120]; thus, studies with more selective BET bromodomain inhibitors are required to fully elucidate the impact of BET inhibition on sirtuin protein expression and reduce o ff -target epigenetic e ff ects. by BET bromodomain inhibitors (e.g., JQ1, I-BET151, and I-BET762) elevates Sirt1 transcript and protein levels in a variety of cell lines (unpublished data) [117-119]. BET bromodomain inhibition by JQ1 also increases Sirt1-dependent p53 deacetylation without directly regulating Sirt1 deacetylase activity [117]. However, BET bromodomain inhibitors decrease the protein expression of Sirt1 and other sirtuins in additional cell lines [118,119] and broadly impact gene expression across many transcriptional networks [120]; thus, studies with more selective BET bromodomain inhibitors are required to fully elucidate the impact of BET inhibition on sirtuin protein expression and reduce o ff -target epigenetic by BET bromodomain inhibitors (e.g., JQ1, I-BET151, and I-BET762) elevates Sirt1 transcript and protein levels in a variety of cell lines (unpublished data) [117-119]. BET bromodomain inhibition by JQ1 also increases Sirt1-dependent p53 deacetylation without directly regulating Sirt1 deacetylase activity [117]. However, BET bromodomain inhibitors decrease the protein expression of Sirt1 and other sirtuins in additional cell lines [118,119] and broadly impact gene expression across many transcriptional networks [120]; thus, studies with more selective BET bromodomain inhibitors are required to fully elucidate the impact of BET inhibition on sirtuin protein expression and reduce o ff -target epigenetic

e

ff

ects.

## 3. Small Molecule Sirtuin Inhibitors 3. Small Molecule Sirtuin Inhibitors 3. Small Molecule Sirtuin Inhibitors

e

ff

ects.

A large class of sirtuin inhibitors consists of small molecules (Table 2) that can be distinguished by their hypothesized or structurally determined sirtuin binding sites, as well as their kinetics of inhibition relative to sirtuin acylated substrates and/or NAD + . Extensive studies based on computational modeling, structural biology approaches, and biochemical data delineate the specific binding sites and inhibition mechanisms of these compounds [121-125]. Such studies inform improvements in the overall potency and isoform selectivity of existing small molecule inhibitors and facilitate testing of inhibitors with high translational potential in aging-related diseases. A large class of sirtuin inhibitors consists of small molecules (Table 2) that can be distinguished by their hypothesized or structurally determined sirtuin binding sites, as well as their kinetics of inhibition relative to sirtuin acylated substrates and/or NAD + . Extensive studies based on computational modeling, structural biology approaches, and biochemical data delineate the speci fi c binding sites and inhibition mechanisms of these compounds [121-125]. Such studies inform improvements in the overall potency and isoform selectivity of existing small molecule inhibitors and facilitate testing of inhibitors with high translational potential in aging-related diseases. A large class of sirtuin inhibitors consists of small molecules (Table 2) that can be distinguished by their hypothesized or structurally determined sirtuin binding sites, as well as their kinetics of inhibition relative to sirtuin acylated substrates and/or NAD + . Extensive studies based on computational modeling, structural biology approaches, and biochemical data delineate the speci fi c binding sites and inhibition mechanisms of these compounds [121-125]. Such studies inform improvements in the overall potency and isoform selectivity of existing small molecule inhibitors and facilitate testing of inhibitors with high translational potential in aging-related diseases. 3. Small Molecule Sirtuin Inhibitors A large class of sirtuin inhibitors consists of small molecules (Table 2) that can be distinguished by their hypothesized or structurally determined sirtuin binding sites, as well as their kinetics of inhibition relative to sirtuin acylated substrates and/or NAD + . Extensive studies based on computational modeling, structural biology approaches, and biochemical data delineate the speci fi c binding sites and inhibition mechanisms of these compounds [121-125]. Such studies inform improvements in the overall potency and isoform selectivity of existing small molecule inhibitors and facilitate testing of inhibitors 3. Small Molecule Sirtuin Inhibitors A large class of sirtuin inhibitors consists of small molecules (Table 2) that can be distinguished by their hypothesized or structurally determined sirtuin binding sites, as well as their kinetics of inhibition relative to sirtuin acylated substrates and/or NAD + . Extensive studies based on computational modeling, structural biology approaches, and biochemical data delineate the speci fi c binding sites and inhibition mechanisms of these compounds [121-125]. Such studies inform improvements in the overall potency and isoform selectivity of existing small molecule inhibitors and facilitate testing of inhibitors 3. Small Molecule Sirtuin Inhibitors A large class of sirtuin inhibitors consists of small molecules (Table 2) that can be distinguished by their hypothesized or structurally determined sirtuin binding sites, as well as their kinetics of inhibition relative to sirtuin acylated substrates and/or NAD + . Extensive studies based on computational modeling, structural biology approaches, and biochemical data delineate the speci fi c binding sites and inhibition mechanisms of these compounds [121-125]. Such studies inform improvements in the overall potency and isoform selectivity of existing small molecule inhibitors and facilitate testing of inhibitors

Table 2. Select small molecule inhibitors of sirtuin deacylases. Table 2. Select small molecule inhibitors of sirtuin deacylases. Table 2. Select small molecule inhibitors of sirtuin deacylases. with high translational potential in aging-related diseases.

with high translational potential in aging-related diseases.

with high translational potential in aging-related diseases.

Table 2.

Select small molecule inhibitors of sirtuin deacylases.

Select small molecule inhibitors of sirtuin deacylases.

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

Table 2.

| Compound Name Compound  Name  Compound  Name  Compound  Name  Compound  Name  Compound   | Structure Structure  Structure  Structure  Structure  Table 2.   | Targeted Sirtuin Isoform Targeted Sirtuin  Isoform  Targeted Sirtuin  Isoform  Targeted Sirtuin  Isoform  Targeted Sirtuin  Isoform  Select small molecule inhibitors of sirtuin deacylases.  Targeted Sirtuin   | Potency (IC 50 ), µ M Potency  (IC50),  μ M  Potency  (IC50),  μ M  Potency  (IC50),  μ M  Potency  (IC50),  μ M  Potency   | Selectivity Selectivity  Selectivity  Selectivity  Selectivity  Selectivity                                                                                                | Cell Active Cell-Active  Citations  Cell-Active  Citations   | Citations Cell-Active  Citations  Cell-Active  Citations  Cell-Active  Citations   |
|------------------------------------------------------------------------------------------|------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|------------------------------------------------------------------------------------|
| ELT-31 ELT-31 ELT-31 ELT-31 ELT-31 Name  ELT-31                                          | Structure                                                        | Sirt1/2/3 Sirt1/2/3 Sirt1/2/3 Sirt1/2/3 Sirt1/2/3 Isoform  Sirt1/2/3                                                                                                                                             | 0.001-0.033 0.001-0.033 0.001-0.033 0.001-0.033 0.001-0.033 (IC50),  μ M  0.001-0.033                                       | Nonselective Nonselective Nonselective Nonselective Nonselective Nonselective                                                                                              | N.D. N.D. N.D. N.D. N.D. N.D.                                | [126] [126] [126] [126] [126] [126]                                                |
| EX-527  EX-527 EX-527  EX-527  EX-527  EX-527                                            |                                                                  | Sirt1  Sirt1  Sirt1 Sirt1  Sirt1  Sirt1                                                                                                                                                                          | 0.12  0.12  0.12 0.12  0.12  0.12                                                                                           | ~20-800 fold  over Sirt2/3  ~20-800 fold  over Sirt2/3  ~20-800 fold over Sirt2/3 ~20-800 fold  over Sirt2/3  ~20-800 fold  over Sirt2/3  ~20-800 fold  over Sirt2/3       | Yes  Yes  Yes Yes  Yes  Yes                                  | [127]  [127]  [127] [127]  [127]  [127]                                            |
| Compound  86 Compound  86 Compound 86 Compound  86 Compound  86 Compound  86             |                                                                  | Sirt2  Sirt2  Sirt2  Sirt2 Sirt2  Sirt2                                                                                                                                                                          | 0.02  0.02  0.02  0.02 0.02  0.02                                                                                           | ~100-400 fold  over Sirt1/3  ~100-400 fold  over Sirt1/3  ~100-400 fold  over Sirt1/3  ~100-400 fold over Sirt1/3 ~100-400 fold  over Sirt1/3  ~100-400 fold  over Sirt1/3 | Yes  Yes  Yes  Yes Yes  Yes                                  | [128]  [128]  [128]  [128] [128]  [128]                                            |
| AGK2  AGK2  AGK2  AGK2 AGK2  AGK2                                                        |                                                                  | Sirt2  Sirt2  Sirt2  Sirt2  Sirt2 Sirt2                                                                                                                                                                          | 3.5  3.5  3.5  3.5  3.5 3.5                                                                                                 | >14 fold over  Sirt1/3  >14 fold over  Sirt1/3  >14 fold over  Sirt1/3  >14 fold over  Sirt1/3  >14 fold over Sirt1/3 >14 fold over  Sirt1/3                               | Yes  Yes  Yes  Yes  Yes Yes                                  | [129]  [129]  [129]  [129]  [129] [129]                                            |
| AK7  AK7  AK7  AK7  AK7 AK7                                                              |                                                                  | Sirt2  Sirt2  Sirt2  Sirt2  Sirt2  Sirt2                                                                                                                                                                         | 15.5  15.5  15.5  15.5  15.5  15.5                                                                                          | >4 fold over  Sirt1/3  >4 fold over  Sirt1/3  >4 fold over  Sirt1/3  >4 fold over  Sirt1/3  >4 fold over  Sirt1/3  >4 fold over Sirt1/3                                    | Yes  Yes  Yes  Yes  Yes  Yes                                 | [130]  [130]  [130]  [130]  [130]  [130]                                           |

, x FOR PEER REVIEW

Molecules

2024

,

29

, x FOR PEER REVIEW

11 of 39

Molecules

2024

,

29

, x FOR PEER REVIEW

11 of 39

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

3.1. Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate Binding Site N.D. = cellular activity not determined. N.D. = cellular activity not determined.

Table 2. Cont.

| Compound Name Molecules  2024 ,  29 Molecules  2024 ,  29 Molecules  2024 ,  29                                                                                                                            | Structure , x FOR PEER REVIEW  , x FOR PEER REVIEW  , x FOR PEER REVIEW   | Targeted Sirtuin Isoform                | Potency (IC 50 ), µ M              | Selectivity                                                                                                                                                                                                                                                                                                                                                              | Cell Active                        | Citations 11 of 39                       |
|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|-----------------------------------------|------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|------------------------------------------|
| SirReal2 SirReal2  SirReal2  SirReal2  SirReal2  SirReal2                                                                                                                                                  | Sirt2                                                                     | Sirt2  Sirt2  Sirt2  Sirt2  Sirt2       | 0.14  0.14 0.14  0.14  0.14  0.14  | >1000 fold over  Sirt1/3/4/5/6  >1000 fold over Sirt1/3/4/5/6 >1000 fold over  Sirt1/3/4/5/6  >1000 fold over  Sirt1/3/4/5/6  >1000 fold over  Sirt1/3/4/5/6  >1000 fold over  Sirt1/3/4/5/6                                                                                                                                                                             | Yes  Yes Yes  Yes  Yes  Yes        | [124]  [124] [124]  [124]  [124]  [124]  |
| MIND4  MIND4 MIND4  MIND4  MIND4  MIND4                                                                                                                                                                    | Sirt2 Sirt2  Sirt2  Sirt2                                                 | Sirt2  Sirt2                            | 3.5  3.5  3.5 3.5  3.5  3.5        | Selective over  Sirt1/3  Selective over  Sirt1/3  Selective over Sirt1/3 Selective over  Sirt1/3  Selective over  Sirt1/3  Selective over  Sirt1/3                                                                                                                                                                                                                       | Yes  Yes  Yes Yes  Yes  Yes        | [131]  [131]  [131] [131]  [131]  [131]  |
| Compound  60 Compound  60 Compound 60 Compound  60 Compound  60 Compound  60                                                                                                                               |                                                                           | Sirt4  Sirt4  Sirt4  Sirt4 Sirt4  Sirt4 | 0.9  0.9  0.9  0.9 0.9  0.9        | Non-selective  over Sirt2; ~3.5- 5.5 fold over  Sirt1/3/5/6  Non-selective  over Sirt2; ~3.5- 5.5 fold over  Sirt1/3/5/6  Non-selective  over Sirt2; ~3.5- 5.5 fold over  Sirt1/3/5/6  Non-selective over Sirt2; ~3.5-5.5 fold over Sirt1/3/5/6 Non-selective  over Sirt2; ~3.5- 5.5 fold over  Sirt1/3/5/6  Non-selective  over Sirt2; ~3.5- 5.5 fold over  Sirt1/3/5/6 | Yes  Yes  Yes  Yes Yes  Yes        | [132]  [132]  [132]  [132] [132]  [132]  |
| Compound  69 Compound  69 Compound  69 Compound 69 Compound  69 Compound  69                                                                                                                               | Sirt4                                                                     | Sirt4  Sirt4  Sirt4  Sirt4  Sirt4       | 16  16  16  16  16 16              | ~2-3 fold over  Sirt1/2/3/5/6    ~2-3 fold over  Sirt1/2/3/5/6    ~2-3 fold over  Sirt1/2/3/5/6    ~2-3 fold over  Sirt1/2/3/5/6    ~2-3 fold over Sirt1/2/3/5/6 ~2-3 fold over  Sirt1/2/3/5/6                                                                                                                                                                           | Yes  Yes  Yes  Yes  Yes Yes        | [132]  [132]  [132]  [132]  [132] [132]  |
| Compound  47 N.D. = cellular activity not determined.  Compound  47 N.D. = cellular activity not determined.  Compound  47 N.D. = cellular activity not determined.  Compound  47 Compound 47 Compound  47 | Sirt5  Sirt5  N.D. = cellular activity not determined.  Sirt5  Sirt5      | Sirt5  Sirt5                            | 0.21  0.21  0.21  0.21  0.21  0.21 | >3800 fold Sirt5  over Sirt1/2/3/6  >3800 fold Sirt5  over Sirt1/2/3/6  >3800 fold Sirt5  over Sirt1/2/3/6  >3800 fold Sirt5  over Sirt1/2/3/6  >3800 fold Sirt5  over Sirt1/2/3/6  >3800 fold Sirt5 over Sirt1/2/3/6                                                                                                                                                    | N.D.  N.D.  N.D.  N.D.  N.D.  N.D. | [133]  [133]  [133]  [133]  [133]  [133] |

3.1. Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate Binding Site

## Competitive inhibition by small molecules at the sirtuin acylated substrate binding cleft increases the apparent K M for acyl-lysine substrates, which lowers the sirtuin deacylCompetitive inhibition by small molecules at the sirtuin acylated substrate binding Competitive inhibition by small molecules at the sirtuin acylated substrate binding 3.1. Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate Binding Site 3.1. Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate Binding Site 3.1. Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate Binding Site

3.1. Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate Binding Site

ase activity when substrate concentrations are not saturating. Hydroxynaphthaldehydes (e.g., sirtinol, salermide, splitomicin, and cambinol) comprise a large class of sirtuin inhibitors that impact the sirtuin catalytic mechanism in this manner with modest potency (average half-maximal inhibitory concentration (IC 50 ) of ~64 µM) [1,134-136] and reported apoptotic activity in cancer cell lines [134,136]. However, the selectivity of hydroxynaphthaldehydes for speci fi c sirtuin isoforms was not rigorously tested [134,136]. The be tt ercharacterized cambinol analogs have been optimized into compounds that selectively inhibit Sirt1 (IC 50 = 27 µM) with >7-fold selectivity for Sirt1 over Sirt2/3, Sirt2 (IC 50 = 13 µM) with >15-fold selectivity for Sirt2 over Sirt1/3, or Sirt3 (IC 50 = 6 µM) with ~5-7-fold selectivity for Sirt3 over Sirt1/2 [137]. Increased acetylation of Sirt1 substrate p53 and Sirt2 substrate α -tubulin demonstrated that selective Sirt1 and Sirt2 inhibitors possessed on-target cellular activity in the NCI-H460 NSCLC cell line [137]. Additional SAR studies of cambinol have generated open-ring cambinol analogs that selectively inhibit Sirt2 with nanomolar potency and moderate selectivity for Sirt2 over Sirt1/3 [138]. These cambinol analogs exhibited on-target Sirt2 inhibition in the NCI-H60 NSCLC cell line, as demonstrated by a concentration-dependent increase in α -tubulin acetylation [138]. The cambinol analogs also induced cytotoxicity in a panel of cancer cell lines [138]. A virtual screen of over 25 million compounds identi fi ed ICL-SIRT078 as a Sirt2 incleft increases the apparent K M for acyl-lysine substrates, which lowers the sirtuin deacylase activity when substrate concentrations are not saturating. Hydroxynaphthaldehydes (e.g., sirtinol, salermide, splitomicin, and cambinol) comprise a large class of sirtuin inhibitors that impact the sirtuin catalytic mechanism in this manner with modest potency (average half-maximal inhibitory concentration (IC 50 ) of ~64 µM) [1,134-136] and reported apoptotic activity in cancer cell lines [134,136]. However, the selectivity of hydroxynaphthaldehydes for speci fi c sirtuin isoforms was not rigorously tested [134,136]. The be tt ercharacterized cambinol analogs have been optimized into compounds that selectively inhibit Sirt1 (IC 50 = 27 µM) with >7-fold selectivity for Sirt1 over Sirt2/3, Sirt2 (IC 50 = 13 µM) with >15-fold selectivity for Sirt2 over Sirt1/3, or Sirt3 (IC 50 = 6 µM) with ~5-7-fold selectivity for Sirt3 over Sirt1/2 [137]. Increased acetylation of Sirt1 substrate p53 and Sirt2 substrate α -tubulin demonstrated that selective Sirt1 and Sirt2 inhibitors possessed on-target cellular activity in the NCI-H460 NSCLC cell line [137]. Additional SAR studies of cambinol have generated open-ring cambinol analogs that selectively inhibit Sirt2 with nanomolar potency and moderate selectivity for Sirt2 over Sirt1/3 [138]. These cambinol analogs exhibited on-target Sirt2 inhibition in the NCI-H60 NSCLC cell line, as demonstrated by a concentration-dependent increase in α -tubulin acetylation [138]. The cambinol analogs also induced cytotoxicity in a panel of cancer cell lines [138]. A virtual screen of over 25 million compounds identi fi ed ICL-SIRT078 as a Sirt2 incleft increases the apparent K M for acyl-lysine substrates, which lowers the sirtuin deacylase activity when substrate concentrations are not saturating. Hydroxynaphthaldehydes (e.g., sirtinol, salermide, splitomicin, and cambinol) comprise a large class of sirtuin inhibitors that impact the sirtuin catalytic mechanism in this manner with modest potency (average half-maximal inhibitory concentration (IC 50 ) of ~64 µM) [1,134-136] and reported apoptotic activity in cancer cell lines [134,136]. However, the selectivity of hydroxynaphthaldehydes for speci fi c sirtuin isoforms was not rigorously tested [134,136]. The be tt ercharacterized cambinol analogs have been optimized into compounds that selectively inhibit Sirt1 (IC 50 = 27 µM) with >7-fold selectivity for Sirt1 over Sirt2/3, Sirt2 (IC 50 = 13 µM) with >15-fold selectivity for Sirt2 over Sirt1/3, or Sirt3 (IC 50 = 6 µM) with ~5-7-fold selectivity for Sirt3 over Sirt1/2 [137]. Increased acetylation of Sirt1 substrate p53 and Sirt2 substrate α -tubulin demonstrated that selective Sirt1 and Sirt2 inhibitors possessed on-target cellular activity in the NCI-H460 NSCLC cell line [137]. Additional SAR studies of cambinol have generated open-ring cambinol analogs that selectively inhibit Sirt2 with nanomolar potency and moderate selectivity for Sirt2 over Sirt1/3 [138]. These cambinol analogs exhibited on-target Sirt2 inhibition in the NCI-H60 NSCLC cell line, as demonstrated by a concentration-dependent increase in α -tubulin acetylation [138]. The cambinol analogs also induced cytotoxicity in a panel of cancer cell lines [138]. A virtual screen of over 25 million compounds identi fi ed ICL-SIRT078 as a Sirt2 inCompetitive inhibition by small molecules at the sirtuin acylated substrate binding cleft increases the apparent K M for acyl-lysine substrates, which lowers the sirtuin deacylase activity when substrate concentrations are not saturating. Hydroxynaphthaldehydes (e.g., sirtinol, salermide, splitomicin, and cambinol) comprise a large class of sirtuin inhibitors that impact the sirtuin catalytic mechanism in this manner with modest potency (average half-maximal inhibitory concentration (IC 50 ) of ~64 µM) [1,134-136] and reported apoptotic activity in cancer cell lines [134,136]. However, the selectivity of hydroxynaphthaldehydes for speci fi c sirtuin isoforms was not rigorously tested [134,136]. The be tt ercharacterized cambinol analogs have been optimized into compounds that selectively inhibit Sirt1 (IC 50 = 27 µM) with >7-fold selectivity for Sirt1 over Sirt2/3, Sirt2 (IC 50 = 13 µM) with >15-fold selectivity for Sirt2 over Sirt1/3, or Sirt3 (IC 50 = 6 µM) with ~5-7-fold selectivity for Sirt3 over Sirt1/2 [137]. Increased acetylation of Sirt1 substrate p53 and Sirt2 substrate α -tubulin demonstrated that selective Sirt1 and Sirt2 inhibitors possessed on-target cellular activity in the NCI-H460 NSCLC cell line [137]. Additional SAR studies of cambinol have generated open-ring cambinol analogs that selectively inhibit Sirt2 with nanomolar potency and moderate selectivity for Sirt2 over Sirt1/3 [138]. These cambinol analogs exhibited on-target Sirt2 inhibition in the NCI-H60 NSCLC cell line, as demonstrated by a concentration-dependent increase in α -tubulin acetylation [138]. The cambinol analogs also induced cytotoxicity in a panel of cancer cell lines [138]. Competitive inhibition by small molecules at the sirtuin acylated substrate binding cleft increases the apparent K M for acyl-lysine substrates, which lowers the sirtuin deacylase activity when substrate concentrations are not saturating. Hydroxynaphthaldehydes (e.g., sirtinol, salermide, splitomicin, and cambinol) comprise a large class of sirtuin inhibitors that impact the sirtuin catalytic mechanism in this manner with modest potency (average half-maximal inhibitory concentration (IC 50 ) of ~64 µM) [1,134-136] and reported apoptotic activity in cancer cell lines [134,136]. However, the selectivity of hydroxynaphthaldehydes for speci fi c sirtuin isoforms was not rigorously tested [134,136]. The be tt ercharacterized cambinol analogs have been optimized into compounds that selectively inhibit Sirt1 (IC 50 = 27 µM) with >7-fold selectivity for Sirt1 over Sirt2/3, Sirt2 (IC 50 = 13 µM) with >15-fold selectivity for Sirt2 over Sirt1/3, or Sirt3 (IC 50 = 6 µM) with ~5-7-fold selectivity for Sirt3 over Sirt1/2 [137]. Increased acetylation of Sirt1 substrate p53 and Sirt2 substrate α -tubulin demonstrated that selective Sirt1 and Sirt2 inhibitors possessed on-target cellular activity in the NCI-H460 NSCLC cell line [137]. Additional SAR studies of cambinol have generated open-ring cambinol analogs that selectively inhibit Sirt2 with nanomolar potency and moderate selectivity for Sirt2 over Sirt1/3 [138]. These cambinol analogs exhibited on-target Sirt2 inhibition in the NCI-H60 NSCLC cell line, as demonstrated by a concentration-dependent increase in α -tubulin acetylation [138]. The cambinol analogs also induced cytotoxicity in a panel of cancer cell lines [138]. Competitive inhibition by small molecules at the sirtuin acylated substrate binding cleft increases the apparent K M for acyl-lysine substrates, which lowers the sirtuin deacylase activity when substrate concentrations are not saturating. Hydroxynaphthaldehydes (e.g., sirtinol, salermide, splitomicin, and cambinol) comprise a large class of sirtuin inhibitors that impact the sirtuin catalytic mechanism in this manner with modest potency (average half-maximal inhibitory concentration (IC 50 ) of ~64 µ M) [1,134-136] and reported apoptotic activity in cancer cell lines [134,136]. However, the selectivity of hydroxynaphthaldehydes for specific sirtuin isoforms was not rigorously tested [134,136]. The better-characterized cambinol analogs have been optimized into compounds that selectively inhibit Sirt1 (IC 50 = 27 µ M) with >7-fold selectivity for Sirt1 over Sirt2/3, Sirt2 (IC 50 = 13 µ M) with >15-fold selectivity for Sirt2 over Sirt1/3, or Sirt3 (IC 50 = 6 µ M) with ~5-7-fold selectivity for Sirt3 over Sirt1/2 [137]. Increased acetylation of Sirt1 substrate p53 and Sirt2 substrate α -tubulin demonstrated that selective Sirt1 and Sirt2 inhibitors possessed on-target cellular activity in the NCI-H460 NSCLC cell line [137]. Additional SAR studies of cambinol have generated open-ring cambinol analogs that selectively inhibit Sirt2 with nanomolar potency and moderate selectivity for Sirt2 over Sirt1/3 [138]. These cambinol analogs exhibited on-target Sirt2 inhibition in the NCI-H60 NSCLC cell line, as demonstrated by a concentration-dependent increase in α -tubulin acetylation [138]. The cambinol analogs also induced cytotoxicity in a panel of cancer cell lines [138].

On-target Sirt2 inhibition by ICL-SIRT078 was observed in the MCF7 breast cancer cell

hibitor with an IC 50 value of 1.45 µM and >50-fold selectivity for Sirt2 over Sirt1/3/5 in vitro [121]. Molecular modeling suggested that the high selectivity of ICL-SIRT078 for Sirt2 over other sirtuins is conferred by a strong network of hydrophobic interactions with the Sirt2 acylated substrate binding cleft, which are disrupted by loops present in Sirt1/3/5[121]. On-target Sirt2 inhibition by ICL-SIRT078 was observed in the MCF7 breast cancer cell hibitor with an IC 50 value of 1.45 µM and >50-fold selectivity for Sirt2 over Sirt1/3/5 in vitro [121]. Molecular modeling suggested that the high selectivity of ICL-SIRT078 for Sirt2 over other sirtuins is conferred by a strong network of hydrophobic interactions with the Sirt2 acylated substrate binding cleft, which are disrupted by loops present in Sirt1/3/5[121]. hibitor with an IC 50 value of 1.45 µM and >50-fold selectivity for Sirt2 over Sirt1/3/5 in vitro [121]. Molecular modeling suggested that the high selectivity of ICL-SIRT078 for Sirt2 over other sirtuins is conferred by a strong network of hydrophobic interactions with the Sirt2 acylated substrate binding cleft, which are disrupted by loops present in Sirt1/3/5[121]. A virtual screen of over 25 million compounds identi fi ed ICL-SIRT078 as a Sirt2 inhibitor with an IC 50 value of 1.45 µM and >50-fold selectivity for Sirt2 over Sirt1/3/5 in vitro [121]. Molecular modeling suggested that the high selectivity of ICL-SIRT078 for Sirt2 over other sirtuins is conferred by a strong network of hydrophobic interactions with the Sirt2 A virtual screen of over 25 million compounds identi fi ed ICL-SIRT078 as a Sirt2 inhibitor with an IC 50 value of 1.45 µM and >50-fold selectivity for Sirt2 over Sirt1/3/5 in vitro [121]. Molecular modeling suggested that the high selectivity of ICL-SIRT078 for Sirt2 over other sirtuins is conferred by a strong network of hydrophobic interactions with the Sirt2 A virtual screen of over 25 million compounds identified ICL-SIRT078 as a Sirt2 inhibitor with an IC 50 value of 1.45 µ Mand >50-fold selectivity for Sirt2 over Sirt1/3/5 in vitro [121]. Molecular modeling suggested that the high selectivity of ICL-SIRT078 for Sirt2 over other sirtuins is conferred by a strong network of hydrophobic interactions

acylated substrate binding cleft, which are disrupted by loops present in Sirt1/3/5[121].

line via increased acetylation of the Sirt2 substrate

α

-tubulin [121]. ICL-SIRT078 treatment

On-target Sirt2 inhibition by ICL-SIRT078 was observed in the MCF7 breast cancer cell

acylated substrate binding cleft, which are disrupted by loops present in Sirt1/3/5[121].

line via increased acetylation of the Sirt2 substrate

α

-tubulin [121]. ICL-SIRT078 treatment

of a rat dopaminergic neural cell line also a

tt

enuated loss of cell viability in induced

line via increased acetylation of the Sirt2 substrate

α

-tubulin [121]. ICL-SIRT078 treatment

On-target Sirt2 inhibition by ICL-SIRT078 was observed in the MCF7 breast cancer cell

of a rat dopaminergic neural cell line also a

tt

enuated loss of cell viability in induced

line via increased acetylation of the Sirt2 substrate

α

-tubulin [121]. ICL-SIRT078 treatment

On-target Sirt2 inhibition by ICL-SIRT078 was observed in the MCF7 breast cancer cell

line via increased acetylation of the Sirt2 substrate

α

-tubulin [121]. ICL-SIRT078 treatment

of a rat dopaminergic neural cell line also a

tt

enuated loss of cell viability in induced

of a rat dopaminergic neural cell line also a

tt

enuated loss of cell viability in induced

of a rat dopaminergic neural cell line also a

tt

enuated loss of cell viability in induced

11 of 39

11 of 39

with the Sirt2 acylated substrate binding cleft, which are disrupted by loops present in Sirt1/3/5[121]. On-target Sirt2 inhibition by ICL-SIRT078 was observed in the MCF7 breast cancer cell line via increased acetylation of the Sirt2 substrate α -tubulin [121]. ICL-SIRT078 treatment of a rat dopaminergic neural cell line also attenuated loss of cell viability in induced Parkinson's disease cell death [121], consistent with prior reports of the benefits of Sirt2 inhibition in Parkinson's disease [139].

Virtual library screening against a Sirt4 homology model, coupled with SAR optimization of sulfur dioxide-containing polycyclic hits, recently resulted in the first reported Sirt4 inhibitors [132]. The most potent inhibitor, Compound 60 (Table 2), inhibited Sirt4 β -hydroxy β -methylglutarylation (de-HMGylation) with an IC 50 value of 0.9 µ M[132]. Although 10 µ Mof Compound 60 was sufficient to inhibit Sirt2 and Sirt4, Compound 60 maintained ~3.5-5.5-fold selectivity for Sirt4 over Sirt1/3/5/6 at 10 µ M[132]. In contrast, less-potent Compound 69 (Table 2) (IC 50 = 16 µ M) exhibited ~two-three-fold selectivity for Sirt4 over Sirt1/2/3/5/6 at 50 µ M[132]. Computational docking predicted that the compounds bind in the Sirt4 acylated substrate binding cleft; follow-up kinetic studies with Compound 69 suggest that the inhibitor is competitive with respect to the acylated substrate [132]. Sirt4 decreases pyruvate dehydrogenase complex activity via delipoylation of dihydrolipoyllysine acetyltransferase [31]. Consistent with on-target cellular Sirt4 inhibition, Compounds 60 and 69 concentration-dependently increased pyruvate dehydrogenase complex activity in C2C12 mouse myoblast cells pretreated with the pyruvate dehydrogenase complex inhibitor Glutamax [132]. These first-in-class Sirt4 inhibitors provide attractive scaffolds for further development of Sirt4 inhibitors with increased potency and sirtuin isoform specificity [132].

Sirt5 is also a recent target of sirtuin inhibitor development [133,140,141]. Since glutarylation is a Sirt5 acylation substrate [29], 3-thioureidopropanoic acid derivatives were developed to competitively inhibit Sirt5 by imitating glutarylated Sirt5 substrates [141]. The most potent derivatives inhibited Sirt5 desuccinylase activity with an IC 50 value of ~3-4 µ M and exhibited ~150-200-fold selectivity for Sirt5 over Sirt1/2/3/6 [141]. Computational docking predicted that the compounds bind in the Sirt5 acylated substrate binding cleft; kinetic assays supported these predictions by demonstrating competitive inhibition against a fluorogenic Sirt5 succinylated substrate [141]. Further SAR studies yielded Compound 58 , an acylated substrate-competitive Sirt5 inhibitor with nanomolar potency (IC 50 = 310 nM) and improved selectivity for Sirt5 over Sirt1/3 [140]. Notably, Compound 58 improved kidney function in two mouse models of sepsis-induced acute kidney injury, though the mechanism by which Sirt5 inhibition exerts kidney-protective effects is unknown [140]. Screening a 5000-member drug-like compound library coupled with SAR optimization of the pyrazolone-containing hit also produced Sirt5 inhibitor Compound 47 (Table 2), with an IC 50 value of 210 nM and >3800-fold selectivity for Sirt5 over Sirt1/2/3/6 [133]. Like the other reported Sirt5 inhibitors [140,141], Compound 47 was predicted to bind the Sirt5 acylated substrate binding pocket and exhibited competitive inhibition against a Sirt5 succinylated substrate [133].

Taken together, small molecules competitively targeting the sirtuin acyl-lysine binding site pose a rational mechanism for inhibiting sirtuin deacylase activity. However, with the potential exception of ICL-SIRT078 (Sirt2) and Compound 47 (Sirt5), most inhibitors targeting the sirtuin acylated substrate binding site require optimization to enhance sirtuin isoform selectivity. Moreover, every member of this inhibitor class would benefit from studies to increase the overall potency and confirm the mechanistic efficacy of sirtuin inhibition within in vivo models of cancer and neurodegeneration.

## 3.2. Small Molecule Sirtuin Inhibitors Targeting the NAD + Binding Site

Small molecule inhibitors interacting with the sirtuin NAD + binding site increase the apparent K M for the NAD + co-substrate, providing an additional mode of sirtuin inhibition. Several small molecule sirtuin inhibitors targeting the NAD + binding site have been identified through various high-throughput screens [127,129,130]. Elixir Pharmaceu-

ticals combined this approach with SAR studies to develop EX-527 (selisistat) (Table 2), which potently inhibited Sirt1 with an IC 50 value of 38-98 nM in vitro [127,142]. Moreover, EX-527 demonstrated 200-fold and ~500-fold selectivity for Sirt1 over Sirt2 and Sirt3, respectively [127]. EX-527 exhibited on-target Sirt1 inhibition in multiple cell lines exposed to DNA-damaging agents, demonstrated by increased acetylation of the Sirt1 substrate p53 [142]. The kinetics of in vitro Sirt1 deacetylation assays suggested that EX-527 is an uncompetitive inhibitor with respect to NAD + and likely binds to the Sirt1-substrate complex after nicotinamide release from the Sirt1 active site [127]. This proposed mechanism of EX-527-mediated Sirt1 inhibition was confirmed using crystallography with Sirt1 bacterial homolog Sir2Tm and human Sirt3, where the active S -enantiomer of EX-527 (EX-243) [142] bound both the C-site (the portion of the sirtuin NAD + binding pocket involved in the cleavage of the NAD + nicotinamide ribosyl bond [143]; Figure 2) and a unique hydrophobic pocket adjacent to the C-site (the extended C-site; Figure 2) and directly interact with the O AADPr sirtuin catalysis product [122]. However, since EX-527 binds and inhibits Sirt2/3 at micromolar concentrations [122,127], the selectivity of EX-527 for Sirt1 is likely attributed to the differential kinetics of the enzyme-product-inhibitor complex formation and subsequent product release rather than specific Sirt1 structural features [122].

Enthusiasm for the clinical potential of EX-527 in neurodegenerative diseases was sparked in part by a study demonstrating that EX-527 attenuated functional motor deficits and decreased striatal huntingtin protein inclusions in a mouse model of Huntington's disease [144]; however, a phase II clinical trial demonstrated that EX-527, while a pharmacologically safe drug, did not improve Huntington's disease patient outcomes within the timeframe of the study [145]. Nonetheless, the search for alternative therapeutic applications of EX-527 is ongoing, as highlighted by a recent study demonstrating that EX-527 attenuates diabetic nephropathy in a rat model of type 2 diabetes [146] and by the recently ended phase I clinical trial exploring the benefits of EX-527-mediated Sirt1 inhibition for infertility (NCT04184323).

Because commonly tested cellular concentrations of EX-527 are sufficient to inhibit other human sirtuins such as Sirt2 [147-150], small molecule Sirt1 inhibitor isoform selectivity can still be improved. To this end, a high-throughput luciferase deacetylation assay for Sirt1 inhibitors, coupled with SAR optimization of the benzoxazine scaffold hit, generated Compound 4.27 , which inhibited Sirt1 with an IC 50 value of 110 nM and slightly improved selectivity for Sirt1 relative to EX-527 [147]. MDA-MB-231 breast cancer cells treated with Compound 4.27 after etoposide pre-treatment exhibited increased p53 acetylation, consistent with Compound 4.27 exhibiting on-target cellular inhibition of Sirt1 [147]. Like EX-527, kinetic analyses revealed that Compound 4.27 is an uncompetitive inhibitor of Sirt1 with respect to NAD + , and molecular docking predicted that Compound 4.27 binds in part of the Sirt1 extended C-site (Figure 2) [147]. Thus, Compound 4.27 is a promising starting point to develop selective Sirt1 inhibitors that can be tested for therapeutic potential in Huntington's disease and other aging pathologies.

To identify more potent Sirt2 inhibitors, Compound B2 , previously found to be a weak Sirt2 inhibitor, was expanded into a library of 200 structural analogs for screening [129]. SAR studies based on the top hit resulted in AGK2 (Table 2), which inhibited Sirt2 deacetylase activity with an IC 50 value of 3.5 µ Mand exhibited >14 fold-selectivity for Sirt2 over Sirt1/3 [129]. Moreover, AGK2 exhibited on-target Sirt2 inhibition in the HeLa cell line, as demonstrated by increased acetylation of the Sirt2 substrate α -tubulin [129]. Molecular docking suggested that AGK2 binds within the Sirt2 C-site (Figure 2), providing a potential mechanism for AGK2-mediated Sirt2 inhibition [129]. Notably, AGK2 treatment of a Drosophila model of Parkinson's disease attenuated α -synuclein-mediated toxicity in dorsomedial neurons, implicating Sirt2 as a therapeutic target for inhibition in Parkinson's disease [129]. AKG2-mediated Sirt2 inhibition also protected against huntingtin-induced toxicity and protein inclusion formation in a primary striatal neuron cell model of Huntington's disease by decreasing sterol production [151].

Because AGK2 cannot cross the blood-brain barrier to exert its neuroprotective effects in vivo, a yeast-based high-throughput screen coupled with in silico screening was employed to generate the Sirt2-selective brain-permeable inhibitor AK7 (Table 2) [130]. Like AGK2, AK7 exhibited selectivity for Sirt2 over Sirt1/3 [130] and was predicted to interact with the Sirt2 C-site (Figure 2) [152]. AK7 also decreased cholesterol biosynthesis [130] and exhibited neuroprotective effects in cell and mouse models of Huntington's disease and Parkinson's disease [130,152,153]. However, AK7-mediated Sirt2 inhibition showed no therapeutic efficacy in mouse models of amyotrophic lateral sclerosis and ischemic stroke [152], indicating that Sirt2 inhibition only mitigates the pathophysiology of specific neurodegenerative diseases.

The in vitro and in vivo activity of nanomolar-potent sirtuin inhibitors targeting the sirtuin NAD + binding site, such as Compound 4.27 (Sirt1), AGK2 (Sirt2), and AK7 (Sirt2), have demonstrated that small molecules targeting the sirtuin C-site and/or extended Csite (Figure 2) are effective sirtuin inhibitors. In particular, these inhibitors have shown promising therapeutic potential for multiple neurodegenerative disorders. Nonetheless, existing inhibitors require further optimization to improve pharmacologic characteristics like brain permeability.

## 3.3. Adenosine Analogs as Small Molecule Sirtuin Inhibitors

Since the sirtuin co-substrate NAD + and the kinase substrate ATP both contain an adenosine moiety, adenosine mimetic kinase inhibitors may also inhibit sirtuins [123]. To this end, a library containing commercially available kinase inhibitors was screened for ability to inhibit Sirt2 in a fluorescent deacetylation assay [123]. Top hits from this screen included several bisindolylmaleimides (BIMs) [123], known ATP-competitive inhibitors of protein kinase C [154]. The best-identified inhibitor, BIM Ro 31-8220, inhibited Sirt2 deacetylase activity with an IC 50 value of 800 nM and exhibited ~four-fold selectivity for Sirt2 over Sirt1 [123]. Molecular docking demonstrated that the indole ring of Ro 31-8220 binds in the same location as the NAD + adenine ring on Sirt2, and an in vitro inhibition assay suggested that Ro 31-8220 inhibits Sirt2 by competing with NAD + binding [123]. The cellular activity of Ro 31-8220 was confirmed via hyperacetylation of the Sirt2 substrate α -tubulin in the A549 lung adenocarcinoma cell line [123]. Despite the promising results of Ro 31-8220 in vitro, cross-reactivity of adenosine mimetic sirtuin inhibitors with kinases remains a significant concern; therefore, a thorough examination of cellular off-target effects, and kinase counterscreens in particular, will be required for any future sirtuin inhibitors developed by adenosine mimesis.

## 3.4. Bivalent Small Molecule Sirtuin Inhibitors Targeting the Acylated Substrate and NAD + Binding Sites

Small molecule inhibitors that bind both the sirtuin acylated substrate cleft and NAD + co-substrate binding site may exhibit extremely potent sirtuin inhibition by substantially disrupting the kinetic parameters (i.e., increased K M ) of the sirtuin catalytic mechanism. For example, fragment-based screening coupled with SAR optimization generated the (5-benzamidonaphthalen-1/2-yloxy)nicotinamide analog Compound 64 as a potent Sirt2 inhibitor (IC 50 = 48 nM) with ~249- and ~915-fold selectivity for Sirt2 over Sirt1 and Sirt3 in vitro [155]. Consistent with its predicted binding to both the C-site and the acylated substrate binding cleft of Sirt2 (Figure 2), kinetic assays suggested that Compound 64 exhibits non-competitive inhibition relative to NAD + and competitive inhibition relative to an acetylated peptide substrate [155]. The cellular activity of Compound 64 was confirmed by increased acetylation of the Sirt2 substrate α -tubulin in a concentration- and timedependent manner in the MCF7 breast cancer cell line [155].

Further SAR optimization of Compound 64 produced 5-((3-amidobenzyl)oxy)nicotinamide analogs that were also predicted to dually bind to the Sirt2 C-site and acylated substrate binding cleft (Figure 2) [128,156]. Many of the new nicotinamide analogs, like Compound 86 (Table 2), retained selectivity for Sirt2 over Sirt1/3 [128,156] and increased potency

against Sirt2 up to ~three-fold compared to Compound 64 [128]. Based on prior studies demonstrating the benefits of Sirt2 inhibition in neurodegenerative diseases with inhibitors targeting the Sirt2 NAD + binding site [129,130,151-153], two nicotinamide analogs with predicted brain permeability were tested in an α -synuclein toxicity model of Parkinson's disease in SH-SY5Y neuroblastoma cells [128]. Both nicotinamide analogs attenuated α -synuclein-induced SH-SY5Y viability loss [128], suggesting that Sirt2 inhibitors with dual binding modes retain translational potential in neurodegenerative diseases [128].

Aside from nicotinamide analogs, library screening with a fluorescent acetylation assay yielded aminothiazole-based sirtuin-rearranging ligands (i.e., SirReals) as Sirt2 inhibitors [124]. SirReal2 (Table 2), the most potent of these compounds, inhibited Sirt2 with an IC 50 value of 140 nM and exhibited high (>1000-fold) selectivity for Sirt2 over Sirt1/3/4/5/6 [124]. Sirt2 selectivity is derived from the unique binding mode of SirReal2 [124]. Crystallization demonstrated that SirReal2 not only engages the extended C-site and protrudes into the Sirt2 acylated substrate binding cleft but also triggers structural reorganization of the Sirt2 active site to bind to an additional 'selectivity pocket' next to the extended C-site (Figure 2) [124]. SirReal2 exhibited on-target Sirt2 inhibition in HeLa cells, demonstrated by increased acetylation of the Sirt2 substrate α -tubulin and decreased protein levels of the mitotic spindle assembly checkpoint protein BubR1 [124], whose proteasomal degradation is inhibited by Sirt2-mediated deacetylation [157]. SAR optimization generated additional SirReals with improved cellular activity and/or solubility [158,159] and a Sirt2 affinity probe [159]. Thiazoles as a scaffold for Sirt2 inhibition have continued to be investigated; the thiazole MIND4 (Table 2) was found to inhibit Sirt2 with an IC 50 value of 3.5 µ Mand independently activate nuclear factor erythroid 2-related factor 2 (NRF2), resulting in synergistic neuroprotection in several models of Huntington's disease [131].

Although most small molecules targeting the sirtuin NAD + binding site and acylated substrate binding cleft are directed against Sirt2, bivalent sirtuin inhibitors targeting other sirtuin isoforms are appearing. For example, potent (low nM IC 50 ) yet non-selective Sirt1-3 inhibitors like ELT-31 (Table 2) have been identified through DNA-encoded library screening coupled with SAR optimization [126]. Analogous to the nanomolar-potent nicotinamide analog Sirt2 inhibitors [128,155], the potent sirtuin inhibition exhibited by the ELT compound family is derived from its simultaneous engagement of the sirtuin C-site and acylated substrate binding cleft (Figure 2) [126]. In addition, Sirt3 inhibitor LC-0296 was developed to target overexpressed Sirt3 in oral squamous cell carcinoma (OSCC) [160]. LC-0296 inhibited Sirt3 with an IC 50 value of 3.6 µ Min vitro and exhibited ~19- and ~9-fold selectivity for Sirt3 over Sirt1 and Sirt2, respectively [160]. Though the mechanism by which LC-0296 binds and inhibits Sirt3 has not been determined, the polycyclic LC-0296 may interact with both the Sirt3 acyl-lysine and NAD + binding sites via moieties resembling acetyl-lysine and nicotinamide [160]. Consistent with mitochondrial Sirt3 inhibition, LC0296 increased global mitochondrial protein acetylation, acetylation of the Sirt3 target glutamate dehydrogenase, and reactive oxygen species levels in head and neck squamous cell carcinoma (HNSCC) cell lines [160]. By disrupting the mitochondrial redox balance, LC-0296 promoted HNSCC cell apoptosis and enhanced HNSCC sensitivity to radiation and cisplatin [160].

Surprisingly, the Zn 2+ -chelating class I/II HDAC inhibitor trichostatin A was found to inhibit Sirt6 via a Zn 2+ -independent mechanism [161,162]. Initially hypothesized to chelate the Zn 2+ in the sirtuin zinc-binding domain (Figure 2) [161], trichostatin A was instead observed to bind the Sirt6 C-site (Figure 2) and the unique Sirt6 extension of the sirtuin acyl-substrate binding cleft [162]. Consistent with this dual binding occupancy, trichostatin Aexhibited competitive inhibition relative to the acylated substrate and noncompetitive inhibition relative to NAD + [161]. Trichostatin A inhibited Sirt6-mediated histone H3K9 and p53 K382 deacetylation with an inhibition constant ( K i ) of 2-4.6 µ Mand did not affect Sirt12/3/5 deacylase activity at concentrations below 50 µ M[161]. However, given its well-characterized primary inhibition of class I HDACs (HDAC1-3) and class II HDACs

(HDAC4, 6, 7, and 9) [163], trichostatin A is not an effective probe for studying Sirt6 cellular functions.

Most bivalent small molecule sirtuin inhibitors (e.g., Compound 64 , SirReal2, and ELT-31) exhibit nanomolar potency towards their specified sirtuin isoforms. Thus, dual inhibition of the sirtuin acylated substrate and NAD + binding is an attractive mechanism for disrupting sirtuin catalytic activity. Therefore, optimizing and diversifying the interactions of bivalent sirtuin inhibitors with the acylated substrate binding cleft and the C-site/extended C-site (Figure 2) are critical next steps for enhancing sirtuin isoform specificity. Moreover, SAR studies of the most potent inhibitors to enhance pharmacological properties like solubility and membrane permeability will be vital in generating therapeutically relevant compounds for in vivo testing in multiple models of aging-related diseases, such as neurodegenerative diseases and cancer.

## 3.5. Allosteric Small Molecule Inhibitors

While most sirtuin inhibitors engage the acylated substrate channel, NAD + binding site, or both, some inhibitors bind to unique allosteric sites on sirtuins. The identification and development of allosteric sirtuin inhibitors has been accelerated by recent advances in the computational tools to predict potential allosteric sites [164]. For example, reversed allostery prediction methods, coupled with site-directed mutagenesis, generated the allosteric Sirt6 inhibitor JYQ-42 [165]. JYQ-42 inhibited Sirt6 deacetylase activity with an IC 50 value of 2.33 µ M[165]. JYQ-42 showed no effect on the deacetylase activity of any other sirtuin except Sirt2, which required a ~37 times greater concentration for inhibition [165]. Molecular modeling suggested that JYQ-42 bound to pocket Z, an allosteric pocket on Sirt6 formed via orthosteric NAD + binding [165]. Mutations of pocket Z residues increased the IC 50 of Sirt6 inhibition by JYQ-42, and in vitro kinetic assays demonstrated non-competitive inhibition of JYQ-42 relative to both the acylated substrate and NAD + , supporting an allosteric binding mechanism [165]. JYQ-42 increased acetylation of the Sirt6 substrates histone H3K9, H3K18, and H3K56 in pancreatic cancer cell lines in a concentration-dependent manner, consistent with on-target Sirt6 inhibition [165]. JYQ-42 also reduced the expression and secretion of the cancer-induced inflammatory mediators IL-6, IL-8, and TNF α [165]. Moreover, JYQ-42 decreased pancreatic cancer cell migration in a time and concentration-dependent manner [165].

Virtual screening of Sirt6 against a library of marine natural products and derivatives, coupled with SAR optimization of a pyrrol-pyridinimidazole-containing hit, produced Compound 8a as an additional Sirt6 inhibitor (IC 50 = 7.5 µ M) with ~11-12-fold selectivity for Sirt6 over Sirt1/2 and >26-fold selectivity for Sirt6 over Sirt3/5 in vitro [166]. Consistent with predicted binding to Sirt6 outside of the acylated substrate and NAD + binding pockets, Compound 8a exhibited non-competitive inhibition relative to both the acetylated substrate and NAD + [166]. Treatment of pancreatic cancer cell lines with Compound 8a increased acetylation of histone H3K9, consistent with cellular activity against Sirt6 [166]. Compound 8a exhibited synergistic cytotoxic effects with gemcitabine in a mouse xenograft model of pancreatic cancer without impacting overall pancreatic function [166], supporting the potential therapeutic efficacy of Sirt6 inhibition in pancreatic cancer identified in previous studies [165].

Since the discovery of the Sirt2 'selectivity pocket' (Figure 2), additional small molecule sirtuin inhibitors have exploited this and other unique hydrophobic regions generated upon inhibitor binding [125,167-171]. For example, SAR refinement of a N -(3-(phenoxymethyl)phenyl) acetamide scaffold yielded compounds that inhibited Sirt2 deacetylase (IC 50 = 42-850 nM) and dedecanoylase (IC 50 = 8.3-17.6 µ M) activity [168,171] and exhibited seven-fold selectivity for Sirt2 over Sirt1/3 in vitro [168]. Crystallography demonstrated these inhibitors induce and bind the Sirt2 'selectivity pocket' (Figure 2) and directly inhibit the binding of acylated substrates to Sirt2 by rearranging portions of the active site [171]. In addition, screening of natural products identified the cytisine derivative NPD11033 as an inhibitor of Sirt2 deacetylase activity, with an IC 50 value of 460 nM and >40-fold selectivity for Sirt2

Molecules

2024

,

29

, x FOR PEER REVIEW

17 of 39

Molecules

2024

,

29

, x FOR PEER REVIEW

Molecules

2024

,

29

, x FOR PEER REVIEW

17 of 39

These studies suggest that sirtuins contain several druggable allosteric sites that can be bound by non-competitive small molecule inhibitors. Without targeting the acylated substrate binding cleft or NAD + binding pocket common to all seven human sirtuins, allosteric inhibitors have the potential to be optimized into compounds that exert superior sirtuin isoform selectivity compared to other small molecule sirtuin inhibitors. However, additional structural studies are required to enhance understanding of the structural requirements for allosteric pocket induction in Sirt2/6 and identify allosteric pockets on the remaining sirtuin isoforms. be bound by non-competitive small molecule inhibitors. Without targeting the acylated substrate binding cleft or NAD + binding pocket common to all seven human sirtuins, allosteric inhibitors have the potential to be optimized into compounds that exert superior sirtuin isoform selectivity compared to other small molecule sirtuin inhibitors. However, additional structural studies are required to enhance understanding of the structural requirements for allosteric pocket induction in Sirt2/6 and identify allosteric pockets on the remaining sirtuin isoforms. 4. Mechanism-Based Sirtuin Inhibitors be bound by non-competitive small molecule inhibitors. Without targeting the acylated substrate binding cleft or NAD + binding pocket common to all seven human sirtuins, allosteric inhibitors have the potential to be optimized into compounds that exert superior sirtuin isoform selectivity compared to other small molecule sirtuin inhibitors. However, additional structural studies are required to enhance understanding of the structural requirements for allosteric pocket induction in Sirt2/6 and identify allosteric pockets on the remaining sirtuin isoforms. be bound by non-competitive small molecule inhibitors. Without targeting the acylated substrate binding cleft or NAD + binding pocket common to all seven human sirtuins, allosteric inhibitors have the potential to be optimized into compounds that exert superior sirtuin isoform selectivity compared to other small molecule sirtuin inhibitors. However, additional structural studies are required to enhance understanding of the structural requirements for allosteric pocket induction in Sirt2/6 and identify allosteric pockets on the remaining sirtuin isoforms.

over Sirt1/3 [125]. A crystal structure of the inhibitor-bound Sirt2 catalytic domain demonstrated that NPD11033 induced 'selectivity pocket' (Figure 2) formation similar to SirReal2, but uniquely formed a direct interaction with critical active site base residue H187[172] through a hydrogen bond [125]. Like SirReal2 [124], NDP11033 decreased BubR1 protein levels in the PANC-1 pancreatic cancer cell line, consistent with on-target cellular Sirt2 inhibition [125]. NDP11033 and SirReal2 also reduced PANC-1 cell viability, suggesting that targeted Sirt2 inhibition via the 'selectivity pocket' (Figure 2) may be beneficial in pancreatic cancer [125]. [124], NDP11033 decreased BubR1 protein levels in the PANC-1 pancreatic cancer cell line, consistent with on-target cellular Sirt2 inhibition [125]. NDP11033 and SirReal2 also reduced PANC-1 cell viability, suggesting that targeted Sirt2 inhibition via the 'selectivity pocket' (Figure 2) may be bene fi cial in pancreatic cancer [125]. These studies suggest that sirtuins contain several druggable allosteric sites that can [124], NDP11033 decreased BubR1 protein levels in the PANC-1 pancreatic cancer cell line, consistent with on-target cellular Sirt2 inhibition [125]. NDP11033 and SirReal2 also reduced PANC-1 cell viability, suggesting that targeted Sirt2 inhibition via the 'selectivity pocket' (Figure 2) may be bene fi cial in pancreatic cancer [125]. These studies suggest that sirtuins contain several druggable allosteric sites that can 17 of 39 [124], NDP11033 decreased BubR1 protein levels in the PANC-1 pancreatic cancer cell line, consistent with on-target cellular Sirt2 inhibition [125]. NDP11033 and SirReal2 also reduced PANC-1 cell viability, suggesting that targeted Sirt2 inhibition via the 'selectivity pocket' (Figure 2) may be bene fi cial in pancreatic cancer [125]. These studies suggest that sirtuins contain several druggable allosteric sites that can

4. Mechanism-Based Sirtuin Inhibitors

Mechanism-based compounds that stall or reverse sirtuin catalysis via active site

## 4. Mechanism-Based Sirtuin Inhibitors Mechanism-based compounds that stall or reverse sirtuin catalysis via active site 4. Mechanism-Based Sirtuin Inhibitors

Mechanism-based compounds that stall or reverse sirtuin catalysis via active site chemistry have emerged as another significant class of sirtuin inhibitors (Table 3). Nicotinamide is a well-known non-competitive inhibitor [173] of all sirtuin isoforms [174]. As a result, various nicotinamide analogs, which interact with the sirtuin active site in a manner analogous to nicotinamide, have been developed as sirtuin deacylase inhibitors. Alternative classes of mechanistic sirtuin inhibitors are broadly classified as linear or cyclic peptidic and pseudopeptidic compounds. These inhibitors contain acyl-substrate warheads that mimic native sirtuin acyl-substrates. chemistry have emerged as another signi fi cant class of sirtuin inhibitors (Table 3). Nicotinamide is a well-known non-competitive inhibitor [173] of all sirtuin isoforms [174]. As a result, various nicotinamide analogs, which interact with the sirtuin active site in a manner analogous to nicotinamide, have been developed as sirtuin deacylase inhibitors. Alternative classes of mechanistic sirtuin inhibitors are broadly classi fi ed as linear or cyclic peptidic and pseudopeptidic compounds. These inhibitors contain acyl-substrate warheads that mimic native sirtuin acyl-substrates. chemistry have emerged as another signi fi cant class of sirtuin inhibitors (Table 3). Nicotinamide is a well-known non-competitive inhibitor [173] of all sirtuin isoforms [174]. As a result, various nicotinamide analogs, which interact with the sirtuin active site in a manner analogous to nicotinamide, have been developed as sirtuin deacylase inhibitors. Alternative classes of mechanistic sirtuin inhibitors are broadly classi fi ed as linear or cyclic peptidic and pseudopeptidic compounds. These inhibitors contain acyl-substrate warheads that mimic native sirtuin acyl-substrates. Mechanism-based compounds that stall or reverse sirtuin catalysis via active site chemistry have emerged as another signi fi cant class of sirtuin inhibitors (Table 3). Nicotinamide is a well-known non-competitive inhibitor [173] of all sirtuin isoforms [174]. As a result, various nicotinamide analogs, which interact with the sirtuin active site in a manner analogous to nicotinamide, have been developed as sirtuin deacylase inhibitors. Alternative classes of mechanistic sirtuin inhibitors are broadly classi fi ed as linear or cyclic peptidic and pseudopeptidic compounds. These inhibitors contain acyl-substrate warheads that mimic native sirtuin acyl-substrates.

Table 3.

Select mechanism-based inhibitors of sirtuin deacylase activity.

Table 3. Select mechanism-based inhibitors of sirtuin deacylase activity. Targeted Table 3. Select mechanism-based inhibitors of sirtuin deacylase activity.

Select mechanism-based inhibitors of sirtuin deacylase activity.

<!-- image -->

<!-- image -->

<!-- image -->

Targeted

Targeted

Potency

Cell

Table 3.

| Lysine Derivative Lysine Derivative  Lysine Derivative  Lysine Derivative     | Warhead Warhead  Warhead  Warhead         | Targeted Sirtuin Isoform(s) Sirtuin Iso- form(s)  (IC 50 Sirtuin Iso- form(s)  (IC 50 Sirtuin Iso- (IC 50   | Potency (IC 50 ), µ M ),  μ M  Potency  ),  μ M  Potency  ),  μ M   | Selectivity Selectivity  Selectivity  Selectivity                                                                                                                               | Cell Active Active  Cell  Active  Cell  Active   | Citations Citations  Citations  Citations   |
|-------------------------------------------------------------------------------|-------------------------------------------|-------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------|---------------------------------------------|
| Thioacetyl Thioacetyl  Thioacetyl  Thioacetyl                                 | form(s)  Sirt1/2/3                        | Sirt1/2/3  Sirt1/2/3 Sirt1/2/3                                                                              | 0.18-29.4  0.18-29.4 0.18-29.4  0.18-29.4                           | Largely non-selective  Largely non-selective Largely non-selective  Largely non-selective                                                                                       | Yes  Yes Yes  Yes                                | [175-183]  [175-183] [175-183]  [175-183]   |
| Thiocarbamoyl  Thiocarbamoyl Thiocarbamoyl  Thiocarbamoyl                     | Sirt1/2/3  Sirt1/2/3  Sirt1/2/3 Sirt1/2/3 | 1.7-159.1  1.7-159.1                                                                                        | 1.7-159.1  1.7-159.1                                                | Non-selective  Non-selective  Non-selective Non-selective                                                                                                                       | Yes  Yes  Yes Yes                                | [184,185]  [184,185]  [184,185] [184,185]   |
| Thiomyristoyl (TM)  Thiomyristoyl (TM)  Thiomyristoyl (TM) Thiomyristoyl (TM) | Sirt2  Sirt2  Sirt2  Sirt2                | 0.03-0.09 *  0.04-3.3 **  0.03-0.09 *  0.04-3.3 **  0.03-0.09 *  0.04-3.3 **  0.03-0.09 *                   | 0.04-3.3 **                                                         | Up to ~3500-7100 fold  over Sirt1/3/5/6/7    Up to ~3500-7100 fold  over Sirt1/3/5/6/7    Up to ~3500-7100 fold  over Sirt1/3/5/6/7    Up to ~3500-7100 fold over Sirt1/3/5/6/7 | Yes  Yes  Yes  Yes                               | [186-188]  [186-188]  [186-188]  [186-188]  |

Soluble TM

Sirt2

0.03

~72-94 fold over Sirt1/3;

~72-94 fold over Sirt1/3;

>1500 fold over Sirt6

~72-94 fold over Sirt1/3;

>1500 fold over Sirt6

>1500 fold over Sirt6

Yes

[189]

Yes

[189]

Yes

[189]

Soluble TM

Sirt2

0.03

Soluble TM

Sirt2

0.03

Thiocarbamoyl

Sirt1/2/3

1.7-159.1

Non-selective

Yes

[184,185]

Thiomyristoyl (TM)

Sirt2

Molecules

Molecules

Molecules

Table 3. Cont.

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

<!-- image -->

N.D.C. = isoform selectivity not determined in cells * = deacetylase activity ** = demyristoylase activity. N.D.C. = isoform selectivity not determined in cells * = deacetylase activity ** = demyristoylase activity. N.D.C. = isoform selectivity not determined in cells * = deacetylase activity ** = demyristoylase activity.

0.03-0.09 *

0.04-3.3 **

Up to ~3500-7100 fold

over Sirt1/3/5/6/7

Yes

[186-188]

| Lysine Derivative                                                                                                                                                                                              | Warhead                                                                                                                                                                                                                                 | Targeted Sirtuin Isoform(s)                                        | Potency (IC 50 ), µ M                                | Selectivity                                                                                                                                                                                                                                       | Cell Active             | Citations                                            |
|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|------------------------------------------------------|
| Soluble TM Soluble TM  2024 ,  29 2024 ,  29 2024 ,  29 Molecules  2024 ,  29                                                                                                                                  | 0.03  , x FOR PEER REVIEW  , x FOR PEER REVIEW  , x FOR PEER REVIEW  , x FOR PEER REVIEW                                                                                                                                                | Sirt2  Sirt2                                                       | 0.03                                                 | ~72-94 fold over Sirt1/3;  >1500 fold over Sirt6  ~72-94 fold over Sirt1/3; >1500 fold over Sirt6                                                                                                                                                 | Yes  Yes                | [189]  [189] 18 of 39  18 of 39  18 of 39  18 of 39  |
| Fatty acyl thiocarbamoyl Fatty acyl thiocar- bamoyl  Fatty acyl thiocar- bamoyl  Fatty acyl thiocar- bamoyl  Fatty acyl thiocar- bamoyl                                                                        |                                                                                                                                                                                                                                         | Sirt2  Sirt2 Sirt2  Sirt2  Sirt2                                   | 0.06-0.15  0.06-0.15 0.06-0.15  0.06-0.15  0.06-0.15 | ~180-2500 fold over  Sirt1/3  ~180-2500 fold over Sirt1/3 ~180-2500 fold over  Sirt1/3  ~180-2500 fold over  Sirt1/3  ~180-2500 fold over  Sirt1/3                                                                                                | Yes  Yes Yes  Yes  Yes  | [190]  [190] [190]  [190]  [190]                     |
| Cyclic thioacetyl  Cyclic thioacetyl Cyclic thioacetyl  Cyclic thioacetyl  Cyclic thioacetyl                                                                                                                   | 0.002  0.01  0.002  0.01  0.002  0.01                                                                                                                                                                                                   | Sirt1  Sirt2  Sirt1  Sirt2  Sirt1 Sirt2 Sirt1  Sirt2  Sirt1  Sirt2 | 0.002 0.01 0.002  0.01                               | 19-62 fold over Sirt2/3;  7-8 fold over Sirt1/3  19-62 fold over Sirt2/3;  7-8 fold over Sirt1/3  19-62 fold over Sirt2/3; 7-8 fold over Sirt1/3 19-62 fold over Sirt2/3;  7-8 fold over Sirt1/3  19-62 fold over Sirt2/3;  7-8 fold over Sirt1/3 | Yes  Yes  Yes Yes  Yes  | [191-193]  [191-193]  [191-193] [191-193]  [191-193] |
| Mitochondrially-tar- geted TM  Mitochondrially-tar- geted TM  Mitochondrially- targeted TM Mitochondrially-tar- geted TM  Mitochondrially-tar- geted TM                                                        | 0.53  0.53  0.53                                                                                                                                                                                                                        | Sirt3  Sirt3  Sirt3  Sirt3 Sirt3                                   | 0.53 0.53                                            | N.D.C.  N.D.C.  N.D.C.  N.D.C. N.D.C.                                                                                                                                                                                                             | Yes  Yes  Yes  Yes Yes  | [194]  [194]  [194]  [194] [194]                     |
| Mitochondrially-tar- geted thiocarbamoyl  Mitochondrially-tar- geted thiocarbamoyl  Mitochondrially-tar- geted thiocarbamoyl  Mitochondrially- targeted thiocarbamoyl Mitochondrially-tar- geted thiocarbamoyl | 1.1  N.D.C. = isoform selectivity not determined in cells * = deacetylase activity ** = demyristoylase activity.  1.1  N.D.C. = isoform selectivity not determined in cells * = deacetylase activity ** = demyristoylase activity.  1.1 | Sirt3  Sirt3  Sirt3  Sirt3  Sirt3                                  | 1.1  1.1                                             | N.D.C.  N.D.C.  N.D.C.  N.D.C.  N.D.C.                                                                                                                                                                                                            | Yes  Yes  Yes  Yes  Yes | [195]  [195]  [195]  [195]  [195]                    |

4.1. Mechanism-Based Sirtuin Inhibition by Nicotinamide

binding pocket (Figure 2) [174] results in nucleophilic a tt ack of the 1 ' -Oalkylamidate intermediate (Figure 5A), the reverse reaction of the fi rst step in the sirtuin catalytic mechanism (Figure 5B), regenerating NAD + and acyl-lysine substrate [114,174,196]. Accordingly, nicotinamide is a non-competitive inhibitor [173] of all sirtuin isoforms [174] with IC 50 values ranging from 50-184 µM for Sirt1/2/3/5/6 in vitro [197,198]. In mammals, physiological concentrations of nicotinamide (estimated to be 11-400 µM [199]) fall within this inhibitory range, suggesting that nicotinamide is an endogenous negative regulator of sirtuins [199]. Consistent with this hypothesis, nicotinamide increased acetylation of the Sirt1 substrate p53 in mice [200] and accelerated aging in yeast, comparable to the phenotype of Sir2 knockout [199]. Rebinding of sirtuin deacylation product nicotinamide [6] to the C-site of the NAD binding pocket (Figure 2) [174] results in nucleophilic a tt ack of the 1 ' -Oalkylamidate intermediate (Figure 5A), the reverse reaction of the fi rst step in the sirtuin catalytic mechanism (Figure 5B), regenerating NAD + and acyl-lysine substrate [114,174,196]. Accordingly, nicotinamide is a non-competitive inhibitor [173] of all sirtuin isoforms [174] with IC 50 values ranging from 50-184 µM for Sirt1/2/3/5/6 in vitro [197,198]. In mammals, physiological concentrations of nicotinamide (estimated to be 11-400 µM [199]) fall within this inhibitory range, suggesting that nicotinamide is an endogenous negative regulator of sirtuins [199]. Consistent with this hypothesis, nicotinamide increased acetylation of the Sirt1 substrate p53 in mice [200] and accelerated aging in yeast, comparable to the phenotype of Sir2 knockout [199]. Rebinding of sirtuin deacylation product nicotinamide [6] to the C-site of the NAD + binding pocket (Figure 2) [174] results in nucleophilic a tt ack of the 1 ' -Oalkylamidate intermediate (Figure 5A), the reverse reaction of the fi rst step in the sirtuin catalytic mechanism (Figure 5B), regenerating NAD + and acyl-lysine substrate [114,174,196]. Accordingly, nicotinamide is a non-competitive inhibitor [173] of all sirtuin isoforms [174] with IC 50 values ranging from 50-184 µM for Sirt1/2/3/5/6 in vitro [197,198]. In mammals, physiological concentrations of nicotinamide (estimated to be 11-400 µM [199]) fall within this inhibitory range, suggesting that nicotinamide is an endogenous negative regulator of sirtuins [199]. Consistent with this hypothesis, nicotinamide increased acetylation of the Sirt1 substrate p53 in mice [200] and accelerated aging in yeast, comparable to the phenotype of Sir2 knockout [199]. Rebinding of sirtuin deacylation product nicotinamide [6] to the C-site of the NAD + binding pocket (Figure 2) [174] results in nucleophilic a tt ack of the 1 ' -Oalkylamidate intermediate (Figure 5A), the reverse reaction of the fi rst step in the sirtuin catalytic mechanism (Figure 5B), regenerating NAD + and acyl-lysine substrate [114,174,196]. Accordingly, nicotinamide is a non-competitive inhibitor [173] of all sirtuin isoforms [174] with IC 50 values ranging from 50-184 µM for Sirt1/2/3/5/6 in vitro [197,198]. In mammals, physiological concentrations of nicotinamide (estimated to be 11-400 µM [199]) fall within this inhibitory range, suggesting that nicotinamide is an endogenous negative regulator of sirtuins [199]. Consistent with this hypothesis, nicotinamide increased acetylation of the Sirt1 substrate p53 in mice [200] and accelerated aging in yeast, comparable to the phenotype of Sir2 knockout [199]. Rebinding of sirtuin deacylation product nicotinamide [6] to the C-site of the NAD + binding pocket (Figure 2) [174] results in nucleophilic attack of the 1 ' -O -alkylamidate intermediate (Figure 5A), the reverse reaction of the first step in the sirtuin catalytic mechanism (Figure 5B), regenerating NAD + and acyl-lysine substrate [114,174,196]. Accordingly, nicotinamide is a non-competitive inhibitor [173] of all sirtuin isoforms [174] with IC 50 values ranging from 50-184 µ Mfor Sirt1/2/3/5/6 in vitro [197,198]. In mammals, physiological concentrations of nicotinamide (estimated to be 11-400 µ M [199]) fall within this inhibitory range, suggesting that nicotinamide is an endogenous negative regulator of sirtuins [199]. Consistent with this hypothesis, nicotinamide increased acetylation of the Sirt1 substrate p53 in mice [200] and accelerated aging in yeast, comparable to the phenotype of Sir2 knockout [199].

## Rebinding of sirtuin deacylation product nicotinamide [6] to the C-site of the NAD 4.1. Mechanism-Based Sirtuin Inhibition by Nicotinamide 4.1. Mechanism-Based Sirtuin Inhibition by Nicotinamide 4.1. Mechanism-Based Sirtuin Inhibition by Nicotinamide 4.1. Mechanism-Based Sirtuin Inhibition by Nicotinamide

+

+

Molecules

2024

,

29

, x FOR PEER REVIEW

19 of 39

Figure 5. The sirtuin catalytic mechanism provides a rationale for using thioacyl-lysine derivatives as sirtuin inhibitors. ( A ) Mechanism of nicotinamide-mediated sirtuin inhibition. ( B ) Native sirtuin catalytic mechanism with acetylated peptide substrate, yielding 2 ' -O -acetyl-ADP-ribose and the deacetylated peptide substrate. ( C ) Stalled sirtuin catalytic mechanism with thioacetylated and thiourea substrates substituted for the acetylated substrate. ( D ) Thiourea-lysine sirtuin inhibitors. From left to right, Sirt5 inhibitor Compound 49 (a succinyl-thiocarbamoyl pseudopeptide) [201], and Sirt5 inhibitor NRD167 (a succinyl-thiocarbamoyl pseudopeptide) [202]. Figure 5. The sirtuin catalytic mechanism provides a rationale for using thioacyl-lysine derivatives as sirtuin inhibitors. ( A ) Mechanism of nicotinamide-mediated sirtuin inhibition. ( B ) Native sirtuin catalytic mechanism with acetylated peptide substrate, yielding 2 ' -O -acetyl-ADP-ribose and the deacetylated peptide substrate. ( C ) Stalled sirtuin catalytic mechanism with thioacetylated and thiourea substrates substituted for the acetylated substrate. ( D ) Thiourea-lysine sirtuin inhibitors. From left to right, Sirt5 inhibitor Compound 49 (a succinyl-thiocarbamoyl pseudopeptide) [201], and Sirt5 inhibitor NRD167 (a succinyl-thiocarbamoyl pseudopeptide) [202].

<!-- image -->

Sirtuin inhibition by nicotinamide exhibits therapeutic e ff ects in several disease models [203-205]. Inhibition of Sirt1/2 by nicotinamide was suggested as a mechanism for the decreased cognitive impairment, reduced phospho-tau species, and increased α -tubulin acetylation observed in a transgenic mouse model of Alzheimer's disease [203]. Nicotinamide also reduced cell viability in prostate and oral cancers through Sirt1 and Sirt3 inhibition, respectively; however, the supra-physiological nicotinamide concentrations' (5-40 mM) need to observe anti-cancer e ff ects in these studies cast doubt on the therapeutic potential of nicotinamide in such models [204,205]. Notably, nicotinamide is also a precursor for NAD + synthesis [206], which may result in a net stimulation of sirtuin activity due to increased NAD + levels (see Section 2.5). Sirtuin inhibition by nicotinamide exhibits therapeutic effects in several disease models [203-205]. Inhibition of Sirt1/2 by nicotinamide was suggested as a mechanism for the decreased cognitive impairment, reduced phospho-tau species, and increased α -tubulin acetylation observed in a transgenic mouse model of Alzheimer's disease [203]. Nicotinamide also reduced cell viability in prostate and oral cancers through Sirt1 and Sirt3 inhibition, respectively; however, the supra-physiological nicotinamide concentrations' (5-40 mM) need to observe anti-cancer effects in these studies cast doubt on the therapeutic potential of nicotinamide in such models [204,205]. Notably, nicotinamide is also a precursor for NAD + synthesis [206], which may result in a net stimulation of sirtuin activity due to increased NAD + levels (see Section 2.5).

To circumvent this inherent limitation of nicotinamide-mediated sirtuin inhibition, various nicotinamide analogs have been developed [207-210]. For example, screening of a nicotinamide and benzamide library identi fi ed a 2-anilinobenzamide compound as a modest Sirt1 inhibitor with ~4 and ~14-fold selectivity for Sirt1 over Sirt2 and Sirt3 [207]; cellular activity was con fi rmed by a concentration-dependent increase in acetylation of the Sirt1 substrate p53 in the HCT116 colorectal cancer cell line [207]. Further SAR studies led to Compound 33a , which inhibited Sirt2 with an IC 50 value of 1 µM and exhibited >300fold selectivity for Sirt2 over Sirt1/3; however, o ff -target inhibition of class I/II HDACs, CYP3A4, and CYP2D6 was also observed [208]. Cellular Sirt2 inhibition was observed through a concentration-dependent increase in the acetylation of Sirt2 substrate α -tubulin in HCT116 cells without changes in p53 acetylation [208]. To circumvent this inherent limitation of nicotinamide-mediated sirtuin inhibition, various nicotinamide analogs have been developed [207-210]. For example, screening of a nicotinamide and benzamide library identified a 2-anilinobenzamide compound as a modest Sirt1 inhibitor with ~4 and ~14-fold selectivity for Sirt1 over Sirt2 and Sirt3 [207]; cellular activity was confirmed by a concentration-dependent increase in acetylation of the Sirt1 substrate p53 in the HCT116 colorectal cancer cell line [207]. Further SAR studies led to Compound 33a , which inhibited Sirt2 with an IC 50 value of 1 µ Mand exhibited >300fold selectivity for Sirt2 over Sirt1/3; however, off-target inhibition of class I/II HDACs, CYP3A4, and CYP2D6 was also observed [208]. Cellular Sirt2 inhibition was observed through a concentration-dependent increase in the acetylation of Sirt2 substrate α -tubulin in HCT116 cells without changes in p53 acetylation [208].

Tenovins have also been identified as potential nicotinamide analog sirtuin inhibitors from a cell-based screen of a small molecule library [209]. Consistent with the enhanced p53 protein stability resulting from retention of acetylation [211], tenovins increased p53 protein levels consistent with Sirt1 inhibition and decreased viability of several cancer cell lines [209]. Increased α -tubulin acetylation suggested that tenovins also inhibit Sirt2 [209]. Kinetic studies of tenovin-mediated sirtuin inhibition suggested a non-competitive mechanism towards both the acyl-lysine substrate and NAD + co-substrate [209], analogous to nicotinamide-mediated sirtuin inhibition [6,173]. Further biochemical and biophysical characterization of tenovins has generated derivatives with improved solubility and retention of Sirt1/2-mediated increases in p53 protein and α -tubulin acetylation levels [212] or increases in p21 protein expression [213].

Alternative nicotinamide analogs have been generated through isostere replacement of the nicotinamide amide bond and the introduction of conformational constraints [210]. Compound 2 , a 3-triazolylpyridine (3-TYP) isostere, was the most potent sirtuin inhibitor identified from these syntheses, inhibiting Sirt3 with an IC 50 value of 38 µ Min vitro and exhibiting ~six-fold selectivity for Sirt3 over Sirt1/2 [210]. Treatment of the HeLa cervical cancer cell line and the SK-MEL-28 melanoma cell line with 3-TYP increased mitochondrial protein acetylation and superoxide levels, respectively, consistent with cellular Sirt3 inhibition [210]. However, 3-TYP is also an inhibitor of methionine aminopeptidase 2 (MetAP2) [214] and indoleamine 2,3-dioxygenase 1 (IDO1) [215] and presently is only used as a chemical probe to elucidate Sirt3 biological functions [216]. Therefore, reducing the off-target inhibition of IDO1 and MetAP2 is necessary to develop 3-TYP derivatives with therapeutic potential.

Taken together, nicotinamide and its chemical analogs provide a logical and biologically relevant mechanism of sirtuin deacylase inhibition. However, given the broadspectrum inhibition of all sirtuin isoforms by nicotinamide [174,197,198], the collateral inhibition of NAD + -consuming enzymes PARPs and CD38 (see Section 2.5) by nicotinamide (IC 50 ~500 µ Mand K i = 920 µ Min vitro, respectively [217,218]) and the relatively low sirtuin isoform selectivity of existing nicotinamide analogs, further studies are required to enhance the overall target specificity of nicotinamide analog sirtuin inhibitors. Moreover, improving the selectivity of nicotinamide-based inhibitors for sirtuins over other enzymes that employ nicotinamide as a substrate, like nicotinamide phosphoribosyltransferase of the NAD + salvage pathway (nicotinamide K M = 57 nM [219]), will be essential for developing sirtuin inhibitors with translational potential for multiple types of cancer.

## 4.2. Mechanism-Based Sirtuin Inhibition by Thioacetyl-Lysine

Experimental manipulation of the native sirtuin deacylase catalytic pathway (Figure 5B) has demonstrated that substituting the oxygen of the acyl-lysine substrate with sulfur results in potent mechanism-based sirtuin inhibition [175-177]. Analogous to the native lysine deacylation mechanism, nucleophilic attack of the nicotinamide ribosyl bond by the thioacyl sulfur (Figure 5C, Table 3) results in rapid nicotinamide production and the formation of a stalled S -alkylamidate intermediate [175,176]. Notably, turnover of the S -alkylamidate to products 1 ' -S H-2 ' -O -ADP ribose and dethioacylated peptide is ~two orders of magnitude slower than the native deacetylation reaction (Figure 5B vs. Figure 5C) [175,176], resulting in potent mechanism-based inhibition of sirtuin catalysis.

The first thioacetyl-lysine peptide sirtuin inhibitor consisted of a peptide modeled after the C -terminus of the Sirt1 substrate p53, with a thioacetyl warhead positioned at a lysine residue near the middle of the peptide sequence (i.e., thioacetyl-p53) [175]. Thioacetyl-p53 inhibited Sirt1 deacetylase activity by decreasing the observed rate constant of deacetylation ~400-fold and did not impact the deacetylase activity of the non-sirtuin HDAC8 [175]. Follow-up studies demonstrated that thioacetyl-p53 exhibited equipotent inhibition of Sirt1/2 (IC 50 ~2 µ M)and~40-fold selectivity for Sirt1/2 over Sirt3 in vitro [177]. Subsequent thioacetyl peptides modeled after sirtuin isoform-specific substrates α -tubulin and acetylCoA synthetase 2 (AceCS2) were developed to target Sirt2 and Sirt3, respectively [177].

Thioacetylα -tubulin inhibited Sirt2 with an IC 50 value of 11.4 µ Mandshowed~10-and~40fold selectivity for Sirt2 over Sirt1 and Sirt3, respectively [177]. Thioacetyl-AceCS2 inhibited Sirt2 and Sirt3 with IC 50 values of ~4.4 µ M [177]; however, the thioacetyl-peptide still exhibited five-fold selectivity for Sirt1 over Sirt2/3, despite being designed to inhibit Sirt3 in vitro [177]. These findings emphasize that Sirt1-3 substrate selectivity, partially governed by varying sirtuin isoform subcellular localization [11], can be superseded by the strong Sirt1-3 sequence conservation [4]. However, truncating a thioacetyl-p53 pentapeptide to a tripeptide resulted in improved inhibition of Sirt1 (IC 50 = 570 nM) and >250-fold selectivity for Sirt1 over Sirt2 [178]. SAR optimization of non-selective Sirt1-3 thioacetyl-tripeptide inhibitors [179] generated thioacetyl-tripeptides that inhibited Sirt3 with an IC 50 value of ~1.6 µ M with ~9-10-fold selectivity for Sirt3 over Sirt1/2 [180]. Therefore, even minor alterations to a thioacetyl-lysine peptide sequence can significantly change the selectivity of its sirtuin isoform inhibition.

Since the cellular efficacy of peptidic inhibitors is limited by susceptibility to proteolytic degradation and poor membrane permeability [220], non-peptidic and pseudopeptidic scaffolds containing a thioacetyl-lysine warhead have been developed as an alternative class of mechanism-based sirtuin inhibitors [181-183]. For example, a Cbz-K(thioacetyl)NH-Ph scaffold was used to generate Compound 1 , a thioacetyl-lysine pseudopeptide with hydrophobic substituents recognized by amino acid residues at the opening of the sirtuin acylated substrate binding cleft [181]. Compound 1 inhibited Sirt1 with an IC 50 value of 2.7 µ Mand demonstrated 8.5 and >300-fold selectivity for Sirt1 over Sirt2 and Sirt3; moreover, a concentration-dependent increase of p53 acetylation in the HCT116 colon cancer cell line suggested that Compound 1 effectively inhibited cellular Sirt1 [181]. Analysis of Sirt1/2 homology models and existing Sirt3 crystal structures, coupled with molecular docking, has also led to Sirt1/2 thioacetyl-lysine pseudopeptide inhibitors with improved potency and cellular activity [182]. In a follow-up study, a fragment-based screening approach was employed to identify optimal N - and C -terminal pseudopeptide modifications for sirtuin inhibition; the resulting thioacetyl-lysine pseudopeptides effectively inhibited Sirt1-3 (IC 50 = 0.24-29.4 µ M) and increased p53 acetylation in epithelial and neuroblastoma cell lines, indicating on-target cellular inhibition of Sirt1 [183]. Additionally, the optimized pseudopeptides exhibited antiproliferative effects on the A549 lung adenocarcinoma and MCF7 breast cancer cell lines by inducing G 1 cell cycle arrest [183].

## 4.3. Mechanism-Based Sirtuin Inhibition by Thioacyl-Lysine Derivatives

Asanalternative to substrate-specific peptide sequences and truncated peptide lengths, Isoform-selective sirtuin inhibition can also be enhanced by switching the acetyl group of a thioacetyl-lysine warhead to mimic sirtuin-specific acyl substrates [186]. To this end, a thiomyristoyl (TM)-lysine pseudopeptide (Table 3) potently inhibited Sirt2 deacetylation (IC 50 = 28 nM) with 3500-fold selectivity for Sirt2 over Sirt1 and no significant activity against Sirt3/5/6/7 [186]. Selective TM pseudopeptide-mediated Sirt2 inhibition was confirmed by increased acetylation of the Sirt2 substrate α -tubulin and no changes in the acetylation of the Sirt1 substrate p53 in several breast cancer cell lines [186]. The TM pseudopeptide demonstrated cytotoxic effects in multiple breast cancer cell lines but had less impact on the growth of normal epithelial cell lines, indicating that breast cancer cells are more susceptible to Sirt2 inhibition [186]. Notably, the TM pseudopeptide decreased tumor size or increased overall survival in two mouse models of breast cancer and induced a G 1 /G 0 cell cycle arrest in MCF7 cells via reduced expression of the oncogene c-Myc, indicating that Sirt2 is an attractive therapeutic target in breast cancer [186]. Subsequent SAR studies generated additional nanomolar-potent TM pseudopeptide inhibitors of Sirt2 deacetylase and demyristoylase activities [188]. Compound 26 , the most potent of these next-generation TM pseudopeptides, exhibited on-target Sirt2 inhibition in MCF7 breast cancer cells, as demonstrated by increased acetylation of perinuclear α -tubulin; Compound 26 also decreased MCF7 cell migration, a proxy for metastatic potential [188].

Glucose conjugation and follow-up SAR studies have also been employed to improve the solubility of TM pseudopeptides [189]. Despite decreased specificity for Sirt2 over Sirt1/3, the optimized TM analogs retained potent inhibition of Sirt2 (IC 50 = 12-400 nM) [189]. Consistent with ~two-fold higher aqueous solubility relative to the original TM inhibitor, MDA-MB231 breast cancer cells treated with 100 µ Mof TM analog NH4-6 (Table 3) demonstrated a ~two-fold greater decrease in viability than cells treated with the original TM pseudopeptide inhibitor at the same concentration [189]. Further SAR studies based on the TM pseudopeptide led to JH-T4, which inhibited Sirt2 deacetylase and demyristoylase activities with IC 50 values of 30 and 40 nM, respectively [187]. JH-T4 also displayed 10- and 500-fold selectivity for Sirt2 over Sirt1 and Sirt3 in vitro [187]. Additionally, JH-T4 increased cellular fatty acylation of the Sirt2 target K-Ras4a and demonstrated cytotoxicity in multiple cancer cell lines [187]. However, the micromolar concentrations of JH-T4 used in cellular assays also inhibited Sirt1 (IC 50 = 300 nM), as demonstrated by increased p53 acetylation in MCF7 cells and off-target cytotoxicity in normal epithelial cells treated with JH-T4 [187].

Since JH-T4 also inhibits Sirt3 deacetylase activity (IC 50 = 2.5-15 µ M) at low µ M concentrations [187,194], the JH-T4 benzyl carbamoyl moiety was substituted for the cationic/lipophilic TPP + group to enhance Sirt3 inhibition via mitochondrial targeting [194]. The resulting inhibitor YC8-02 demonstrated improved Sirt3 inhibition in vitro and enhanced mitochondrial penetration in the Karpas 422 diffuse large B cell lymphoma (DLBCL) cell line compared to JH-T4; however, YC8-02 still inhibited Sirt1 and Sirt2 with IC 50 values of 2.8 µ Mand 62 nM, respectively [194]. YC8-02 increased global mitochondrial protein acetylation and induced metabolic perturbations in Karpas 422 cells; YC8-02 also inhibited tumor growth of DLBCL xenografts in mice [194]. Similar metabolic results were observed with genetic Sirt3 depletion in Karpas 422 cells, suggesting that the effects of YC8-02 in DLBCL models are mediated by Sirt3 inhibition [194]. Additional studies demonstrated that YC8-02 induced metabolic perturbations in leukemic stem cells [221] and increased acetylation of the Sirt3 substrate isocitrate dehydrogenase 2 in MDA-MB-231 cells [222], consistent with its presumed role as a mitochondrially targeted Sirt3 inhibitor.

Since myristoyl-lysine substrates are preferred Sirt6 deacylation targets [33], the TM warhead was integrated into pentapeptide Sirt6 inhibitors based on substrates TNF α or H3K9 [223]. Several pentapeptides inhibited Sirt6 demyristoylase activity (IC 50 = 1.7-42.2 µ M); however, inhibition of Sirt1-3 deacetylase activity was observed at comparable concentrations [223]. Treatment of HEK293T cells with the pentapeptide Sirt6 inhibitors increased TNF α fatty acylation, suggesting that the inhibitors possess on-target cellular activity despite overall low potency and specificity [223].

Analogous to TM, the acyl-lysine substrate specificity of Sirt5 [22] can be mimicked by thiosuccinyl (TS) or thiomalonyl as the warhead in peptide-based Sirt5 inhibitors [197]. Accordingly, a 12-mer peptide with a TS warhead, based on the sequence of Sirt5 substrate H3K9, inhibited Sirt5 desuccinylation with an IC 50 value of 5 µ M and >20-fold selectivity for Sirt5 over Sirt1-3 [197]. In contrast to prior studies with thioacetyl peptide inhibitors [178,180], shortening the length of TS peptides did not improve potency towards Sirt5 [197]. Thioglutaryl pseudopeptides have also been developed as alternative Sirt5 inhibitors with IC 50 values of 120-730 nM and ~240-5000-fold selectivity for Sirt5 over Sirt1/2/3/6 [224].

Overall, thioacyl analogs represent an attractive scaffold to develop potent and isoformspecific inhibitors that stall the sirtuin catalytic mechanism, especially in cancer. Despite their observed efficacy in vitro and in select preclinical disease models, concerns still exist regarding the safety and durability of thioacyl-lysine sirtuin inhibitors as therapeutic agents. First, thioamides can exert cytotoxic effects via S -oxidation [225-227]. Furthermore, although thioacyl analogs effectively inhibit sirtuin deacylation, the stalled S -alkylamidate is eventually turned over to the dethioacylated substrate and 1 ' -SH -2 ' -O -acyl-ADP-ribose, terminating mechanism-based sirtuin inhibition (Figure 5C) [175,176].

## 4.4. Mechanism-Based Sirtuin Inhibition by Thiourea-Lysine Derivatives

To address the concerns raised by thioacyl analogs [175,176,225-227], thiourea analogs represent an ideal alternative; compared to thioamides, thiourea-containing compounds are more easily synthesized [190], exhibit increased cell penetrance [228], are less cytotoxic [225], and are not turned over to sirtuin deacylation products [51,185,190]. Our observation that urea-containing homocitrulline stalls the deacylase activity of yeast sirtuin Hst2 [25] provided the rationale for testing whether thiocarbamoyl-lysine, a thiourea-lysine derivative, could inhibit sirtuins in a mechanism-based manner similar to thioacyl-lysines [185]. Indeed, in the first step of sirtuin catalysis, thiourea-lysine analogs are converted to a stalled S -isothiourea intermediate incapable of proceeding further, resulting in persistent mechanism-based sirtuin inhibition (Figure 5B) [51,185,190]. Consistent with this scheme, a peptidomimetic containing thiocarbamoyl-lysine inhibited Sirt1-3 deacetylase activity with IC 50 values of 57-159 µ M, albeit ~13-55-fold less potently than its thioacetyl-lysine counterpart [185]. Using the same tripeptide mimetic scaffold, derivatization of the original thiocarbamoyl modification (-CSNH 2 ) to a methyl thiocarbamoyl (-CSNHCH 3 ) (Table 3) resulted in a pseudopeptide inhibitor with ~2-26-fold improved potency compared to the parent thiocarbamoyl pseudopeptide inhibitor, inhibiting Sirt1-3 deacetylation with IC 50 values of 2-50 µ M[184]. Notably, the methyl thiocarbamoyl pseudopeptide inhibitor demonstrated ~eight-fold greater antioxidant activity than the analogous thioacetyl pseudopeptide inhibitor when tested at 200 µ Min a superoxide radical scavenging assay [184]. The methyl thiocarbamoyl pseudopeptide inhibitor also increased p53 acetylation in the HCT116 colon cancer cell line in a concentration-dependent manner, consistent with ontarget cellular activity against Sirt1 [184].

Despite its therapeutic efficacy in breast cancer models [186], the poor aqueous solubility and laborious synthesis of the TM pseudopeptide Sirt2 inhibitor inspired conversion to thiocarbamoyl pseudopeptide analogs with improved solubility and decreased hydrophobicity [190]. Like the TM pseudopeptide, fatty-acyl thiocarbamoyl-lysine analogs AF8, AF10, and AF12, with respective alkyl chains of 7, 9, and 11 carbons (Table 3), inhibited Sirt2 with IC 50 values of 61-150 nM and exhibited ~180-2500-fold selectivity for Sirt2 over Sirt1/3 [190]; AF8 and AF10 also exhibited superior predicted solubility relative to the TM-lysine pseudopeptide [190]. Consistent with selective Sirt2 inhibition, HCT116 cells co-treated with trichostatin A and AF8 or AF10 exhibited increased acetylation of the Sirt2 substrate α -tubulin without significant increases in acetylation of the Sirt1 substrate p53 [190]. As observed with the TM pseudopeptide inhibitor [186], AF8 and AF10 selectively inhibited the growth of multiple cancer cell lines but not normal epithelial cells [190]. Treatment of HCT116 cells with AF8 or AF10 potently inhibited colony formation and significantly reduced tumor growth in a colon cancer xenograft mouse model [190], suggesting that thiocarbamoyl pseudopeptides may exhibit therapeutic potential for Sirt2 inhibition in cancer.

Myristoyl-thiocarbamoyl pseudopeptides like Compound 17 (Table 3), pan-Sirt1-3 inhibitors with IC 50 values of 0.4-2 µ Min vitro, have been adapted to include mitochondrialtargeting peptide sequences to confer selective cellular inhibition of the primary mitochondrial deacetylase Sirt3 [195]. Treatment of HEK293T cells with Compound 17 did not increase p53 or α -tubulin acetylation, consistent with negligible cellular activities against Sirt1 and Sirt2 [195]. However, the claim of selective cellular Sirt3 inhibition by Compound 17 , as demonstrated by increased acetylation of the Sirt3 substrate MnSOD [195], is insufficiently supported because MnSOD acetylation was examined without comparison to total MnSOD protein levels [195]; therefore, the observed increase in MnSOD acetylation could be due to increased MnSOD protein levels unrelated to inhibition of Sirt3 deacetylase activity [195]. Therefore, further validation of the cellular activity of Compound 17 and similar mitochondrial-targeted Sirt3-selective probes will be necessary before employing them to dissect the roles of Sirt3 in health and aging-related diseases.

Consistent with the Sirt5 substrate preference for negatively-charged deacylation targets [229], derivatization of methyl thiocarbamoyl Sirt1-3 pseudopeptide inhibitors to

carboxyethyl thiocarbamoyl resulted in a pseudopeptide inhibitor that inhibited Sirt5 with an IC 50 value of 5 µ Mand exhibited 20- and 480-fold selectivity for Sirt5 over Sirt1 and Sirt6, respectively [184]. SAR studies based on thioglutaryl-lysine generated thiourea pseudopeptides [201]. The best of these pseudopeptides, succinyl thiocarbamoyl Compound 49 (Figure 5D), inhibited Sirt5 desuccinylation and deglutarylation with IC 50 values of 170 nM and 110 nM, respectively [201]. Moreover, Compound 49 demonstrated >7-fold selectivity for Sirt5 over Sirt1/2/3/6 [201]. However, Compound 49 exhibited poor cellular activity [202]. As a result, the ethyl ester prodrug NRD167 (Figure 5D) was developed, which selectively decreased the proliferation of Sirt5-dependent acute myeloid leukemia (AML) cell lines with IC 50 values of 5-8 µ M, induced apoptosis at similar concentrations, and disrupted cellular oxidative phosphorylation and glycolysis, recapitulating the effects observed upon Sirt5 knockdown in Sirt5-dependent AML cells [202].

Despite being important for recognition by the Sirt5 substrate binding cleft [230], the carboxylate moiety of Compound 49 was hypothesized to be responsible for reduced cellular penetrance and activity (compare Compound 49 and NRD167 in Figure 5D) [202,229]. Succinyl thiocarbamoyl pseudopeptides containing carboxylate isosteres were developed to overcome this limitation [230]. The best compound, substituting the carboxylate for a masked tetrazole, penetrated cells and inhibited Sirt5-dependent AML cell line proliferation at half-maximal growth inhibitory concentrations (GI 50 ) of 9-24 µ M[230]. Additional SAR studies of Compound 49 focused on adding aryl fluorosulfate moieties to enhance inhibitor electrophilicity [231]. This class of thiourea pseudopeptides facilitated mechanism-based Sirt5 inhibition by forming covalent bonds at select Sirt5 tyrosine residues and producing inhibitor-Sirt5 covalent conjugates detectable via liquid chromatography-mass spectrometry [231]. Notably, aryl-fluorosulfate thiourea pseudopeptide Compound 17 covalently bound Sirt5 overexpressed in HEK293T cell lysates, native Sirt5 in living HeLa cells, and native Sirt5 in mouse cardiac tissue, demonstrating that covalent mechanism-based sirtuin inhibitors are an exciting new means to inhibit sirtuin activity [231].

Thiourea derivatives are an attractive class of isoform-selective sirtuin inhibitors with multiple practical advantages compared to thioacyl sirtuin inhibitors [184,190,225,228]. Moreover, these inhibitors harbor nanomolar potency, well-characterized cellular activity, and improved drug-like characteristics. Thus, thiourea derivatives exhibit high translational potential for investigating the therapeutic efficacy of selective sirtuin inhibition in a variety of cancers. Further preclinical studies in mouse models of cancer will aid further inhibitor optimization and begin to dissect the roles that specific sirtuins play in cancer pathogenesis.

## 4.5. Mechanism-Based Sirtuin Inhibition by Alternative Compounds

Some non-sulfur acetyl-lysine peptides and peptidomimetics also exhibit mechanismbased sirtuin inhibition [232-234]. For example, a fluorescent Sirt1 pseudopeptide substrate was modified to generate acetyl analogs containing electron-withdrawing or anionstabilizing groups to increase inhibitor nucleophilicity and promote interaction with both the acylated substrate binding cleft and the NAD + binding pocket [232]. The most potent acetyl analog identified was Compound 2k , which contained an ethoxycarbonyl moiety at the α -carbon of the acetamide group [232]. Compound 2k inhibited Sirt1 with an IC 50 value of 3.9 µ M and demonstrated 17- and >77-fold selectivity for Sirt1 over Sirt2 and Sirt3, respectively [232]. Mechanism-based inhibition was facilitated by the pseudopeptide analog forming a thermodynamically favored acetyl-lysine/ADP-ribose conjugate detectable via mass spectrometry [232]. However, the occupation of both the acyl-lysine and NAD + binding pockets was not structurally confirmed [232]. Compound 2k induced a concentration-dependent increase in acetylation of the Sirt1 substrate p53 in HCT116 colon cancer cells co-treated with etoposide, consistent with on-target cellular Sirt1 inhibition [232]. Carboxamides, isosteres of the acetyl group, have also demonstrated mechanism-based sirtuin inhibition [233,234]. In particular, when incorporated into peptides or pseudopeptides, the acetyl-lysine analog L -2-amino-7-carboxamidoheptanoic acid (L-ACAH) inhibited Sirt1-3 and yielded a long-lived catalytic intermediate detectable via

mass spectrometry, suggesting mechanism-based sirtuin inhibition [233]. Derivatization of L-ACAH to ethyl L-ACAH increased inhibitory potency towards Sirt1-3 by ~2-7-fold, while carboxymethyl and dodecyl L-ACAH analogs were Sirt5 and Sirt6 inhibitors [234]. Although these non-sulfur-based sirtuin inhibitors generally exhibit weak potency and sirtuin isoform selectivity [232-234], they remain intriguing alternatives to thioamide-based compounds and warrant further investigation.

## 4.6. Mechanism-Based Sirtuin Inhibition with Cyclic Peptides

Another method to circumvent the limitations of peptide-based inhibitors [220] is cyclization to form cyclic peptides, which exhibit increased resistance to peptidases and improved cell permeability [235]. Cyclic peptides can also enhance target affinity and selectivity due to conformational restriction and maximization of favorable interactions with protein targets [235]. Therefore, cyclic peptides have increased translational potential compared to other sirtuin peptidic inhibitors. Linear thioacetyl-lysine Sirt1-3 peptide inhibitors were converted into monocyclic and bicyclic analogs, whereby several cyclized peptides exhibited increased Sirt1/2 inhibitory potency and/or isoform selectivity compared to the parent linear Sirt1-3 peptide inhibitors [191,192]. Monocyclic thioacetyl-lysine peptide Compound 10 (Table 3), one of the most potent Sirt2 inhibitors identified, inhibited Sirt2 with an IC 50 value of 10.1 nM with ~seven- and eight-fold selectivity for Sirt2 over Sirt1 and Sirt3, respectively [191]. Consistent with the advantages of cyclic peptide inhibitors [235], a bicyclic Sirt1-3 thioacetyl-lysine peptide inhibitor exhibited increased proteolytic stability relative to the linear peptide inhibitor [192]. The bicyclic inhibitor also demonstrated on-target cellular activity, as demonstrated by increased acetylation of the Sirt1 substrate p53 in the HCT116 colon cancer cell line and decreased proliferation of a melanoma cell line that is susceptible to Sirt1/3 inhibition [192]. Additional optimization generated cyclic thioacetyl-lysine tripeptides which exhibited dual Sirt1/2 inhibition (IC 50 = 220-890 nM) and ~4-8-fold selectivity for Sirt1/2 over Sirt3 [193]. Like previously developed cyclic peptide inhibitors [192], several of the dual Sirt1/2 inhibitors demonstrated increased proteolytic stability [193].

Cyclic peptide sirtuin inhibitors with thiourea warheads are also being explored [236-238]. For example, cyclic peptides with carboxyethylthiocarbamoyl warheads were designed to selectively target Sirt5 [236]; the best cyclic peptides inhibited Sirt5 with an IC 50 value of ~2.2 µ M and demonstrated ≥ 60-fold selectivity for Sirt5 over Sirt1/2/3/6 [236]. A cyclic peptide with a tetradecyl thiocarbamoyl warhead also inhibited Sirt6 with an IC 50 value of 319 nM and ~20-, ~11-, and >940-fold selectivity for Sirt6 over Sirt2, Sirt3, and Sirt5, respectively; however, the inhibitor exhibited only ~two-fold selectivity for Sirt6 over Sirt1 [238]. Cyclic peptides containing thiourea and carboxamide warheads were also developed as the first mechanism-based Sirt7 inhibitors [237]. These cyclic peptides inhibited tRNA-activated Sirt7 deacetylase activity (IC 50 = 5-11.1 µ M), yet inhibited Sirt1/2/3/6 with similar or increased potency (IC 50 = 0.55-11.2 µ M) [237]. Nonetheless, these cyclic peptides provide a strong foundation for future optimization to increase sirtuin isoform selectivity as additional structural and biochemical data for Sirt7 become available.

## 5. Peptidic Non-Mechanism-Based Sirtuin Inhibitors

The first cyclic peptide sirtuin inhibitors, inspired by prior studies with linear peptides in yeast, slowed sirtuin deacylation without chemically modifying the sirtuin active site [239]. Previously, trifluoroacetyl-lysine warheads were found to bind sirtuins with ~six-fold greater affinity than acetyl-lysine and slow the rate of nicotinamide formation by ~five orders of magnitude compared to thioacetyl-lysine warheads and acetyllysine [25,176,240]. This inspired the use of a cell-free translation and peptide screening platform to generate peptides containing trifluoroacetyl-lysine warheads as Sirt2 inhibitors [239]. This screen generated two cyclic peptide inhibitors with trifluoroacetyl-lysine warheads that inhibited Sirt2 with IC 50 values of 3.2-3.7 nM and demonstrated ~9-150-fold selectivity for Sirt2 over Sirt1/3 [239]. Recently, the same cell-free translation peptide

Molecules

2024

,

29

, x FOR PEER REVIEW

26 of 39

that inhibited Sirt2 with IC

50

values of 3.2-3.7 nM and demonstrated ~9-150-fold selectiv-

ity for Sirt2 over Sirt1/3 [239]. Recently, the same cell-free translation peptide screen used

screen used to identify the first cyclic peptide inhibitors [239] was employed to develop non-mechanism-based cyclic peptide and peptide lariat sirtuin inhibitors [241]. Peptide lariat Compound 41 (Figure 6) was found to be a novel Sirt7 inhibitor (IC 50 = 2.7 µ M) that exhibited ~55-fold selectivity for Sirt7 over Sirt6, with minimal impacts on Sirt1/2/3/5 at 10 µ M[241]. HEK293T cells treated with Compound 41 demonstrated increased acetylation of the Sirt7 substrate histone H3K18, consistent with on-target cellular Sirt7 inhibition [241]. Compound 41 represents a substantial improvement in Sirt7 inhibitor potency and sirtuin isoform specificity [237,241]. Therefore, Compound 41 demonstrates that peptide lariats are an exciting new avenue for sirtuin inhibitors, especially since their precise mode of sirtuin inhibition is unknown [241]. to identify the fi rst cyclic peptide inhibitors [239] was employed to develop non-mechanism-based cyclic peptide and peptide lariat sirtuin inhibitors [241]. Peptide lariat Compound 41 (Figure 6) was found to be a novel Sirt7 inhibitor (IC 50 = 2.7 µM) that exhibited ~55-fold selectivity for Sirt7 over Sirt6, with minimal impacts on Sirt1/2/3/5 at 10 µM [241]. HEK293T cells treated with Compound 41 demonstrated increased acetylation of the Sirt7 substrate histone H3K18, consistent with on-target cellular Sirt7 inhibition [241]. Compound 41 represents a substantial improvement in Sirt7 inhibitor potency and sirtuin isoform speci fi city [237,241]. Therefore, Compound 41 demonstrates that peptide lariats are an exciting new avenue for sirtuin inhibitors, especially since their precise mode of sirtuin inhibition is unknown [241].

Figure 6. Peptide lariats represent a novel class of peptidic non-mechanism-based sirtuin inhibitors. Peptide lariat Compound 41 [241]. Figure 6. Peptide lariats represent a novel class of peptidic non-mechanism-based sirtuin inhibitors. Peptide lariat Compound 41 [241].

<!-- image -->

## 6. Conclusions 6. Conclusions

To aid investigators studying the cellular functions of sirtuin isoforms, we recommend several compounds (Table 4) for use as chemical probes and foundations for development into more e ff ective therapeutics, while considering their limitations. Notably, no activators of Sirt2, Sirt4, or Sirt7 have been reported (Table 4). The suggested Sirt1 activators SRT1720 or SRT2104 [59,60] show high potency, isoform selectivity, and biological activity in preclinical and early clinical trials. However, SRT1720 and SRT2104 e ffi cacy is limited to scenarios where Sirt1 substrates include hydrophobic and aromatic amino acids' C -terminal to the acyl-lysine residue [39,86,87]. Moreover, SRT1720 and SRT2104 exhibit o ff -target e ff ects on non-sirtuin proteins, including AMPK, receptors, enzymes, transporters, and ion channels [85,90]. In contrast, due to its unique inhibition mechanism, the Sirt1 inhibitor EX-527 [127] has nanomolar potency and translational potential in Huntington's disease but is often used in cellular studies at micromolar concentrations that also inhibit Sirt2 [147-150]. The recommended Sirt2 inhibitors TM [186], AF8/AF10 [190], and SirReal2 [124] display nanomolar potency, high isoform selectivity, and e ffi cacy in various cancer models. However, further optimization of compound solubility is required To aid investigators studying the cellular functions of sirtuin isoforms, we recommend several compounds (Table 4) for use as chemical probes and foundations for development into more effective therapeutics, while considering their limitations. Notably, no activators of Sirt2, Sirt4, or Sirt7 have been reported (Table 4). The suggested Sirt1 activators SRT1720 or SRT2104 [59,60] show high potency, isoform selectivity, and biological activity in preclinical and early clinical trials. However, SRT1720 and SRT2104 efficacy is limited to scenarios where Sirt1 substrates include hydrophobic and aromatic amino acids' C -terminal to the acyl-lysine residue [39,86,87]. Moreover, SRT1720 and SRT2104 exhibit off-target effects on non-sirtuin proteins, including AMPK, receptors, enzymes, transporters, and ion channels [85,90]. In contrast, due to its unique inhibition mechanism, the Sirt1 inhibitor EX-527 [127] has nanomolar potency and translational potential in Huntington's disease but is often used in cellular studies at micromolar concentrations that also inhibit Sirt2 [147-150]. The recommended Sirt2 inhibitors TM [186], AF8/AF10 [190], and SirReal2 [124] display nanomolar potency, high isoform selectivity, and efficacy in various cancer models. However, further optimization of compound solubility is required for TM and AF8/AF10 [186,190].

for TM and AF8/AF10 [186,190]. Table 4. Recommended sirtuin modulators for cellular studies.

| Targeted Sirtuin Isoform   | Activators                                      | Inhibitors                                                                |
|----------------------------|-------------------------------------------------|---------------------------------------------------------------------------|
| Sirt1                      | SRT1720 [59], SRT2104 [60]                      | EX-527 [127]                                                              |
| Sirt2                      | N/A *                                           | TM[186], AF8/10 [189,190], SirReal2 [124]                                 |
| Sirt3                      | ADTL-SA1215 [62]                                | Mitochondrially targeted myristoyl-thiocarbamoyl pseudopeptides [194,195] |
| Sirt4                      | N/A *                                           | Compound 60 / 69 [132]                                                    |
| Sirt5                      | 1,4-dihydropyridine derivative Compound 31 [63] | Succinyl-thiocarbamoyl pseudopeptides [201,202]                           |

Table 4. Cont.

| Targeted Sirtuin Isoform   | Activators              | Inhibitors                       |
|----------------------------|-------------------------|----------------------------------|
| Sirt6                      | LPA [65], MDL-811 [107] | N/A *                            |
| Sirt7                      | N/A *                   | Peptide lariat Compound 41 [240] |

N/A * indicates not available, where no suitable sirtuin modulator is recommended.

The suggested Sirt3 activator ADTL-SA1215 [62] possesses high potency and is effective in cell and mouse models of breast cancer. However, this Sirt3 activator was only recently reported [62], and follow-up studies from other laboratories using this activator have yet to be published. On the other hand, mitochondrially targeted myristoyl-thiocarbamoyl pseudopeptides like YC8-02 [194] and Compound 17 [195], the recommended Sirt3 inhibitors, demonstrate a first-in-class ability to target a sirtuin isoform within a subcellular organelle. Nonetheless, these inhibitors lack selectivity for Sirt3 over Sirt1/2 in vitro, so further confirmation of mitochondrial-targeting and selective Sirt3 inhibition is required at the cellular level for both compounds [194,195]. The suggested Sirt4 inhibitors Compounds 60 and 69 [132] are first-in-class modulators of Sirt4 deacylase activity [132]. Sirt4 inhibitor development would be accelerated by in-depth structural and kinetic studies confirming the modes by which Compounds 60 and 69 bind and inhibit Sirt4, which have not been reported [132].

The suggested Sirt5 activator, 1,4-dihydropyridine derivative Compound 30 , exhibits good selectivity for Sirt5 [63]; however, like ADTL-SA1215, the efficacy of this compound in the hands of others remains to be tested [63]. Succinyl-thiocarbamoyl peptides like Compound 49 [201] and NRD167 [202], the recommended Sirt5 inhibitors, exhibit nanomolar potency and efficacy in Sirt5-dependent acute myeloid leukemia cell lines. However, the effects of these inhibitors in mouse or other preclinical models of acute myeloid leukemia have not yet been reported [201,202]. The suggested Sirt6 activators LPA [65] and MDL811 [107] substantially increase Sirt6 deacetylase activity. In particular, MDL-811 exhibits high efficacy in mouse models of cancer and neural damage [107]. In contrast to typical sirtuin inhibition modes, the recommended Sirt6 inhibitor, JYQ-42 [165], allosterically decreases Sirt6 deacetylase activity and demonstrates translational promise in pancreatic cancer cell models. However, SAR optimization of JQY-42 has yet to be conducted [165], suggesting that enhancing JQY-42 binding to the unique Sirt6 pocket Z will be necessary to increase its inhibitory potency from the micromolar to the nanomolar range. Despite requiring further optimization to increase overall potency and selectivity [239], the recent development of peptide lariat Compound 41 , the recommended Sirt7 inhibitor [239], opens up an exciting avenue of peptide lariat sirtuin inhibitor research.

With universal cellular expression yet unique subcellular localizations and functions [10,11], sirtuins are paradoxically reported to promote and prevent aging-related diseases [46-48]. Existing small molecule activators, small molecule inhibitors, and mechanismbased inhibitors of sirtuins allow unprecedented insight into the kinetics of sirtuin deacylase activity and provide rationales for the selective activation or inhibition of sirtuins in particular biological and disease contexts. As the sirtuin field evolves, continued efforts to enhance potency, isoform specificity, cellular activity, solubility, and pharmacokinetic properties will allow for further dissection of individual sirtuin isoform roles in the pathophysiology of aging. Moreover, such studies will lead to the identification of improved scaffolds for developing safe, bioactive, sirtuin-selective drugs with therapeutic benefits for aging-related diseases.

Author Contributions: Conceptualization, C.J.G. and B.C.S.; writing-original draft preparation; C.J.G. and B.C.S.; writing-review and editing; K.L.B. and B.C.S.; visualization, K.L.B., C.J.G. and B.C.S.; supervision, B.C.S.; funding acquisition, B.C.S. All authors have read and agreed to the published version of the manuscript.

Funding: This work was supported by the National Institutes of Health grants R01 DK119359 (B.C.S.) and F30 CA278386 (K.L.B.). K.L.B. is a member of the Medical Scientist Training Program at the Medical College of Wisconsin, which is supported in part by the National Institutes of Health Training Grant T32 GM080202. K.L.B. is also supported by the Medical College of Wisconsin Cancer Center. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

Institutional Review Board Statement: Not applicable.

Informed Consent Statement: Not applicable.

Data Availability Statement: No new data were created or analyzed in this study. Data sharing is not applicable to this article.

Conflicts of Interest: The authors declare no conflicts of interest.

## Abbreviations

AceCS2 (acetyl-CoA synthetase 2); AML (acute myeloid leukemia); AMPK (AMP-activated protein kinase); BET (bromodomain and extraterminal domain); BIM (bisindolylmaleimide); deHMGylation (deβ -hydroxy β -methylglutarylation); DLBCL (diffuse large B cell lymphoma); EC50 (half-maximal effective concentration); eIF5a (eukaryotic translation initiation factor 5a); FFA (free fatty acid); FOXO3a (fork-head box O3a); GI50 (half-maximal growth inhibitory concentration); GOT1 (glutamate oxaloacetate transaminase 1); GSK (GlaxoSmithKline); HCC (hepatocellular carcinoma); HDAC(histone deacetylase); HNSCC (head and neck squamous cell carcinoma); IC50 (half-maximal inhibitory concentration); IDO1 (indoleamine 2,3-dioxygenase 1); Ki (inhibition constant); KM (Michaelis constant); L-ACAH (L-2-amino-7-carboxamidoheptanoic acid); LFA (long-chain fatty acid); LPA (oleoyl-lysophosphatidic acid); MetAP2 (methionine aminopeptidase 2); MnSOD (manganese superoxide dismutase); NAD (nicotinamide adenine dinucleotide); NRF2 (nuclear factor erythroid 2-related factor 2); NSCLC (non-small cell lung cancer); OAADPr (O-acyl-ADP-ribose); OSCC (oral squamous cell carcinoma); OSCP (oligomycin-sensitivity conferring protein); PARP (poly-ADP-ribose polymerase); PDHA1 (pyruvate dehydrogenase subunit A1); PGC-1 α (proliferator-activated receptor γ coactivator 1 α ); SAR (structure-activity relationship); SBD (STAC-binding domain); SirReal (sirtuinrearranging ligand); Sir2 (Silent Information Regulator 2); STAC (sirtuin-activating compound); TM (thiomyristoyl); TPP+ (triphenylphosphonium); TRPM2 (transient receptor potential cation channel, subfamily M, member 2); TS (thiosuccinyl); maximal reaction velocity (Vmax).

## References

- 1. Grozinger, C.M.; Chao, E.D.; Blackwell, H.E.; Moazed, D.; Schreiber, S.L. Identification of a class of small molecule inhibitors of the sirtuin family of NAD-dependent deacetylases by phenotypic screening. J. Biol. Chem. 2001 , 276 , 38837-38843. [CrossRef] [PubMed]
- 2. Rine, J.; Herskowitz, I. Four genes responsible for a position effect on expression from HML and HMR in Saccharomyces cerevisiae. Genetics 1987 , 116 , 9-22. [CrossRef]
- 3. Loo, S.; Rine, J. Silencers and Domains of Generalized Repression. Science 1994 , 264 , 1768-1771. [CrossRef] [PubMed]
- 4. Frye, R.A. Phylogenetic classification of prokaryotic and eukaryotic Sir2-like proteins. Biochem. Biophys. Res. Commun. 2000 , 273 , 793-798. [CrossRef] [PubMed]
- 5. Michan, S.; Sinclair, D. Sirtuins in mammals: Insights into their biological function. Biochem. J. 2007 , 404 , 1-13. [CrossRef] [PubMed]
- 6. Landry, J.; Slama, J.T.; Sternglanz, R. Role of NAD(+) in the deacetylase activity of the SIR2-like proteins. Biochem. Biophys. Res. Commun. 2000 , 278 , 685-690. [CrossRef]
- 7. Jackson, M.D.; Denu, J.M. Structural identification of 2 ' - and 3 ' -O -acetyl-ADP-ribose as novel metabolites derived from the Sir2 family of beta-NAD + -dependent histone/protein deacetylases. J. Biol. Chem. 2002 , 277 , 18535-18544. [CrossRef]
- 8. Anderson, K.A.; Madsen, A.S.; Olsen, C.A.; Hirschey, M.D. Metabolic control by sirtuins and other enzymes that sense NAD + , NADH, or their ratio. Biochim. Biophys. Acta Bioenerg. 2017 , 1858 , 991-998. [CrossRef]
- 9. Seto, E.; Yoshida, M. Erasers of histone acetylation: The histone deacetylase enzymes. Cold Spring Harb. Perspect. Biol. 2014 , 6 , a018713. [CrossRef]
- 10. Michishita, E.; Park, J.Y.; Burneskis, J.M.; Barrett, J.C.; Horikawa, I. Evolutionarily conserved and nonconserved cellular localizations and functions of human SIRT proteins. Mol. Biol. Cell 2005 , 16 , 4623-4635. [CrossRef]

- 11. Bheda, P.; Jing, H.; Wolberger, C.; Lin, H. The Substrate Specificity of Sirtuins. Annu. Rev. Biochem. 2016 , 85 , 405-429. [CrossRef] [PubMed]
- 12. Tanno, M.; Sakamoto, J.; Miura, T.; Shimamoto, K.; Horio, Y. Nucleocytoplasmic shuttling of the NAD + -dependent histone deacetylase SIRT1. J. Biol. Chem. 2007 , 282 , 6823-6832. [CrossRef] [PubMed]
- 13. Eldridge, M.J.G.; Pereira, J.M.; Impens, F.; Hamon, M.A. Active nuclear import of the deacetylase Sirtuin-2 is controlled by its C-terminus and importins. Sci. Rep. 2020 , 10 , 2034. [CrossRef] [PubMed]
- 14. Cooper, H.M.; Spelbrink, J.N. The human SIRT3 protein deacetylase is exclusively mitochondrial. Biochem. J. 2008 , 411 , 279-285. [CrossRef] [PubMed]
- 15. Hallows, W.C.; Albaugh, B.N.; Denu, J.M. Where in the cell is SIRT3? Functional localization of an NAD + -dependent protein deacetylase. Biochem. J. 2008 , 411 , e11-e13. [CrossRef] [PubMed]
- 16. Scher, M.B.; Vaquero, A.; Reinberg, D. SirT3 is a nuclear NAD + -dependent histone deacetylase that translocates to the mitochondria upon cellular stress. Genes Dev. 2007 , 21 , 920-928. [CrossRef] [PubMed]
- 17. Sundaresan, N.R.; Samant, S.A.; Pillai, V.B.; Rajamohan, S.B.; Gupta, M.P. SIRT3 is a stress-responsive deacetylase in cardiomyocytes that protects cells from stress-mediated cell death by deacetylation of Ku70. Mol. Cell. Biol. 2008 , 28 , 6384-6401. [CrossRef] [PubMed]
- 18. Nishida, Y.; Rardin, M.J.; Carrico, C.; He, W.; Sahu, A.K.; Gut, P.; Najjar, R.; Fitch, M.; Hellerstein, M.; Gibson, B.W.; et al. SIRT5 Regulates both Cytosolic and Mitochondrial Protein Malonylation with Glycolysis as a Major Target. Mol. Cell 2015 , 59 , 321-332. [CrossRef]
- 19. Sanders, B.D.; Jackson, B.; Marmorstein, R. Structural basis for sirtuin function: What we know and what we don't. Biochim. Biophys. Acta 2010 , 1804 , 1604-1616. [CrossRef]
- 20. Jin, L.; Wei, W.; Jiang, Y.; Peng, H.; Cai, J.; Mao, C.; Dai, H.; Choy, W.; Bemis, J.E.; Jirousek, M.R.; et al. Crystal Structures of Human SIRT3 Displaying Substrate-induced Conformational Changes. J. Biol. Chem. 2009 , 284 , 24394-24405. [CrossRef]
- 21. Smith, B.C.; Settles, B.; Hallows, W.C.; Craven, M.W.; Denu, J.M. SIRT3 substrate specificity determined by peptide arrays and machine learning. ACS Chem. Biol. 2011 , 6 , 146-157. [CrossRef]
- 22. Du, J.; Zhou, Y.; Su, X.; Yu, J.J.; Khan, S.; Jiang, H.; Kim, J.; Woo, J.; Kim, J.H.; Choi, B.H.; et al. Sirt5 is a NAD-dependent protein lysine demalonylase and desuccinylase. Science 2011 , 334 , 806-809. [CrossRef]
- 23. Wang, M.; Lin, H. Understanding the Function of Mammalian Sirtuins and Protein Lysine Acylation. Annu. Rev. Biochem. 2021 , 90 , 245-285. [CrossRef]
- 24. Feldman, J.L.; Dittenhafer-Reed, K.E.; Kudo, N.; Thelen, J.N.; Ito, A.; Yoshida, M.; Denu, J.M. Kinetic and Structural Basis for Acyl-Group Selectivity and NAD + Dependence in Sirtuin-Catalyzed Deacylation. Biochemistry 2015 , 54 , 3037-3050. [CrossRef]
- 25. Smith, B.C.; Denu, J.M. Acetyl-lysine analog peptides as mechanistic probes of protein deacetylases. J. Biol. Chem. 2007 , 282 , 37256-37265. [CrossRef]
- 26. Zhang, X.; Cao, R.; Niu, J.; Yang, S.; Ma, H.; Zhao, S.; Li, H. Molecular basis for hierarchical histone deβ -hydroxybutyrylation by SIRT3. Cell Discov. 2019 , 5 , 35. [CrossRef] [PubMed]
- 27.
- Bao, X.; Wang, Y.; Li, X.; Li, X.M.; Liu, Z.; Yang, T.; Wong, C.F.; Zhang, J.; Hao, Q.; Li, X.D. Identification of 'erasers' for lysine crotonylated histone marks using a chemical proteomics approach. Elife 2014 , 3 , e02999. [CrossRef]
- 28. Feldman, J.L.; Baeza, J.; Denu, J.M. Activation of the protein deacetylase SIRT6 by long-chain fatty acids and widespread
- deacylation by mammalian sirtuins. J. Biol. Chem. 2013 , 288 , 31350-31356. [CrossRef] [PubMed]
- 29. Tan, M.; Peng, C.; Anderson, K.A.; Chhoy, P.; Xie, Z.; Dai, L.; Park, J.; Chen, Y.; Huang, H.; Zhang, Y.; et al. Lysine glutarylation is a protein posttranslational modification regulated by SIRT5. Cell Metab. 2014 , 19 , 605-617. [CrossRef]
- 30. Anderson, K.A.; Huynh, F.K.; Fisher-Wellman, K.; Stuart, J.D.; Peterson, B.S.; Douros, J.D.; Wagner, G.R.; Thompson, J.W.; Madsen, A.S.; Green, M.F.; et al. SIRT4 Is a Lysine Deacylase that Controls Leucine Metabolism and Insulin Secretion. Cell Metab. 2017 , 25 , 838-855.e815. [CrossRef] [PubMed]
- 31. Mathias, R.A.; Greco, T.M.; Oberstein, A.; Budayeva, H.G.; Chakrabarti, R.; Rowland, E.A.; Kang, Y.; Shenk, T.; Cristea, I.M. Sirtuin 4 is a lipoamidase regulating pyruvate dehydrogenase complex activity. Cell 2014 , 159 , 1615-1625. [CrossRef] [PubMed]
- 32. Feldman, J.L.; Dittenhafer-Reed, K.E.; Denu, J.M. Sirtuin catalysis and regulation. J. Biol. Chem. 2012 , 287 , 42419-42427. [CrossRef] [PubMed]
- 33. Jiang, H.; Khan, S.; Wang, Y.; Charron, G.; He, B.; Sebastian, C.; Du, J.; Kim, R.; Ge, E.; Mostoslavsky, R.; et al. SIRT6 regulates TNFα secretion through hydrolysis of long-chain fatty acyl lysine. Nature 2013 , 496 , 110-113. [CrossRef] [PubMed]
- 34. Gil, R.; Barth, S.; Kanfi, Y.; Cohen, H.Y. SIRT6 exhibits nucleosome-dependent deacetylase activity. Nucleic Acids Res. 2013 , 41 , 8537-8545. [CrossRef] [PubMed]
- 35. Liu, W.H.; Zheng, J.; Feldman, J.L.; Klein, M.A.; Kuznetsov, V.I.; Peterson, C.L.; Griffin, P.R.; Denu, J.M. Multivalent interactions drive nucleosome binding and efficient chromatin deacetylation by SIRT6. Nat. Commun. 2020 , 11 , 5244. [CrossRef] [PubMed]
- 36. Kuznetsov, V.I.; Liu, W.H.; Klein, M.A.; Denu, J.M. Potent Activation of NAD + -Dependent Deacetylase Sirt7 by Nucleosome Binding. ACS Chem. Biol. 2022 , 17 , 2248-2261. [CrossRef] [PubMed]
- 37. Tong, Z.; Wang, Y.; Zhang, X.; Kim, D.D.; Sadhukhan, S.; Hao, Q.; Lin, H. SIRT7 Is Activated by DNA and Deacetylates Histone H3 in the Chromatin Context. ACS Chem. Biol. 2016 , 11 , 742-747. [CrossRef]
- 38. Tong, Z.; Wang, M.; Wang, Y.; Kim, D.D.; Grenier, J.K.; Cao, J.; Sadhukhan, S.; Hao, Q.; Lin, H. SIRT7 Is an RNA-Activated Protein Lysine Deacylase. ACS Chem. Biol. 2017 , 12 , 300-310. [CrossRef]

- 39. Dai, H.; Case, A.W.; Riera, T.V.; Considine, T.; Lee, J.E.; Hamuro, Y.; Zhao, H.; Jiang, Y.; Sweitzer, S.M.; Pietrak, B.; et al. Crystallographic structure of a small molecule SIRT1 activator-enzyme complex. Nat. Commun. 2015 , 6 , 7645. [CrossRef]
- 40. Cohen, H.Y.; Miller, C.; Bitterman, K.J.; Wall, N.R.; Hekking, B.; Kessler, B.; Howitz, K.T.; Gorospe, M.; de Cabo, R.; Sinclair, D.A. Calorie restriction promotes mammalian cell survival by inducing the SIRT1 deacetylase. Science 2004 , 305 , 390-392. [CrossRef]
- 41. Kim, J.E.; Chen, J.; Lou, Z. DBC1 is a negative regulator of SIRT1. Nature 2008 , 451 , 583-586. [CrossRef]
- 42. Verdin, E. AROuSing SIRT1: Identification of a novel endogenous SIRT1 activator. Mol. Cell 2007 , 28 , 354-356. [CrossRef]
- 43. Pan, M.; Yuan, H.; Brent, M.; Ding, E.C.; Marmorstein, R. SIRT1 contains N- and C-terminal regions that potentiate deacetylase activity. J. Biol. Chem. 2012 , 287 , 2468-2476. [CrossRef] [PubMed]
- 44. Kalous, K.S.; Wynia-Smith, S.L.; Smith, B.C. Sirtuin Oxidative Post-translational Modifications. Front. Physiol. 2021 , 12 , 763417. [CrossRef]
- 45. Flick, F.; Lüscher, B. Regulation of Sirtuin Function by Posttranslational Modifications. Front. Pharmacol. 2012 , 3 , 29. [CrossRef] [PubMed]
- 46. Costa-Machado, L.F.; Fernandez-Marcos, P.J. The sirtuin family in cancer. Cell Cycle 2019 , 18 , 2164-2196. [CrossRef]
- 47. Bindu, S.; Pillai, V.B.; Gupta, M.P. Role of Sirtuins in Regulating Pathophysiology of the Heart. Trends Endocrinol. Metab. 2016 , 27 , 563-573. [CrossRef] [PubMed]
- 48. Yeong, K.Y.; Berdigaliyev, N.; Chang, Y. Sirtuins and Their Implications in Neurodegenerative Diseases from a Drug Discovery Perspective. ACS Chem. Neuro 2020 , 11 , 4073-4091. [CrossRef]
- 49. Bonkowski, M.S.; Sinclair, D.A. Slowing ageing by design: The rise of NAD(+) and sirtuin-activating compounds. Nat. Rev. Mol. Cell Biol. 2016 , 17 , 679-690. [CrossRef]
- 50. Karaman Mayack, B.; Sippl, W.; Ntie-Kang, F. Natural Products as Modulators of Sirtuins. Molecules 2020 , 25 , 3287. [CrossRef]
- 51. Jiang, Y.; Liu, J.; Chen, D.; Yan, L.; Zheng, W. Sirtuin Inhibition: Strategies, Inhibitors, and Therapeutic Potential. Trends Pharmacol. Sci. 2017 , 38 , 459-472. [CrossRef] [PubMed]
- 52. Dai, H.; Sinclair, D.A.; Ellis, J.L.; Steegborn, C. Sirtuin activators and inhibitors: Promises, achievements, and challenges. Pharmacol. Ther. 2018 , 188 , 140-154. [CrossRef] [PubMed]
- 53. Loharch, S.; Chhabra, S.; Kumar, A.; Swarup, S.; Parkesh, R. Discovery and characterization of small molecule SIRT3-specific inhibitors as revealed by mass spectrometry. Bioorg. Chem. 2021 , 110 , 104768. [CrossRef] [PubMed]
- 54. Kaeberlein, M.; McVey, M.; Guarente, L. The SIR2/3/4 complex and SIR2 alone promote longevity in Saccharomyces cerevisiae by two different mechanisms. Genes Dev. 1999 , 13 , 2570-2580. [CrossRef]
- 55. Lin, S.-J.; Defossez, P.-A.; Guarente, L. Requirement of NAD and SIR2 for Life-Span Extension by Calorie Restriction in Saccharomyces cerevisiae. Science 2000 , 289 , 2126-2128. [CrossRef]
- 56. Satoh, A.; Brace, C.S.; Rensing, N.; Cliften, P.; Wozniak, D.F.; Herzog, E.D.; Yamada, K.A.; Imai, S. Sirt1 extends life span and delays aging in mice through the regulation of Nk2 homeobox 1 in the DMH and LH. Cell Metab. 2013 , 18 , 416-430. [CrossRef]
- 57. Kanfi, Y.; Naiman, S.; Amir, G.; Peshti, V.; Zinman, G.; Nahum, L.; Bar-Joseph, Z.; Cohen, H.Y. The sirtuin SIRT6 regulates lifespan in male mice. Nature 2012 , 483 , 218-221. [CrossRef]
- 58. Howitz, K.T.; Bitterman, K.J.; Cohen, H.Y.; Lamming, D.W.; Lavu, S.; Wood, J.G.; Zipkin, R.E.; Chung, P.; Kisielewski, A.; Zhang, L.L.; et al. Small molecule activators of sirtuins extend Saccharomyces cerevisiae lifespan. Nature 2003 , 425 , 191-196. [CrossRef]
- 59. Milne, J.C.; Lambert, P.D.; Schenk, S.; Carney, D.P.; Smith, J.J.; Gagne, D.J.; Jin, L.; Boss, O.; Perni, R.B.; Vu, C.B.; et al. Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes. Nature 2007 , 450 , 712-716. [CrossRef] [PubMed]
- 60. Ng, Y.P.; Bemis, E.J.; Disch, S.J.; Vu, B.C.; Oalmann, J.C.; Lynch, V.A.; Carney, P.D.; Riera, V.T.; Song, J.; Smith, J.J.; et al. The Identification of the SIRT1 Activator SRT2104 as a Clinical Candidate. Lett. Drug Des. Discov. 2013 , 10 , 793-797. [CrossRef]
- 61. Pillai, V.B.; Samant, S.; Sundaresan, N.R.; Raghuraman, H.; Kim, G.; Bonner, M.Y.; Arbiser, J.L.; Walker, D.I.; Jones, D.P.; Gius, D.; et al. Honokiol blocks and reverses cardiac hypertrophy in mice by activating mitochondrial Sirt3. Nat. Commun. 2015 , 6 , 6656.
- [CrossRef]
- 62. Zhang, J.; Zou, L.; Shi, D.; Liu, J.; Zhang, J.; Zhao, R.; Wang, G.; Zhang, L.; Ouyang, L.; Liu, B. Structure-Guided Design of a Small-Molecule Activator of Sirtuin-3 that Modulates Autophagy in Triple Negative Breast Cancer. J. Med. Chem. 2021 , 64 , 14192-14216. [CrossRef]
- 63. Suenkel, B.; Valente, S.; Zwergel, C.; Weiss, S.; Di Bello, E.; Fioravanti, R.; Aventaggiato, M.; Amorim, J.A.; Garg, N.; Kumar, S.; et al. Potent and Specific Activators for Mitochondrial Sirtuins Sirt3 and Sirt5. J. Med. Chem. 2022 , 65 , 14015-14031. [CrossRef]
- 64. You, W.; Rotili, D.; Li, T.M.; Kambach, C.; Meleshin, M.; Schutkowski, M.; Chua, K.F.; Mai, A.; Steegborn, C. Structural Basis of Sirtuin 6 Activation by Synthetic Small Molecules. Angew. Chem. Int. Ed. Engl. 2017 , 56 , 1007-1011. [CrossRef]
- 65. Klein, M.A.; Liu, C.; Kuznetsov, V.I.; Feltenberger, J.B.; Tang, W.; Denu, J.M. Mechanism of activation for the sirtuin 6 protein deacylase. J. Biol. Chem. 2020 , 295 , 1385-1399. [CrossRef] [PubMed]
- 66. He, T.; Shang, J.; Gao, C.; Guan, X.; Chen, Y.; Zhu, L.; Zhang, L.; Zhang, C.; Zhang, J.; Pang, T. A novel SIRT6 activator ameliorates neuroinflammation and ischemic brain injury via EZH2/FOXC1 axis. Acta Pharm. Sin. B 2021 , 11 , 708-726. [CrossRef] [PubMed]
- 67. Chen, X.; Sun, W.; Huang, S.; Zhang, H.; Lin, G.; Li, H.; Qiao, J.; Li, L.; Yang, S. Discovery of Potent Small-Molecule SIRT6 Activators: Structure-Activity Relationship and Anti-Pancreatic Ductal Adenocarcinoma Activity. J. Med. Chem. 2020 , 63 , 10474-10495. [CrossRef]

- 68. Jang, M.; Cai, L.; Udeani, G.O.; Slowing, K.V.; Thomas, C.F.; Beecher, C.W.W.; Fong, H.H.S.; Farnsworth, N.R.; Kinghorn, A.D.; Mehta, R.G.; et al. Cancer Chemopreventive Activity of Resveratrol, a Natural Product Derived from Grapes. Science 1997 , 275 , 218-220. [CrossRef] [PubMed]
- 69. Kaeberlein, M.; McDonagh, T.; Heltweg, B.; Hixon, J.; Westman, E.A.; Caldwell, S.D.; Napper, A.; Curtis, R.; DiStefano, P.S.; Fields, S.; et al. Substrate-specific activation of sirtuins by resveratrol. J. Biol. Chem. 2005 , 280 , 17038-17045. [CrossRef]
- 70. Borra, M.T.; Smith, B.C.; Denu, J.M. Mechanism of human SIRT1 activation by resveratrol. J. Biol. Chem. 2005 , 280 , 17187-17195. [CrossRef]
- 71. Ray, U.; Roy, S.S.; Chowdhury, S.R. Lysophosphatidic Acid Promotes Epithelial to Mesenchymal Transition in Ovarian Cancer Cells by Repressing SIRT1. Cell Physiol. Biochem. 2017 , 41 , 795-805. [CrossRef] [PubMed]
- 72. Pirola, L.; Fröjdö, S. Resveratrol: One molecule, many targets. IUBMB Life 2008 , 60 , 323-332. [CrossRef] [PubMed]
- 73. Liu, M.; Wilk, S.A.; Wang, A.; Zhou, L.; Wang, R.H.; Ogawa, W.; Deng, C.; Dong, L.Q.; Liu, F. Resveratrol inhibits mTOR signaling by promoting the interaction between mTOR and DEPTOR. J. Biol. Chem. 2010 , 285 , 36387-36394. [CrossRef] [PubMed]
- 74. Price, N.L.; Gomes, A.P.; Ling, A.J.; Duarte, F.V.; Martin-Montalvo, A.; North, B.J.; Agarwal, B.; Ye, L.; Ramadori, G.; Teodoro, J.S.; et al. SIRT1 is required for AMPK activation and the beneficial effects of resveratrol on mitochondrial function. Cell Metab. 2012 , 15 , 675-690. [CrossRef] [PubMed]
- 75. Park, S.J.; Ahmad, F.; Philp, A.; Baar, K.; Williams, T.; Luo, H.; Ke, H.; Rehmann, H.; Taussig, R.; Brown, A.L.; et al. Resveratrol ameliorates aging-related metabolic phenotypes by inhibiting cAMP phosphodiesterases. Cell 2012 , 148 , 421-433. [CrossRef]
- 76. Walle, T.; Hsieh, F.; DeLegge, M.H.; Oatis, J.E., Jr.; Walle, U.K. High absorption but very low bioavailability of oral resveratrol in humans. Drug Metab. Dispos. 2004 , 32 , 1377-1382. [CrossRef] [PubMed]
- 77. Gertz, M.; Nguyen, G.T.; Fischer, F.; Suenkel, B.; Schlicker, C.; Fränzel, B.; Tomaschewski, J.; Aladini, F.; Becker, C.; Wolters, D.; et al. A molecular mechanism for direct sirtuin activation by resveratrol. PLoS ONE 2012 , 7 , e49761. [CrossRef]
- 78. Yang, H.; Baur, J.A.; Chen, A.; Miller, C.; Adams, J.K.; Kisielewski, A.; Howitz, K.T.; Zipkin, R.E.; Sinclair, D.A. Design and synthesis of compounds that extend yeast replicative lifespan. Aging Cell 2007 , 6 , 35-43. [CrossRef]
- 79. Biasutto, L.; Mattarei, A.; Marotta, E.; Bradaschia, A.; Sassi, N.; Garbisa, S.; Zoratti, M.; Paradisi, C. Development of mitochondriatargeted derivatives of resveratrol. Bioorg. Med. Chem. Lett. 2008 , 18 , 5594-5597. [CrossRef]
- 80. Nayagam, V.M.; Wang, X.; Tan, Y.C.; Poulsen, A.; Goh, K.C.; Ng, T.; Wang, H.; Song, H.Y.; Ni, B.; Entzeroth, M.; et al. SIRT1 modulating compounds from high-throughput screening as anti-inflammatory and insulin-sensitizing agents. J. Biomol. Screen 2006 , 11 , 959-967. [CrossRef]
- 81. Bemis, J.E.; Vu, C.B.; Xie, R.; Nunes, J.J.; Ng, P.Y.; Disch, J.S.; Milne, J.C.; Carney, D.P.; Lynch, A.V.; Jin, L.; et al. Discovery of oxazolo[4,5-b]pyridines and related heterocyclic analogs as novel SIRT1 activators. Bioorg. Med. Chem. Lett. 2009 , 19 , 2350-2353. [CrossRef] [PubMed]
- 82. Vu, C.B.; Bemis, J.E.; Disch, J.S.; Ng, P.Y.; Nunes, J.J.; Milne, J.C.; Carney, D.P.; Lynch, A.V.; Smith, J.J.; Lavu, S.; et al. Discovery of Imidazo[1,2-b]thiazole Derivatives as Novel SIRT1 Activators. J. Med. Chem. 2009 , 52 , 1275-1283. [CrossRef]
- 83. Zhao, L.; Cao, J.; Hu, K.; He, X.; Yun, D.; Tong, T.; Han, L. Sirtuins and their Biological Relevance in Aging and Age-Related Diseases. Aging Dis. 2020 , 11 , 927-945. [CrossRef] [PubMed]
- 84. Nguyen, G.T.; Schaefer, S.; Gertz, M.; Weyand, M.; Steegborn, C. Structures of human sirtuin 3 complexes with ADP-ribose and with carba-NAD + and SRT1720: Binding details and inhibition mechanism. Acta Crystallogr. D Biol. Crystallogr. 2013 , 69 , 1423-1432. [CrossRef]
- 85. Pacholec, M.; Bleasdale, J.E.; Chrunyk, B.; Cunningham, D.; Flynn, D.; Garofalo, R.S.; Griffith, D.; Griffor, M.; Loulakis, P.; Pabst, B.; et al. SRT1720, SRT2183, SRT1460, and resveratrol are not direct activators of SIRT1. J. Biol. Chem. 2010 , 285 , 8340-8351. [CrossRef]
- 86. Dai, H.; Kustigian, L.; Carney, D.; Case, A.; Considine, T.; Hubbard, B.P.; Perni, R.B.; Riera, T.V.; Szczepankiewicz, B.; Vlasuk, G.P.; et al. SIRT1 activation by small molecules: Kinetic and biophysical evidence for direct interaction of enzyme and activator. J. Biol. Chem. 2010 , 285 , 32695-32703. [CrossRef] [PubMed]
- 87. Hubbard, B.P.; Gomes, A.P.; Dai, H.; Li, J.; Case, A.W.; Considine, T.; Riera, T.V.; Lee, J.E.; E, S.Y.; Lamming, D.W.; et al. Evidence for a common mechanism of SIRT1 regulation by allosteric activators. Science 2013 , 339 , 1216-1219. [CrossRef]
- 88. Minor, R.K.; Baur, J.A.; Gomes, A.P.; Ward, T.M.; Csiszar, A.; Mercken, E.M.; Abdelmohsen, K.; Shin, Y.K.; Canto, C.; ScheibyeKnudsen, M.; et al. SRT1720 improves survival and healthspan of obese mice. Sci. Rep. 2011 , 1 , 70. [CrossRef]
- 89. Mitchell, S.J.; Martin-Montalvo, A.; Mercken, E.M.; Palacios, H.H.; Ward, T.M.; Abulwerdi, G.; Minor, R.K.; Vlasuk, G.P.; Ellis, J.L.; Sinclair, D.A.; et al. The SIRT1 activator SRT1720 extends lifespan and improves health of mice fed a standard diet. Cell Rep. 2014 , 6 , 836-843. [CrossRef]
- 90. Park, S.J.; Ahmad, F.; Um, J.H.; Brown, A.L.; Xu, X.; Kang, H.; Ke, H.; Feng, X.; Ryall, J.; Philp, A.; et al. Specific Sirt1 Activatormediated Improvement in Glucose Homeostasis Requires Sirt1-Independent Activation of AMPK. EBioMedicine 2017 , 18 , 128-138. [CrossRef]
- 91. Hoffmann, E.; Wald, J.; Lavu, S.; Roberts, J.; Beaumont, C.; Haddad, J.; Elliott, P.; Westphal, C.; Jacobson, E. Pharmacokinetics and tolerability of SRT2104, a first-in-class small molecule activator of SIRT1, after single and repeated oral administration in man. Br. J. Clin. Pharmacol. 2013 , 75 , 186-196. [CrossRef] [PubMed]

- 92. Libri, V.; Brown, A.P.; Gambarota, G.; Haddad, J.; Shields, G.S.; Dawes, H.; Pinato, D.J.; Hoffman, E.; Elliot, P.J.; Vlasuk, G.P.; et al. Apilot randomized, placebo controlled, double blind phase I trial of the novel SIRT1 activator SRT2104 in elderly volunteers. PLoS ONE 2012 , 7 , e51395. [CrossRef]
- 93. Venkatasubramanian, S.; Noh, R.M.; Daga, S.; Langrish, J.P.; Joshi, N.V.; Mills, N.L.; Hoffmann, E.; Jacobson, E.W.; Vlasuk, G.P.; Waterhouse, B.R.; et al. Cardiovascular effects of a novel SIRT1 activator, SRT2104, in otherwise healthy cigarette smokers. J. Am. Heart Assoc. 2013 , 2 , e000042. [CrossRef]
- 94. Baksi, A.; Kraydashenko, O.; Zalevkaya, A.; Stets, R.; Elliott, P.; Haddad, J.; Hoffmann, E.; Vlasuk, G.P.; Jacobson, E.W. A phase II, randomized, placebo-controlled, double-blind, multi-dose study of SRT2104, a SIRT1 activator, in subjects with type 2 diabetes. Br. J. Clin. Pharmacol. 2014 , 78 , 69-77. [CrossRef]
- 95. van der Meer, A.J.; Scicluna, B.P.; Moerland, P.D.; Lin, J.; Jacobson, E.W.; Vlasuk, G.P.; van der Poll, T. The Selective Sirtuin 1 Activator SRT2104 Reduces Endotoxin-Induced Cytokine Release and Coagulation Activation in Humans. Crit. Care Med. 2015 , 43 , e199-e202. [CrossRef] [PubMed]
- 96. Krueger, J.G.; Su á rez-Fariñas, M.; Cueto, I.; Khacherian, A.; Matheson, R.; Parish, L.C.; Leonardi, C.; Shortino, D.; Gupta, A.; Haddad, J.; et al. A Randomized, Placebo-Controlled Study of SRT2104, a SIRT1 Activator, in Patients with Moderate to Severe Psoriasis. PLoS ONE 2015 , 10 , e0142081. [CrossRef]
- 97. Sands, B.E.; Joshi, S.; Haddad, J.; Freudenberg, J.M.; Oommen, D.E.; Hoffmann, E.; McCallum, S.W.; Jacobson, E. Assessing Colonic Exposure, Safety, and Clinical Activity of SRT2104, a Novel Oral SIRT1 Activator, in Patients with Mild to Moderate Ulcerative Colitis. Inflamm. Bowel Dis. 2016 , 22 , 607-614. [CrossRef] [PubMed]
- 98. Mai, A.; Valente, S.; Meade, S.; Carafa, V.; Tardugno, M.; Nebbioso, A.; Galmozzi, A.; Mitro, N.; De Fabiani, E.; Altucci, L.; et al. Study of 1,4-dihydropyridine structural scaffold: Discovery of novel sirtuin activators and inhibitors. J. Med. Chem. 2009 , 52 , 5496-5504. [CrossRef]
- 99. Valente, S.; Mellini, P.; Spallotta, F.; Carafa, V.; Nebbioso, A.; Polletta, L.; Carnevale, I.; Saladini, S.; Trisciuoglio, D.; Gabellini, C.; et al. 1,4-Dihydropyridines Active on the SIRT1/AMPK Pathway Ameliorate Skin Repair and Mitochondrial Function and Exhibit Inhibition of Proliferation in Cancer Cells. J. Med. Chem. 2016 , 59 , 1471-1491. [CrossRef]
- 100. Zwergel, C.; Aventaggiato, M.; Garbo, S.; Di Bello, E.; Fassari, B.; Noce, B.; Castiello, C.; Lambona, C.; Barreca, F.; Rotili, D.; et al. Novel 1,4-Dihydropyridines as Specific Binders and Activators of SIRT3 Impair Cell Viability and Clonogenicity and Downregulate Hypoxia-Induced Targets in Cancer Cells. J. Med. Chem. 2023 , 66 , 9622-9641. [CrossRef]
- 101. Hu, T.; Shukla, S.K.; Vernucci, E.; He, C.; Wang, D.; King, R.J.; Jha, K.; Siddhanta, K.; Mullen, N.J.; Attri, K.S.; et al. Metabolic Rewiring by Loss of Sirt5 Promotes Kras-Induced Pancreatic Cancer Progression. Gastroenterology 2021 , 161 , 1584-1600. [CrossRef]
- 102. Schlicker, C.; Boanca, G.; Lakshminarasimhan, M.; Steegborn, C. Structure-based development of novel sirtuin inhibitors. Aging 2011 , 3 , 852-872. [CrossRef]
- 103. Iachettini, S.; Trisciuoglio, D.; Rotili, D.; Lucidi, A.; Salvati, E.; Zizza, P.; Di Leo, L.; Del Bufalo, D.; Ciriolo, M.R.; Leonetti, C.; et al. Pharmacological activation of SIRT6 triggers lethal autophagy in human cancer cells. Cell Death Dis. 2018 , 9 , 996. [CrossRef]
- 104. Xu, J.; Shi, S.; Liu, G.; Xie, X.; Li, J.; Bolinger, A.A.; Chen, H.; Zhang, W.; Shi, P.Y.; Liu, H.; et al. Design, synthesis, and pharmacological evaluations of pyrrolo[1,2-a]quinoxaline-based derivatives as potent and selective sirt6 activators. Eur. J. Med. Chem. 2023 , 246 , 114998. [CrossRef] [PubMed]
- 105. Huang, Z.; Zhao, J.; Deng, W.; Chen, Y.; Shang, J.; Song, K.; Zhang, L.; Wang, C.; Lu, S.; Yang, X.; et al. Identification of a cellularly active SIRT6 allosteric activator. Nat. Chem. Biol. 2018 , 14 , 1118-1126. [CrossRef]
- 106. Shang, J.-l.; Ning, S.-b.; Chen, Y.-y.; Chen, T.-x.; Zhang, J. MDL-800, an allosteric activator of SIRT6, suppresses proliferation and enhances EGFR-TKIs therapy in non-small cell lung cancer. Acta Pharmacol. Sin. 2021 , 42 , 120-131. [CrossRef]
- 107. Shang, J.; Zhu, Z.; Chen, Y.; Song, J.; Huang, Y.; Song, K.; Zhong, J.; Xu, X.; Wei, J.; Wang, C.; et al. Small-molecule activating SIRT6 elicits therapeutic effects and synergistically promotes anti-tumor activity of vitamin D(3) in colorectal cancer. Theranostics 2020 , 10 , 5845-5864. [CrossRef]
- 108. Imai, S.; Guarente, L. NAD + and sirtuins in aging and disease. Trends Cell Biol. 2014 , 24 , 464-471. [CrossRef] [PubMed]
- 109. Imai, S.I.; Guarente, L. It takes two to tango: NAD(+) and sirtuins in aging/longevity control. NPJ Aging Mech. Dis. 2016 , 2 , 16017. [CrossRef] [PubMed]
- 110. Nikiforov, A.; Dölle, C.; Niere, M.; Ziegler, M. Pathways and subcellular compartmentation of NAD biosynthesis in human cells: From entry of extracellular precursors to mitochondrial NAD generation. J. Biol. Chem. 2011 , 286 , 21767-21778. [CrossRef]
- 111. Cant ó , C.; Houtkooper, R.H.; Pirinen, E.; Youn, D.Y.; Oosterveer, M.H.; Cen, Y.; Fernandez-Marcos, P.J.; Yamamoto, H.; Andreux, P.A.; Cettour-Rose, P.; et al. The NAD(+) precursor nicotinamide riboside enhances oxidative metabolism and protects against high-fat diet-induced obesity. Cell Metab. 2012 , 15 , 838-847. [CrossRef] [PubMed]
- 112. Bai, P.; Cant ó , C.; Oudart, H.; Bruny á nszki, A.; Cen, Y.; Thomas, C.; Yamamoto, H.; Huber, A.; Kiss, B.; Houtkooper, R.H.; et al. PARP-1 inhibition increases mitochondrial metabolism through SIRT1 activation. Cell Metab. 2011 , 13 , 461-468. [CrossRef] [PubMed]
- 113. Camacho-Pereira, J.; Tarrag ó , M.G.; Chini, C.C.S.; Nin, V.; Escande, C.; Warner, G.M.; Puranik, A.S.; Schoon, R.A.; Reid, J.M.; Galina, A.; et al. CD38 Dictates Age-Related NAD Decline and Mitochondrial Dysfunction through an SIRT3-Dependent Mechanism. Cell Metab. 2016 , 23 , 1127-1139. [CrossRef] [PubMed]
- 114. Jackson, M.D.; Schmidt, M.T.; Oppenheimer, N.J.; Denu, J.M. Mechanism of nicotinamide inhibition and transglycosidation by Sir2 histone/protein deacetylases. J. Biol. Chem. 2003 , 278 , 50985-50998. [CrossRef] [PubMed]

- 115. Sauve, A.A.; Moir, R.D.; Schramm, V.L.; Willis, I.M. Chemical activation of Sir2-dependent silencing by relief of nicotinamide inhibition. Mol. Cell 2005 , 17 , 595-601. [CrossRef]
- 116. Kalous, K.S.; Wynia-Smith, S.L.; Summers, S.B.; Smith, B.C. Human sirtuins are differentially sensitive to inhibition by nitrosating agents and other cysteine oxidants. J. Biol. Chem. 2020 , 295 , 8524-8536. [CrossRef]
- 117. Kokkola, T.; Suuronen, T.; Pesonen, M.; Filippakopoulos, P.; Salminen, A.; Jarho, E.M.; Lahtela-Kakkonen, M. BET Inhibition Upregulates SIRT1 and Alleviates Inflammatory Responses. Chembiochem 2015 , 16 , 1997-2001. [CrossRef]
- 118. Tenhunen, J.; Kokkola, T.; Huovinen, M.; Rahnasto-Rilla, M.; Lahtela-Kakkonen, M. Impact of structurally diverse BET inhibitors on SIRT1. Gene 2020 , 741 , 144558. [CrossRef]
- 119. Järvenpää, J.; Rahnasto-Rilla, M.; Lahtela-Kakkonen, M.; Küblbeck, J. Profiling the regulatory interplay of BET bromodomains and Sirtuins in cancer cell lines. Biomed. Pharmacother. 2022 , 147 , 112652. [CrossRef]
- 120. Andrieu, G.; Belkina, A.C.; Denis, G.V. Clinical trials for BET inhibitors run ahead of the science. Drug Discov. Today Technol. 2016 , 19 , 45-50. [CrossRef]
- 121. Di Fruscia, P.; Zacharioudakis, E.; Liu, C.; Moniot, S.; Laohasinnarong, S.; Khongkow, M.; Harrison, I.F.; Koltsida, K.; Reynolds, C.R.; Schmidtkunz, K.; et al. The discovery of a highly selective 5,6,7,8-tetrahydrobenzo[4,5]thieno[2,3-d]pyrimidin-4(3H)-one SIRT2 inhibitor that is neuroprotective in an in vitro Parkinson's disease model. ChemMedChem 2015 , 10 , 69-82. [CrossRef]
- 122. Gertz, M.; Fischer, F.; Nguyen, G.T.T.; Lakshminarasimhan, M.; Schutkowski, M.; Weyand, M.; Steegborn, C. Ex-527 inhibits Sirtuins by exploiting their unique NAD + -dependent deacetylation mechanism. Proc. Natl. Acad. Sci. USA 2013 , 110 , E2772-E2781. [CrossRef] [PubMed]
- 123. Trapp, J.; Jochum, A.; Meier, R.; Saunders, L.; Marshall, B.; Kunick, C.; Verdin, E.; Goekjian, P.; Sippl, W.; Jung, M. Adenosine mimetics as inhibitors of NAD + -dependent histone deacetylases, from kinase to sirtuin inhibition. J. Med. Chem. 2006 , 49 , 7307-7316. [CrossRef] [PubMed]
- 124. Rumpf, T.; Schiedel, M.; Karaman, B.; Roessler, C.; North, B.J.; Lehotzky, A.; Ol á h, J.; Ladwein, K.I.; Schmidtkunz, K.; Gajer, M.; et al. Selective Sirt2 inhibition by ligand-induced rearrangement of the active site. Nat. Commun. 2015 , 6 , 6263. [CrossRef]
- 125. Kudo, N.; Ito, A.; Arata, M.; Nakata, A.; Yoshida, M. Identification of a novel small molecule that inhibits deacetylase but not defatty-acylase reaction catalysed by SIRT2. Philos. Trans. R. Soc. Lond. B Biol. Sci. 2018 , 373 , 20170070. [CrossRef] [PubMed]
- 126. Disch, J.S.; Evindar, G.; Chiu, C.H.; Blum, C.A.; Dai, H.; Jin, L.; Schuman, E.; Lind, K.E.; Belyanskaya, S.L.; Deng, J.; et al. Discovery of thieno[3,2-d]pyrimidine-6-carboxamides as potent inhibitors of SIRT1, SIRT2, and SIRT3. J. Med. Chem. 2013 , 56 , 3666-3679. [CrossRef]
- 127. Napper, A.D.; Hixon, J.; McDonagh, T.; Keavey, K.; Pons, J.F.; Barker, J.; Yau, W.T.; Amouzegh, P.; Flegg, A.; Hamelin, E.; et al. Discovery of indoles as potent and selective inhibitors of the deacetylase SIRT1. J. Med. Chem. 2005 , 48 , 8045-8054. [CrossRef]
- 128. Ai, T.; Wilson, D.J.; More, S.S.; Xie, J.; Chen, L. 5-((3-Amidobenzyl)oxy)nicotinamides as Sirtuin 2 Inhibitors. J. Med. Chem. 2016 , 59 , 2928-2941. [CrossRef]
- 129. Outeiro, T.F.; Kontopoulos, E.; Altmann, S.M.; Kufareva, I.; Strathearn, K.E.; Amore, A.M.; Volk, C.B.; Maxwell, M.M.; Rochet, J.C.; McLean, P.J.; et al. Sirtuin 2 inhibitors rescue alpha-synuclein-mediated toxicity in models of Parkinson's disease. Science 2007 , 317 , 516-519. [CrossRef]
- 130. Taylor, D.M.; Balabadra, U.; Xiang, Z.; Woodman, B.; Meade, S.; Amore, A.; Maxwell, M.M.; Reeves, S.; Bates, G.P.; Luthi-Carter, R.; et al. A brain-permeable small molecule reduces neuronal cholesterol by inhibiting activity of sirtuin 2 deacetylase. ACS Chem. Biol. 2011 , 6 , 540-546. [CrossRef]
- 131. Quinti, L.; Casale, M.; Moniot, S.; Pais, T.F.; Van Kanegan, M.J.; Kaltenbach, L.S.; Pallos, J.; Lim, R.G.; Naidu, S.D.; Runne, H.; et al. SIRT2- and NRF2-Targeting Thiazole-Containing Compound with Therapeutic Activity in Huntington's Disease Models. Cell Chem. Biol. 2016 , 23 , 849-861. [CrossRef]
- 132. Pannek, M.; Alhalabi, Z.; Tomaselli, D.; Menna, M.; Fiorentino, F.; Robaa, D.; Weyand, M.; Puhlmann, M.; Tomassi, S.; Barreca, F.; et al. Specific Inhibitors of Mitochondrial Deacylase Sirtuin 4 Endowed with Cellular Activity. J. Med. Chem. 2024 , 67 , 1843-1860. [CrossRef]
- 133. Yao, J.; Yin, Y.; Han, H.; Chen, S.; Zheng, Y.; Liang, B.; Wu, M.; Shu, K.; Debnath, B.; Lombard, D.B.; et al. Pyrazolone derivatives as potent and selective small-molecule SIRT5 inhibitors. Eur. J. Med. Chem. 2023 , 247 , 115024. [CrossRef]
- 134. Lara, E.; Mai, A.; Calvanese, V.; Altucci, L.; Lopez-Nieva, P.; Martinez-Chantar, M.L.; Varela-Rey, M.; Rotili, D.; Nebbioso, A.; Ropero, S.; et al. Salermide, a Sirtuin inhibitor with a strong cancer-specific proapoptotic effect. Oncogene 2009 , 28 , 781-791. [CrossRef]
- 135. Bedalov, A.; Gatbonton, T.; Irvine, W.P.; Gottschling, D.E.; Simon, J.A. Identification of a small molecule inhibitor of Sir2p. Proc. Natl. Acad. Sci. USA 2001 , 98 , 15113-15118. [CrossRef]
- 136. Heltweg, B.; Gatbonton, T.; Schuler, A.D.; Posakony, J.; Li, H.; Goehle, S.; Kollipara, R.; Depinho, R.A.; Gu, Y.; Simon, J.A.; et al. Antitumor activity of a small-molecule inhibitor of human silent information regulator 2 enzymes. Cancer Res. 2006 , 66 , 4368-4377. [CrossRef]
- 137. Mahajan, S.S.; Scian, M.; Sripathy, S.; Posakony, J.; Lao, U.; Loe, T.K.; Leko, V.; Thalhofer, A.; Schuler, A.D.; Bedalov, A.; et al. Development of pyrazolone and isoxazol-5-one cambinol analogues as sirtuin inhibitors. J. Med. Chem. 2014 , 57 , 3283-3294. [CrossRef]

- 138. Chowdhury, S.; Sripathy, S.; Webster, A.A.; Park, A.; Lao, U.; Hsu, J.H.; Loe, T.; Bedalov, A.; Simon, J.A. Discovery of Selective SIRT2 Inhibitors as Therapeutic Agents in B-Cell Lymphoma and Other Malignancies. Molecules 2020 , 25 , 455. [CrossRef] [PubMed]
- 139. Garske, A.L.; Smith, B.C.; Denu, J.M. Linking SIRT2 to Parkinson's Disease. ACS Chem. Biol. 2007 , 2 , 529-532. [CrossRef] [PubMed]
- 140. Mou, L.; Yang, L.; Hou, S.; Wang, B.; Wang, X.; Hu, L.; Deng, J.; Liu, J.; Chen, X.; Jiang, Y.; et al. Structure-Activity Relationship Studies of 2,4,5-Trisubstituted Pyrimidine Derivatives Leading to the Identification of a Novel and Potent Sirtuin 5 Inhibitor against Sepsis-Associated Acute Kidney Injury. J. Med. Chem. 2023 , 66 , 11517-11535. [CrossRef] [PubMed]
- 141. Yang, F.; Su, H.; Deng, J.; Mou, L.; Wang, H.; Li, R.; Dai, Q.-Q.; Yan, Y.-H.; Qian, S.; Wang, Z.; et al. Discovery of new human Sirtuin 5 inhibitors by mimicking glutaryl-lysine substrates. Eur. J. Med. Chem. 2021 , 225 , 113803. [CrossRef]
- 142. Solomon, J.M.; Pasupuleti, R.; Xu, L.; McDonagh, T.; Curtis, R.; DiStefano, P.S.; Huber, L.J. Inhibition of SIRT1 catalytic activity increases p53 acetylation but does not alter cell survival following DNA damage. Mol. Cell. Biol. 2006 , 26 , 28-38. [CrossRef]
- 143. Min, J.; Landry, J.; Sternglanz, R.; Xu, R.-M. Crystal Structure of a SIR2 Homolog-NAD Complex. Cell 2001 , 105 , 269-279. [CrossRef]
- 144. Smith, M.R.; Syed, A.; Lukacsovich, T.; Purcell, J.; Barbaro, B.A.; Worthge, S.A.; Wei, S.R.; Pollio, G.; Magnoni, L.; Scali, C.; et al. A potent and selective Sirtuin 1 inhibitor alleviates pathology in multiple animal and cell models of Huntington's disease. Hum. Mol. Genet. 2014 , 23 , 2995-3007. [CrossRef]
- 145. Süssmuth, S.D.; Haider, S.; Landwehrmeyer, G.B.; Farmer, R.; Frost, C.; Tripepi, G.; Andersen, C.A.; Di Bacco, M.; Lamanna, C.; Diodato, E.; et al. An exploratory double-blind, randomized clinical trial with selisistat, a SirT1 inhibitor, in patients with Huntington's disease. Br. J. Clin. Pharmacol. 2015 , 79 , 465-476. [CrossRef]
- 146. Kundu, A.; Richa, S.; Dey, P.; Kim, K.S.; Son, J.Y.; Kim, H.R.; Lee, S.Y.; Lee, B.H.; Lee, K.Y.; Kacew, S.; et al. Protective effect of EX-527 against high-fat diet-induced diabetic nephropathy in Zucker rats. Toxicol. Appl. Pharmacol. 2020 , 390 , 114899. [CrossRef]
- 147. Spinck, M.; Bischoff, M.; Lampe, P.; Meyer-Almes, F.-J.; Sievers, S.; Neumann, H. Discovery of Dihydro-1,4-Benzoxazine Carboxamides as Potent and Highly Selective Inhibitors of Sirtuin-1. J. Med. Chem. 2021 , 64 , 5838-5849. [CrossRef] [PubMed]
- 148. Ekblad, T.; Schüler, H. Sirtuins are Unaffected by PARP Inhibitors Containing Planar Nicotinamide Bioisosteres. Chem. Biol. Drug Des. 2016 , 87 , 478-482. [CrossRef] [PubMed]
- 149. Peck, B.; Chen, C.-Y.; Ho, K.-K.; Di Fruscia, P.; Myatt, S.S.; Coombes, R.C.; Fuchter, M.J.; Hsiao, C.-D.; Lam, E.W.F. SIRT Inhibitors Induce Cell Death and p53 Acetylation through Targeting Both SIRT1 and SIRT2. Mol. Cancer Ther. 2010 , 9 , 844-855. [CrossRef] [PubMed]
- 150. Hughes, K.J.; Meares, G.P.; Hansen, P.A.; Corbett, J.A. FoxO1 and SIRT1 regulate beta-cell responses to nitric oxide. J. Biol. Chem. 2011 , 286 , 8338-8348. [CrossRef] [PubMed]
- 151. Luthi-Carter, R.; Taylor, D.M.; Pallos, J.; Lambert, E.; Amore, A.; Parker, A.; Moffitt, H.; Smith, D.L.; Runne, H.; Gokce, O.; et al. SIRT2 inhibition achieves neuroprotection by decreasing sterol biosynthesis. Proc. Natl. Acad. Sci. USA 2010 , 107 , 7927-7932. [CrossRef]
- 152. Chen, X.; Wales, P.; Quinti, L.; Zuo, F.; Moniot, S.; Herisson, F.; Rauf, N.A.; Wang, H.; Silverman, R.B.; Ayata, C.; et al. The sirtuin-2 inhibitor AK7 is neuroprotective in models of Parkinson's disease but not amyotrophic lateral sclerosis and cerebral ischemia. PLoS ONE 2015 , 10 , e0116919. [CrossRef] [PubMed]
- 153. Chopra, V.; Quinti, L.; Kim, J.; Vollor, L.; Narayanan, K.L.; Edgerly, C.; Cipicchio, P.M.; Lauver, M.A.; Choi, S.H.; Silverman, R.B.; et al. The sirtuin 2 inhibitor AK-7 is neuroprotective in Huntington's disease mouse models. Cell Rep. 2012 , 2 , 1492-1497. [CrossRef] [PubMed]
- 154. Davis, P.D.; Hill, C.H.; Lawton, G.; Nixon, J.S.; Wilkinson, S.E.; Hurst, S.A.; Keech, E.; Turner, S.E. Inhibitors of protein kinase C. 1. 2,3-bisarylmaleimides. J. Med. Chem. 1992 , 35 , 177-184. [CrossRef] [PubMed]
- 155. Cui, H.; Kamal, Z.; Ai, T.; Xu, Y.; More, S.S.; Wilson, D.J.; Chen, L. Discovery of potent and selective sirtuin 2 (SIRT2) inhibitors using a fragment-based approach. J. Med. Chem. 2014 , 57 , 8340-8357. [CrossRef]
- 156. Ai, T.; Wilson, D.J.; Chen, L. 5-((3-Amidobenzyl)oxy)nicotinamides as SIRT2 Inhibitors: A Study of Constrained Analogs. Molecules 2023 , 28 , 7655. [CrossRef] [PubMed]
- 157. North, B.J.; Rosenberg, M.A.; Jeganathan, K.B.; Hafner, A.V.; Michan, S.; Dai, J.; Baker, D.J.; Cen, Y.; Wu, L.E.; Sauve, A.A.; et al. SIRT2 induces the checkpoint kinase BubR1 to increase lifespan. EMBO J. 2014 , 33 , 1438-1453. [CrossRef]
- 158. Schiedel, M.; Rumpf, T.; Karaman, B.; Lehotzky, A.; Ol á h, J.; Gerhardt, S.; Ov á di, J.; Sippl, W.; Einsle, O.; Jung, M. Aminothiazoles as Potent and Selective Sirt2 Inhibitors: A Structure-Activity Relationship Study. J. Med. Chem. 2016 , 59 , 1599-1612. [CrossRef] [PubMed]
- 159. Schiedel, M.; Rumpf, T.; Karaman, B.; Lehotzky, A.; Gerhardt, S.; Ov á di, J.; Sippl, W.; Einsle, O.; Jung, M. Structure-Based Development of an Affinity Probe for Sirtuin 2. Angew. Chem. Int. Ed. Engl. 2016 , 55 , 2252-2256. [CrossRef]
- 160. Alhazzazi, T.Y.; Kamarajan, P.; Xu, Y.; Ai, T.; Chen, L.; Verdin, E.; Kapila, Y.L. A Novel Sirtuin-3 Inhibitor, LC-0296, Inhibits Cell Survival and Proliferation, and Promotes Apoptosis of Head and Neck Cancer Cells. Anticancer. Res. 2016 , 36 , 49-60.
- 161. Wood, M.; Rymarchyk, S.; Zheng, S.; Cen, Y. Trichostatin A inhibits deacetylation of histone H3 and p53 by SIRT6. Arch. Biochem. Biophys. 2018 , 638 , 8-17. [CrossRef]
- 162. You, W.; Steegborn, C. Structural Basis of Sirtuin 6 Inhibition by the Hydroxamate Trichostatin A: Implications for Protein Deacylase Drug Development. J. Med. Chem. 2018 , 61 , 10922-10928. [CrossRef]

- 163. Cacabelos, R.; Torrellas, C. Chapter 32-Pharmacoepigenomics. In Medical Epigenetics ; Tollefsbol, T.O., Ed.; Academic Press: Boston, MA, USA, 2016; pp. 585-617.
- 164. Ni, D.; Chai, Z.; Wang, Y.; Li, M.; Yu, Z.; Liu, Y.; Lu, S.; Zhang, J. Along the allostery stream: Recent advances in computational methods for allosteric drug discovery. Wiley Interdiscip. Rev. Comput. Mol. Sci. 2022 , 12 , e1585. [CrossRef]
- 165. Zhang, Q.; Chen, Y.; Ni, D.; Huang, Z.; Wei, J.; Feng, L.; Su, J.-C.; Wei, Y.; Ning, S.; Yang, X.; et al. Targeting a cryptic allosteric site of SIRT6 with small-molecule inhibitors that inhibit the migration of pancreatic cancer cells. Acta Pharm. Sin. B 2021 , 12 , 876-889. [CrossRef] [PubMed]
- 166. Song, N.; Guan, X.; Zhang, S.; Wang, Y.; Wang, X.; Lu, Z.; Chong, D.; Wang, J.Y.; Yu, R.; Yu, W.; et al. Discovery of a pyrrolepyridinimidazole derivative as novel SIRT6 inhibitor for sensitizing pancreatic cancer to gemcitabine. Cell Death Dis. 2023 , 14 , 499. [CrossRef]
- 167. Sundriyal, S.; Moniot, S.; Mahmud, Z.; Yao, S.; Di Fruscia, P.; Reynolds, C.R.; Dexter, D.T.; Sternberg, M.J.E.; Lam, E.W.F.; Steegborn, C.; et al. Thienopyrimidinone Based Sirtuin-2 (SIRT2)-Selective Inhibitors Bind in the Ligand Induced Selectivity Pocket. J. Med. Chem. 2017 , 60 , 1928-1945. [CrossRef] [PubMed]
- 168. Yang, L.; Ma, X.; Yuan, C.; He, Y.; Li, L.; Fang, S.; Xia, W.; He, T.; Qian, S.; Xu, Z.; et al. Discovery of 2-((4,6-dimethylpyrimidin-2yl)thio)-N-phenylacetamide derivatives as new potent and selective human sirtuin 2 inhibitors. Eur. J. Med. Chem. 2017 , 134 , 230-241. [CrossRef]
- 169. Moniot, S.; Forgione, M.; Lucidi, A.; Hailu, G.S.; Nebbioso, A.; Carafa, V.; Baratta, F.; Altucci, L.; Giacch é , N.; Passeri, D.; et al. Development of 1,2,4-Oxadiazoles as Potent and Selective Inhibitors of the Human Deacetylase Sirtuin 2: Structure-Activity Relationship, X-ray Crystal Structure, and Anticancer Activity. J. Med. Chem. 2017 , 60 , 2344-2360. [CrossRef]
- 170. Yang, L.-L.; Wang, H.-L.; Zhong, L.; Yuan, C.; Liu, S.-Y.; Yu, Z.-J.; Liu, S.; Yan, Y.-H.; Wu, C.; Wang, Y.; et al. X-ray crystal structure guided discovery of new selective, substrate-mimicking sirtuin 2 inhibitors that exhibit activities against non-small cell lung cancer cells. Eur. J. Med. Chem. 2018 , 155 , 806-823. [CrossRef] [PubMed]
- 171. Yang, L.L.; Xu, W.; Yan, J.; Su, H.L.; Yuan, C.; Li, C.; Zhang, X.; Yu, Z.J.; Yan, Y.H.; Yu, Y.; et al. Crystallographic and SAR analyses reveal the high requirements needed to selectively and potently inhibit SIRT2 deacetylase and decanoylase. Medchemcomm 2019 , 10 , 164-168. [CrossRef]
- 172. Smith, B.C.; Denu, J.M. Sir2 protein deacetylases: Evidence for chemical intermediates and functions of a conserved histidine. Biochemistry 2006 , 45 , 272-282. [CrossRef]
- 173. Borra, M.T.; Langer, M.R.; Slama, J.T.; Denu, J.M. Substrate Specificity and Kinetic Mechanism of the Sir2 Family of NAD + -Dependent Histone/Protein Deacetylases. Biochemistry 2004 , 43 , 9877-9887. [CrossRef]
- 174. Avalos, J.L.; Bever, K.M.; Wolberger, C. Mechanism of Sirtuin Inhibition by Nicotinamide: Altering the NAD + Cosubstrate Specificity of a Sir2 Enzyme. Mol. Cell 2005 , 17 , 855-868. [CrossRef] [PubMed]
- 175. Fatkins, D.G.; Monnot, A.D.; Zheng, W. Nepsilon-thioacetyl-lysine: A multi-facet functional probe for enzymatic protein lysine Nepsilon-deacetylation. Bioorg. Med. Chem. Lett. 2006 , 16 , 3651-3656. [CrossRef] [PubMed]
- 176. Smith, B.C.; Denu, J.M. Mechanism-Based Inhibition of Sir2 Deacetylases by Thioacetyl-Lysine Peptide. Biochemistry 2007 , 46 , 14478-14486. [CrossRef] [PubMed]
- 177. Fatkins, D.G.; Zheng, W. Substituting N(epsilon)-thioacetyl-lysine for N(epsilon)-acetyl-lysine in peptide substrates as a general approach to inhibiting human NAD(+)-dependent protein deacetylases. Int. J. Mol. Sci. 2008 , 9 , 1. [CrossRef] [PubMed]
- 178. Kiviranta, P.H.; Suuronen, T.; Wall é n, E.A.A.; Leppänen, J.; Tervonen, J.; Kyrylenko, S.; Salminen, A.; Poso, A.; Jarho, E.M. N ϵ -Thioacetyl-Lysine-Containing Tri-, Tetra-, and Pentapeptides as SIRT1 and SIRT2 Inhibitors. J. Med. Chem. 2009 , 52 , 2153-2156. [CrossRef] [PubMed]
- 179. Hirsch, B.M.; Gallo, C.A.; Du, Z.; Wang, Z.; Zheng, W. Discovery of potent, proteolytically stable, and cell permeable human sirtuin peptidomimetic inhibitors containing N ε -thioacetyl-lysine. MedChemComm 2010 , 1 , 233-238. [CrossRef]
- 180. Chen, B.; Wang, J.; Huang, Y.; Zheng, W. Human SIRT3 tripeptidic inhibitors containing N( ε )-thioacetyl-lysine. Bioorg. Med. Chem. Lett. 2015 , 25 , 3481-3487. [CrossRef] [PubMed]
- 181. Suzuki, T.; Asaba, T.; Imai, E.; Tsumoto, H.; Nakagawa, H.; Miyata, N. Identification of a cell-active non-peptide sirtuin inhibitor containing N-thioacetyl lysine. Bioorg. Med. Chem. Lett. 2009 , 19 , 5670-5672. [CrossRef]
- 182. Huhtiniemi, T.; Salo, H.S.; Suuronen, T.; Poso, A.; Salminen, A.; Leppänen, J.; Jarho, E.; Lahtela-Kakkonen, M. Structure-based design of pseudopeptidic inhibitors for SIRT1 and SIRT2. J. Med. Chem. 2011 , 54 , 6456-6468. [CrossRef]
- 183. Mellini, P.; Kokkola, T.; Suuronen, T.; Salo, H.S.; Tolvanen, L.; Mai, A.; Lahtela-Kakkonen, M.; Jarho, E.M. Screen of Pseudopeptidic Inhibitors of Human Sirtuins 1-3: Two Lead Compounds with Antiproliferative Effects in Cancer Cells. J. Med. Chem. 2013 , 56 , 6681-6695. [CrossRef]
- 184. Zang, W.; Hao, Y.; Wang, Z.; Zheng, W. Novel thiourea-based sirtuin inhibitory warheads. Bioorg. Med. Chem. Lett. 2015 , 25 , 3319-3324. [CrossRef] [PubMed]
- 185. Hirsch, B.M.; Hao, Y.; Li, X.; Wesdemiotis, C.; Wang, Z.; Zheng, W. A mechanism-based potent sirtuin inhibitor containing N ε -thiocarbamoyl-lysine (TuAcK). Bioorg. Med. Chem. Lett. 2011 , 21 , 4753-4757. [CrossRef]
- 186. Jing, H.; Hu, J.; He, B.; Negr ó n Abril, Y.L.; Stupinski, J.; Weiser, K.; Carbonaro, M.; Chiang, Y.-L.; Southard, T.; Giannakakou, P.; et al. A SIRT2-Selective Inhibitor Promotes c-Myc Oncoprotein Degradation and Exhibits Broad Anticancer Activity. Cancer Cell 2016 , 29 , 297-310. [CrossRef]

- 187. Spiegelman, N.A.; Hong, J.Y.; Hu, J.; Jing, H.; Wang, M.; Price, I.R.; Cao, J.; Yang, M.; Zhang, X.; Lin, H. A Small-Molecule SIRT2 Inhibitor That Promotes K-Ras4a Lysine Fatty-Acylation. ChemMedChem 2019 , 14 , 744-748. [CrossRef]
- 188. Nielsen, A.L.; Rajabi, N.; Kudo, N.; Lundø, K.; Moreno-Yruela, C.; Bæk, M.; Fontenas, M.; Lucidi, A.; Madsen, A.S.; Yoshida, M.; et al. Mechanism-based inhibitors of SIRT2: Structure-activity relationship, X-ray structures, target engagement, regulation of α -tubulin acetylation and inhibition of breast cancer cell migration. RSC Chem. Biol. 2021 , 2 , 612-626. [CrossRef] [PubMed]
- 189. Hong, J.Y.; Price, I.R.; Bai, J.J.; Lin, H. A Glycoconjugated SIRT2 Inhibitor with Aqueous Solubility Allows Structure-Based Design of SIRT2 Inhibitors. ACS Chem. Biol. 2019 , 14 , 1802-1810. [CrossRef]
- 190. Farooqi, A.S.; Hong, J.Y.; Cao, J.; Lu, X.; Price, I.R.; Zhao, Q.; Kosciuk, T.; Yang, M.; Bai, J.J.; Lin, H. Novel Lysine-Based Thioureas as Mechanism-Based Inhibitors of Sirtuin 2 (SIRT2) with Anticancer Activity in a Colorectal Cancer Murine Model. J. Med. Chem. 2019 , 62 , 4131-4141. [CrossRef] [PubMed]
- 191. Huang, Y.; Liu, J.; Yan, L.; Zheng, W. Simple N( ε )-thioacetyl-lysine-containing cyclic peptides exhibiting highly potent sirtuin inhibition. Bioorg. Med. Chem. Lett. 2016 , 26 , 1612-1617. [CrossRef]
- 192. Li, R.; Yan, L.; Sun, X.; Zheng, W. A bicyclic pentapeptide-based highly potent and selective pan-SIRT1/2/3 inhibitor harboring N( ε )-thioacetyl-lysine. Bioorg. Med. Chem. 2020 , 28 , 115356. [CrossRef]
- 193. Chen, D.; Zheng, W. Cyclic peptide-based potent and selective SIRT1/2 dual inhibitors harboring N( ε )-thioacetyl-lysine. Bioorg. Med. Chem. Lett. 2016 , 26 , 5234-5239. [CrossRef]
- 194. Li, M.; Chiang, Y.-L.; Lyssiotis, C.A.; Teater, M.R.; Hong, J.Y.; Shen, H.; Wang, L.; Hu, J.; Jing, H.; Chen, Z.; et al. Non-oncogene Addiction to SIRT3 Plays a Critical Role in Lymphomagenesis. Cancer Cell 2019 , 35 , 916-931.e919. [CrossRef]
- 195. Troelsen, K.S.; Bæk, M.; Nielsen, A.L.; Madsen, A.S.; Rajabi, N.; Olsen, C.A. Mitochondria-targeted inhibitors of the human SIRT3 lysine deacetylase. RSC Chem. Biol. 2021 , 2 , 627-635. [CrossRef]
- 196. Sauve, A.A.; Schramm, V.L. Sir2 regulation by nicotinamide results from switching between base exchange and deacetylation chemistry. Biochemistry 2003 , 42 , 9249-9256. [CrossRef] [PubMed]
- 197. He, B.; Du, J.; Lin, H. Thiosuccinyl Peptides as Sirt5-Specific Inhibitors. J. Am. Chem. Soc. 2012 , 134 , 1922-1925. [CrossRef]
- 198. Hu, J.; He, B.; Bhargava, S.; Lin, H. A fluorogenic assay for screening Sirt6 modulators. Org. Biomol. Chem. 2013 , 11 , 5213-5216. [CrossRef]
- 199. Bitterman, K.J.; Anderson, R.M.; Cohen, H.Y.; Latorre-Esteves, M.; Sinclair, D.A. Inhibition of silencing and accelerated aging by nicotinamide, a putative negative regulator of yeast sir2 and human SIRT1. J. Biol. Chem. 2002 , 277 , 45099-45107. [CrossRef]

200. Luo, J.; Nikolaev, A.Y.; Imai, S.; Chen, D.; Su, F.; Shiloh, A.; Guarente, L.; Gu, W. Negative control of p53 by Sir2alpha promotes

cell survival under stress.

Cell

2001

,

107

, 137-148. [CrossRef] [PubMed]

- 201. Rajabi, N.; Auth, M.; Troelsen, K.R.; Pannek, M.; Bhatt, D.P.; Fontenas, M.; Hirschey, M.D.; Steegborn, C.; Madsen, A.S.; Olsen, C.A. Mechanism-Based Inhibitors of the Human Sirtuin 5 Deacylase: Structure-Activity Relationship, Biostructural, and Kinetic Insight. Angew. Chem. Int. Ed. Engl. 2017 , 56 , 14836-14841. [CrossRef] [PubMed]
- 202. Yan, D.; Franzini, A.; Pomicter, A.D.; Halverson, B.J.; Antelope, O.; Mason, C.C.; Ahmann, J.M.; Senina, A.V.; Vellore, N.A.; Jones, C.L.; et al. SIRT5 Is a Druggable Metabolic Vulnerability in Acute Myeloid Leukemia. Blood Cancer Discov. 2021 , 2 , 266-287. [CrossRef]
- 203. Green, K.N.; Steffan, J.S.; Martinez-Coria, H.; Sun, X.; Schreiber, S.S.; Thompson, L.M.; LaFerla, F.M. Nicotinamide restores cognition in Alzheimer's disease transgenic mice via a mechanism involving sirtuin inhibition and selective reduction of Thr231-phosphotau. J. Neurosci. 2008 , 28 , 11500-11510. [CrossRef]
- 204. Jung-Hynes, B.; Nihal, M.; Zhong, W.; Ahmad, N. Role of sirtuin histone deacetylase SIRT1 in prostate cancer. A target for prostate cancer management via its inhibition? J. Biol. Chem. 2009 , 284 , 3823-3832. [CrossRef]
- 205. Alhazzazi, T.Y.; Kamarajan, P.; Joo, N.; Huang, J.Y.; Verdin, E.; D'Silva, N.J.; Kapila, Y.L. Sirtuin-3 (SIRT3), a novel potential therapeutic target for oral cancer. Cancer 2011 , 117 , 1670-1678. [CrossRef]
- 206. Petrack, B.; Greengard, P.; Craston, A.; Kalinsky, H.J. Nicotinamide deamidase in rat liver and the biosynthesis of NAD. Biochem. Bioph. Res. Commun. 1963 , 13 , 472-477. [CrossRef]
- 207. Suzuki, T.; Imai, K.; Nakagawa, H.; Miyata, N. 2-Anilinobenzamides as SIRT inhibitors. ChemMedChem 2006 , 1 , 1059-1062. [CrossRef]
- 208. Suzuki, T.; Khan, M.N.; Sawada, H.; Imai, E.; Itoh, Y.; Yamatsuta, K.; Tokuda, N.; Takeuchi, J.; Seko, T.; Nakagawa, H.; et al. Design, synthesis, and biological activity of a novel series of human sirtuin-2-selective inhibitors. J. Med. Chem. 2012 , 55 , 5760-5773. [CrossRef] [PubMed]
- 209. Lain, S.; Hollick, J.J.; Campbell, J.; Staples, O.D.; Higgins, M.; Aoubala, M.; McCarthy, A.; Appleyard, V.; Murray, K.E.; Baker, L.; et al. Discovery, in vivo activity, and mechanism of action of a small-molecule p53 activator. Cancer Cell 2008 , 13 , 454-463. [CrossRef] [PubMed]
- 210. Galli, U.; Mesenzani, O.; Coppo, C.; Sorba, G.; Canonico, P.L.; Tron, G.C.; Genazzani, A.A. Identification of a sirtuin 3 inhibitor that displays selectivity over sirtuin 1 and 2. Eur. J. Med. Chem. 2012 , 55 , 58-66. [CrossRef]
- 211. Ito, A.; Lai, C.H.; Zhao, X.; Saito, S.; Hamilton, M.H.; Appella, E.; Yao, T.P. p300/CBP-mediated p53 acetylation is commonly induced by p53-activating agents and inhibited by MDM2. EMBO J. 2001 , 20 , 1331-1340. [CrossRef] [PubMed]
- 212. McCarthy, A.R.; Pirrie, L.; Hollick, J.J.; Ronseaux, S.; Campbell, J.; Higgins, M.; Staples, O.D.; Tran, F.; Slawin, A.M.; Lain, S.; et al. Synthesis and biological characterisation of sirtuin inhibitors based on the tenovins. Bioorg. Med. Chem. 2012 , 20 , 1779-1793. [CrossRef]

- 213. McCarthy, A.R.; Sachweh, M.C.; Higgins, M.; Campbell, J.; Drummond, C.J.; van Leeuwen, I.M.; Pirrie, L.; Ladds, M.J.; Westwood, N.J.; La í n, S. Tenovin-D3, a novel small-molecule inhibitor of sirtuin SirT2, increases p21 (CDKN1A) expression in a p53-independent manner. Mol. Cancer Ther. 2013 , 12 , 352-360. [CrossRef]
- 214. Kallander, L.S.; Lu, Q.; Chen, W.; Tomaszek, T.; Yang, G.; Tew, D.; Meek, T.D.; Hofmann, G.A.; Schulz-Pritchard, C.K.; Smith, W.W.; et al. 4-Aryl-1,2,3-triazole: A Novel Template for a Reversible Methionine Aminopeptidase 2 Inhibitor, Optimized To Inhibit Angiogenesis in Vivo. J. Med. Chem. 2005 , 48 , 5644-5647. [CrossRef] [PubMed]
- 215. Röhrig, U.F.; Awad, L.; Grosdidier, A.; Larrieu, P.; Stroobant, V.; Colau, D.; Cerundolo, V.; Simpson, A.J.G.; Vogel, P.; Van den Eynde, B.J.; et al. Rational Design of Indoleamine 2,3-Dioxygenase Inhibitors. J. Med. Chem. 2010 , 53 , 1172-1189. [CrossRef] [PubMed]
- 216. Zhang, J.; Xiang, H.; Liu, J.; Chen, Y.; He, R.R.; Liu, B. Mitochondrial Sirtuin 3: New emerging biological function and therapeutic target. Theranostics 2020 , 10 , 8315-8342. [CrossRef] [PubMed]
- 217. Dom í nguez-G ó mez, G.; D í az-Ch á vez, J.; Ch á vez-Blanco, A.; Gonzalez-Fierro, A.; Jim é nez-Salazar, J.E.; Dami á n-Matsumura, P.; G ó mez-Quiroz, L.E.; Dueñas-Gonz á lez, A. Nicotinamide sensitizes human breast cancer cells to the cytotoxic effects of radiation and cisplatin. Oncol. Rep. 2015 , 33 , 721-728. [CrossRef]
- 218. Berthelier, V.; Tixier, J.M.; Muller-Steffner, H.; Schuber, F.; Deterre, P. Human CD38 is an authentic NAD(P)+ glycohydrolase. Biochem. J. 1998 , 330 Pt 3 , 1383-1390. [CrossRef] [PubMed]
- 219. Burgos, E.S.; Schramm, V.L. Weak Coupling of ATP Hydrolysis to the Chemical Equilibrium of Human Nicotinamide Phosphoribosyltransferase. Biochemistry 2008 , 47 , 11086-11096. [CrossRef]
- 220. Wang, L.; Wang, N.; Zhang, W.; Cheng, X.; Yan, Z.; Shao, G.; Wang, X.; Wang, R.; Fu, C. Therapeutic peptides: Current applications and future directions. Signal Transduct. Target. Ther. 2022 , 7 , 48. [CrossRef] [PubMed]
- 221. O'Brien, C.; Berman, J.; Culp-Hill, R.; Reisz, J.; Ling, T.; Rondeau, V.; Li, M.; Yang, M.; Hong, J.Y.; Lin, H.; et al. Sirtuin 3 Inhibition Targets AML Stem Cells through Perturbation of Fatty Acid Oxidation. Blood 2021 , 138 , 2240. [CrossRef]
- 222. Hong, J.Y.; Fernandez, I.; Anmangandla, A.; Lu, X.; Bai, J.J.; Lin, H. Pharmacological Advantage of SIRT2-Selective versus pan-SIRT1-3 Inhibitors. ACS Chem. Biol. 2021 , 16 , 1266-1275. [CrossRef] [PubMed]
- 223. He, B.; Hu, J.; Zhang, X.; Lin, H. Thiomyristoyl peptides as cell-permeable Sirt6 inhibitors. Org. Biomol. Chem. 2014 , 12 , 7498-7502. [CrossRef] [PubMed]
- 224. Deng, J.; Liu, Z.-M.; Zhu, K.-R.; Cui, G.-L.; Liu, L.-X.; Yan, Y.-H.; Ning, X.-L.; Yu, Z.-J.; Li, G.-B.; Qi, Q.-R. New ε -N-thioglutaryllysine derivatives as SIRT5 inhibitors: Chemical synthesis, kinetic and crystallographic studies. Bioorg. Chem. 2023 , 135 , 106487. [CrossRef]
- 225. Neal, R.A.; Halpert, J. Toxicology of Thiono-Sulfur Compounds. Annu. Rev. Pharmacol. Toxicol. 1982 , 22 , 321-339. [CrossRef] [PubMed]
- 226. Ruse, M.J.; Waring, R.H. The effect of methimazole on thioamide bioactivation and toxicity. Toxicol. Lett. 1991 , 58 , 37-41. [CrossRef] [PubMed]
- 227. Cox, D.N.; Davidson, V.P.; Judd, C.E.; Stodgell, C.; Traiger, G.J. The effect of partial hepatectomy on the metabolism, distribution, and nephrotoxicity of para-methylthiobenzamide in the rat. Toxicol. Appl. Pharmacol. 1992 , 113 , 246-252. [CrossRef]
- 228. Leblond, J.; Mignet, N.; Largeau, C.; Seguin, J.; Scherman, D.; Herscovici, J. Lipopolythiourea transfecting agents: Lysine thiourea derivatives. Bioconjug. Chem. 2008 , 19 , 306-314. [CrossRef]
- 229. Zhou, Y.; Zhang, H.; He, B.; Du, J.; Lin, H.; Cerione, R.A.; Hao, Q. The bicyclic intermediate structure provides insights into the desuccinylation mechanism of human sirtuin 5 (SIRT5). J. Biol. Chem. 2012 , 287 , 28307-28314. [CrossRef]
- 230. Rajabi, N.; Hansen, T.N.; Nielsen, A.L.; Nguyen, H.T.; Bæk, M.; Bolding, J.E.; Bahlke, O.Ø.; Petersen, S.E.G.; Bartling, C.R.O.; Strømgaard, K.; et al. Investigation of Carboxylic Acid Isosteres and Prodrugs for Inhibition of the Human SIRT5 Lysine Deacylase Enzyme. Angew. Chem. Int. Ed. 2022 , 61 , e202115805. [CrossRef]
- 231. Bolding, J.E.; Mart í n-Gago, P.; Rajabi, N.; Gamon, L.F.; Hansen, T.N.; Bartling, C.R.O.; Strømgaard, K.; Davies, M.J.; Olsen, C.A. Aryl Fluorosulfate Based Inhibitors That Covalently Target the SIRT5 Lysine Deacylase. Angew. Chem. Int. Ed. 2022 , 61 , e202204565. [CrossRef]
- 232. Asaba, T.; Suzuki, T.; Ueda, R.; Tsumoto, H.; Nakagawa, H.; Miyata, N. Inhibition of Human Sirtuins by in Situ Generation of an Acetylated Lysine -ADP -Ribose Conjugate. J. Am. Chem. Soc. 2009 , 131 , 6989-6996. [CrossRef]
- 233. Hirsch, B.M.; Du, Z.; Li, X.; Sylvester, J.A.; Wesdemiotis, C.; Wang, Z.; Zheng, W. Potent sirtuin inhibition bestowed by l-2-amino-7-carboxamidoheptanoic acid (l-ACAH), a N ε -acetyl-lysine analog. MedChemComm 2011 , 2 , 291-299. [CrossRef]
- 234. He, Y.; Yan, L.; Zang, W.; Zheng, W. Novel sirtuin inhibitory warheads derived from the N ε -acetyl-lysine analog l-2-amino-7carboxamidoheptanoic acid. Org. Biomol. Chem. 2015 , 13 , 10442-10450. [CrossRef] [PubMed]
- 235. Costa, L.; Sousa, E.; Fernandes, C. Cyclic Peptides in Pipeline: What Future for These Great Molecules? Pharmaceuticals 2023 , 16 , 996. [CrossRef]
- 236. Jiang, Y.; Zheng, W. Cyclic Tripeptide-based Potent and Selective Human SIRT5 Inhibitors. Med. Chem. 2020 , 16 , 358-367. [CrossRef] [PubMed]
- 237. Li, S.; Wu, B.; Zheng, W. Cyclic tripeptide-based potent human SIRT7 inhibitors. Bioorg. Med. Chem. Lett. 2019 , 29 , 461-465. [CrossRef]
- 238. Liu, J.; Zheng, W. Cyclic peptide-based potent human SIRT6 inhibitors. Org. Biomol. Chem. 2016 , 14 , 5928-5935. [CrossRef] [PubMed]

- 239. Morimoto, J.; Hayashi, Y.; Suga, H. Discovery of Macrocyclic Peptides Armed with a Mechanism-Based Warhead: IsoformSelective Inhibition of Human Deacetylase SIRT2. Angew. Chem. Int. Ed. 2012 , 51 , 3423-3427. [CrossRef]
- 240. Smith, B.C.; Denu, J.M. Sir2 Deacetylases Exhibit Nucleophilic Participation of Acetyl-Lysine in NAD + Cleavage. J. Am. Chem. Soc. 2007 , 129 , 5802-5803. [CrossRef]
- 241. Bolding, J.E.; Nielsen, A.L.; Jensen, I.; Hansen, T.N.; Ryberg, L.A.; Jameson, S.T.; Harris, P.; Peters, G.H.J.; Denu, J.M.; Rogers, J.M.; et al. Substrates and Cyclic Peptide Inhibitors of the Oligonucleotide-Activated Sirtuin 7. Angew. Chem. Int. Ed. 2023 , 62 , e202314597. [CrossRef]

Disclaimer/Publisher's Note: The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.