<!-- image -->

<!-- image -->

## Article

## Transgenic Expression of Haemonchus contortus Cytochrome P450 Hco-cyp-13A11 Decreases Susceptibility to Particular but Not All Macrocyclic Lactones in the Model Organism Caenorhabditis elegans

<!-- image -->

Natalie Jakobs, Esra Yilmaz and Jürgen Krücken *

Institute for Parasitology and Tropical Veterinary Medicine, Freie Universität Berlin, 14163 Berlin, Germany * Correspondence: juergen.kruecken@fu-berlin.de; Tel.: +49-(0)-30-838-62318

Abstract: The number of reported macrocyclic lactones (ML) resistance cases across all livestock hosts is steadily increasing. Different studies in the parasitic nematode Haemonchus contortus assume the participation of cytochrome P450s (Cyps) enzymes in ML resistance. Still, functional data about their individual contribution to resistance or substrate specificity is missing. Via microinjection, transgenic Caenorhabditis elegans expressing HCON\_00141052 ( transgene-Hco-cyp-13A11 ) from extrachromosomal arrays were generated. After 24 h of exposure to different concentrations of ivermectin (IVM), ivermectin aglycone (IVMa), selamectin (SEL), doramectin (DRM), eprinomectin (EPR), and moxidectin (MOX), motility assays were performed to determine the impact of the H. contortus Cyp to the susceptibility of the worms against each ML. While transgene -Hco-cyp-13A11 significantly decreased susceptibility to IVM (four-fold), IVMa (2-fold), and SEL (3-fold), a slight effect for DRM and no effect for MOX, and EPR was observed. This substrate specificity of Hco-cyp-13A11 could not be explained by molecular modeling and docking studies. Hco-Cyp-13A11 molecular models were obtained for alleles from isolates with different resistance statuses. Although 14 amino acid polymorphisms were detected, none was resistance specific. In conclusion, Hco-cyp-13A11 decreased IVM, IVMa, and SEL susceptibility to a different extent, but its potential impact on ML resistance is not driven by polymorphisms.

Keywords: macrocyclic lactones; drug resistance; cytochrome P450; drug metabolism; parasitology; parasite metabolism

## 1. Introduction

Anthelmintic therapy remains the most important management strategy to treat parasitic nematode infections in humans and livestock. Due to their broad-spectrum activity and high efficacy, macrocyclic lactones (MLs), including ivermectin (IVM), moxidectin (MOX), selamectin (SEL), eprinomectin (EPR), and doramectin (DRM), represent the most widely used drug class [1]. Both subfamilies of MLs, the milbemycins (MOX and milbemycin oxime) and the avermectins (IVM, SEL, EPR, and DRM), share the core structure, a 16-member macrocyclic lactone ring fused with a benzofuran and a spiroketal moiety. Structural differences between the subfamilies relate to the presence and absence of specific substituents [1].

The extensive and inappropriate use of these drugs has led to the worldwide emergence of ML resistance [2,3]. This increasingly impairs the treatment of trichostrongyloid parasitic nematodes, particularly Haemonchus contortus , one of the most pathogenic gastrointestinal nematodes infecting small ruminants [4]. Intensive research on ML resistance revealed a multi-genic mechanism, including target-site [5-8] and non-target-site associated changes [9-12]. The latter mainly include pharmacokinetic-related defense via drug biotransformation enzymes [12,13] and efflux transporters [11,14-17].

<!-- image -->

Citation: Jakobs, N.; Yilmaz, E.; Krücken, J. Transgenic Expression of Haemonchus contortus Cytochrome P450 Hco-cyp-13A11 Decreases Susceptibility to Particular but Not All Macrocyclic Lactones in the Model Organism Caenorhabditis elegans . Int. J. Mol. Sci. 2022 , 23 , 9155. https://doi.org/10.3390/ijms23169155

Academic Editors: Patrick M. Dansette and Efstratios Stratikos

Received: 27 May 2022

Accepted: 28 July 2022

Published: 15 August 2022

Publisher's Note: MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.

<!-- image -->

Copyright: © 2022 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (https:// creativecommons.org/licenses/by/ 4.0/).

Drug metabolism can generally be divided into two phases. In phase I, xenobiotics can undergo modification via oxidation, reduction, or hydrolysis reactions to insert or uncover hydrophilic groups. Depending on the hydrophilicity of the resulting products, phase II can follow, which involves conjugation with endogenous compounds such as glucuronic acid, glucose, or glutathione [18]. Although expression changes of genes encoding phase II UDP-glycosyltransferases (UGTs) between susceptible and resistant H. contortus isolates were reported for the benzimidazole albendazole [19], the contribution of UGTs to ML resistance has to date not been registered. In contrast, IVM-selected Caenorhabditis elegans strains induced the overexpression of two-phase II glutathione S -transferases [20].

The largest family of biotransformation enzymes comprises cytochrome P450 enzymes (Cyps). These phase I enzymes are present in almost all living organisms and contribute to drug resistance via various catalytic reactions [21]. The human genome encodes 57 Cyps, with five Cyps dominating drug metabolism [22], and within this group, Cyp-3A4 is present in the largest quantities within the human body [23]. The C. elegans genome even encodes 80 Cyps [24]. In particular, the members of the family Cyp35 were shown to be xenobiotically inducible [25,26]. Within the H. contortus genome, a smaller number of 42 Cyps was identified [27] without the extensive gene duplications observed in C. elegans [24]. A study by Yilmaz et al. further suggested differences in constitutive expression of Cyps for H. contortus isolates with different resistance status since the Cyp34/35 ortholog (HCON\_00022640) was identified to be constitutively higher expressed in the multi-resistant H. contortus White River (WR) isolate [28].

Investigating IVM metabolite formation in mammalian liver microsomes has already elucidated the contribution of human Cyp isoforms to IVM metabolism, particularly cytochrome P450 3A4 [29-31]. Inhibition of human liver-derived Cyp-3A4 with troleandomycin specifically diminished IVM biotransformation by >90, thereby revealing Cyp-3A4 as a critical enzyme in IVM biotransformation [30]. The importance of Cyp-3A4 in ML metabolism was also confirmed by conducting in vitro studies analyzing MOX metabolism by various mammalian species. Cyp-3A4 hydroxylates IVM and MOX at the alkyl side chains of the spiroketal moiety [30,32]. Although in vitro metabolism of MOX has also been reported in adult H. contortus , neither in vitro nor ex vivo IVM biotransformation products were found [33,34]. Differences in oxidative metabolism between ML-susceptible and -resistant H. contortus by measuring Cyp-mediated transformation of model substrates were not identified [35].

Nevertheless, several studies supported the potential contribution of Cyps to ML resistance, particularly for IVM. When inhibiting Cyp activity with the widely used inhibitor piperonyl butoxide and simultaneous exposure to IVM, ML-susceptible and -resistant Cooperia oncophora and Osteragia ostertagi showed complete inhibition of larval development and a significant reduction in larval migration [36]. Furthermore, IVM- and MOX-selected C. elegans strains showing decreased ML sensitivity displayed the constitutive overexpression of various Cyps [20]. While overexpression of Cel-cyp-14A2 and Cel-cyp-14A5 was observed for IVM and MOX selected strains, Cel-cyp-35A1 appeared to be specific for MOX and Cel-cyp-37B1 specific for IVM selected strains [20]. A later study by Yilmaz et al. (2019) excluded Cel-cyp-14A5 in ML metabolism using a loss-of-function variant of C. elegans cyp-14A5 . In addition, in adults of the drug-susceptible H. contortus isolate ISE only minorly, and no significant IVM-inducible changes in the expression of different candidate Cyps were observed [19].

By investigating the genetic bases of ML resistance, whole-genome analysis of a triple resistant Teladorsagia circumcincta isolate introgressed into a susceptible background elucidated various resistance-associated genes. Besides differential expression of efflux transporters, the TELCIR\_18530 ortholog of the C. elegans Cyp-13A subfamily was approximately 20-fold overexpressed in resistant worms (see S7 Table in [16]). The C. elegans Cyp13A subfamily has already been demonstrated to be inducible by the xenobiotic rifampicin [37]. However, analysis of constitutive Cyp expression in different life-cycle stages of the drugsusceptible H. contortus isolate ISE revealed the highest HCON\_00141052 (orthologue of

contrast,

low

or

no

expression

of

this

Cyp

has

been

observed

in

L4

and

adult

worms

[24].

Whether

Hco

-

cyp

-

13A11

can

potentially

contribute

to

an

increased

ML

resistance

level

in

H.

contortus

remains

unclear.

Therefore,

the

present

work

investigated

the

impact

of

Hco

-

cyp

-

ML

susceptibility

by

transgenic

expression

in

the

model

organism

C.

elegans

.

To

examine

whether

different

structural

features

of

MLs

have

effects

on

the

substrate

specificity

of

Cel-cyp-13A11 and Cel-cyp-13A12 ) expression in eggs. In contrast, low or no expression of this Cyp has been observed in L4 and adult worms [24]. Whether Hco-cyp-13A11 can potentially contribute to an increased ML resistance level in H. contortus remains unclear. Hco -cyp -13A11 , IVM, IVMa, MOX, EPR, SEL, and DRM were tested. The exon sequence of Hco -cyp -13A11 was compared between H. contortus isolates with different resistance statuses to identify single nucleotide polymorphisms correlating with ML resistance.

Therefore, the present work investigated the impact of Hco-cyp-13A11 on ML susceptibility by transgenic expression in the model organism C. elegans . To examine whether different structural features of MLs have effects on the substrate specificity of Hco-cyp-13A11 , IVM, IVMa, MOX, EPR, SEL, and DRM were tested. The exon sequence of Hco-cyp-13A11 was compared between H. contortus isolates with different resistance statuses to identify single nucleotide polymorphisms correlating with ML resistance. Their potential impact regarding changes in ML binding of Hco-cyp-13A11 was elucidated by molecular modeling and docking. Their potential impact regarding changes in ML binding of Hco -cyp -13A11 was elucidated by molecular modeling and docking. 2. Results 2.1. Caenorhabditis elegans Motility Assays with Ivermectin, Moxidectin, Ivermectin Aglycone, Doramectin, Selamectin, and Eprinomectin Haemonchus contortus cyp -13A11 was expressed in the model organism C. elegans N2

## 2. Results Bristol wild

-

type

(WT)

by

introducing

extrachromosomal

transgene

arrays

using

on

the

gene

2.1. Caenorhabditis elegans Motility Assays with Ivermectin, Moxidectin, Ivermectin Aglycone, Doramectin, Selamectin, and Eprinomectin microinjection of a plasmid construct to determine the impact of Hco -cyp -13A11 susceptibility towards various MLs via drug metabolism (Figure S1). The candidate

Haemonchus contortus cyp-13A11 was expressed in the model organism C. elegans N2 Bristol wild-type (WT) by introducing extrachromosomal transgene arrays using microinjection of a plasmid construct to determine the impact of Hco-cyp-13A11 on the susceptibility towards various MLs via drug metabolism (Figure S1). The candidate gene sequence was derived from the highly IVM and BZ resistant WR isolate. To identify transgenic worms, a plasmid driving pharyngeal gfp expression in C. elegans was co-injected. Expression of the Cyp was under the control of the C. elegans intestine epithelium-specific gut esterase 1 promotor ( ges-1p ) [38], since IVM uptake was demonstrated to occur via active pharyngeal pumping through the gut epithelium [39] and xenobiotic metabolism is primarily assumed to take part in the gut epithelium of nematodes [40]. The WT was used as the control line since Gerhard et al. [39] did not observe significant differences between WT and mock-transduced nematodes using almost identical constructs. sequence was derived from the highly IVM and BZ resistant WR isolate. To identify transgenic worms, a plasmid driving pharyngeal gfp expression in C. elegans was co -injected. Expression of the Cyp was under the control of the C. elegans intestine epithelium -specific gut esterase 1 promotor ( ges -1p ) [38], since IVM uptake was demonstrated to occur via active pharyngeal pumping through the gut epithelium [39] and xenobiotic metabolism is primarily assumed to take part in the gut epithelium of nematodes [40]. The WT was used as the control line since Gerhard et al. [39] did not observe significant differences between WT and mock -transduced nematodes using almost identical constructs. The transgenic line showed a semi -stable transmission pattern with GFP expression in the offspring, varying between 50 and 70%. In addition, no noticeable variability of

The transgenic line showed a semi-stable transmission pattern with GFP expression in the offspring, varying between 50 and 70%. In addition, no noticeable variability of fluorescence intensity was observed between GFP-positive individuals (Figure 1). For motility assays, only individuals expressing GFP in the pharynx were used. Escherichia coli OP50 as a food source was added, with an approximate OD 600 of 0.5 as recently described to stimulate drug uptake by the pharynx [39]. The expression of the candidate Hco-cyp-13A11 was confirmed by RT-PCR, targeting the full-length sequence. fluorescence intensity was observed between GFP -positive individuals (Figure 1). For motility assays, only individuals expressing GFP in the pharynx were used. Escherichia coli OP50 as a food source was added, with an approximate OD 600 of 0.5 as recently described to stimulate drug uptake by the pharynx [39]. The expression of the candidate Hco -cyp -13A11 was confirmed by RT -PCR, targeting the full -length sequence.

Figure 1. Fluorescence photographs of a single transgenic line on an NGM-agar plate. ( A ). An individual Hco-cyp-13A11 adult transgene C. elegans expressing GFP in the pharynx with transgenic progeny (black arrows). ( B ). Population of adult GFP positive Hco-cyp-13A11 transgene C. elegans .

<!-- image -->

Astatistically significant increase ( p < 0.0001) in EC 50 values was observed in thrashing assays investigating IVM, IVMa, and SEL for worms transgenic for Hco-cyp-13A11 ( transgene-Hco-cyp-13A11 ) compared to the N2 wild-type control. The highest increase in EC 50 value was observed for IVM with an approximate fold-change of 4.3, followed by 2.6 for SEL and IVMa with a 1.7-fold change (Figure 2 and Table 1). In contrast, exposure to DRMled only to a slight effect ( p = 0.005), and MOX and EPR did not lead to significant differences in concentration-response curves between the transgenic and the WT lines.

2.2. Comparison of Hco-cyp-13A11 Sequences between Susceptible and Resistant H. contortus Isolates to Identify Potential Macrocyclic Lactone Resistance-Associated Single-Nucleotide Polymorphisms (SNPs)

The encoded candidate gene sequence of Hco-cyp-13A11 was compared between two susceptible (inbred-susceptible Edinburgh (ISE) and McMaster (McM)) and two ML resistant (WR and Berlin-selected isolate (BSI)) isolates to identify potential resistance-associated single nucleotide polymorphisms (SNPs). Although a reference sequence has already been published for ISE [41], the Cyp gene for this isolate was analyzed analogously to the others using Sanger sequencing. Genomic DNA was extracted from H. contortus third-stage larvae to amplify sequences spanning one to multiple exons of Hco-cyp-13A11 . Protein coding sequences (CDS) were finally assembled according to the Hco-cyp-13A11 published ISE sequence [42] and used for analysis. Compared to the gene sequence published, the multiple sequence alignment (Figure S3) revealed complete CDSs for all isolates. The Hco-cyp-13A11 sequences obtained for ISE, McM, and BSI corresponded to a 1555 base pair (bp) and the WRsequence to a 1549 bp open reading frame. Comparing the sequenced and reference ISE sequences, differences in 12 nucleotide positions were identified. The comparison of the two ISE sequences with the sequences obtained from the other isolates revealed two SNPs for McM, eleven for BSI, and 31 SNPs and six deletions for WR (Figure S3).

To further elucidate the impact of each SNP on potential peptide and protein structure alterations, the translated amino acid (AA) sequences were considered. The cyp-13A11 polypeptide contains 517 residues for ISE, McM, and BSI, whereas the deletion of two adjacent codons (nucleotide positions from 1065 to 1070) encoding codons CAA and CAG in the genomic sequence reduced the number to 515 AAs for the WR isolate. The calculated molecular weights ranged between 59.4-59.6 kDa and corresponding theoretical pI values between 8.6 and 9.1 (Table 2). Post-translational modifications or functional motifs were predicted by ExPASy-ScanProsite, revealing the same cysteine heme-iron-binding motif FGlGPRQCIG (residues 454-463 for ISE, McM, BSI; residues 452-461 for McM) for all analyzed Cyps. This motif is identical to the orthologous C. elegans cyp-13A11 and cyp-13A12 heme-binding sites (Table 2). However, functionally or structurally validated data such as crystal structures are currently unavailable for the C. elegans orthologous Cyps. Keller et al. (2014) revealed that Cel-cyp-13A12 regioselectively metabolizes arachidonic acid when expressed in insect cells, and it was proposed to act as a polyunsaturated fatty acid epoxygenase. The most closely related human orthologous gene to Cel-cyp-13A12 (32% AA identity) and Hco-cyp-13A11 (28% AA identity) based on a BLAST search represents Cyp-3A4.

13A11

(

transgene

-

Hco

-

cyp

-

13A11

)

compared

to

the

N2

wild

-

type

control.

The

highest

increase

in

EC

followed

by

2.6

for

SEL

and

IVMa

with

a

1.7

-

fold

change

(Figure

2

and

Table

1).

In

contrast,

exposure

to

DRM

led

only

to

a

slight

effect

(

p

=

0.005),

and

MOX

and

EPR

did

not

lead

to

significant

differences

in

concentration

-

response

curves

between

transgenic

and

the

WT

lines.

Figure 2. Modulation of macrocyclic lactone susceptibility in Caenorhabditis elegans by transgenic expression of Hco -cyp -13A11 . ( A ) Concentration -response curves to ivermectin, ivermectin aglycone, selamectin, doramectin, eprinomectin, and moxidectin of the N2 wild -type and the transgene -Hco -cyp -13A11 under control of the gut epithelium -specific promotor ges -1 . The corresponding chemical drug structure is presented below the graph and highlighted according to the structural differences to the other macrocyclic lactones. Blue : carbohydrate moiety; purple : Figure 2. Modulation of macrocyclic lactone susceptibility in Caenorhabditis elegans by transgenic expression of Hco-cyp-13A11 . ( A ) Concentration-response curves to ivermectin, ivermectin aglycone, selamectin, doramectin, eprinomectin, and moxidectin of the N2 wild-type and the transgene-Hcocyp-13A11 under control of the gut epithelium-specific promotor ges-1 . The corresponding chemical drug structure is presented below the graph and highlighted according to the structural differences to the other macrocyclic lactones. Blue : carbohydrate moiety; purple : hydroxyl/oxime substituent; green : alkyl-side chain; yellow : altered spiroketal function. Adult worms were incubated for 24 h in S-medium with Escherichia coli OP-50 containing different drug concentrations. The motility of individual worms was assessed as body bends per minute. The negative control, only containing 1% DMSO, was set to 10 -20 Mbefore log 10 transformation of concentrations. Values represent the means ± standard error of the mean of at least four biological replicates. At least 12 worms per concentration were counted for each replicate, and the mean normalized to the DMSO mean. The bottom and top values for four-parameter logistic regression were constrained to values between 0 and 100%. ( B ) Forest plot visualizing resulting EC 50 values [nM], corresponding 95% confidence interval for the N2 control ( black ) and the transgenic Hco-cyp-13A11 C. elegans strain ( blue ) from concentration-response curves presented in ( A ). Hco-cyp-13A11 genotype is gutCyp-13A11Ex1 [ Celges-1p::Hco-cyp-13A11::FLAG::Cel-unc-54\_3 ' -UTR; Cel-myo-2p::gfp::Cel-unc-54\_3 ' UTR ]; control strain is N2 Bristol wild-type; IVM: ivermectin; MOX: moxidectin; EPR: eprinomectin; IVMa: ivermectin aglycone; SEL: selamectin; DRM: doramectin.

<!-- image -->

50

value

was

observed

for

IVM

with

approximate

fold

-

change

of

4.3,

Table 1. Effect of Hco-cyp-13A11 transgenic expression on macrocyclic lactone susceptibility in Caenorhabditis elegans N2 background recorded by counting individual motility (body bends in liquid medium).

| Strain          | Drug                | EC 50 (95% CI 1 ) [nM]   |   R 2 2 | p Value   | Fold Change in EC 50 vs N2   |
|-----------------|---------------------|--------------------------|---------|-----------|------------------------------|
| N2 3            | Ivermectin          | 1.64 (1.43-1.87)         |   0.972 | <0.0001   |                              |
| Hco-cyp-13A11 4 | Ivermectin          | 7.01 (5.53-8.89)         |   0.874 |           | 4.27                         |
| N2              | Ivermectin aglycone | 163.6 (152.6-175.4)      |   0.984 |           | 1.68                         |
| Hco-cyp-13A11   | Ivermectin aglycone | 275.5 (259.7-292.2)      |   0.98  | <0.0001   |                              |
| N2              | Selamectin          | 156.7 (136.8-179.5)      |   0.983 |           |                              |
| Hco-cyp-13A11   | Selamectin          | 405.3 (386.1-425.3)      |   0.95  | <0.0001   | 2.58                         |
| N2              | Doramectin          | 7.75 (7.30-8.24)         |   0.987 | 0.0005    |                              |
| Hco-cyp-13A11   | Doramectin          | 8.60 (8.11-9.12)         |   0.986 |           | 1.10                         |
| N2              | Eprinomectin        | 1.48 (1.17-1.86)         |   0.994 |           |                              |
| Hco-cyp-13A11   | Eprinomectin        | 1.89 (1.39-2.56)         |   0.972 | 0.1165    | 1.27                         |
| N2              | Moxidectin          | 5.95 (5.38-6.58)         |   0.966 |           |                              |
| Hco-cyp-13A11   | Moxidectin          | 6.40 (5.81-7.05)         |   0.955 | 0.3212    | 1.07                         |

Table 2. Prediction of Hco-Cyp-13A11 protein parameters for different Haemonchus contortus isolates and the orthologous Caenorhabditis elegans cyp-13A11 and cyp-13A12 .

|                  | ISE Reference a         | ISE b                  | McM c                  | WR d                   | BSI e                  | C. elegans cyp-13A11 f   | C. elegans cyp-13A12 g   |
|------------------|-------------------------|------------------------|------------------------|------------------------|------------------------|--------------------------|--------------------------|
| No AA h          | 517                     | 517                    | 517                    | 515                    | 517                    | 517                      | 518                      |
| MW[kDa] h        | 59.56                   | 59.39                  | 59.58                  | 59.39                  | 59.58                  | 59.47                    | 59.08                    |
| Theo. pI h       | 8.76                    | 8.97                   | 8.97                   | 9.09                   | 8.62                   | 6.36                     | 6.08                     |
| Cyp heme motif i | 454-463 fj : FGlGPRQCIG | 454-463 j : FGlGPRQCIG | 454-463 j : FGlGPRQCIG | 452-461 j : FGlGPRQCIG | 454-463 j : FGlGPRQCIG | 454-463 j : FGlGPRQCIG   | 455-464 j : FGlGPRQCIG   |

The human Cyp-3A4 is the most abundant Cyp in the human liver and small intestine and oxidizes structurally diverse substrates [41,43]. Its importance in drug metabolism and pharmacologic drug development fostered structure elucidation by crystallography [43-47]. Using the superposition of different Cyp structures from various species, Cyps have been described to contain 19 structurally conserved regions (SCRs), including defined secondary structure elements such as alpha helices α A-L and beta sheets β 1-4 [48]. Assignment of a multiple sequence alignment to a crystal structure of the human Cyp-3A4 was conducted using ESPript (Figure 3) to identify Hco-cyp-13A11 secondary structure elements. As a PDB reference, the entry 1TQN was chosen since it is the only Cyp-3A4 structure available that includes the cofactor protoporphyrin containing an iron(II) without binding other substrates or inhibitors [43]. Comparing the SCRs between the Haemonchus and human Cyp, the Hco-cyp-13A11 residues for all isolates matched 38% to the Cyp-3A4 sequence. The highest residue coverages were observed for SCRs 2 (66%), 5 (57%), 11 (50%), 12 (53%), 15 (62%) and 16 (47%) (Figure 3). SCR11 α I and SCR16 α L belong to a conserved four-helix bundle which forms together with the SCR11 α J and SCR12 α K helices and the SCR15 meander the conserved core of the Cyp [49]. SCR16 comprises the CIG motif located in the cysteine pocket serving as a ligand to the heme iron [49], which is identical between the Haemonchus and human Cyp. The second conserved motif is ExxR, which is present at the C-terminal end of the SCR12 α K [48,49]. While the Cyp-3A4 shows a threonine at

the second position in this motif (ETLR), Hco-cyp-13A11 encodes a serine (ESLR) (Figure 3). The rest of the SCRs showed identities < 35%, while SCR4 and 19 share no residues with Cyp-3A4. The meander insertion proposed to form a reductase interacting surface [50] showed no AA identity but an identical length of six AAs for Cyp-3A4 and Hco-cyp-13A11 (Figure 3).

In total, 14 amino acid polymorphisms were observed between all analyzed isolates. Six positions were located in the SCRs 7, 10, 14, 15, and 16 based on the multiple peptide sequence alignment (Figure 3). The variations within the SCRs occur unspecifically without any pattern specific for the resistant or susceptible isolates. In addition, two AA deletions were found for the WR isolate in a non-structurally conserved region. Since one AA change is located in the meander loop and another directly at the end of the Cys-heme binding domain for WR and BSI, the substrate pocket or even the cofactor position inside the core structure could be affected. To investigate the impact of the AA variations on the overall 3D protein structure or ML binding, homology modeling and ML docking for each isolate were conducted.

## 2.3. Homology Models of Haemonchus contortus Cyp-13A11

To examine the impact of AA alterations on the protein structure and potential macrocyclic lactone binding, homology models for Hco-cyp-13A11 WR, McM, BSI, and ISE (own sequence) were constructed. A tertiary structure homology model was identified using BlastP with the protein data bank (PDB) as database and SwissModel Template Search web tool (https://swissmodel.expasy.org/interactive accessed on 5 February 2022) which identified several human Cyp-3A4 enzyme structures available in PDB. To select the best model for molecular docking studies, only holo-protein crystal structures were further investigated containing the enzyme bound to different ligands or inhibitors expected to allow the docking of MLs. The pre-selected reference structures (PDB: 6MA7, 5A1R, and 4D6Z) were further examined concerning sequence identity towards all Hco-cyp-13A11 sequences, resolution of the crystal structure, and the structure of the bound inhibitors (Table S3). After performing a multiple sequence alignment using Modeller version 10.1 [51], the crystal structure PDB: 6MA7 was chosen as the template for all homology models. A pairwise alignment was conducted for each Hco-cyp-13A11 sequence with the template, and 50 homology models were generated for each Modeller run. The homology models were ranked according to their discrete optimized protein energy (DOPE) score. Next, the quality of the homology models with the best DOPE score was assessed with the programs PROCHECK [52], ProSa-web (protein structure analysis) [53,54], and QMEAN (Qualitative Model Energy Analysis) [55,56] to execute individual model refinement.

The Ramachandran plot analysis showed that approximately 99% of all residues were within the generously allowed regions for all homology models (Table S4). The residues corresponding to the outlier regions and possessing sterically disallowed conformations refer to non-conserved loops and could not be optimized via individual loop refinement. However, no major difference between the quality scores of the four homology models was observed.

Generally, all homology models retained the overall protein fold of the Cyp-3A4 template with predominantly α -helices, a small portion of β -sheets (Figure 4A), and a similar solvent accessible area (ISE: 28056 Å 2 ; McM: 26790 Å 2 ; BSI: 28052 Å 2 ; WR: 27721 Å 2 as calculated with PyMol version 2.0 [57]). All structural elements of the SCRs predicted by multiple sequence alignment analysis could be confirmed for the homology models.

The highest values of local structure similarities with about 65-90% to the target sequence were computed for the structurally conserved regions between CYPs, particularly the heme-binding core. The lowest local similarities values (5-50%) were determined for the N-terminal tail, the BC-loop, the FG-loop, the GH-loop, and HI-loop regions, since the Hco-cyp-13A11 peptide sequence comprises within these regions more residues than the Cyp-3A4 template (Figure 3). In particular, the HI-loop, directed towards the surface of the protein structure, is characterized by eleven additional residues. This causes different

particularly

determined

for

the

N

-

terminal

tail,

the

BC

-

loop,

the

FG

-

loop,

the

GH

-

loop,

and

HI

-

loop

regions,

since

the

Hco

-

cyp

-

13A11

peptide

sequence

comprises

within

these

regions

more

residues

than

the

Cyp

-

3A4

template

(Figure

3).

In

particular,

the

HI

-

loop,

directed

towards

the

surface

of

the

protein

structure,

is

characterized

by

eleven

additional

residues.

This

causes

different

conformations

simulated

for

this

region

between

the

four

conformations simulated for this region between the four homology models. Furthermore, thirteen extra AA are found at the end of the N-terminal domain compared to the human Cyp-3A4, suggesting a functional relevance as an N-terminal anchor of this region. homology models. Furthermore, thirteen extra AA are found at the end of the N -terminal domain compared to the human Cyp -3A4, suggesting a functional relevance as an N -terminal anchor of this region.

Figure 3. A multiple peptide sequence alignment of Hco-Cyp-13A11 from different Haemonchus contortus isolates, including the ISE reference sequence (HCON\_00141052) [42] and human Cyp-3A4. The Hco-Cyp-13A11 amino acid sequence was predicted using ExPASy-Translate, and the alignment was obtained by Clustal Omega (version 1.2.4) [58]. Secondary structure elements were predicted with ESPript (version 3.0) [59] using the human Cyp-3A4 [43] (PDB: 1TQN), and residues were colored according to their physicochemical properties by ESPript. The annotation of structurally conserved regions (SCR) is indicated according to the multiple sequence alignment of human Cyp structures [48]. Numbers above the alignment refer to positions in the human Cyp-3A4. The cysteine heme-binding signature was predicted by ExPASy-ScanProsite. Black boxes highlight amino acid variation between different isolates for the specific position.

<!-- image -->

<!-- image -->

| WR BSI ISE                                                | GMIELGNMYELTDIS GLVL VY VGLVL GLVL GMIELGNMYELTDIS GLVL GMIE NMYELTDIS PVGLVL      |
|-----------------------------------------------------------|------------------------------------------------------------------------------------|
| ISE reference McM CYP-344 (ITQN) WR BSI ISE ISE reference | YYWQLDYW SCR4 ac 100 110 DMIKTVLV ECYSVFTNRRPF sg IIGGDVE VFEAQ ss EAQ ss ss GGDVE |
| CYP-344 (ITQN) ISE McM WR ISE ISE reference McM           | aG 210 220 230 240 IP FLR SQFFMAIA VIY TIDE SQFFMAIA TIDE P E EAQDAAGVVSQE         |
| WR McM                                                    | VI LA SCR13                                                                        |
| CYP-344 (ITQN) WR                                         | aK 330 340 350 360 370 VLQ DVEINGMF IPK NS STTLG VEE TTLG VEE                      |
|                                                           | EE ss VE                                                                           |
|                                                           | SETINYE DISVM ETLSSM Ls                                                            |
|                                                           | SCR16 meander loop Cys-pocket 400 440                                              |
| BSI ISE ISE reference McM                                 | SCR15 meander insertion 410 420 430 FSKKNKDNIDPYIYTPE RV SAAFLSE VL SAAFLSE        |
|                                                           | SAAFLSE LA SAAFLS EE SAAELS EE                                                     |
| WR ISE ISE reference McM                                  | ONF TELTA WTELTA TELTA                                                             |
|                                                           | SCR18 SCR19 Cysteine heme-binding signature                                        |
|                                                           | 480                                                                                |
|                                                           | 470                                                                                |
|                                                           | EKPVVLKVE PE TVQLQ ET VQLQ                                                         |
|                                                           | VQLQ                                                                               |

the

heme

-

binding

core.

The

lowest

local

similarities

values

(5-50%)

were

The

extent

to

which

the

additional

methyl

group

of

Leu388

influences

substrate

recognition

remains

unclear

here.

Interestingly,

the

B

-

factor

analysis

of

this

linker

region

revealed

higher

flexibility

for

the

susceptible

ISE

and

McM

isolates

compared

to

the

resistant

BSI

and

WR

(Figure

S4).

In

addition,

determination

of

the

electrostatic

inside

the

catalytic

pocket

using

PyMol

revealed

that

the

pocket

surface

is

primarily

hydrophobic

or

positively

charged

for

all

homology

models

(Figure

5D).

Figure 4. Homology model of Hco -Cyp -13A11. ( A ) Ribbon representation of the molecular model of Haemonchus contortus Cyp -13A11/12 ISE isolate. The model was created using Modeller and the human Cyp -3A4 (PDB: 6MA7) as a template. The protein shows the conserved overall fold of Cyps, with a large helical domain and a smaller beta -sheet region. The structurally conserved regions (SCRs) are highlighted in blue, whereas the variable regions are shown in gray. ( B ) The overview of the ISE Hco -Cyp -13A11 homology model displays the distribution of the 16 residue positions (pink sticks) varying between the ISE, McM, BSI, and WR isolates. Figure 4. Homology model of Hco-Cyp-13A11. ( A ) Ribbon representation of the molecular model of Haemonchus contortus Cyp-13A11/12 ISE isolate. The model was created using Modeller and the human Cyp-3A4 (PDB: 6MA7) as a template. The protein shows the conserved overall fold of Cyps, with a large helical domain and a smaller beta-sheet region. The structurally conserved regions (SCRs) are highlighted in blue, whereas the variable regions are shown in gray. ( B ) The overview of the ISE Hco-Cyp-13A11 homology model displays the distribution of the 16 residue positions (pink sticks) varying between the ISE, McM, BSI, and WR isolates.

<!-- image -->

A B-factor analysis used to identify the flexibility of side chains for each homology model revealed higher flexibility for these poorly conserved regions in comparison to human Cyp-3A4 (Figure S4). Interestingly, the BC-loop regions for the susceptible isolates ISE and McM (Figure S4A,B) were predicted to be more flexible compared to the resistant BSI and WR isolates (Figure S4C,D), although this region showed no AA variation due

to an SNP. The BC-loop region is described as an essential secondary structure within the CYPs influencing substrate specificity and binding [48].

When comparing all models regarding the predicted AA substitutions, variation primarily occurs randomly in domains towards the outer surface (Figure 4B). No SNP resulting in AA variation drastically influenced the overall protein fold between the HcoCyp-13A11 sequences.

However, two variable residues are directly located within the heme-core structure and could potentially affect heme-binding (residue 464) or the recognition and binding of potential substrates (residue 390). The position numbers refer to the ISE reference sequence if not stated otherwise. Therefore, the coordination of the heme-cofactor, particularly the heme propionates, was further investigated, as slight changes of the heme cofactor position within the catalytic pocket can affect its size [60]. Regarding position 464 within the highly conserved Cys-loop region, the isolates WR and ISE show a valine, whereas BSI and McM exhibit a methionine equal to the Cyp-3A4 template, potentially enabling it to form a hydrogen (H) bond and reducing the binding area. Nevertheless, the analysis of the hemebinding site for each model revealed no participation of this specific residue in coordinating the heme or influencing the secondary Cys-loop structure. Rather, the superimposition of all homology models and the Cyp-3A4 template structure indicated a highly conserved Cysloop fold with Cys461 (ISE, McM, BSI) or Cys459 (WR) at the proximal ligand binding site coordinating the iron heme center (Table S5). Hence, the planar heme cofactor is partially located between the α I and α L helices with the propionates directed to the BC-loop and β 1-4 sheet.

In addition, slight differences in residues interacting with the heme propionates were observed between all homology models based on PyMol and the Protein-Ligand Interaction Profiler web tool (version 2.2.0) [61] analyses (Table S5). Key residues forming hydrogen bonds with the propionates represent Arg99, Trp124, Arg128, Asn393, Arg395, Arg459 (WR: Arg99, Trp124, Arg128, Asn391, Arg393, and Arg457) (Figure 5). However, one propionate residue is additionally stabilized by a hydrogen-bond, with His115 perpendicularly located above the heme center for ISE and WR isolates (Figure 5A,C). In contrast, no interaction with His115 was computed for the McM and BSI models (Figure 5B). In addition, the heme within the WR model was not predicted to interact with Phe452 as determined for ISE, McM, and BSI (Phe454).

The second variable residue in proximity to the heme-core structure at position 390 (ISE, McM, BSI) is substituted in the WR isolate (Val388Leu) and potentially influences the substrate recognition and/or binding located within the linker region of α K and β 1-4. The larger residue side chain directly ranges into the active site pocket but without changing the electrostatics of the cavity. The active site volume for WR with a computed size of 916 Å 3 was therefore the smallest cavity volume compared to the other homology models (ISE: 1392 Å 3 ; McM: 1024 Å 3 ; BSI: 1284 Å 3 ).

The extent to which the additional methyl group of Leu388 influences substrate recognition remains unclear here. Interestingly, the B-factor analysis of this linker region revealed higher flexibility for the susceptible ISE and McM isolates compared to the resistant BSI and WR (Figure S4). In addition, determination of the electrostatic potential inside the catalytic pocket using PyMol revealed that the pocket surface is primarily hydrophobic or positively charged for all homology models (Figure 5D).

Int.

J.

Mol.

Sci.

2022

,

23

,

x

FOR

PEER

REVIEW

11

of

28

Figure 5. Heme cofactor coordination within Hco -Cyp -13A11 homology models. Coordination of heme propionates and polar interaction sites for Hco -Cyp -13A11 ISE ( A ), McM and BSI ( B ), and WR ( C ) determined with the PyMol Plugin ShowContacts. Hydrogen bonds are presented in yellow, electrostatic clashes in red, and further close contacts (<4 Å) in purple. ( D ) Representative electrostatic surface of the active site cavity. Electrostatic potentials were calculated with PyMOL. Red and blue colors represent the negative and positive electrostatic potentials, respectively. Figure 5. Heme cofactor coordination within Hco-Cyp-13A11 homology models. Coordination of heme propionates and polar interaction sites for Hco-Cyp-13A11 ISE ( A ), McM and BSI ( B ), and WR ( C ) determined with the PyMol Plugin ShowContacts. Hydrogen bonds are presented in yellow, electrostatic clashes in red, and further close contacts (<4 Å) in purple. ( D ) Representative electrostatic surface of the active site cavity. Electrostatic potentials were calculated with PyMOL. Red and blue colors represent the negative and positive electrostatic potentials, respectively.

<!-- image -->

## 2.4. Molecular Modeling of Putative Macrocyclic Lactone-Binding Sites in Hco-Cyp-13A11

To obtain insights into the impact of AA variations towards substrate binding and understanding the observed differences for the MLs in the motility assays, in silico docking was conducted. After generating 3D models of IVM, IVMa, SEL, DRM, EPR, and MOX, the molecules were individually docked into every Hco-Cyp-13A11 homology model using AutoDock Vina (version 1.1.2) [62]. The substrate position with the lowest docking score was selected for iterative energy minimization for each ligand-protein complex. In addition, docking was performed for the Cyp-3A4 template sequence as a reference since the human Cyp-3A4 is known to metabolize IVM [30].

For the human Cyp-3A4, all MLs except EPR docked within the active site (Figure S5) with the highest affinities for IVM ( -10.1 kcal/mol) and IVMa ( -8.6 kcal/mol). In the case of IVM, the hydroxyl group of the disaccharide moiety was most closely located to the heme iron (distance 3.3 Å). Ivermectin aglycone showed the closest contact with the terminal methyl-group of the alkyl side chain at the spiroketal moiety (distance 4.8 Å). Although SEL, DRM, and MOX also docked within the catalytic pocket, the resulting docking energies were considerably higher (Table 3). Similar to IVM, a hydroxyl group of the DRM disaccharide showed the shortest distance (2.9 Å) to the heme iron. In contrast, the methyl group of the benzofuran substituent of SEL (2.9 Å) and the methyl group of the spiroketal moiety of MOX (5.4 Å) were directed towards the heme center. Apart from SEL and DRM, which both were calculated to interact with Ser119, no substrate formed H-bonds with the active site residues that have been reported in the literature (Tables S5 and S6) [43]. Eprinomectin was predicted to dock at the outer surface of Cyp-3A4 ( -8.4 kcal/mol) in a location parallel to the BC-loop and the β -bulge region and lateral of the FG-loop.

Calculations for the Hco-Cyp-13A11 models demonstrated considerable differences in docking energies compared to the human Cyp-3A4. IVM, IVMa, SEL, and MOX were predicted to bind with generally lower energies ranging from -5.9 to -11.3 kcal/mol within the active site for all homology models (Table 3, Figures 6 and S6-S8). In addition, no considerable differences between the models nor a correlation between differences in models and the resistance status were observed (Table 3). However, the MLs were predicted to have different interaction sites (Table S6) within the four models and orientations within the catalytic pocket for the lowest docking energy positions.

Investigating IVM, the spiroketal moiety was located towards the heme cofactor in the ISE, BSI, and WR homology models. At the same time, the best docking position for McM indicated IVM to be directed towards the center with the benzofuran substituent. The analysis of IVMa predicted that this molecule could interact with the heme center via the alkyl side chain of the spiroketal moiety while it gets potentially metabolized at the benzofuran by the WR Hco-Cyp-13A11. In the case of SEL, the molecule was directed with the sugar substituent to the catalytic center of McM and WR while it was flipped within BSI and ISE. There, SEL potentially interacts via the cyclohexyl side chain with the heme center. Moxidectin showed the fewest commonalities regarding the predicted binding topology between the homology models. Docking for WR and McM presented the methoxime moiety above the heme iron. For the BSI model, MOX most likely interacts with the methyl of the spiroketal group. In contrast, MOX was directed with the benzofuran group to the catalytic site for the ISE model. For both EPR and DRM, docking into the active site was not feasible for the WR model. The best-fit position was predicted to occur on the protein surface. Both molecules were simulated to bind to the FG-loop's peripheral site, a highly hydrophobic region that is predicted to reside within the lipid bilayer and the N -terminal membrane anchor α -helix [63].

Table 3. Docking characteristics of IVM, IVMa, SEL, EPR, DRM, and MOX in the human Cyp-3A4 (PDB: 6MA7) structure and Hco-Cyp-13A11 homology models. The docking energy was computed with AutoDock Vina (version 1.1.2) tools. The docking values in bold represent docking at the surface of the protein, as no docking into the active site could be simulated. The solvent-accessible surface area for the best docking position was estimated using PyMol. The number of hydrogen (H) bonds and hydrophobic interaction sites were determined with the web tool PLIP [61].

|                                        |                     | Hco-Cyp-13A11 b c d   | Hco-Cyp-13A11 b c d   | Hco-Cyp-13A11 b c d   | Hco-Cyp-13A11 b c d   |
|----------------------------------------|---------------------|-----------------------|-----------------------|-----------------------|-----------------------|
|                                        | Cyp-3A4 (PDB: 6MA7) | ISE a                 | McM                   | WR                    | BSI                   |
| Ivermectin                             |                     |                       |                       |                       |                       |
| Docking energy [kcal/mol]              | - 10.1              | - 10.5                | - 8.5                 | - 6.5                 | - 8.1                 |
| Solvent accessible surface Area [Å 2 ] | 993                 | 1084                  | 1058                  | 962                   | 1076                  |
| Number of H-bonds                      | 3                   | 2                     | 4                     | 8                     | 2                     |
| Hydrophobic interactions               | 22                  | 11                    | 12                    | 13                    | 12                    |
| Ivermectin aglycone                    |                     |                       |                       |                       |                       |
| Docking energy [kcal/mol]              | - 8.6               | - 11.0                | - 11.3                | - 10.5                | - 11.2                |
| Solvent accessible surface Area [Å 2 ] | 745                 | 756                   | 762                   | 753                   | 745                   |
| Number of H-bonds                      | 1                   | 3                     | 2                     | 1                     | 3                     |
| Hydrophobic interactions               | 7                   | 9                     | 2                     | 7                     | 10                    |
| Selamectin                             |                     |                       |                       |                       |                       |
| Docking energy [kcal/mol]              | - 0.5               | - 8.2                 | - 9.3                 | - 9.1                 | - 8.8                 |
| Solvent accessible surface Area [Å 2 ] | 949                 | 953                   | 918                   | 918                   | 927                   |
| Number of H-bonds                      | 1                   | 1                     | 2                     | 6                     | 3                     |
| Hydrophobic interactions               | 15                  | 11                    | 13                    | 11                    | 11                    |
| Doramectin                             |                     |                       |                       |                       |                       |
| Docking energy [kcal/mol]              | - 4.1               | - 10.6                | - 8.6                 | - 11.1                | - 9.8                 |
| Solvent accessible surface Area [Å 2 ] | 1139                | 1111                  | 1134                  | 1148                  | 1136                  |
| Number of H-bonds                      | 3                   | 3                     | 4                     | 3                     | 4                     |
| Hydrophobic interactions               | 16                  | 13                    | 9                     | 10                    | 4                     |
| Eprinomectin                           |                     |                       |                       |                       |                       |
| Docking energy [kcal/mol]              | - 8.4               | - 8.0                 | - 7.7                 | - 8.5                 | - 10.8                |
| Solvent accessible surface Area [Å 2 ] | 1057                | 1143                  | 1068                  | 1104                  | 1155                  |
| Number of H-bonds                      | 4                   | 3                     | 7                     | 3                     | 4                     |
| Hydrophobic interactions               | 5                   | 15                    | 15                    | 4                     | 7                     |
| Moxidectin                             |                     |                       |                       |                       |                       |
| Docking energy [kcal/mol]              | - 1.6               | - 11.0                | - 5.9                 | - 8.9                 | - 7.8                 |
| Solvent accessible surface Area [Å 2 ] | 875                 | 870                   | 881                   | 870                   | 876                   |
| Number of H-bonds                      | 4                   | 1                     | 3                     | 5                     | 3                     |
| Hydrophobic interactions               | 13                  | 10                    | 7                     | 8                     | 6                     |

- a Inbred-susceptible isolate. b McMaster isolate. c White River isolate. d Berlin-selected isolate.

The same binding locations (Figure S8) and docking energies (Table 3) for EPR and DRMwere identified for the BSI Hco-Cyp-13A11 model. While the DRM binding site for both WR and BSI models shared Arg209 and Arg497 (WR: Arg495) as interacting residues, EPR showed no conformity (Figures 6 and S8, Table S6). In contrast to the WR and BSI models, docking simulations for the ISE and McM models resulted in EPR and DRM binding within the active site (Figures S6 and S7). Both models for the susceptible isolates already predicted higher cavity volumes than the resistant isolate models.

PEER

REVIEW

14

Figure 6. Molecular docking analysis for Hco -Cyp -13A11 Berlin -selected isolate (BSI). ( A , B ) Overlayed structures of Hco -Cyp -13A11 BSI in complex with IVM (blue), IVMa (cyan), SEL (green), DRM (orange), EPR (red), and MOX (yellow). The corresponding 2D ML ligand -protein interaction diagrams of Hco -Cyp -13A11 with IVM ( C ), IVMa ( D ), SEL( E ) and MOX ( H ) show the closest residues within a 4 Å radius. DRM ( F ) docked within a channel leading to the active site. EPR ( G ) showed the lowest docking energy and docked at the outer surface of the BC -loop. 2D interaction plots were generated using Maestro Elements (version 4.6.117). Figure 6. Molecular docking analysis for Hco-Cyp-13A11 Berlin-selected isolate (BSI). ( A , B ) Overlayed structures of Hco-Cyp-13A11 BSI in complex with IVM (blue), IVMa (cyan), SEL (green), DRM(orange), EPR (red), and MOX (yellow). The corresponding 2D ML ligand-protein interaction diagrams of Hco-Cyp-13A11 with IVM ( C ), IVMa ( D ), SEL( E ) and MOX ( H ) show the closest residues within a 4 Å radius. DRM ( F ) docked within a channel leading to the active site. EPR ( G ) showed the lowest docking energy and docked at the outer surface of the BC-loop. 2D interaction plots were generated using Maestro Elements (version 4.6.117).

<!-- image -->

## 3. Discussion

Cyps have already been reported to contribute to resistance in cancer (Rochat, 2005; Rodriguez-Antona and Ingelman-Sundberg, 2006) and against insecticides [64,65]. Even though the primary resistance mechanism against the broad-spectrum anthelmintics benzimidazoles (BZs) is linked to SNPs of the beta-tubulin gene of nematodes [66-68], an increased oxidation level of different BZs was observed in C. elegans , H. contortus, and Fasciola hepatica [69-73].

In C. elegans , the ablation of cytochrome P450 activity with the widely used Cyp inhibitor piperonyl butoxide or the use of a severely impaired strain in cytochrome reductase activity resulted in slightly increased susceptibility to IVM but not MOX [74]. The authors further excluded that Cyp-14A5 contributes to changes in IVM susceptibility, although it was previously shown to be moderately upregulated in IVM and MOX-selected C. elegans strains [20]. The present study aimed to investigate the impact of the transgenically expressed Hco-cyp-13A11 in the intestine of C. elegans regarding the susceptibility of different MLs.

Using extrachromosomal arrays to express transgenes represents a fast and efficient technique for analyzing gene functions. Within this study, microinjection was performed into the C. elegans N2 background since C. elegans Cel-cyp-13A11 and Cel-cyp-13A12 lossof-function strains were unavailable to perform rescue experiments. Various studies have already demonstrated the suitability of transgene expression to investigate drug targets against parasitic nematodes [15,39,75-77].

The analyzed Hco-cyp-13A11 was derived from the highly resistant White River isolate and its expression in the gut epithelium has decreased susceptibility for IVM, IVMa, and SEL. In contrast, no effect was observed for EPR, DRM, and MOX compared to the N2 wild-type control line. This study demonstrates for the first time that ML sensitivity in C. elegans can be modulated by transgenic expression of a parasitic nematode Cyp.

Interestingly, neither a constitutive expression nor a direct link to the modulation of MLsusceptibility or xenobiotic inducibility of Hco-cyp-13A11 has been reported so far. The introgression of two highly resistant H. contortus isolates into a susceptible background identified a quantitative trait locus (QTL) for IVM resistance. This QTL was localized on chromosome V ranging from 37 to 42 Mbp, and the same region was under selection using two geographically and genetically divergent IVM resistant populations [78]. Although Hco-cyp-13A11 is located on chromosome V (area of 14.2 Mbp) in proximity to the QTL region (37-42 Mbp), it seems not to be part of the QTL under selection. However, there are several possible mechanisms by which Hco-cyp-13A11 might nevertheless be involved in modulating the susceptibility of worms to MLs. Due to the high genomic variability of H. contortus , different genomic regions might be involved in other isolates and it might be difficult to obtain statistical support for regions with only a small effect. The major QTL regions might be under selection but exert its effects through effectors encoded in other regions that might include Hco-cyp-13A11 . Alternate splicing events have not been considered here but might lead to changes in the protein structure, augmenting the gene function or translation efficacy as reported for C. elegans [79] and potentially increasing the resistance level. For example, Brugia malayi revealed a sex-dependent differential splicing of the potassium channel slo-1 causing significant differences in emodepside sensitivity between females and males [80]. Moreover, alternate splicing of the acetylcholine receptor subunit acr-8 resulted in truncated transcripts that were found to be expressed explicitly in levamisole resistant H. contortus , Trichostrongylus colubriformis, and T. circumcinta [81,82].

Investigations on the orthologues C. elegans Cyp cyp-13A12 co-expressed with the Cyp reductase emb-8 in insect cells revealed its role as an epoxygenase of polyunsaturated fatty acids [83]. Although further functional information and tissue expression levels are missing, the miscellaneous role of this particular C. elegans Cyp does not exclude the hypothesis that Cel-cyp-13A12 might also contribute to ML resistance. This should be investigated in the future by creating loss-of-function alleles and examining the effects of overexpression of Cel-cyp-13A12 but also investigating its closely related paralog Cel-cyp-13A11 .

Within this study, the transgenic expression of Hco-cyp-13A11 had the most potent effect for IVM with a 4.3-fold increase of the EC 50 value, followed by SEL with 2.6-fold and a modest increase for IVMa with a 1.7-fold change. Different studies have compared the interaction of IVM and MOX with recombinantly expressed Pgps, and in all cases, the interaction of IVM was much stronger than for MOX [1,84,85]. However, only one study compares the effects of a similar spectrum of MLs on Pgps. Kaschny et al. [86] expressed Cylicocylus elongatus pgp-9 in Saccharomyces cerevisiae yeast cells. This data revealed a strong interaction with IVM and EPR, a moderate interaction with MOX, and no interaction with SEL and DRM [86]. Ivermectin aglycone was not tested in this study. Although the different order of effect strength for the different MLs shows specific interactions with the particular Pgp, the fact that IVM always showed the strongest effects is remarkable. However, this needs more diverse studies since most investigations only compared IVM to MOX[39,84,85].

Except for the milbemycin MOX, all tested drugs belonged to the class of avermectins differing in the sugar moiety, the alkyl side chain, a methoxime function, or a functionalized benzofuran. In particular, the sugar group was shown to strongly affect the physicochemical properties of these drugs, for example, the hydrophobicity results in different logP values [1]. This further influences pharmacokinetics and pharmacodynamics, mainly demonstrated for IVM and MOX, potentially explaining the observed differences for IVM, SEL, and IVMa [6,20,85,87,88]. Ivermectin and MOX were shown to bind to the same glutamate-gated chloride channel (GluCl) subunit, but with different affinities towards the binding site [87]. The analysis of the GluCl crystal structure reinforced the previous observations made for IVM interactions and proposed a model for the IVM binding site. The missing disaccharide and the additional methoxime moiety for MOX suggest different interactions for MOX within this structure [89]. A difference between IVM and MOX can also be assumed for ML metabolism by H. contortus . While MOX was reported to be metabolized [33], no IVM metabolites have been identified [34]. Since the latter study by Vokral et al. (2013) used a susceptible H. contortus isolate to analyze metabolism, the production of IVM metabolites cannot be hypothesized for resistant isolates containing SNPs or overexpressing xenobiotic metabolizing enzymes.

The transgenic expression of Hco-cyp-13A11 did not affect the susceptibility to MOX, EPR, and DRM, thereby confirming the fact that there are considerable pharmacokinetic differences between different MLs. However, the sugar moiety does not seem to be the pivotal functional group in deciding whether a ML can interact with Hco-cyp-13A11 , since IVM, EPR, and DRM all possess a disaccharide group in contrast to MOX, but Hco-cyp-13A11 expression was only protective against IVM. Further structural differences are present at the alkyl side chain of MOX and an additional carbon-carbon double bond at the spiroketal moiety of EPR and DRM.

To better understand why Hco-cyp-13A11 shows a substrate specificity towards the tested MLs, comparison in the observed differences in EC 50 values, molecular modeling and docking studies were performed. Although analysis of Hco-cyp-13A11 in C. elegans was carried out only for the sequence variant derived from the White River isolate, in silico analyses were extended to Hco-cyp-13A11 sequences obtained from three additional isolates differing in resistance status. Human Cyps have already been shown to exhibit polymorphisms in multiple allelic variants associated with differences in metabolizing drugs [88,90]. In total, the exon sequences of Hco-cyp-13A11 from four H. contortus isolates revealed 14 polymorphic positions in the amino acid sequences. However, none of these polymorphisms occurred explicitly in the ML-susceptible or -resistant isolates. This can certainly be linked to the high genetic diversity of H. contortus with a mutation rate ten times higher than in vertebrates, resulting in considerable variation within a laboratory strain as observed for the sequences derived from the partially inbred ISE [42,91,92]. Additional different selection pressures can affect the genes selected and how they respond to MLs, thereby leading to different phenotypes within different ML-resistant strains [93,94].

Nonetheless, two AA changes were predicted in SCRs directly located towards the active site of the Cyp and potentially influencing substrate binding. Molecular modeling of Hco-Cyp-13A11 calculated that the other AA variations occurred mainly towards the protein's surface. This may lead to slight changes in electrostatic interactions with membrane lipids, affecting Cyp behavior as proposed for human Cyps [95]. The more positively charged AA are located on the protein's surface, the stronger it is attracted to anionic membranes compared to neutral ones therefore changing the orientation of Cyp towards the membrane and its immersion depth in the membrane. Hence the flexibility in the membrane-immersed parts of the catalytic domain such as the FG-loop and reduction in access for substrates to their channels [95]. However, molecular dynamic simulations should be conducted to evaluate the impact of all AA changes on the substrate access channel gating.

In general, the molecular models for Hco-Cyp-13A11 revealed high sequence identities for the helices I, L, J, K, and the two sets of beta-sheets compared to the human Cyp-3A4. Furthermore, the ExxR and the CIG sequence motifs which are known to stabilize the core, and the heme-binding are also preserved for Hco-Cyp-13A11 [48]. This core structure was shown to be highly conserved in Cyps of all phyla [48,96]. Although the primary sequence forming the helices E and D showed a low sequence coverage for Hco-Cyp-13A11 among the isolates, which were predicted to be well-conserved core structures in mammals [49], the secondary helix structure was preserved.

Interestingly, Hco-Cyp-13A11 exhibits more residues within the G helix (6 AA) and the HI-loop (11 AA) than Cyp-3A4. Even though the exact function of the HI-loop at the surface of the protein remains unclear, the quantity of AA within Hco-Cyp-13A11 can affect the opening and closing of the substrate access channels through altered flexibility and conformational dynamic [64,97].

The short meander insertion region of Hco-Cyp-13A11 indicates that this protein can be potentially classified as a class II Cyp. The classification refers to the electron transfer mechanism to the catalytic site with class II Cyps only requiring a flavin adenine nucleotide containing P450 reductase to transfer electrons. In contrast to class II, class I Cyps require a reductase and an iron-sulfur redoxin, while class III Cyps do not require any electron donor [50]. Together with α J/ α J', the meander insertion region is proposed to display a reductase interaction face [51]. Keller et al. (2014) has already revealed the importance for the orthologous Cel-cyp-13A12 to be co-expressed with the reductase emb-8 to build a monooxygenase system.

The low sequence identity for the non-core regions between human Cyp-3A4 and Hco-Cyp-13A11 is not surprising. Homologous Cyps are not necessarily expected to have the same spectrum of specificity or affinity. It can be assumed that the nematode Cyp metabolizes a particular drug that is no substrate to Cyp-3A4 and the other way around.

Human Cyp-3A4 metabolizes several large substrates, including erythromycin and cyclosporin [23,98]. The present study indicated favorable docking scores for the MLs IVM, IVMa, SEL, DRM, EPR, and MOX when in silico complexed with the human Cyp-3A4 crystal structure (PDB: 6MA7). Except for EPR, all drugs could dock to the active site cavity within the protein. Cyp-3A4 has already been associated with cross-resistance to several drugs [23]. Independent biotransformation studies for IVM [30] and MOX [32] conducted with hepatic microsomes described the ability of the Cyp-3A family and particular Cyp-3A4 to metabolize these drugs. The best IVM docking score within this study was accomplished with the disaccharide moiety directed to the heme center. Indeed, IVM metabolite analysis showed a demethylation product at the sugar group [30,31]. The in silico calculations for MOX exhibited the closest contact towards the heme with the methyl-group positioned at the spiroketal function, and a corresponding hydroxylation product has also been identified [99,100]. However, these metabolite formation studies pointed out more than one metabolization site for IVM and MOX.

Multiple docking modes within Cyp-3A4 were also reported for erythromycin and ketoconazole and depended on the binding of an effector molecule and the molecular size of

the substrate [101]. It should be noted that the analysis in the present study focused only on the best docking score for each drug. Ivermectin, MOX, IVMa, and SEL were docked with different conformations into the Cyp-3A4 active site exhibiting differing docking scores. Therefore, it can be assumed that Cyp-3A4 can also metabolize IVMa, SEL, and DRM. In contrast to the other MLs, EPR showed the highest docking score on the solvent-accessible site between the BC-loop and the beta bulge region. The BS-loop represents one of the most flexible parts of the protein, influencing the formation of substrate access channels to bind drugs selectively [95,97]. Thus, it currently cannot be excluded that EPR cannot bind within the active site.

The Hco-Cyp-13A11 models generated here suggested that the protein can form complexes with each ML, but docking scores for SEL, DRM, and MOX were considerably higher than those obtained for human Cyp-3A4. For Hco-Cyp-13A11, no major differences regarding drug docking scores or drug interacting residues could be observed between the models for predicted sequences from susceptible and resistant isolates. Analyzing predicted hydrogen-bond formation with MLs in the active site, Lys100, Arg113, and Ser394 were identified as potential hot-spot residues for substrate recognition in all Hco-Cyp13A11 models. Interestingly, the models for WR and BSI showed docking of DRM and EPR at the solvent-accessible site of the FG-loop. The observed peripheral docking area has already been described for the human Cyp-3A4 when exposed to the highly hydrophobic bromoergocryptine and ritonavir. It is assumed that these molecules first bind outside and are then translocated to the catalytic cavity due to conformational changes [64]. Currently, it cannot be excluded that Hco-Cyp-13A11 WR and BSI would also allow translocation of EPR and DRM to the active site.

In contrast, the active site of ISE and McM models was sufficient to complex both molecules. It remains unclear if the smaller active sites of the WR and BSI models, presumably caused by the long-distance effects of polymorphisms at positions outside of the active site, are reliable or caused by artifacts in the modeling process.

The FG-loop, which has been reported to be immersed into the membrane in many Cyps, represents, similar to the BC-loop, one of the most flexible regions in Cyps and is capable of opening and closing substrate access channels [95]. Nevertheless, the amphiphilic character of EPR and DRM would enable them to enter the active site from the membrane environment, which cannot be excluded since channel formation is dependent on different environmental factors such as membrane composition or cholesterol concentration within the membrane [23,95].

Although no IVM biotransformation products for the susceptible ISE isolate were detected [34], the importance of Cyps increasing the ML resistance should not be underestimated. This is mainly because MOX oxidation products were identified for H. contortus [33], and the docking study revealed the ability of multiple MLs to bind to Hco-cyp-13A11 .

However, the identified SNPs leading to 14 amino acid changes within Hco-cyp-13A11 between the analyzed isolates ISE, McM, WR, and BSI do not seem to contribute to specific responses of resistant and susceptible isolates towards MLs. It needs to be investigated if Hco-cyp-13A11 might be differentially expressed upon ML exposure as already shown for HCON\_00038960 [19]. Hco-cyp-13A11 could be one among several players leading to ML resistance, as several studies already support a multigenic basis for ML resistance [42,78,102,103].

Besides Cyps, flavin-containing monooxygenases, glutathioneS -transferases or glycosyltransferases contribute to drug metabolism in humans, ruminants, and nematodes by being upregulated or reinforcing the metabolic cascade [13]. Particularly, members of the drug transporter family P-glycoproteins such as H. contortus and Parascaris univalens pgp-2 [17,104] Cylicocyclus elongatus , H. contortus and Parascaris univalens pgp-9 [86,104,105], T. circumcincta pgp-10 [16], Parascaris univalens and H. contortus pgp-11 [15,19], and Hcopgp-13 [106], were linked to IVM resistance. A structural model analysis of Hco-pgp-13 additionally predicted a high-affinity binding site in the inner chamber of the protein for IVM [106]. Similarly, high binding affinities were also calculated for the Cel-pgp-1 , which

was also predicted to bind EPR, DOR, SEL, MOX, IVMa, and abamectin within the same cavity [107]. P-glycoproteins and cytochrome P450 enzymes, especially the human Cyp-3A family, possess broad substrate specificities and were shown to overlap in substrate and inhibitor specificity [23,108,109]. Both human Pgps and Cyps are frequently expressed in the same cells, such as hepatocytes and enterocytes, demonstrating their functional link within the drug detoxification cascade [23]. Hence, Pgps can alter the intracellular concentration of Cyp inducers or substrates and result in changing the magnitude of inductive and catalytic response. Though detailed data about the tissue-specific expression of Pgps and Cyps in H. contortus are missing, similar coherent expression patterns can be deduced from different studies. Both Hco-pgp-13 [106], Hco-pgp-2 [17], and Hco-cyp-34A5 , Hco-cyp-13A10 , Hco-cyp-33C9 , Hco-cyp-43A1 [24] show higher expression levels in the intestine of adult H. contortus than in other tissues. However, inducibility upon IVM exposure was only shown for Hco-pgp-13 so far [19].

In conclusion, this is the first report of a transgenically expressed parasitic nematode Cyp decreasing IVM, IVMa, and SEL susceptibility. However, molecular docking studies could not verify substrate specificity or resistance-specific mutations for Hco-cyp-13A11 resulting in different affinities towards the MLs. Therefore, Hco-cyp-13A11 is assumed to contribute to ML resistance in H. contortus only when combined with other pathways.

## 4. Materials and Methods

## 4.1. Chemicals

Stock solutions of 10 µ M IVM (Sigma-Aldrich, I8898, Taufkirchen, Germany), 10 µ M MOX (Sigma-Aldrich, 33746), 100 µ M IVMa (Santa Cruz, SC-202189, Heidelberg, Germany), 10 µ MEPR(Cayman, 28182, Ann Arbor, MI, USA), 100 µ MSEL (Sigma-Aldrich, 32476, Darmstadt, Germany) and 10 µ M DRM (Sigma-Aldrich, 33993, Germany) were prepared in 100% DMSO. For thrashing assays with the C. elegans N2 Bristol and transgenic strain expressing Hco-cyp-13A11, stock solutions were serially diluted to obtain final concentrations ranging from 0.1 to 1 µ MIVM, 0.25 to 2.5 µ MMOX, 10 to 100 µ MIVMa, 0.05 to 2 µ MEPR, 20 to 200 µ MSEL, and 0.5 to 2 µ MDRM.

## 4.2. Plasmid Construction for Transgenesis

For transgenesis, a plasmid containing the 1551 bp H. contortus White River (WR) cyp-13A11 (old designation: HCOI00827700, now: HCON\_00141052, BioProject PRJEB506) placed downstream of a 2000 bp C. elegans ges-1 promoter fragment followed by a FlagTag and 822 bp of the C. elegans 3 ' -UTR of the unc-54 gene was constructed (Figure S1). To amplify ges-1 and the unc-54 3 ' -UTR, genomic DNA was isolated from C. elegans Bristol N2 using NucleoSpin Tissue XS kit (Macherey Nagel) according to the manufacturer's instructions. To amplify Hco-cyp-13A11 , RNA was isolated from fourth-stage larvae and cDNA synthesized as described previously [28]. Each fragment needed to assemble an expression plasmid was amplified via PCR or RT-PCR. Corresponding primer sequences were designed using the NEBuild-er ® Assembly tool (version 2.3.1). Reactions contained 0.02 U/ µ L Phusion Hot Start II High-Fidelity DNA polymerase (Thermo Scientific, Waltham, MA, USA), 0.2 µ MdNTPs, 0.5 µ Mof each primer in 50 µ L 1 × Phusion HF buffer. Reactions were denatured at 98 · C for 2 min, followed by 50 cycles of 98 · C for 5 s, a primer-pair specific annealing temperature for 30 s, and 72 · C for 2 min. Primer sequences and specific annealing temperatures are provided in the supporting data (Supplementay Table S1). The purified PCR products (DNA Clean & Concentrator™-5, Macherey Nagel) were used for assembly with the NEBuilder ® HiFi DNA Assembly Master Mix and were cloned with NEBuilder HiFi DNA Assembly Cloning Kit (New England Biolabs) according to the manufacturer's instructions. The pUC19 vector (New England Biolabs) was linearized with SmaI (Thermo Scientific) before assembly. The resulting plasmid was confirmed by Sanger sequencing (LGC Genomics, Berlin, Germany).

## 4.3. Transformation of Caenorhabitis elegans

The C. elegans Bristol N2 strain obtained from Caenorhabditis Genetics Centre (CGC, University of Minnesota, Minneapolis, MN, USA) was used as wild-type and maintained under standard conditions [110]. The plasmid for the expression of H. contortus cyp-13A11 was diluted in water and injected into the germline of young adult C. elegans hermaphrodites at a concentration of 50 ng/ µ L as described previously [39,75]. A plasmid carrying a pharyngeal GFP-expression marker (pPD118.33, Addgene plasmid 1596: L3790, P myo2 -gfp , Fire Lab 1995 Vector Kit) was co-injected as a transformation marker at a concentration of 12.5 ng/ µ L. Successfully transformed worms were identified in the F1 progeny of injected worms by GFP fluorescence and isolated on new agar plates (Figure S2). RT-PCR confirmed the expression of Hco-cyp-13A11 . For this purpose, GFP positive transgenes, all descending from the same F1 worm, were collected, homogenized with a pestle, and RNA isolated with the NucleoSpin RNA kit (Macherey Nagel) according to the manufacturer's instructions. Then, cDNA was synthesized from approximately 1 µ g of RNA using the Maxima first-strand cDNA synthesis kit (Thermo Fisher) according to the manufacturer's instructions. PCR was conducted in a 50 µ L reaction mixture containing 2 µ L cDNA or no-RT control template, 0.02 U/ µ L Phusion Hot Start II High Fidelity DNA polymerase (Thermo Scientific), 0.2 mM dNTPs, 0.25 µ M of each gene-specific expression plasmid primer (Table S1) in 50 µ L 1 × Phusion HF buffer. The reaction was denatured at 98 · C for 2 min, followed by 45 cycles of 98 · C for 15 s, annealing at 57.8 · C for 30 s and 72 · C for 2 min. The presence of the Hco-cyp-13A11 PCR product was verified by gel electrophoresis and Sanger sequencing.

## 4.4. Synchronization of Caenorhabditis elegans Developmental Stages

For synchronization, 100 µ L of pelleted worms were exposed to 2 mL of a lysis solution consisting of distilled water, 1 M NaOH, and chlorine-based household bleach in a 1:5:4 ratio. The obtained eggs were washed four times with a 15 mL M9 buffer (3 mg/mL KH 2 PO 4 , 6 mg/mL Na 2 HPO 4 , 5 mg/mL NaCl, 1 mM MgSO 4 ) and centrifuged for 2.5 min at 780 rcf. Finally, the eggs were resuspended in 2 mL M9 buffer, placed on nematode growth medium (NGM) agar plates seeded with Escherichia coli OP50 (Brenner, 1974), and left to hatch at 20 · C.

## 4.5. Motility Assays

Aminimum of 12 young adult transgenic individuals in 100 µ L M9 buffer were added to 1880 µ L S Medium (prepared by mixing 1 l S Basal (5.85 mg/mL NaCl, 6 mg/mL K 2 HPO 4 , 0.005 mg/mL cholesterol), 10 ml trace metal solution (1.86 mg/mL disodium EDTA, 0.69 mg/mL FeSO 4 · 7H 2 O, 0.2 mg/mL MnCl 2 · 4H 2 O, 0.29 mg/mL ZnSO 4 · 7H 2 O, 0.025 mg/mL CuSO 4 ), 3 ml 1M CaCl 2 and 3 mL 1M MgSO 4 ) containing E. coli OP50 (OD600 approximately 0.5) and 20 µ L drug solution per concentration into a 6 well plate. DMSOwas used as vehicle control and accounted for 1% of the total volume of each well. Assays were run, protected from light, at 20 · C with constant shaking at 150 rpm for 24 h. The worms were then transferred to an NGM agar plate overlaid with M9 buffer and allowed to adapt to light for 1 min before the movement was quantified under a stereo microscope by counting the number of body bends for 1 min. Each concentration was repeated at least four times independently with three worms per block (n ≥ 12 worms for each concentration). To compare the thrashes per minute of the N2 control strain and the transgenic Hco-cyp-13A11 C. elegans strain for the DMSO control, a Whitney Mann U test was used (Figure S3) (GraphPad Prism 5.0.3). For the concentration-response curves, the mean number of movements was normalized to the mean of the no-drug control of the same strain in the same block to obtain the relative motility as a percentage. For concentration-response curves, EC50 values were determined by four-parameter logistic regression using GraphPad Prism 5.0.3, and statistical differences in EC50 values were calculated using the extra sum of the squares F test. The parameters' top and bottom were constrained to values between 0 and 100%.

## 4.6. Parasite Isolates

Four isolates of H. contortus with differing susceptibility to MLs and other anthelmintics were used for Hco-cyp-13A11 (HCON\_00141052) sequence comparison.

- (i) McM(McMaster): susceptible to all anthelmintics.
- (ii) ISE (Inbred-susceptible Edinburgh; MHco3): susceptible to all anthelmintics.
- (iii) BSI (Berlin selected isolate): highly IVM, MOX, thiabendazole (TBZ), and levamisole (LEV) resistant.
- (iv) WR(White River; MHco4): highly IVM and BZ resistant; moderately LEV resistant.

All isolates have been maintained by the regular passage in helminth-naïve lambs at the Institute for Parasitology and Tropical Veterinary Medicine of the Freie Universität in Berlin for several years. The WR isolate was regularly challenged by treating the infected animals with ivermectin and fenbendazole.

## 4.7. Sequence Comparison of Hco-cyp-13A11 from Different Haemonchus contortus Isolates

According to the manufacturer's instructions, DNA was isolated from infective thirdstage larvae using the NucleoSpin Tissue kit (Macherey Nagel). To amplify exons encoding the CYP gene, primer pairs were designed at the 5 ' - and 3 ' -ends of either single exon sequences or approximately over-spanning three exon sequences (Table S2). All PCRs were conducted using Phusion Hot Start II High Fidelity DNA polymerase (Thermo Scientific). Reactions contained 0.02 U/ µ L polymerase, 0.2 mM dNTPs, 0.25 µ Mofeach primer in 20 µ L 1 × Phusion HF buffer. Reactions were denatured at 98 · C for 2 min, followed by 42 cycles of 98 · C for 20 s, a primer-pair specific annealing temperature (Table S2) for 30 s, and 72 · C for 1 min. Amplification products were cloned into the pSC-B-amp/kan vector (StrataClone Blunt Cloning Kit, Agilent Technologies). Plasmids (GenUP™Plasmid Kit, biotechrabbit) with inserts of the expected size were identified by restriction analysis, and clones with such inserts were sequenced by primer walking at LGC Genomics (Berlin). Exon sequences encoding Hco-cyp-13A11 have been deposited in the GenBank database under the accession numbers OM99853 (ISE), OM995854 (McM), OM995855 (WR), OM995856 (BSI).

## 4.8. Initial Analysis of DNA and Deduced Protein Sequences

The multiple sequence alignments were obtained by Clustal Omega version 1.2.4 (Available online: https://www.ebi.ac.uk/Tools/msa/clustalo/ accessed on 12 February 2021). The translation of Hco-cyp-13A11 protein-coding genomic DNA fragments into a polypeptide sequences was performed using the ExPASy-Translate tool (Available online: https://web.expasy.org/translate/ accessed on 18 Februry 2021). The protein parameters (molecular weight, length, theoretical isoelectric point (pI)) were calculated with ExPASy-ProtParam (Available online:https://web.expasy.org/protparam/ accessed on 19 February 2021) and ExPASy-ScanProsite (Available online: https://prosite.expasy.org/ scanprosite/ accessed on 19 February 2021) was used to detect functional motifs in the sequence. The prediction of secondary structure elements was carried out by the web tool ESPript version 3.0 (Available online: https://espript.ibcp.fr/ESPript/cgi-bin/ESPript.cgi accessed on 19 February 2021) [59] using the crystal structure of the human Cyp-3A4 (PDB: 1TQN) as a structure prediction reference [43].

## 4.9. Haemonchus contortus Cyp-13A11 Homology Modeling

Based on the crystal structure of Homo sapiens Cyp-3A4 (PDB: 6MA7) [111] as a template, 3D models of Hco-Cyp-13A11 were built using Modeller version 10.1 [51]. After pairwise alignment of each Hco-Cyp-13A11 isolate sequence with the Cyp-3A4 target sequence, 50 homology models were calculated in each Modeller run. The homology models were then individually improved based on the Modeller loop refining method by calculating 50 loop-refined models to decrease the number of Ramachandran-plot outlier residues. Subsequently, model qualities were assessed by DOPE score, the QMEAN (qualitative Model Energy Analysis) scoring function [55,56], ProSA-web (Protein structure analysis) [53,54], and Errat [112]. Next, an iron-oxo haem molecule was included by

identifying the best docking position with AutoDock Vina version 1.1.2 [63]. Therefore, the homology models were prepared using AutoDock 4 [113] by adding polar hydrogens and Kollman charges. Furthermore, the heme ligand was processed within AutoDock 4 by keeping the structure rigid and adding polar hydrogens and Gasteiger charges. The grid box used to dock the heme ligand was built manually within AutoDock 4 by centering the box to the iron coordinating cysteine sulfur (ISE, McM, BSI: Cys461; WR: Cys459). The lowest energy position calculated by AutoDock Vina was selected for each model, and interacting residues within the protein were analyzed using PyMOL version 2.0 [57]. Finally, the iron-oxo heme was connected to the homology model by creating a covalent bond between the heme iron and the sulfur atom of the previously mentioned cysteine residues using PyMOL. Heme cofactor coordinating residues were analyzed using the Pimolin plugin ShowContacts and the PLIP Protein-Ligand Interaction Profiler web tool version 2.2.0 [62]. The determination of the active site volume was performed with the web tool CASTp [114] (version 3.0, probe radius 1.4 Å).

## 4.10. Molecular Docking of Macrocyclic lactones into the Hco-Cyp-13A11 Homology Models

AutoDock Vina was used in all docking experiments with the homology models described previously. The analyses performed were restricted to molecular docking without performing an additional molecular dynamics simulation. Before docking, the hemecontaining receptor was prepared with AutoDock Tools by adding polar hydrogens and Kollman charges. Thereby, AutoDock Tools automatically set the heme iron charge to zero. In addition, water molecules present in the protein structure were removed.

The molecular structures of the MLs used for ligand docking were all downloaded from PubChem (PubChem CID: IVM (6321424); IVMa (126455999); SEL (9578507); DRM (9832750); EPR (6450531); and MOX (9832912)). In the case of IVM and EPR with occurring mixtures of B1a and B1b, only the structure of the predominant variant B1a was used for docking simulations. The 3D structures used for docking were generated with ChemDraw3D (version 20.0.0.41, level: ultra), and energy conformation was minimized performing the MMF94 (Merck molecular force field) approach. The ligands were then processed within AutoDock 4 by keeping the structures flexible and adding polar hydrogens and Kollman charges.

All MLs were primarily docked to the active site cavity of the models, which were identified using the PyMol APBS (Adaptive Poisson-Boltzmann Solver) [115] plugin calculating the macromolecular electrostatics and the CAVER3 [116]plugin computing tunnels and channels in protein structures. In the next step, the initial grid box (40 Å × 40 Å × 40 Å) used for each ligand docking was centered on the heme cofactor of each homology model. In the AutoDock Vina configuration files, the parameter num\_modes was set to 100, the exhaustiveness to 12, and energy range to 9. Based on the lowest docking energy resulting from the calculation and visual evaluation with PyMol, grid boxes were iteratively improved, and analysis with the same configuration files was performed again until obtaining the lowest (most negative) docking energies possible. If docking did not result in binding to the active site, the best docking position on the molecular surface was screened. The evaluation of interacting residues was performed by PyMol, the open-source PLIP Protein-Ligand Interaction Profiler web tool [62], and the ligand interaction tool of Maestro Elements version 4.6.117. The latter tool was also used to generate 2D ligand-protein interaction diagrams.

Supplementary Materials: The following are available online at https://www.mdpi.com/article/10 .3390/ijms23169155/s1.

Author Contributions: Conceptualization, N.J., E.Y. and J.K.; methodology, N.J. and E.Y.; software, N.J.; validation, N.J.; formal analysis, N.J.; writing-original draft preparation, N.J.; writing-review and editing, J.K.; visualization, N.J.; supervision, J.K.; project administration, J.K.; funding acquisition, J.K. All authors have read and agreed to the published version of the manuscript.

Funding: This research was funded by Deutsche Forschungsgemeinschaft (German Research Foundation, DFG) through the Research Training Group GRK 2046 'Parasite Infections: From experimental models to natural systems' (project number 251133687/GRP2046).

Institutional Review Board Statement: All animal experiments were in agreement with the European directive 2010/63/EU and the German law ('Tierschutzgesetz') and were approved by the responsible local authorities (LAGeSo Berlin) under the reference number H0337/17. Open Access Funding provided by the Freie Universität Berlin.

Informed Consent Statement: Not applicable.

Data Availability Statement: Plasmids are available upon request. The data from this study are available on request from the corresponding author.

Conflicts of Interest: The authors declare that they have no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.

## References

- 1. Prichard, R.; M é nez, C.; Lespine, A. Moxidectin and the avermectins: Consanguinity but not identity. Int. J. Parasitol. Drugs Drug Resist. 2012 , 2 , 134-153. [CrossRef]
- 2. Wolstenholme, A.J.; Kaplan, R.M. Resistance to Macrocyclic Lactones. Curr. Pharm. Biotechnol. 2012 , 13 , 873-887. [CrossRef] [PubMed]
- 3. Kaplan, R.M. Biology, Epidemiology, Diagnosis, and Management of Anthelmintic Resistance in Gastrointestinal Nematodes of Livestock. Vet. Clin. N. Am.-Food Anim. Pract. 2020 , 36 , 17-30. [PubMed]
- 4. Besier, R.; Kahn, L.; Sargison, N.; Van Wyk, J. Diagnosis, Treatment and Management of Haemonchus contortus in Small Ruminants. Adv. Parasitol. 2016 , 93 , 181-238. [CrossRef] [PubMed]
- 5. Dent, J.A.; Smith, M.M.; Vassilatis, D.K.; Avery, L. The genetics of ivermectin resistance in Caenorhabditis elegans . Proc. Natl. Acad. Sci. USA 2000 , 97 , 2674-2679. [CrossRef]
- 6. Njue, A.I.; Hayashi, J.; Kinne, L.; Feng, X.P.; Prichard, R.K. Mutations in the extracellular domains of glutamate-gated chloride channel α 3 and β subunits from ivermectin-resistant Cooperia oncophora affect agonist sensitivity. J. Neurochem. 2004 , 89 , 1137-1147. [CrossRef]
- 7. Glendinning, S.K.; Buckingham, S.D.; Sattelle, D.B.; Wonnacott, S.; Wolstenholme, A.J. Glutamate-Gated Chloride Channels of Haemonchus contortus Restore Drug Sensitivity to Ivermectin Resistant Caenorhabditis elegans . PLoS ONE 2011 , 6 , e22390. [CrossRef]
- 8. Wolstenholme, A.J. Glutamate-gated Chloride Channels. J. Biol. Chem. 2012 , 287 , 40232-40238. [CrossRef]
- 9. Urdaneta-Marquez, L.; Bae, S.H.; Janukavicius, P.; Beech, R.; Dent, J.; Prichard, R. A dyf-7 haplotype causes sensory neuron defects and is associated with macrocyclic lactone resistance worldwide in the nematode parasite Haemonchus contortus . Int. J. Parasitol. 2014 , 44 , 1063-1071. [CrossRef]
- 10. Jones, L.M.; Flemming, A.J.; Urwin, P.E. NHR-176 regulates cyp-35d1 to control hydroxylation-dependent metabolism of thiabendazole in Caenorhabditis elegans . Biochem. J. 2015 , 466 , 37-44. [CrossRef]
- 11. Mat é , L.; Ballent, M.; Cant ó n, C.; Ceballos, L.; Lifschitz, A.; Lanusse, C.; Alvarez, L.; Liron, J. Assessment of P-glycoprotein gene expression in adult stage of Haemonchus contortus in vivo exposed to ivermectin. Vet. Parasitol. 2018 , 264 , 1-7. [CrossRef] [PubMed]
- 12. Matouškov á , P.; Lecov á , L.; Laing, R.; Dimunov á , D.; Vogel, H.; Stuchl í kov á , L.; Nguyen, L.T.; Kellerov á , P.; Vokˇr á l, I.; Lamka, J.; et al. UDP-glycosyltransferase family in Haemonchus contortus : Phylogenetic analysis, constitutive expression, sex-differences and resistance-related differences. Int. J. Parasitol. Drugs Drug. Resist 2018 , 8 , 420-429. [CrossRef] [PubMed]
- 13. Matouškov á , P.; Vokˇr á l, I.; Lamka, J.; Sk á lov á , L. The Role of Xenobiotic-Metabolizing Enzymes in Anthelmintic Deactivation and Resistance in Helminths. Trends Parasitol. 2016 , 32 , 481-491. [CrossRef]
- 14. Janssen, I.J.I.; Krücken, J.; Demeler, J.; von Samson-Himmelstjerna, G. Transgenically expressed Parascaris P-glycoprotein-11 can modulate ivermectin susceptibility in Caenorhabditis elegans . Int. J. Parasitol. Drugs Drug Resist. 2015 , 5 , 44-47. [CrossRef]
- 15. Janssen, I.J.I.; Krücken, J.; Demeler, J.; von Samson-Himmelstjerna, G. Caenorhabditis elegans : Modest increase of susceptibility to ivermectin in individual P-glycoprotein loss-of-function strains. Exp. Parasitol. 2013 , 134 , 171-177. [CrossRef]
- 16. Choi, Y.-J.; Bisset, S.A.; Doyle, S.R.; Hallsworth-Pepin, K.; Martin, J.; Grant, W.N.; Mitreva, M. Genomic introgression mapping of field-derived multiple-anthelmintic resistance in Teladorsagia circumcincta . PLoS Genet. 2017 , 13 , e1006857. [CrossRef]
- 17. Godoy, P.; Lian, J.; Beech, R.N.; Prichard, R.K. Haemonchus contortus P-glycoprotein-2: In situ localisation and characterisation of macrocyclic lactone transport. Int. J. Parasitol. 2015 , 45 , 85-93. [CrossRef]
- 18. Cvilink, V.; Lamka, J.; Sk á lov á , L. Xenobiotic metabolizing enzymes and metabolism of anthelminthics in helminths. Drug Metab. Rev. 2009 , 41 , 8-26. [CrossRef]
- 19. Kellerov á , P.; Matouškov á , P.; Lamka, J.; Vokˇr á l, I.; Szot á kov á , B.; Zaj í ˇ ckov á , M.; Pas á k, M.; Sk á lov á , L. Ivermectin-induced changes in the expression of cytochromes P450 and efflux transporters in Haemonchus contortus female and male adults. Vet. Parasitol. 2019 , 273 , 24-31. [CrossRef]

- 20. M é nez, C.; Alberich, M.; Kansoh, D.; Blanchard, A.; Lespine, A. Acquired Tolerance to Ivermectin and Moxidectin after Drug Selection Pressure in the Nematode Caenorhabditis elegans. Antimicrob. Agents Chemother. 2016 , 60 , 4809-4819. [CrossRef]
- 21. Anzenbacher, P.; Anzenbacherov á , E. Cytochromes P450 and metabolism of xenobiotics. Cell Mol. Life Sci. 2001 , 58 , 737-747. [CrossRef]
- 22. Guengerich, F.P. Cytochrome P450 research and The Journal of Biological Chemistry. J. Biol. Chem. 2019 , 294 , 1671-1680. [CrossRef] [PubMed]
- 23. Zhou, S.-F. Drugs Behave as Substrates, Inhibitors and Inducers of Human Cytochrome P450 3A4. Curr. Drug Metab. 2008 , 9 , 310-322. [CrossRef] [PubMed]
- 24. Laing, R.; Bartley, D.J.; Morrison, A.A.; Rezansoff, A.; Martinelli, A.; Laing, S.T.; Gilleard, J.S. The cytochrome P450 family in the parasitic nematode Haemonchus contortus . Int. J. Parasitol. 2015 , 45 , 243-251. [CrossRef] [PubMed]
- 25. Menzel, R.; Rödel, M.; Kulas, J.; Steinberg, C.E. CYP35: Xenobiotically induced gene expression in the nematode Caenorhabditis elegans . Arch. Biochem. Biophys. 2005 , 438 , 93-102. [CrossRef] [PubMed]
- 26. Menzel, R.; Bogaert, T.; Achazi, R. A Systematic Gene Expression Screen of Caenorhabditis elegans Cytochrome P450 Genes Reveals CYP35 as Strongly Xenobiotic Inducible. Arch. Biochem. Biophys. 2001 , 395 , 158-168. [CrossRef]
- 27. Laing, R.; Kikuchi, T.; Martinelli, A.; Tsai, I.J.; Beech, R.N.; Redman, E.; Holroyd, N.; Bartley, D.J.; Beasley, H.; Britton, C.; et al. The genome and transcriptome of Haemonchus contortus , a key model parasite for drug and vaccine discovery. Genome Biol. 2013 , 14 , R88. [CrossRef]
- 28. Yilmaz, E.; Ramünke, S.; Demeler, J.; Krücken, J. Comparison of constitutive and thiabendazole-induced expression of five cytochrome P450 genes in fourth-stage larvae of Haemonchus contortus isolates with different drug susceptibility identifies one gene with high constitutive expression in a multi-resis. Int. J. Parasitol. Drugs Drug. Resist. 2017 , 7 , 362-369. [CrossRef]
- 29. Zeng, Z.; Andrew, N.W.; Woda, J.M.; Halley, B.A.; Crouch, A.L.S.; Wang, R.W. Role of Cytochrome P450 Isoforms in the Metabolism of Abamectin and Ivermectin in Rats. J. Agric. Food Chem. 1996 , 44 , 3374-3378. [CrossRef]
- 30. Zeng, Z.; Andrew, N.W.; Arison, B.H.; Luffer-Atlas, D.; Wang, R.W. Identification of cytochrome P4503A4 as the major enzyme responsible for the metabolism of ivermectin by human liver microsomes. Xenobiotica 1998 , 28 , 313-321. [CrossRef]

31.

Lee Chiu, S.-H.; Taub, R.; Sestokas, E.; Lu, A.Y.H.; Jacob, T.A. Comparative in Vivo and in Vitro Metabolism of Ivermectin in

Steers, Sheep, Swine, and Rat.

Drug Metab. Rev.

1987

,

18

, 289-302. [CrossRef] [PubMed]

- 32. Dupuy, J.; Escudero, E.; Eeckhoutte, C.; Sutra, J.; Galtier, P.; Alvinerie, M. In Vitro Metabolism of 14C-Moxidectin by Hepatic Microsomes from Various Species. Veter-Res. Commun. 2001 , 25 , 345-354. [CrossRef]
- 33. Alvinerie, M.; Dupuy, J.; Eeckhoutte, C.; Sutra, J.F.; Kerboeuf, D. In vitro metabolism of moxidectin in Haemonchus contortus adult stages. Parasitol. Res. 2001 , 87 , 702-704. [CrossRef] [PubMed]
- 34. Vokˇr á l, I.; Jedliˇckov á , V.; Jir á sko, R.; Stuchl í kov á , L.; B á rt í kov á , H.; Sk á lov á , L.; Lamka, J.; Holˇcapek, M.; Szot á kov á , B. The metabolic fate of ivermectin in host ( Ovis aries ) and parasite ( Haemonchus contortus ). Parasitology 2013 , 140 , 361-367. [CrossRef] [PubMed]
- 35. Kotze, A.C. Oxidase activities in macrocyclic-resistant and -susceptible Haemonchus contortus . J. Parasitol. 2000 , 86 , 873-876. [CrossRef]
- 36. AlGusbi, S.; Krücken, J.; Ramünke, S.; von Samson-Himmelstjerna, G.; Demeler, J. Analysis of putative inhibitors of anthelmintic resistance mechanisms in cattle gastrointestinal nematodes. Int. J. Parasitol. 2014 , 44 , 647-658. [CrossRef]
- 37. Chakrapani, B.P.S.; Kumar, S.; Subramaniam, J.R. Development and evaluation of an in vivo assay in Caenorhabditis elegans for screening of compounds for their effect on cytochrome P450 expression. J. Biosci. 2008 , 33 , 269-277. [CrossRef] [PubMed]
- 38. Kennedy, B.P.; Aamodt, E.J.; Allen, F.L.; Chung, M.A.; Heschl, M.F.; McGhee, J. The Gut Esterase Gene (ges-1) from the Nematodes Caenmorhabditis elegans and Caenorhabditis briggsae . J. Mol. Biol. 1993 , 229 , 890-908. [CrossRef]
- 39. Gerhard, A.; Krücken, J.; Neveu, C.; Charvet, C.; Harmache, A.; von Samson-Himmelstjerna, G. Pharyngeal Pumping and Tissue-Specific Transgenic P-Glycoprotein Expression Influence Macrocyclic Lactone Susceptibility in Caenorhabditis elegans . Pharmaceuticals 2021 , 14 , 153. [CrossRef]
- 40. McGhee, J. The C. elegans Intestine ; WormBook: Minneapolis, MN, USA, 2007; pp. 1-36.
- 41. Guengerich, F.P. CYTOCHROME P-450 3A4: Regulation and Role in Drug Metabolism. Annu. Rev. Pharmacol. Toxicol. 1999 , 39 , 1-17. [CrossRef]
- 42. Doyle, S.R.; Tracey, A.; Laing, R.; Holroyd, N.; Bartley, D.; Bazant, W.; Beasley, H.; Beech, R.; Britton, C.; Brooks, K.; et al. Genomic and transcriptomic variation defines the chromosome-scale assembly of Haemonchus contortus , a model gastrointestinal worm. Commun. Biol. 2020 , 3 , 656. [CrossRef]
- 43. Yano, J.K.; Wester, M.R.; Schoch, G.A.; Griffin, K.J.; Stout, C.D.; Johnson, E.F. The Structure of Human Microsomal Cytochrome P450 3A4 Determined by X-ray Crystallography to 2.05-Å Resolution. J. Biol. Chem. 2004 , 279 , 38091-38094. [CrossRef]
- 44. Samuels, E.R.; Sevrioukova, I. Structure-Activity Relationships of Rationally Designed Ritonavir Analogues: Impact of SideGroup Stereochemistry, Headgroup Spacing, and Backbone Composition on the Interaction with CYP3A4. Biochemistry 2019 , 58 , 2077-2087. [CrossRef]
- 45. Williams, P.A.; Cosme, J.; Vinkovi'c, D.M.; Ward, A.; Angove, H.C.; Day, P.J.; Vonrhein, C.; Tickle, I.J.; Jhoti, H. Crystal Structures of Human Cytochrome P450 3A4 Bound to Metyrapone and Progesterone. Science 2004 , 305 , 683-686. [CrossRef] [PubMed]
- 46. Sevrioukova, I.F.; Poulos, T.L. Pyridine-Substituted Desoxyritonavir Is a More Potent Inhibitor of Cytochrome P450 3A4 than Ritonavir. J. Med. Chem. 2013 , 56 , 3733-3741. [CrossRef] [PubMed]

- 47. Sevrioukova, I.F.; Poulos, T.L. Anion-Dependent Stimulation of CYP3A4 Monooxygenase. Biochemistry 2015 , 54 , 4083-4096. [CrossRef]
- 48. Sirim, D.; Widmann, M.; Wagner, F.; Pleiss, J. Prediction and analysis of the modular structure of cytochrome P450 monooxygenases. BMC Struct. Biol. 2010 , 10 , 34. [CrossRef] [PubMed]
- 49. Werck-Reichhart, D.; Feyereisen, R. Cytochromes P450: A success story. Genome Biol. 2000 , 1 , 1-9. [CrossRef] [PubMed]
- 50. Hasemann, C.A.; Kurumbail, R.G.; Boddupalli, S.S.; Peterson, J.A.; Deisenhofer, J. Structure and function of cytochromes P450:a comparative analysis of three crystal structures. Structure 1995 , 3 , 41-62. [CrossRef]
- 51. Webb, B.; Sali, A. Comparative protein structure modeling using MODELLER. Curr. Protoc. Bioinforma. 2016 , 2016 , 5.6.1-5.6.37.
- 52. Laskowski, R.A.; MacArthur, M.W.; Thornton, J.M. PROCHECK: Validation of protein-structure coordinates. In International Tables for Crystallography ; Springer: Berlin/Heidelberg, Germany, 2012; pp. 684-687.
- 53. Sippl, M.J. Recognition of errors in three-dimensional structures of proteins. Proteins. Proteins Struct. Funct. Bioinform. 1993 , 17 , 355-362. [CrossRef]
- 54. Wiederstein, M.; Sippl, M.J. ProSA-web: Interactive web service for the recognition of errors in three-dimensional structures of proteins. Nucleic Acids Res. 2007 , 35 (Suppl. S2), 407-410. [CrossRef] [PubMed]
- 55. Benkert, P.; Künzli, M.; Schwede, T. QMEAN server for protein model quality estimation. Nucleic Acids Res. 2009 , 37 , W510-W514. [CrossRef]
- 56. Benkert, P.; Tosatto, S.; Schomburg, D. QMEAN: A comprehensive scoring function for model quality assessment. Proteins: Struct. Funct. Bioinform. 2007 , 71 , 261-277. [CrossRef] [PubMed]
- 57. Schrödinger, L.; DeLano, W. PyMOL[Internet]. 2020. Available online: http://www.pymol.org/pymol (accessed on 1 January 2021).
- 58. Madeira, F.; Park, Y.M.; Lee, J.; Buso, N.; Gur, T.; Madhusoodanan, N.; Basutkar, P.; Tivey, A.R.N.; Potter, S.C.; Finn, R.D.; et al. The EMBL-EBI search and sequence analysis tools APIs in 2019. Nucleic Acids Res. 2019 , 47 , W636-W641. [CrossRef] [PubMed]
- 59. Robert, X.; Gouet, P. Deciphering key features in protein structures with the new ENDscript server. Nucleic Acids Res. 2014 , 42 , W320-W324. [CrossRef]
- 60. Ortiz de Montellano, P.R. Cytochrome P450: Structure, Mechanism and Biochemistry , 3rd ed.; Kluwer Academi/Plenum: New York, NY, USA, 2005.
- 61. Adasme, M.F.; Linnemann, K.L.; Bolz, S.N.; Kaiser, F.; Salentin, S.; Haupt, V.J.; Schroeder, M. PLIP 2021: Expanding the scope of the protein-ligand interaction profiler to DNA and RNA. Nucleic Acids Res. 2021 , 49 , W530-W534. [CrossRef] [PubMed]
- 62. Trott, O.; Olson, A.J. AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J. Comput. Chem. 2009 , 31 , 455-461. [CrossRef] [PubMed]
- 63. Sevrioukova, I.F.; Poulos, T.L. Understanding the mechanism of cytochrome P450 3A4: Recent advances and remaining problems. Dalton Trans. 2012 , 42 , 3116-3126. [CrossRef]
- 64. Jones, R.T.; Bakker, S.E.; Stone, D.; Shuttleworth, S.N.; Boundy, S.; McCart, C.; Daborn, P.J.; Ffrench-Constant, R.H.; Elsen, J.M.V.D. Homology modelling of Drosophila cytochrome P450 enzymes associated with insecticide resistance. Pest Manag. Sci. 2010 , 66 , 1106-1115. [CrossRef] [PubMed]
- 65. David, J.-P.; Ismail, H.M.; Chandor-Proust, A.; Paine, M.J.I. Role of cytochrome P450s in insecticide resistance: Impact on the control of mosquito-borne diseases and use of insecticides on Earth. Philos. Trans. R. Soc. B Biol. Sci. 2013 , 368 , 20120429. [CrossRef] [PubMed]
- 66. Kwa, M.S.; Veenstra, J.G.; Roos, M.H. Benzimidazole resistance in Haemonchus contortus is correlated with a conserved mutation at amino acid 200 in β -tubulin isotype 1. Mol. Biochem. Parasitol. 1994 , 63 , 299-303. [CrossRef]
- 67. Kwa, M.; Kooyman, F.; Boersema, J.; Roos, M. Effect of Selection for Benzimidazole Resistance in Haemonchus contortus on β -Tubulin Isotype 1 and Isotype 2 Genes. Biochem. Biophys. Res. Commun. 1993 , 191 , 413-419. [CrossRef]

68.

Hahnel, S.R.; Zdraljevic, S.; Rodriguez, B.C.; Zhao, Y.; McGrath, P.T.; Andersen, E.C. Extreme allelic heterogeneity at a

Caenorhab-

ditis elegans

beta-tubulin locus explains natural resistance to benzimidazoles.

PLoS Pathog.

2018

,

14

, e1007226. [CrossRef]

- 69. Robinson, M.W.; Fairweather, I.; Lawson, J.; Trudgett, A.; Hoey, E.M. The comparative metabolism of triclabendazole sulphoxide by triclabendazole-susceptible and triclabendazole-resistant Fasciola hepatica. Parasitol. Res. 2003 , 92 , 205-210. [CrossRef]
- 70. Alvarez, L.I.; Solana, H.D.; Mottier, M.L.; Virkel, G.L.; Fairweather, I.; Lanusse, C.E. Altered drug influx/efflux and enhanced metabolic activity in triclabendazole-resistant liver flukes. Parasitology 2005 , 131 , 501-510. [CrossRef] [PubMed]
- 71. Solana, H.D.; Rodriguez, J.A.; Lanusse, C.E. Comparative metabolism of albendazole and albendazole sulphoxide by different helminth parasites. Parasitol. Res. 2001 , 87 , 275-280. [CrossRef]
- 72. Stuchl í kov á , L.R.; Matouškov á , P.; Vokˇr á l, I.; Lamka, J.; Szot á kov á , B.; Seˇckaˇrov á , A.; Dimunov á , D.; Nguyen, L.T.; V á rady, M.; Sk á lov á , L. Metabolism of albendazole, ricobendazole and flubendazole in Haemonchus contortus adults: Sex differences, resistance-related differences and the identification of new metabolites. Int. J. Parasitol. Drugs Drug Resist. 2018 , 8 , 50-58. [CrossRef] [PubMed]
- 73. Laing, S.T.; Ivens, A.; Laing, R.; Ravikumar, S.; Butler, V.; Woods, D.J.; Gilleard, J.S. Characterization of the xenobiotic response of Caenorhabditis elegans to the anthelmintic drug albendazole and the identification of novel drug glucoside metabolites. Biochem. J. 2010 , 432 , 505-516. [CrossRef]
- 74. Yilmaz, E.; Gerst, B.; McKay-Demeler, J.; Krücken, J. Minimal modulation of macrocyclic lactone susceptibility in Caenorhabditis elegans following inhibition of cytochrome P450 monooxygenase activity. Exp. Parasitol. 2019 , 200 , 61-66. [CrossRef] [PubMed]

- 75. Miltsch, S.M.; Krücken, J.; Demeler, J.; Janssen, I.J.I.; Krüger, N.; Harder, A.; von Samson-Himmelstjerna, G. Decreased emodepside sensitivity in unc-49 γ -aminobutyric acid (GABA)-receptor-deficient Caenorhabditis elegans. Int. J. Parasitol. 2012 , 42 , 761-770. [CrossRef] [PubMed]
- 76. Welz, C.; Krüger, N.; Schniederjans, M.; Miltsch, S.M.; Krücken, J.; Guest, M.; Holden-Dye, L.; Harder, A.; von SamsonHimmelstjerna, G. SLO-1-Channels of Parasitic Nematodes Reconstitute Locomotor Behaviour and Emodepside Sensitivity in Caenorhabditis elegans slo-1 Loss of Function Mutants. PLoS Pathog. 2011 , 7 , e1001330. [CrossRef] [PubMed]
- 77. Kwa, M.S.; Veenstra, J.G.; Van Dijk, M.; Roos, M.H. β -Tubulin Genes from the Parasitic NematodeHaemonchus contortusModulate Drug Resistance in Caenorhabditis elegans . J. Mol. Biol. 1995 , 246 , 500-510. [CrossRef] [PubMed]
- 78. Doyle, S.R.; Illingworth, C.J.R.; Laing, R.; Bartley, D.J.; Redman, E.; Martinelli, A.; Holroyd, N.; Morrison, A.A.; Rezansoff, A.; Tracey, A.; et al. Population genomic and evolutionary modelling analyses reveal a single major QTL for ivermectin drug resistance in the pathogenic nematode, Haemonchus contortus. BMC Genom. 2019 , 20 , 218. [CrossRef]
- 79. Yang, Y.-F.; Zhang, X.; Ma, X.; Zhao, T.; Sun, Q.; Huan, Q.; Wu, S.; Du, Z.; Qian, W. Trans -splicing enhances translational efficiency in C. elegans . Genome Res. 2017 , 27 , 1525-1535. [CrossRef]
- 80. Kashyap, S.S.; Verma, S.; Voronin, D.; Lustigman, S.; Kulke, D.; Robertson, A.P.; Martin, R.J. Emodepside has sex-dependent immobilizing effects on adult Brugia malayi due to a differentially spliced binding pocket in the RCK1 region of the SLO-1 K channel. PLoS Pathog. 2019 , 15 , e1008041. [CrossRef]
- 81. Fauvin, A.; Charvet, C.; Issouf, M.; Cortet, J.; Cabaret, J.; Neveu, C. cDNA-AFLP analysis in levamisole-resistant Haemonchus contortus reveals alternative splicing in a nicotinic acetylcholine receptor subunit. Mol. Biochem. Parasitol. 2010 , 170 , 105-107. [CrossRef]
- 82. Neveu, C.; Charvet, C.L.; Fauvin, A.; Cortet, J.; Beech, R.N.; Cabaret, J. Genetic diversity of levamisole receptor subunits in parasitic nematode species and abbreviated transcripts associated with resistance. Pharm. Genom. 2010 , 20 , 414-425. [CrossRef]
- 83. Keller, J.; Ellieva, A.; Ma, D.K.; Ju, J.; Nehk, E.; Konkel, A.; Falck, J.R.; Schunck, W.-H.; Menzel, R. CYP-13A12 of the nematode Caenorhabditis elegans is a PUFA-epoxygenase involved in behavioural response to reoxygenation. Biochem. J. 2014 , 464
- , 61-71. [CrossRef]
- 84. Bygarski, E.E.; Prichard, R.K.; Ardelli, B.F. Resistance to the macrocyclic lactone moxidectin is mediated in part by membrane transporter P-glycoproteins: Implications for control of drug resistant parasitic nematodes. Int. J. Parasitol. Drugs Drug Resist. 2014 , 4 , 143-151. [CrossRef]
- 85. Ardelli, B.F.; Stitt, L.E.; Tompkins, J.B.; Prichard, R.K. A comparison of the effects of ivermectin and moxidectin on the nematode Caenorhabditis elegans . Veter-Parasitol. 2009 , 165 , 96-108. [CrossRef] [PubMed]
- 86. Kaschny, M.; Demeler, J.; Janssen, I.J.I.; Kuzmina, T.A.; Besognet, B.; Kanellos, T.; Kerboeuf, D.; von Samson-Himmelstjerna, G.; Krücken, J. Macrocyclic Lactones Differ in Interaction with Recombinant P-Glycoprotein 9 of the Parasitic Nematode Cylicocylus elongatus and Ketoconazole in a Yeast Growth Assay. PLoS Pathog. 2015 , 11 , e1004781. [CrossRef] [PubMed]
- 87. Forrester, S.G.; Prichard, R.K.; Beech, R. NRN. A glutamate-gated chloride channel subunit from Haemonchus contortus : Expression in a mammalian cell line, ligand binding, and modulation of anthelmintic binding by glutamate. Biochem. Pharmacol. 2002 , 63 , 1061-1068. [CrossRef]
- 88. Ingelman-Sundberg, M. Pharmacogenetics of cytochrome P450 and its applications in drug therapy: The past, present and future. Trends Pharmacol. Sci. 2004 , 25 , 193-200. [CrossRef] [PubMed]
- 89. Hibbs, R.E.; Gouaux, E. Principles of activation and permeation in an anion-selective Cys-loop receptor. Nature 2011 , 474 , 54-60. [CrossRef] [PubMed]
- 90. Rodriguez-Antona, C.; Ingelman-Sundberg, M. Cytochrome P450 pharmacogenetics and cancer. Oncogene 2006 , 25 , 1679-1691. [CrossRef]
- 91. Blouin, M.S.; Yowell, C.A.; Courtney, C.H.; Dame, J.B. Host movement and the genetic structure of populations of parasitic nematodes. Genetics 1995 , 141 , 1007-1014. [CrossRef]
- 92. Gilleard, J.; Redman, E. Genetic Diversity and Population Structure of Haemonchus contortus ; Advances in Parasitology; Elsevier Ltd.: Amsterdam, The Netherlands, 2016; Volume 93, pp. 31-68. [CrossRef]
- 93. Gill, J.; Lacey, E. Avermectin \ milbemycin resistance in trichostrongyloid nematodes. Int. J. Parasitol. 1998 , 28 , 863-877. [CrossRef]
- 94. Gill, J.H.; Kerr, C.A.; Shoop, W.L.; Lacey, E. Evidence of multiple mechanisms of avermectin resistance in Haemonchus contortus -Comparison of selection protocols. Int. J. Parasitol. 1998 , 28 , 783-789. [CrossRef]
- 95. Šrejber, M.; Navr á tilov á , V.; Palonc ý ov á , M.; Bazgier, V.; Berka, K.; Anzenbacher, P.; Otyepka, M. Membrane-attached mammalian cytochromes P450: An overview of the membrane's effects on structure, drug binding, and interactions with redox partners. J. Inorg. Biochem. 2018 , 183 , 117-136. [CrossRef]
- 96. Oezguen, N.; Kumar, S. Analysis of Cytochrome P450 Conserved Sequence Motifs between Helices E and H: Prediction of Critical Motifs and Residues in Enzyme Functions. J. Drug Metab. Toxicol. 2011 , 2 , 1000110. [CrossRef] [PubMed]
- 97. Gora, A.; Brezovsky, J.; Damborsky, J. Gates of Enzymes. Chem. Rev. 2013 , 113 , 5871-5923. [CrossRef] [PubMed]
- 98. Rendic, S. Summary of information on human CYP enzymes: Human P450 metabolism data. Drug Metab. Rev. 2002 , 34 , 83-448. [CrossRef] [PubMed]
- 99. Stout, S.J.; Dacunha, A.R.; Wu, S.S.; Zulalian, J.; Afzal, J. Moxidectin: Characterization of Cattle, Sheep, and Rat in vitro and in vivo Metabolites by Liquid Chromatography/Tandem Mass Spectrometry. J. Agric. Food Chem. 1994 , 42 , 388-392. [CrossRef]

- 100. Afzal, J.; Burke, A.B.; Batten, P.L.; DeLay, R.L.; Miller, P. Moxidectin: Metabolic Fate and Blood Pharmacokinetics of 14C-Labeled Moxidectin in Horses. J. Agric. Food Chem. 1997 , 45 , 3627-3633. [CrossRef]
- 101. Ekroos, M.; Sjögren, T. Structural basis for ligand promiscuity in cytochrome P450 3A4. Proc. Natl. Acad. Sci. USA 2006 , 103 , 13682-13687. [CrossRef]
- 102. Redman, E.; Sargison, N.; Whitelaw, F.; Jackson, F.; Morrison, A.; Bartley, D.J.; Gilleard, J.S. Introgression of Ivermectin Resistance Genes into a Susceptible Haemonchus contortus Strain by Multiple Backcrossing. PLoS Pathog. 2012 , 8 , e1002534. [CrossRef]
- 103. Kotze, A.C.; Hunt, P.W.; Skuce, P.; von Samson-Himmelstjerna, G.; Martin, R.J.; Sager, H.; Krücken, J.; Hodgkinson, J.; Lespine, A.; Jex, A.R.; et al. Recent advances in candidate-gene and whole-genome approaches to the discovery of anthelmintic resistance markers and the description of drug/receptor interactions. Int. J. Parasitol. Drugs Drug Resist. 2014 , 4 , 164-184. [CrossRef]
- 104. Gerhard, A.P.; Krücken, J.; Heitlinger, E.; Janssen, I.J.I.; Basiaga, M.; Korna's, S.; Beier, C.; Nielsen, M.K.; Davis, R.E.; Wang, J.; et al. The P-glycoprotein repertoire of the equine parasitic nematode Parascaris univalens. Sci. Rep. 2020 , 10 , 13586. [CrossRef]
- 105. Godoy, P.; Che, H.; Beech, R.N.; Prichard, R.K. Characterisation of P-glycoprotein-9.1 in Haemonchus contortus. Parasites Vectors 2016 , 9 , 52. [CrossRef]
- 106. David, M.; Lebrun, C.; Duguet, T.; Talmont, F.; Beech, R.; Orlowski, S.; Andr é , F.; Prichard, R.K.; Lespine, A. Structural model, functional modulation by ivermectin and tissue localization of Haemonchus contortus P-glycoprotein-13. Int. J. Parasitol. Drugs Drug Resist. 2018 , 8 , 145-157. [CrossRef] [PubMed]
- 107. David, M.A.; Orlowski, S.; Prichard, R.K.; Hashem, S.; Andr é , F.; Lespine, A. In silico analysis of the binding of anthelmintics to Caenorhabditis elegans P-glycoprotein 1. Int. J. Parasitol. Drugs Drug Resist. 2016 , 6 , 299-313. [CrossRef] [PubMed]
- 108. Kirn, R.B.; Wandel, C.; Leake, B.; Cvetkovic, M.; Fromm, M.F.; Dempsey, P.J.; Roden, M.M.; Belas, F.; Chaudhary, A.K.; Roden, D.M.; et al. Interrelationship Between Substrates and Inhibitors of Human CYP3A and P-Glycoprotein. Pharm. Res. 1999 , 16 , 408-414. [CrossRef] [PubMed]
- 109. Kurnik, D.; Wood, A.J.J.; Wilkinson, G.R. The erythromycin breath test reflects P-glycoprotein function independently of cytochrome P450 3A activity. Clin. Pharmacol. Ther. 2006 , 80 , 228-234. [CrossRef]
- 110. Stiernagle, T. Maintenance of C. elegans ; WormBook: Minneapolis, MN, USA, 2006; pp. 1-11.
- 111. Sevrioukova, I. Interaction of Human Drug-Metabolizing CYP3A4 with Small Inhibitory Molecules. Biochemistry 2019 , 58 , 930-939. [CrossRef] [PubMed]
- 112. Colovos, C.; Yeates, T.O. Verification of protein structures: Patterns of nonbonded atomic interactions. Protein Sci. 1993 , 2 , 1511-1519. [CrossRef]
- 113. Morris, G.M.; Ruth, H.; Lindstrom, W.; Sanner, M.F.; Belew, R.K.; Goodsell, D.S.; Olson, A.J. Software news and updates AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility. J. Comput. Chem. 2009 , 30 , 2785-2791. [CrossRef]
- 114. Tian, W.; Chen, C.; Lei, X.; Zhao, J.; Liang, J. CASTp 3.0: Computed atlas of surface topography of proteins. Nucleic Acids Res. 2018 , 46 , W363-W367. [CrossRef]
- 115. Jurrus, E.; Engel, D.; Star, K.; Monson, K.; Brandi, J.; Felberg, L.E.; Brookes, D.H.; Wilson, L.; Chen, J.; Liles, K.; et al. Improvements to the APBS biomolecular solvation software suite. Protein Sci. 2018 , 27 , 112-128. [CrossRef]
- 116. Pavelka, A.; Sebestova, E.; Kozlikova, B.; Brezovsky, J.; Sochor, J.; Damborsky, J. CAVER: Algorithms for Analyzing Dynamics of Tunnels in Macromolecules. IEEE/ACM Trans. Comput. Biol. Bioinform. 2016 , 13 , 505-517. [CrossRef]