<!-- image -->

## RESEARCH ARTICLE

<!-- image -->

## Mixed and non-competitive enzyme inhibition: underlying mechanisms and mechanistic irrelevance of the formal two-site model

## Alessandro Pesaresi

Istituto di Cristallografia -Consiglio Nazionale delle Ricerche, Trieste, Italy

## ABSTRACT

The formal mechanism of linear mixed and non-competitive enzyme inhibition implies the binding of inhibitors to both the active site of the free enzyme in competition with the substrate, and to an allosteric site on the enzyme-substrate complex. However, it is evident from a review of the scientific literature that the two-site mechanism is frequently mistaken as the actual underlying mechanism of mixed inhibition. In this study, we conducted a comprehensive assessment of the mechanistic relevance of this type of inhibition using a statistical approach. By combining a statistical analysis of the inhibition cases documented in the BRENDA database with a theoretical investigation of inhibition models, we conclude that mixed inhibitors exclusively bind to the active site of enzymes. Hence ruling out any implication of allosteric sites and depriving the two-site model of any mechanistic relevance.

## Introduction

Prompted to ' write a definition of mixed-type enzyme inhibition ' , ChatGPT elaborated the following composition:

Mixed-type enzyme inhibition is a form of enzyme inhibition where the inhibitor can bind to both the free enzyme and the enzyme-substrate complex, resulting in distinct effects on both the enzyme ' s catalytic activity and its affinity for the substrate. In mixed-type inhibition, the inhibitor can interact with the enzyme at either the active or a separate allosteric site. [ … ]

Mixed-type enzyme inhibition is a complex regulatory mechanism that can have important implications in various biological processes and the development of therapeutic strategies targeting enzyme activity. Understanding the nature of mixed-type inhibition is crucial for studying enzyme kinetics, drug discovery, and the design of effective enzyme inhibitors.

Because the algorithms of ChatGPT have been trained using a massive amount of text data, the response it provided can be considered as a representative description of mixed inhibition that aligns with the general understanding presented in articles, books and other scientific publications.

Also, it seems safe to assume that most biochemists, drug designers and, in general, most of those who concern themselves with enzyme inhibition, would substantially agree with this definition.

Yet, the above statement features a misleading inaccuracy: it is not true that mixed inhibitors necessarily bind to topologically distinct sites on the free enzyme and the enzyme-substrate complex. In defence of ChatGTP, it must be said that this error reflects a very common misconception about the significance of kinetic mechanisms. Reversible inhibition, in fact, is classified as competitive, mixed or uncompetitive, on the basis of the effect that the inhibitor exerts on Kcat and Kcat / Km , which classically is measured

CONTACT

<!-- image -->

/C223

Alessandro Pesaresi

alessandro.pesaresi@ic.cnr.it

Istituto di Cristallografia

-

Consiglio Nazionale delle Ricerche, Trieste, Italy

<!-- image -->

<!-- image -->

Supplemental data for this article can be accessed online at https://doi.org/10.1080/14756366.2023.2245168.

2023 The Author(s). Published by Informa UK Limited, trading as Taylor & Francis Group.

This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. The terms on which this article has been published allow the posting of the Accepted Manuscript in a repository by the author(s) or with their consent.

## ARTICLE HISTORY

Received 13 June 2023

Revised 14 July 2023

Accepted 1 August 2023

## KEYWORDS

Mixed inhibition; noncompetitive inhibition; inhibition mechanism

as an intercept and/or a slope effect of a double-reciprocal plot, respectively (Figure 1). One aspect that is often overlooked however, is that there is no bijective relation between the type of inhibition, whose definition is only operational, and the underlying molecular mechanisms 1 . The steady-state analysis of initial velocities can never establish a molecular mechanism beyond doubt, therefore, it is critical to point out that the three formal mechanisms of inhibition (Figure 1(A)) only represent the simplest mechanisms that are consistent with the experimental observations 1 .

This means that the type of inhibition cannot be taken as a sufficient proof of any given molecular mechanism. For example, although it is true that competitive inhibition is generally caused by inhibitors that, in agreement with the conventional mechanism (Figure 1(A)) bind the active site in competition with the substrate, exceptions where competition results from the binding of allosteric sites have also been reported 2,3 . In contrast, only on rare occasions do uncompetitive inhibitors actually bind the ES complex 4,5 . The most common sources of uncompetitive inhibition being, by far, reactions with more than one substrate with compulsory-order mechanisms 6,7 where the inhibitor binds to the binding site of the constant substrate.

With mixed and pure non-competitive inhibition the picture is even more complex and tricky: the conventional mechanism, reported in all textbooks and acknowledged by ChatGPT (Figure 1(A)), implies the unlikely event of one inhibitor binding, with somewhat similar affinity, to two distinct sites, i.e. the active site and an allosteric site of the ES complex.

Seasoned enzymologists are well aware that this two-site model is mainly just a convenient formal way to define mixed inhibition and that it should not be intended as a real molecular mechanism.

Figure 1. Inhibition mechanisms and patterns. (A) Formal inhibition mechanisms. Competitive inhibition (a), mixed-type inhibition (b), uncompetitive inhibition (c). (B) . Inhibition pattern of competitive and mixed-type inhibition. In a 1/V versus 1/[S] plot of initial rates, competitive inhibitors generate a bundle of lines (green) that intercept the non-inhibited reaction trace (blue) on the 1/V axis. Mixed-type inhibitors reduce both 1/V and V/KM by binding the free enzyme (E) and the ES complex. They generate bundles of lines that intercept on the left of the 1/V axis at 1/[S] values equal to KIC / KIU . For pure non-competitive inhibitors ( KIC ¼ KIU , orange lines), the intercept is on the 1/[S] axis. If KIC < KIU , (red lines, KIC / KIU ¼ 0.5), the intercept is above the 1/[S] axis, and it is below if KIC > KIU , (yellow lines, KIC / KIU ¼ 2).

<!-- image -->

Nevertheless, general biochemistry textbooks usually do not go beyond this description, ingraining the wrong belief that mixed and non-competitive inhibitors actually act by binding the active site and a topologically distinct allosteric site. Sometimes, even educational research papers 8,9 and enzymology manuals 10,11 indulge in the same oversimplification.

Resorting to authoritative sources certainly reduces the risk of misapprehension, but for those who do not have a strong background in enzymology, attaining a proper understanding of mixed inhibition and its molecular determinants remains arduous. For example, Cornish-Bowden in ' Fundamental of Enzyme Kinetics ' clearly explains the limits of the formal two-site mechanism and suggests that mixed inhibition occurs mainly as a case of product inhibition with iso-mechanism enzymes 12 . Johnson, in ' Kinetic Analysis for the New Enzymology ' , adds that mixed inhibition, analogous to uncompetitive inhibition, occurs mainly with multisubstrate reactions, either as a form of product inhibition or, for ternary-complex mechanisms, when a dead-end inhibitor binds before the variable substrate. 7 Copeland instead, in ' Evaluation of Enzyme Inhibition in Drug Discovery ' , mentions five mechanisms through which active site-directed inhibitors can cause inhibition patterns typical of mixed inhibition. 3 However, these five mechanisms are treated as exceptions to a general rule. With the general rule being, again, the formal two-site mechanism.

As this brief example shows, the picture that emerges is so complicated and confusing that, as a matter of fact, many authors just rely on the widely acknowledged idea that the inhibition is of the mixedtype if, and only if, the inhibitor binds both the active site of the free enzyme and an allosteric site of the enzyme-substrate complex.

We conducted a brief survey by searching Google Scholar with the query ' mixed inhibition mechanism ' . It turned out that in 16 out of the first 30 papers that report cases of mixed inhibition, the authors explicitly assumed the two-site model as the actual underlying mechanism 13 -28 . It is worth noting that this misconception is common in medium-low tier journals as well as in high impact ones 29 -33 .

Often, especially in drug design projects, the ultimate objective of kinetic analysis is to elucidate a molecular mechanism for the obvious reason that it can help to rationalise the improvement of lead compounds. The belief that mixed inhibition implies the binding

of an active and an allosteric site is so deeply enrooted, however, that many researchers just take it as consolidated truth with no need for further investigations. On the other hand, establishing the actual inhibition mechanism would require thorough biochemical or biophysical studies that are not always within the scope of those works. Therefore, there is a question, of fundamental importance for medicinal chemists and drug designers, that still stands: to what extent, if any, does the two-site mechanism account for the reported cases of mixed-type and non-competitive inhibition?

Here, we try to provide an answer to that one question employing a statistical approach: forty years ago, Cornish-Bowden noted that uncompetitive inhibition almost always occurs in the form of mixed inhibition and that in mixed inhibition, the uncompetitive component never prevails over the competitive counterpart (i.e. Kic /C20 Kiu ), 34 . This intriguing anomaly - it would imply that mixed inhibitors bind the active site always more effectively than the allosteric site - evidently do not find explanation in the framework of the two-site mechanism. Indeed, there are five main mechanisms, or circumstances, by which an active site-bound inhibitor might appear to be mixed-type or non-competitive when analysed by steady-state kinetics (four of which have been revised in reference 35 ) In this study, we demonstrated that in the mixed inhibition caused by these five mixed inhibition-mimicking (MIM) mechanisms, Kic is necessarily always less than or equal to Kiu .

The combination of this latter finding with the results of a statistical analysis, which confirmed the overwhelming preponderance of the competitive component of mixed inhibition, strongly suggests that the five MIM mechanisms are the actual underlying mechanisms of mixed-type inhibition, and hence, that, with virtually no exception, mixed inhibitors bind only the substrate site of the free enzyme. This excludes the implication of any interaction with any allosteric site and, in conclusion, points to a substantial mechanistic irrelevance of the formal two-site model of mixed inhibition.

## Methods

## Simulation of mixed inhibition-mimicking mechanisms

To simulate MIM mechanisms, initial velocities were generated using KinTek Explorer 36,37 , and the inhibition constants Kic and Kiu were determined by the Dixon and Cornish-Bowden plots,

respectively. Velocities were detected 60 s after the start of the reactions. For each mechanism, the reaction scheme, kinetic constants and concentration of reagents are given below.

## Case I -tight binding inhibitors

<!-- image -->

k1 ¼ 2 /C1 10 7 mol /C0 1 s /C0 1 , k-1 ¼ 50s /C0 1 , k2 ¼ 50s /C0 1 . Kic ¼ 0.01 -10nM. [E] ¼ 1nM, [S] ¼ 10 -400 m M, [I] ¼ 0 -10 nM.

## Case II -time dependent inhibition

<!-- image -->

k1 ¼ 1 /C1 10 8 mol /C0 1 s /C0 1 , k-1 ¼ 1s /C0 1 , k2 ¼ 1000s /C0 1 , koff ¼ 10 /C0 5 -10 3 s /C0 1 , koff / kon ¼ 10 /C0 6 M. [E] ¼ 100pM, [S] ¼ 1 -200 m M, [I] ¼ 0.5 -50 m M.

## Case III -Multisubstrate reactions

<!-- image -->

k1 ¼ 1 /C1 10 8 mol /C0 1 s /C0 1 , k-1 ¼ 2500s /C0 1 , k2 ¼ 1 /C1 10 8 mol /C0 1 s /C0 1 , k-2 ¼ 400s /C0 1 , and k2 ¼ 100s /C0 1 . [E] ¼ 1nM, [A] ¼ 0.1 -1000 m M, [B] ¼ 2.5 -200 m M, [I] ¼ 0 -10nM. Ks ¼ 25 m M, KM(A) ¼ 1 m M, KM(B) ¼ 5 m M, Ki ¼ 10 nM, kca t ¼ 100s /C0 1 ,

Kcat/k-1 ¼ 25.

## Case IV -iso-mechanisms

<!-- image -->

[E] ¼ 0.1 nM, [S] ¼ 2 -100 m M, [I] ¼ 0 -100nM, Kd ¼ 10nM. k1 ¼ 10 8 mol /C0 1 s /C0 1 , k-1 ¼ 100 s /C0 1 , k3 ¼ 100 mol /C0 1 s /C0 1 , k-3 ¼ 10 8 s /C0 1 . kinetic constant val-

The combinations of the k2 , k-2 , k4 and k-4 ues used to simulate the different Fiso values are given in Table 1.

## Case V -exo-site

<!-- image -->

[E] ¼ 0.1 nM, [S] ¼ 0.2 -100 m M, [I] ¼ 0 -100nM, Kd ¼ 10 nM. k1 ¼ 10 8 mol /C0 1 s /C0 1 , k-1 ¼ 1s /C0 1 , k2 ¼ 10 6 -50s /C0 1 , k-2 ¼ 1s /C0 1 , k3 ¼ 50 -10 6 s /C0 1 .

Table 1. Combination of the k2 , k-2 , k4 and k-4 constants used to obtain the different Fiso values.

| Fiso              | 0    |   0.1 |   0.2 |   0.5 |   0.7 | 1    |
|-------------------|------|-------|-------|-------|-------|------|
| k2 (mol -1 s -1 ) | 100  |   111 |   125 |   200 |   333 | 10 6 |
| k-2 (s -1 )       | 100  |   111 |   125 |   200 |   333 | 10 6 |
| k4 (mol -1 s -1 ) | 10 6 |  1000 |   500 |   200 |   143 | 100  |
| k-4 (s -1 )       | 10 6 |  1000 |   500 |   200 |   143 | 100  |

Figure 2. Distribution of the Kiu -Kic ratio frequencies of mixed-type inhibitors. For inhibition dominated by the competitive component ( Kiu > Kic , red bars), the data are presented as Kiu / Kic . Data of mixed-inhibitions with Kic > Kiu (blue bars) are presented as Kic / Kiu ratio. Pure non-competitive inhibition (PNC) is shown in green. The competitive component is prevalent in 70% of cases, pure non-competitive inhibition represents 20% of cases, while the dominance of the uncompetitive component is observed only in 10% of cases. The total number of cases analysed was 467.

<!-- image -->

## Results

## Statistics of enzyme inhibition

The quantitative statistical analysis of the available kinetic data is a precondition to turn anecdotal evidence into more solid objective observations and hence to draw conclusions on the significance of mixed-inhibition. Unfortunately, none of the extant repositories stores information about the mechanism of inhibition in a structured and systematic way. In the BRENDA database 38 , for example, which contains entries for more than 220,000 enzyme inhibitors and 200,000 reference citations, this kind of information has only occasionally been reported by the database curators and can be found as scattered notes in the commentary fields of a minority of the entries. Because these entry notes are not searchable, to build a reliable statistic, the relevant data were mined from the downloadable BRENDA text file. A search in this almost five million-line document of the terms ' competitive ' , ' noncompetitive ' , ' mixed ' and ' uncompetitive ' inhibitors returned a total of 7563 hits. On the basis of the occurrence of each term and in agreement with what had already been reported 29 , competitive inhibition was the most common type of inhibition, accounting for 73% of total cases. Mixed/non-competitive and uncompetitive inhibition accounted for 22% and 5% of cases, respectively. To deepen the ' Kic versus Kiu issue ' , the 464 entries containing the term ' mixed ' or ' mixed-type ' inhibition/inhibitor were further analysed. These entries point to 266 reference articles, from which it has been possible to retrieve 467 Kic -Kiu couples (Figure 2). If mixed inhibition was caused by the binding of inhibitors to two distinct sites, the dominance of the competitive or of the uncompetitive component would be equally probable, and a symmetric and rather

Figure 3. Simulation of MIM mechanisms of type 1 and 2. (A) . Tight inhibition. ( a ) Double reciprocal plot of the uninhibited reaction (blue trace) and of reactions inhibited by four competitive inhibitors with different Ki and at different concentrations so that the [I]/ Ki ratio is the same for all reactions. As expected for an active site-bound inhibitor, as long as the inhibitor concentration is at least /C24 10 times larger than the enzyme concentration (for green trace [I]/[E] ¼ 10), the inhibition looks competitive. If [I] drops below 10 /C1 [E], the intercept shifts towards the first quadrant of the plot, mimicking a mixed-type inhibition whose Kiu / Kic ratio asymptotically tends to 1 as the inhibitor-to-enzyme ratio decreases (b). ( B ). Slow-dissociating inhibitors. For reactions initiated after a long enzyme-inhibitor incubation, the observed initial velocities depend critically on the dissociation rate constant of the inhibitor ( koff ). In the presence of active site-bound inhibitors, this mechanism generates an apparent mixed inhibition with Kiu / Kic, which asymptotically tends to 1 as the dissociation slows.

<!-- image -->

flat distribution of the Kic / Kiu and Kiu / Kic ratios around a central value of 1 would be expected. The analysis of these Kic -Kiu values, however, showed that the dominance of the competitive component is overwhelmingly preponderant ( Kic /C20 Kiu in 90% of the cases) and that pure non-competitive inhibition ( Kic ¼ Kiu ) occurs with an unrealistically high frequency ( Kic / Kiu ¼ 1in20%ofcases),(Figure 2).

It is worth noting that the large majority of the gathered Ki s were originally obtained by the graphical analysis of initial velocity data, a method that is known to be rather inaccurate 39,40 .Ifto account for this inaccuracy some allowance is granted and inhibition is considered to be pure non-competitive up to a Kic / Kiu ratio between 0.5 and 2 (rather than exactly equal to 1), then the frequency of pure non-competitive inhibition jumps to 40%, while the occurrence of mixed inhibition cases with Kic > Kiu drops to a mere 5%. Interesting to note is that 66 % of the 467 Kic -Kiu couples were relative to multi-substrate enzymes.

## Investigation into the mixed inhibition-mimicking mechanisms

In this section, we sought to demonstrate whether MIM mechanisms cause mixed inhibition with Kic sthatarenecessarilyalwayslessthan, or equal to, Kiu s. We first derived a mathematical relation between Kic and Kiu for each of the five mechanisms (for the sake of brevity this part was moved in the Supplemental Material). Then, to verify the correctness of the derived equations, MIM mechanisms were simulated using KinTek Explorer 36,37 . To determine Kic sand Kiu s, as in a real experiment, the computed initial velocities were analysed by the conventional double-reciprocal plot of 1/V vs 1/[S].

## Case I -tight-binding inhibitors

To determine the Ki of an inhibitor, reactions must be measured in the presence of inhibitor concentrations close to the Ki . This

implies that for very potent inhibitors, typically with Ki in the nM range or less, the inhibitor might turn out to be not in concentration higher enough than the enzyme, hence violating the first of the two steady-state assumptions. Methods to analyse tight inhibition have been proposed 41,42 , but if this circumstance is overlooked and kinetic data are analysed according to the classic steady-state method, active-site directed inhibitors can easily be mistaken for mixed-type inhibitors 43,44 .

Figure 3(A) shows the inhibition caused by competitive inhibitors of different potencies supplied at concentrations between 10 /C1 [E] and 0.1 /C1 [E]. In the simulation, for all inhibitors, the [I]/ Ki ratio was set to 10. Therefore, in principle, for all of the reactions, the degree of inhibitionshouldbethesame.However,asthe[I]/[E]ratiodropsbelow 10, the inhibitor depletion that occurs upon E-I binding can no longer be neglected, and the fraction of the inactive inhibitor-bound enzyme starts to deviate from the expected linear behaviour. In the doublereciprocalplot,thisisvisualisedasbothaslopeandanintercept effect that shifts the point of line intersection towards the first quadrant, thus mimicking a mixed inhibition pattern. The apparent Kiu , which evidently emerges as a mere artefact, only depends on the [I]/[E] ratio: the smaller the ratio is, the smaller the Kiu . As [I] tends to 0, however, the inhibition fades off, and the trace of the inhibited reaction tends to overlap with that of the uninhibited reaction. Therefore, under no circumstance, the intersection point can be below the 1/[S] axis, and hence, the apparent Kiu turns out to be necessarily greater than, or at best equal to, Kic .

## Case II -time-dependent inhibition

The steady-state model assumes that the initial velocities are a direct measure of the steady-state rates. In the case of slow-binding inhibitors, however, the onset of the inhibition is delayed, with the consequence that initial velocities underestimate the actual

Scheme 1. Mechanism of ternary-complex reaction with compulsory order.

<!-- image -->

inhibition 45 . A common strategy to tackle slow inhibition is to preincubatetheenzymeandinhibitorandtoaddthesubstrateasthelast reactant so that the enzyme-inhibitor equilibrium is attained before the reaction is started. In this case, however, if the enzyme-inhibitor dissociation is slow, the initial velocities underestimate the steady-state turnover, and the conventional steady-state analysis causes competitive inhibitors to be mistaken for mixed-type or non-competitive. Although the effects of slow inhibitor dissociation on substrate-initiated reactions have long been described 45 , the mechanism leading to the emergence of an apparent uncompetitive inhibition component, to the best of our knowledge, has explicitly been elucidated only very recently 46 . As in all cases where the steady-state assumptions are not met, the output of the conventional data analysis is largely dependent on the experimental setup. In this specific case, the extent of the fictitious uncompetitive component strictly depends on the enzymeinhibitor dissociation rate and on the dead time for the detection of initial velocities. Readers interested in deepening the mechanistic aspect of this kinetic artefact are referred to the cited literature 46 . Here, it suffices to state that the apparent Kiu asymptotically tends to Kic as the enzyme-inhibitor dissociation gets slower, eventually mimicking pure non-competitive inhibition ( Kic ¼ Kiu )when koff is less than /C24 1 0 /C0 3 -10 /C0 4 s /C0 1 (Figure 3(B)), so that necessarily Kiu /C21 Kic .

## Case III -Multisubstrate reactions

The Michaelis -Menten model was derived for single substrate reactions, although these are quite rare in biochemistry and are confined to a few isomerizations. It has been evaluated that some 60% of all the known enzyme-catalyzed reactions involve the conversion of two substrates into two products according to the equation A þ B ¼ P þ Q 6 . For such reactions, referred to as Bi-Bi reactions, there are three common mechanisms: the substituted-enzyme (or ping-pong) mechanism and the ternary-complex mechanism, either with compulsory or random order of substrate binding. Depending on the actual mechanism, the steady-state analysis of a dead-end inhibitor directed against the binding site of substrate A and assayed against substrate B (i.e. at varying B concentrations and at fixed A) can give mixed-type, pure non-competitive or uncompetitive inhibition. This is a very well-established fact 47 -49 that likely constitutes the most common source of non competitive inhibition. Here, we sought to demonstrate whether in this mixed inhibition case too, the uncompetitive inhibition constant, K app IU ð B Þ , is necessarily always greater than or equal to the competitive constant, K app IC ð B Þ :

For a ternary-complex reaction with compulsory order of binding, as depicted in Scheme 1, it can be shown (see Supplemental Material)that K app IC ð B Þ and K app IU ð B Þ depend on the concentration of substrate A (i.e. the constant substrate) according to the following rules:

K app IC B ðÞ¼ Kic þ A ½/C138 Ks Kic K app IU B ðÞ¼ Kic þ A ½/C138 K M ð A Þ Kic

where Kic is the true dissociation constant for the enzyme-inhibitor complex (i.e. the competitive inhibition constant measured when substrate B is constant and A is variable), Ks is the dissociation constant of the enzyme-substrate A complex and KM(A) is the

Michaelis constant for substrate A. At any given concentration of substrate A, the ratio between the apparent uncompetitive and competitive components is:

K app IU B ðÞ K app IC B ðÞ ¼ Kic þ A ½/C138 K M ð A Þ Kic Kic þ A ½/C138 KS Kic

which asymptotically tends to 1 as the concentration of substrate A tends to 0 and to Ks / KM(A ) as [A] tends to infinity. This K app IU B ðÞ = K app IC B ðÞ ratio dependence on [A] was confirmed by steadystate analysis of the KinTek Explorer simulations (Figure 4(A)). By definition, Ks / KM(A ) is equal to kcat / k-1 , which for a compulsory order mechanism is always greater than 1 (see Supplemental Material), hence leading to the conclusion that in this type of mixed inhibition, the uncompetitive component is always necessarily greater than the competitive component ( K app IU ð B Þ > K app IC ð B Þ ).

For ternary-complex mechanisms with random-order, the two inhibition components are identical, and it can be demonstrated that they are equal to:

K app IC B ðÞ¼ K app IU B ðÞ¼ Kic þ A ½/C138 Ks Kic

Therefore, the K app IU B ðÞ / K app IC B ðÞ ratio is always equal to 1, and the apparent inhibition against substrate B always appears to be of the pure non-competitive type.

## Case IV -iso-mechanisms

The cyclic nature of enzymatic catalysis implies that once the reaction products are released, the enzyme reverts to the form competent for the binding of the substrate. The Michaelis -Menten model assumes that the isomerisation between the substrate form and the product form (F in Scheme 2) of the enzyme is instantaneous and therefore can be neglected. This, indeed, is true in most cases. There are, however, exceptions, such as isomerases that assume different conformations to bind different isomers, enzymes engaging in general acid or base catalysis that need reprotonation, or hydrolytic enzymes that require rehydratation 50,51 . If the isomerisation step that restores the initial enzyme form is slow enough to affect catalytic turnover, the enzyme is said to have an iso-mechanism (Scheme 2) 52 .

Compelling analytical tools to investigate iso-mechanisms were developed only in the 1990s, mainly by Dexter Northrop, who exploited the distinctive inhibition pattern of these enzymes. In the absence of an iso-mechanism, the reaction product is typically a competitive inhibitor of the substrate because both reactants bind the same form of enzyme. In contrast, for enzymes with iso-mechanisms, the substrate and product bind two distinct enzyme forms; therefore, the product inhibition pattern is of the mixed-type 50,53,54 .Analogous considerations apply to the case of dead-end inhibitors that exclusively target the product form of the active site.

As shown in Scheme 2, for iso-mechanisms, the overall turnover rate, ( kcat ), is made up of two segments, the catalytic step governed by the net rate constant kchem and the isomerisation step, with net rate constant kiso. The binding of an inhibitor to the product form of theenzyme(FinScheme 2) causes a mixed-type inhibition with:

Kic ¼ kd k 4 þ k /C0 4 k 4 Kiu ¼ kd kiso kcat

with Kd representing the dissociation constant of the FI complex (see Supplemental Material). As correctly predicted by Northrop 50,51 under

Figure 4. Simulation of MIM mechanisms of type 3, 4 and 5. (A). Multisubstrate reactions. For bi-bi reactions with a sequential ordered mechanism, competitive inhibitors directed against the binding site of substrate A, when B is the variable substrate, generate an apparent mixed inhibition with a Kiu / Kic ratio that tends asymptotically to 1 when [A]/ kmA tends to 0 and to kcat / k-1 when [A] tends to 1 . (B) . For iso-mechanism enzymes, inhibitors that target the product-binding isomer of the free enzyme (F in the reaction scheme) generate mixed-type inhibition with a Kiu / Kic ratio equal to 1/Fiso. (C) . With exo-site enzymes, active-site directed inhibitors cause mixed inhibition with Kiu / Kic that tends asymptotically to 1 as Fiso tends to 1.

<!-- image -->

<!-- image -->

Scheme 2. Reaction scheme of iso-mechanism enzymes.

<!-- image -->

the initial velocity condition, the Kic / Kiu ratio equals Fiso (Figure 4(B)), a parameter that quantifies the kinetic significance of the isomerisation segment and is defined as:

Fiso ¼ 1 kFiso þ 1 kRiso 1 kcat

where kFiso and kRiso are the rate constants for the forward and reverse isomerisation steps, respectively.

Unlike the previous MIM mechanisms, in this case, the reduction in the turnover rate caused by active site-bound inhibitors is not just an artefact. Because Fiso depicts an intrinsic feature of the catalytic mechanism, all inhibitors of a given enzyme would cause a mixed inhibition with an identical Kiu / Kic ratio.

The acetylcholinesterase inhibitors developed as potential antiAlzheimer ' s drugs perhaps constitute the most prominent example of mixed dead-end inhibition of an iso-mechanism enzyme 55 . However, there are no reliable data on the prevalence of isomechanisms, so it is difficult to estimate their contribution to the overall occurrence of mixed inhibition. It is worth noting that the product form of the free enzyme is a short-lived molecular species generated along with the catalysis rather than a stable enzyme isoform. This implies that isomerisation in the forward direction (F ! E) is necessarily faster than in the reverse direction (F E) 52 , which is consistent with the finding that the chemical segment is always the rate-limiting step 50 . Consequently, even though the Kiu s generated by iso-mechanisms cannot be algebraically shown to be necessarily greater than the Kic s, mechanistic considerations explain why Fiso is practically always less than 1 and,

Scheme 3. Reaction scheme of exo-site enzymes.

consequently, why, in this case too, the mixed inhibition is dominated by the competitive component.

## Case V -exo-sites

With most enzymes, it is not possible to make a clear distinction between the portion of the active site involved in the binding of the substrate and the actual catalytic site because the catalytic site also contributes to substrate stabilisation, at least to some extent. In this respect, enzymes that catalyse the covalent modification of large macromolecular substrates as endonucleases 56 , protein kinases/phosphatases 57 , proteases 58 and in general, all post-translational modification enzymes, represent a notable exception. Often, this type of enzyme derives most of the affinity for their substrates from a region of the protein surface that is distinct and remote from the catalytic site. The substrate recognition site, in this case, is called an exo-site 58 .

For exo-site enzymes, the binding of the substrate typically proceeds through two sequential steps (Scheme 3), the first leading to an ' encounter complex ' , here denoted as ' SE ' , with the substrate bound only to the exo-site. SE then decays monomolecularly to form the catalytically competent complex ' ES ' , with the consensus sequence of the substrate accommodated within the active site and ready to undergo catalytic conversion.

An active site-directed inhibitor, I, can bind both the free enzyme E and the encounter complex SE but not ES. In other words, the inhibitor does not compete with the initial enzymesubstrate recognition but impedes the isomerisation of the SE complex which leads to the formation of the catalytically

competent complex ES. This causes a reduction in the ' degree of competition ' that in a double reciprocal plot is visualised as a mixed-type inhibition pattern 3 .

To model this reaction mechanism, it is convenient to regard kcat as composed of two segments, the isomerisation of the SE complex and the chemical catalytic step, governed by the constant rates kiso and kchem , respectively (Scheme 3).

Analogous to the previous iso-mechanism case, the coefficient Fiso, here defined as kcat / kiso , quantifies the kinetic relevance of the isomerisation step. Fiso assumes values between 0 when kiso is so large that the isomerisation is kinetically irrelevant and 1 when it is slow enough to be completely rate-limiting.

Simulation of this reaction with KinTek Explorer (Figure 4(C)) showed that the Kiu / Kic ratio is equal to 1/Fiso.

From which follows that if Fiso tends to 0, the inhibition is competitive, as expected for an active site-bound inhibitor. If Fiso ¼ 1, the catalytic segment of the reaction can be neglected, and the inhibition turns pure non-competitive ( Kic ¼ Kiu ). For 0 < Fkiso < 1, the inhibition is of the mixed-type with Kic < Kiu , so that necessarily Kiu /C21 Kic (see Supplemental Material).

The reduction of the affinity for the substrate and of the apparent maximal velocity are both actual. However, as for the previous iso-mechanism case, there is no allosteric site involved, and neither Kic or Kiu are true dissociation constants.

Although enzymes with exosites are often of utmost pharmacological relevance 3,59 , the proper study of their inhibition mechanism, which necessarily has to be performed using physiologic substrates, has been limited by the remarkable effort required to produce enough proteinaceous substrates and to measure the time course of their chemical modification. Indeed, this type of study has been reported for just a handful of enzymes, mostly serine proteases of the coagulation cascade process 58,60,61 , so that experimental evidence is scant and the contribution of exo-site enzyme inhibition to the statistics of chapter 3 is negligible.

## Discussion

While the definition given by ChatGPT of mixed inhibition is incorrect, it is easy to agree with the concluding remark: ' Understanding the nature of mixed-type inhibition is crucial for studying enzyme kinetics, drug discovery, and the design of effective enzyme inhibitors ' .

The scouting of scientific literature, however, and, ironically enough, the ChatGPT response itself, clearly showed that the proper understanding of mixed inhibition is not something that should be taken for granted. The intrinsic complexity of this type of reversible inhibition made it difficult to devise a clear schematic mechanistic explanation. In turn, this has resulted in descriptions that are often oversimplifying, incomplete and ultimately, misleading. Moreover, to worsen the problem, the great diversity typically displayed by enzymes has discouraged authors from making definitive assertions on the mechanistic irrelevance of the two-site model. With the overall outcome that many of those who deal with the study of enzyme inhibition, perhaps the majority of them, developed an incorrect understanding of the causative molecular determinants of mixed inhibition.

The scope of this work was to assess the actual mechanistic significance of the formal two-site model. The finding that five MIM mechanisms exist that generate mixed inhibition patterns with Kiu s that are inevitably always larger than or equal to their cognate Kic s, combined with the evidence that in the reported cases of mixed inhibition practically never is Kic > Kiu , constitutes solid statistical evidence that these MIM mechanisms constitute the actual underlying mechanisms of mixed inhibition.

With the understanding that a molecular mechanism can be established only on the basis of a thorough specific study, the analysis presented herein shows that the two-site formal mechanism substantially plays no role in the generation of mixed inhibition and hence, that the molecular nature of reversible enzyme inhibition is basically just competitive.

This also explains the lack of structural evidence for the binding of mixed inhibitors to allosteric sites: on the basis of the statistic on enzyme inhibition reported in the result section, the Protein Data Bank 62 should contain at least 4 -5000 structures of enzymes in complex with mixed or non-competitive inhibitors. However, in an attempt to assess the occurrence, among mixed inhibitors, of ES complex binders, we found that none of the 197 entries that currently contain the term ' noncompetitive inhibitor ' or ' mixedtype inhibitor ' bound sites other than the enzyme active site.

In the first three cases discussed in the results section, i.e. tight inhibition, slow dissociating inhibitors and multiple-substrate enzymes, the inhibition is only apparently of the mixed type: in an in vivo scenario, a dead end inhibitor acting through these mechanisms, as for example a drug, would simply cause an inhibition of the competitive type that can be overcome by high substrate concentrations. Regardless, multiple-substrate enzymes remain the most frequent cause of the reported mixed inhibition, as correctly suggested by Johnson 7 . It is worth noting that because highly potent inhibitors are by necessity tight and/or slow-dissociating binders, the MIM mechanism of type I and II might be of special importance in drug design.

In cases IV and V (i.e. iso-mechanism and exo-site enzymes) although the inhibitors only bind to the same site bound by the substrate, the inhibition actually affects both the enzyme ' s substrate affinity and limiting velocity and cannot be ' resolved ' by higher substrate concentrations. As correctly pointed out by Cornish-Bowden 12 , iso-mechanism enzymes likely constitute the main source of biologically relevant mixed inhibition. It should be stressed, however, that the real incidence of mixed inhibition of exo-site enzymes is certainly largely underestimated.

Perhaps, the most convenient approach to disentangle the complexity of mixed inhibition is to dismiss the classic analysis if initial rates in favour of the more up-to-date computer assisted global fitting of full reaction time-courses implemented by software such as KinTek Explorer 36,37 or ENZO 63 . The fitting of kinetic data based on numerical integration of rate equations streamlines the otherwise difficult study of mechanisms that deviate from the steady-state assumptions, as are the cases, quite common in pharmacology, of tight-binders and time-dependent inhibitors. In general, once the mechanism of the target enzyme has been resolved, the direct simulation of enzymatic reactions allow to effortlessly distinguish between the five MIM mechanisms.

## Disclosure statement

No potential conflict of interest was reported by the author(s).

## Funding

The author(s) reported there is no funding associated with the work featured in this article.

## References

- 1. Symbolism and terminology in enzyme kinetics. Recommendations 1981. Biochem J. 1983;213(3):561 -571.
- 8 A. PESARESI
- 2. Luo L, Parrish CA, Nevins N, McNulty DE, Chaudhari AM, Carson JD, Sudakin V, Shaw AN, Lehr R, Zhao H, et al. ATPcompetitive inhibitors of the mitotic kinesin KSP that function via an allosteric mechanism. Nat Chem Biol. 2007;3(11): 722 -726.
- 3. Copeland RA. Evaluation of enzyme inhibitors in drug discovery. Ch. 3, 2nd ed. Hoboken: John Wiley & Sons, Inc.; 2013.
- 4. Osterman AL, Rodionova I, Li X, Sergienko E, Ma CT, Catanzaro A, Pettigrove ME, Reed RW, Gupta R, Rohde KH, et al. Novel antimycobacterial compounds suppress NAD biogenesis by targeting a unique pocket of NaMN adenylyltransferase. ACS Chem Biol. 2019;14(5):949 -958.
- 5. Copeland RA, Anderson PS. Enzymes and Enzyme Inhibitors ' in Textbook of Drug Design and Discovery. 3rd ed., P. Krogsgaard-Larsen, T. Liljefors, U. Madsen, editors. New York: Taylor and Francis; 2002
- 6. Cornish-Bowden A. Fundamentals of enzyme kinetics. Ch. 7, 3rd ed. London: Portland Press; 2004.
- 7. Johnson KA. Kinetic analysis for the new enzymology. Ch. 6. Austin (TX): KinTek Corporation; 2019.
- 8. Arya A, Kumar A. Inconsistencies in some common terms and notations in enzymology: Textbook examples and suggestions. Biochem Mol Biol Educ. 2019;47(2):140 -144.
- 9. Srinivasan B. Words of advice: teaching enzyme kinetics. Febs J. 2021;288(7):2068 -2083.
- 10. Strelow J, Dewe W, Iversen PW, Brooks HB, Radding JA, McGee J, Weidner J, et al. Mechanism of Action Assays for Enzymes. In Markossian S, Grossman A, Brimacombe K, editors. Assay Guidance Manual. Bethesda (MD): Eli Lilly & Company and the National Center for Advancing Translational Sciences; 2012. https://www.ncbi.nlm.nih.gov/ books/NBK92001/
- 11. Sharma R. Enzyme inhibition: mechanisms and scope. In: Sharma RR, editor. Enzyme inhibition and bioapplications. Rijeka (Croatia): InInTech; 2012.
- 12. Cornish-Bowden A. Fundamentals of enzyme kinetics Ch. 5, 3rd ed. London: Portland Press; 2004.
- 13. Kumawat A, Raheem S, Ali F, Dar TA, Chakrabarty S, Rizvi MA. Organoselenium Compounds as acetylcholinesterase inhibitors: evidence and mechanism of mixed inhibition. J Phys Chem B. 2021;125(6):1531 -1541. vol.
- 14. Qi D, Li H, Liang C, Peng P, Yang Z, Gao Y, Li Z, Zhang Q, Liu Z. Herb-drug interaction of Xingnaojing injection and Edaravone via pharmacokinetics, mixed inhibition of UGTs, and molecular docking. Phytomedicine. 2023;112:154696.
- 15. Le Corre L, Padovani D. Mechanism-based and computational modeling of hydrogen sulfide biogenesis inhibition: interfacial inhibition. Sci Rep. 2023;13(1):7287.
- 16. Converti A, Perego P, Torre P, Silverio da Silva S. Mixed inhibitions by methanol, furfural and acetic acid on xylitol production by Candida guilliermondii. Biotechnol Lett. 2000; 22(23):1861 -1865.
- 17. Deri B, Kanteev M, Goldfeder M, Lecina D, Guallar V, Adir N, Fishman A. The unravelling of the complex pattern of tyrosinase inhibition. Sci Rep. 2016;6 (6):34993.

18.

19.

- Kim SB, Cho HJ, Kim YS, Kim DD, Yoon IS. Modulation of cytochrome P450 activity by 18 b -glycyrrhetic acid and its consequence on buspirone pharmacokinetics in rats. Phytother Res. 2015;29(8):1188 -1194.
- Wang Y, Liu X, Schneider B, Zverina EA, Russ K, Wijeyesakere SJ, Fierke CA, Richardson RJ, Philbert MA. Mixed inhibition of adenosine deaminase activity by 1,3-dinitrobenzene: a

20.

model for understanding cell-selective neurotoxicity in chemically-induced energy deprivation syndromes in brain. Toxicol Sci. 2012;125(2):509 -521.

- Wang Z, Tan X, Lu G, Liu Y, Naidu R, He W. Soil properties influence kinetics of soil acid phosphatase in response to arsenic toxicity. Ecotoxicol Environ Saf. 2018;147:266 -274.
- 21. Misko TA, Wijerathna SR, Radivoyevitch T, Berdis AJ, Ahmad MF, Harris ME, Dealwis CG. Inhibition of yeast ribonucleotide reductase by Sml1 depends on the allosteric state of the enzyme. FEBS Lett. 2016;590(12):1704 -1712.
- 22. Nakahara H, Okada S, Mochida K. Kinetic studies on pancreatic lipase activity in micellar systems. I. Inhibition by sodium deoxycholate micelles. Chem Pharm Bull (Tokyo)). 1982;30(8):2673 -2681.

23.

24.

25.

26.

27.

28.

29.

30.

- Prescott TA, Panaretou B, Veitch NC, Simmonds MS. A yeast chemical genetics approach identifies the compound 3,4,5trimethoxybenzyl isothiocyanate as a calcineurin inhibitor. FEBS Lett. 2014;588(3):455 -458.

Bounias M, Morgan MRJ. An in vitro kinetic study of the mixed inhibition of honeybee hemolymph PNPa -D-glucosidase by sucrose. Experientia. 1981;37(11):1147 -1149.

- Dhital S, Gidley MJ, Warren FJ. Inhibition of a -amylase activity by cellulose: Kinetic analysis and nutritional implications. Carbohydr Polym. 2015;123:305 -312.
- Liu J, Liu Y, He X, Teng B, McRae JM. Valonea Tannin: Tyrosinase Inhibition Activity, Structural Elucidation and Insights into the Inhibition Mechanism. Molecules. 2021; 26(9):2747.

Pogacean F, Baldea I, Olenic L, Pruneanu S, Biris AS. Kinetic Determination of Drug Particles Concentration via EnzymeCatalyzed Decomposition of Hydrogen Peroxide. Part Sci Technol. 2011;29(6):493 -502.

Zhang C, Schilir /C18 o T, Gea M, Bianchi S, Spinello A, Magistrato A, Gilardi G, Di Nardo G. Molecular Basis for Endocrine Disruption by Pesticides Targeting Aromatase and Estrogen Receptor. Int J Environ Res Public Health. 2020;17(16):5664.

Alam MT, Olin-Sandoval V, Stincone A, Keller MA, Zelezniak A, Luisi BF, Ralser M. The self-inhibitory nature of metabolic networks and its alleviation through compartmentalization. Nat Commun. 2017;8:16018.

- Holdgate GA, Meek TD, Grimley RL. Mechanistic enzymology in drug discovery: a fresh perspective. Nat Rev Drug Discov. 2018;17(2):115 -132.
- 31. Robin T, Reuveni S, Urbakh M. Single-molecule theory of enzymatic inhibition. Nat Commun. 2018;9(1):779.
- 32. Cha SH, Hong J, McGuffie M, Yeom B, VanEpps JS, Kotov NA. Shape-Dependent Biomimetic Inhibition of Enzyme by Nanoparticles and Their Antibacterial Activity. ACS Nano. 2015;9(9):9097 -9105.
- 33. Swinney D. Biochemical mechanisms of drug action: what does it take for success? Nat Rev Drug Discov. 2004;3(9): 801 -808.
- 34. Cornish-Bowden A. Why is uncompetitive inhibition so rare? A possible explanation, with implications for the design of drugs and pesticides. FEBS Lett. 1986;203(1):3 -6.
- 35. Blat Y. Non-competitive inhibition by active site binders. Chem Biol Drug Des. 2010;75(6):535 -540.
- 36. Johnson KA, Simpson ZB, Blom T. Global kinetic explorer: a new computer program for dynamic simulation and fitting of kinetic data. Anal Biochem. 2009;387(1):20 -29.

37.

- Johnson KA, Simpson ZB, Blom T. FitSpace explorer: an algorithm to evaluate multidimensional parameter space in fitting kinetic data. Anal Biochem. 2009;387(1):30 -41.
- 38. Chang A, Jeske L, Ulbrich S, Hofmann J, Koblitz J, Schombu I, Neumann-Schaal M, Jahn D, Schomburg D. BRENDA, the ELIXIR core data resource in 2021: new developments and updates. Nucleic Acids Res. 2021;49(D1):D498 -D508.
- 39. Mannervik B, Jakobson I, Warholm M. Error structure as a function of substrate and inhibitor concentration in enzyme kinetic experiments. Biochem J. 1986;235(3):797 -804.
- 40. Johnson KA. A century of enzyme kinetic analysis, 1913 to 2013. FEBS Lett. 2013;587(17):2753 -2766.
- 41. Cornish-Bowden A. Fundamentals of enzyme kinetics. Ch. 6, 3rd ed. London: Portland Press; 2004.
- 42. Copeland RA. Enzymes: a practical introduction to structure, mechanism, and data analysis. Ch. 9, 2nd ed. New York: John Wiley & Sons, Inc.; 2000.
- 43. Waley SG. The kinetics of slow-binding and slow, tight-binding inhibition: The effects of substrate depletion. Biochem J. 1993;294 (Pt 1)(Pt 1):195 -200. :
- 44. Turner PM, Lerea KM, Kull FJ. The ribonuclease inhibitors from porcine thyroid and liver are slow, tight-binding inhibitors of bovine pancreatic ribonuclease A. Biochem Biophys Res Commun. 1983;114(3):1154 -1160.
- 45. Morrison JF, Walsh CT. The behavior and significance of slow-binding enzyme inhibitors. Adv Enzymol Relat Areas Mol Biol. 1988;61:201 -301.
- 46. Lamba D, Pesaresi A. Kinetic modeling of time-dependent enzyme inhibition by pre-steady-state analysis of progress curves: the case study of the anti-Alzheimer ' s drug galantamine. Int J Mol Sci. 2022;23:5072.
- 47. Cleland WW. The kinetics of enzyme-catalyzed reactions with two or more substrates or products. I. Nomenclature and rate equations. Biochim Biophys Acta. 1963;67:104 -137.
- 48. Cleland WW. The kinetics of enzyme-catalyzed reactions with two or more substrates or products. II. Inhibition: nomenclature and theory. Biochim Biophys Acta. 1963;67: 173 -187.
- 49. Cleland WW. The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection. Biochim Biophys Acta. 1963;67:188 -196.
- 50. Rebholz L, Northrop DB. Kinetics of enzymes with iso-mechanisms: dead-end inhibition of fumarase and carbonic anhydrase II. Arch Biochem Biophys. 1994;312(1):227 -233.

- 51. Rebholz KL, Northrop DB. Kinetics of iso mechanisms. Methods Enzymol. 1995;249:211 -240.
- 52. Adediran SA, Morrison MJ, Pratt RF. Detection of an enzyme isomechanism by means of the kinetics of covalent inhibition. Biochim Biophys Acta Proteins Proteom. 2021;1869(9): 140681.
- 53. Rebholz KL, Northrop DB. Kinetics of enzymes with isomechanisms: analysis of product inhibition. Biochem J. 1993; 296 (Pt 2)(Pt 2):355 -360. :
- 54. Northrop DB. Problems in defining limits: how slow is very slow? Biochem J. 1994;301(Pt 2):623.
- 55. Krupka RM, Laidler KJ. Molecular mechanisms for hydrolytic enzyme action. I. apparent non-competitive inhibition, with special reference to acetylcholinesterase. Richard M. J Am Chem Soc. 1961;83(6):1445 -1447.
- 56. Colandene JD, Topal MD. The domain organization of NaeI endonuclease: separation of binding and catalysis. Proc Natl Acad Sci U S A. 1998;95(7):3531 -3536.
- 57. Horiuchi KY, Scherle PA, Trzaskos JM, Copeland RA. Competitive inhibition of MAP kinase activation by a peptide representing the alpha C helix of ERK. Biochemistry. 1998;37(25):8879 -8885.
- 58. Jordan SP, Mao SS, Lewis SD, Shafer JA. Reaction pathway for inhibition of blood coagulation factor Xa by tick anticoagulant peptide. Biochemistry. 1992;31(23):5374 -5380.
- 59. Ebert T, Tran N, Schurgers L, Stenvinkel P, Shiels PG. Ageing - oxidative stress, PTMs and disease. Mol Aspects Med. 2022; 86:101099.
- 60. Krishnaswamy S. Exosite-driven substrate specificity and function in coagulation. J Thromb Haemost. 2005;3(1):54 -67.
- 61. Ogawa T, Verhamme IM, Sun MF, Bock PE, Gailani D. Exosite-mediated substrate recognition of factor IX by factor XIa. The factor XIa heavy chain is required for initial recognition of factor IX. J Biol Chem. 2005;280(25):23523 -23530.
- 62. Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat GN, Weissig H, Shindyalov IN, Bourne PE. The Protein Data Bank. Nucleic Acids Res. 2000;28(1):235 -242.
- 63. Bevc S, Konc J, Stojan J, Hodo /C20 s /C20 cek M, Penca M, Praprotnik M, Jane /C20 zi /C20 c D. ENZO: a web tool for derivation and evaluation of kinetic models of enzyme catalyzed reactions. PLoS One. 2011;6(7):e22265.